<!--
  ~ The Exomiser - A tool to annotate and prioritize genomic variants
  ~
  ~ Copyright (c) 2016-2021 Queen Mary University of London.
  ~ Copyright (c) 2012-2016 Charité Universitätsmedizin Berlin and Genome Research Ltd.
  ~
  ~ This program is free software: you can redistribute it and/or modify
  ~ it under the terms of the GNU Affero General Public License as
  ~ published by the Free Software Foundation, either version 3 of the
  ~ License, or (at your option) any later version.
  ~
  ~ This program is distributed in the hope that it will be useful,
  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  ~ GNU Affero General Public License for more details.
  ~
  ~ You should have received a copy of the GNU Affero General Public License
  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  -->

<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>
    <title>The Exomiser - A Tool to Annotate and Prioritise Whole-Exome Sequencing Data</title>
    <meta charset="UTF-8"/>
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <style>
        /*!
 * Bootstrap v3.3.7 (http://getbootstrap.com)
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 *//*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */html{font-family:sans-serif;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,hgroup,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block;vertical-align:baseline}audio:not([controls]){display:none;height:0}[hidden],template{display:none}a{background-color:transparent}a:active,a:hover{outline:0}abbr[title]{border-bottom:1px dotted}b,strong{font-weight:700}dfn{font-style:italic}h1{margin:.67em 0;font-size:2em}mark{color:#000;background:#ff0}small{font-size:80%}sub,sup{position:relative;font-size:75%;line-height:0;vertical-align:baseline}sup{top:-.5em}sub{bottom:-.25em}img{border:0}svg:not(:root){overflow:hidden}figure{margin:1em 40px}hr{height:0;-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}pre{overflow:auto}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}button,input,optgroup,select,textarea{margin:0;font:inherit;color:inherit}button{overflow:visible}button,select{text-transform:none}button,html input[type=button],input[type=reset],input[type=submit]{-webkit-appearance:button;cursor:pointer}button[disabled],html input[disabled]{cursor:default}button::-moz-focus-inner,input::-moz-focus-inner{padding:0;border:0}input{line-height:normal}input[type=checkbox],input[type=radio]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;padding:0}input[type=number]::-webkit-inner-spin-button,input[type=number]::-webkit-outer-spin-button{height:auto}input[type=search]{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;-webkit-appearance:textfield}input[type=search]::-webkit-search-cancel-button,input[type=search]::-webkit-search-decoration{-webkit-appearance:none}fieldset{padding:.35em .625em .75em;margin:0 2px;border:1px solid silver}legend{padding:0;border:0}textarea{overflow:auto}optgroup{font-weight:700}table{border-spacing:0;border-collapse:collapse}td,th{padding:0}/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */@media print{*,:after,:before{color:#000!important;text-shadow:none!important;background:0 0!important;-webkit-box-shadow:none!important;box-shadow:none!important}a,a:visited{text-decoration:underline}a[href]:after{content:" (" attr(href) ")"}abbr[title]:after{content:" (" attr(title) ")"}a[href^="javascript:"]:after,a[href^="#"]:after{content:""}blockquote,pre{border:1px solid #999;page-break-inside:avoid}thead{display:table-header-group}img,tr{page-break-inside:avoid}img{max-width:100%!important}h2,h3,p{orphans:3;widows:3}h2,h3{page-break-after:avoid}.navbar{display:none}.btn>.caret,.dropup>.btn>.caret{border-top-color:#000!important}.label{border:1px solid #000}.table{border-collapse:collapse!important}.table td,.table th{background-color:#fff!important}.table-bordered td,.table-bordered th{border:1px solid #ddd!important}}@font-face{font-family:'Glyphicons Halflings';src:url(../fonts/glyphicons-halflings-regular.eot);src:url(../fonts/glyphicons-halflings-regular.eot?#iefix) format('embedded-opentype'),url(../fonts/glyphicons-halflings-regular.woff2) format('woff2'),url(../fonts/glyphicons-halflings-regular.woff) format('woff'),url(../fonts/glyphicons-halflings-regular.ttf) format('truetype'),url(../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular) format('svg')}.glyphicon{position:relative;top:1px;display:inline-block;font-family:'Glyphicons Halflings';font-style:normal;font-weight:400;line-height:1;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.glyphicon-asterisk:before{content:"\002a"}.glyphicon-plus:before{content:"\002b"}.glyphicon-eur:before,.glyphicon-euro:before{content:"\20ac"}.glyphicon-minus:before{content:"\2212"}.glyphicon-cloud:before{content:"\2601"}.glyphicon-envelope:before{content:"\2709"}.glyphicon-pencil:before{content:"\270f"}.glyphicon-glass:before{content:"\e001"}.glyphicon-music:before{content:"\e002"}.glyphicon-search:before{content:"\e003"}.glyphicon-heart:before{content:"\e005"}.glyphicon-star:before{content:"\e006"}.glyphicon-star-empty:before{content:"\e007"}.glyphicon-user:before{content:"\e008"}.glyphicon-film:before{content:"\e009"}.glyphicon-th-large:before{content:"\e010"}.glyphicon-th:before{content:"\e011"}.glyphicon-th-list:before{content:"\e012"}.glyphicon-ok:before{content:"\e013"}.glyphicon-remove:before{content:"\e014"}.glyphicon-zoom-in:before{content:"\e015"}.glyphicon-zoom-out:before{content:"\e016"}.glyphicon-off:before{content:"\e017"}.glyphicon-signal:before{content:"\e018"}.glyphicon-cog:before{content:"\e019"}.glyphicon-trash:before{content:"\e020"}.glyphicon-home:before{content:"\e021"}.glyphicon-file:before{content:"\e022"}.glyphicon-time:before{content:"\e023"}.glyphicon-road:before{content:"\e024"}.glyphicon-download-alt:before{content:"\e025"}.glyphicon-download:before{content:"\e026"}.glyphicon-upload:before{content:"\e027"}.glyphicon-inbox:before{content:"\e028"}.glyphicon-play-circle:before{content:"\e029"}.glyphicon-repeat:before{content:"\e030"}.glyphicon-refresh:before{content:"\e031"}.glyphicon-list-alt:before{content:"\e032"}.glyphicon-lock:before{content:"\e033"}.glyphicon-flag:before{content:"\e034"}.glyphicon-headphones:before{content:"\e035"}.glyphicon-volume-off:before{content:"\e036"}.glyphicon-volume-down:before{content:"\e037"}.glyphicon-volume-up:before{content:"\e038"}.glyphicon-qrcode:before{content:"\e039"}.glyphicon-barcode:before{content:"\e040"}.glyphicon-tag:before{content:"\e041"}.glyphicon-tags:before{content:"\e042"}.glyphicon-book:before{content:"\e043"}.glyphicon-bookmark:before{content:"\e044"}.glyphicon-print:before{content:"\e045"}.glyphicon-camera:before{content:"\e046"}.glyphicon-font:before{content:"\e047"}.glyphicon-bold:before{content:"\e048"}.glyphicon-italic:before{content:"\e049"}.glyphicon-text-height:before{content:"\e050"}.glyphicon-text-width:before{content:"\e051"}.glyphicon-align-left:before{content:"\e052"}.glyphicon-align-center:before{content:"\e053"}.glyphicon-align-right:before{content:"\e054"}.glyphicon-align-justify:before{content:"\e055"}.glyphicon-list:before{content:"\e056"}.glyphicon-indent-left:before{content:"\e057"}.glyphicon-indent-right:before{content:"\e058"}.glyphicon-facetime-video:before{content:"\e059"}.glyphicon-picture:before{content:"\e060"}.glyphicon-map-marker:before{content:"\e062"}.glyphicon-adjust:before{content:"\e063"}.glyphicon-tint:before{content:"\e064"}.glyphicon-edit:before{content:"\e065"}.glyphicon-share:before{content:"\e066"}.glyphicon-check:before{content:"\e067"}.glyphicon-move:before{content:"\e068"}.glyphicon-step-backward:before{content:"\e069"}.glyphicon-fast-backward:before{content:"\e070"}.glyphicon-backward:before{content:"\e071"}.glyphicon-play:before{content:"\e072"}.glyphicon-pause:before{content:"\e073"}.glyphicon-stop:before{content:"\e074"}.glyphicon-forward:before{content:"\e075"}.glyphicon-fast-forward:before{content:"\e076"}.glyphicon-step-forward:before{content:"\e077"}.glyphicon-eject:before{content:"\e078"}.glyphicon-chevron-left:before{content:"\e079"}.glyphicon-chevron-right:before{content:"\e080"}.glyphicon-plus-sign:before{content:"\e081"}.glyphicon-minus-sign:before{content:"\e082"}.glyphicon-remove-sign:before{content:"\e083"}.glyphicon-ok-sign:before{content:"\e084"}.glyphicon-question-sign:before{content:"\e085"}.glyphicon-info-sign:before{content:"\e086"}.glyphicon-screenshot:before{content:"\e087"}.glyphicon-remove-circle:before{content:"\e088"}.glyphicon-ok-circle:before{content:"\e089"}.glyphicon-ban-circle:before{content:"\e090"}.glyphicon-arrow-left:before{content:"\e091"}.glyphicon-arrow-right:before{content:"\e092"}.glyphicon-arrow-up:before{content:"\e093"}.glyphicon-arrow-down:before{content:"\e094"}.glyphicon-share-alt:before{content:"\e095"}.glyphicon-resize-full:before{content:"\e096"}.glyphicon-resize-small:before{content:"\e097"}.glyphicon-exclamation-sign:before{content:"\e101"}.glyphicon-gift:before{content:"\e102"}.glyphicon-leaf:before{content:"\e103"}.glyphicon-fire:before{content:"\e104"}.glyphicon-eye-open:before{content:"\e105"}.glyphicon-eye-close:before{content:"\e106"}.glyphicon-warning-sign:before{content:"\e107"}.glyphicon-plane:before{content:"\e108"}.glyphicon-calendar:before{content:"\e109"}.glyphicon-random:before{content:"\e110"}.glyphicon-comment:before{content:"\e111"}.glyphicon-magnet:before{content:"\e112"}.glyphicon-chevron-up:before{content:"\e113"}.glyphicon-chevron-down:before{content:"\e114"}.glyphicon-retweet:before{content:"\e115"}.glyphicon-shopping-cart:before{content:"\e116"}.glyphicon-folder-close:before{content:"\e117"}.glyphicon-folder-open:before{content:"\e118"}.glyphicon-resize-vertical:before{content:"\e119"}.glyphicon-resize-horizontal:before{content:"\e120"}.glyphicon-hdd:before{content:"\e121"}.glyphicon-bullhorn:before{content:"\e122"}.glyphicon-bell:before{content:"\e123"}.glyphicon-certificate:before{content:"\e124"}.glyphicon-thumbs-up:before{content:"\e125"}.glyphicon-thumbs-down:before{content:"\e126"}.glyphicon-hand-right:before{content:"\e127"}.glyphicon-hand-left:before{content:"\e128"}.glyphicon-hand-up:before{content:"\e129"}.glyphicon-hand-down:before{content:"\e130"}.glyphicon-circle-arrow-right:before{content:"\e131"}.glyphicon-circle-arrow-left:before{content:"\e132"}.glyphicon-circle-arrow-up:before{content:"\e133"}.glyphicon-circle-arrow-down:before{content:"\e134"}.glyphicon-globe:before{content:"\e135"}.glyphicon-wrench:before{content:"\e136"}.glyphicon-tasks:before{content:"\e137"}.glyphicon-filter:before{content:"\e138"}.glyphicon-briefcase:before{content:"\e139"}.glyphicon-fullscreen:before{content:"\e140"}.glyphicon-dashboard:before{content:"\e141"}.glyphicon-paperclip:before{content:"\e142"}.glyphicon-heart-empty:before{content:"\e143"}.glyphicon-link:before{content:"\e144"}.glyphicon-phone:before{content:"\e145"}.glyphicon-pushpin:before{content:"\e146"}.glyphicon-usd:before{content:"\e148"}.glyphicon-gbp:before{content:"\e149"}.glyphicon-sort:before{content:"\e150"}.glyphicon-sort-by-alphabet:before{content:"\e151"}.glyphicon-sort-by-alphabet-alt:before{content:"\e152"}.glyphicon-sort-by-order:before{content:"\e153"}.glyphicon-sort-by-order-alt:before{content:"\e154"}.glyphicon-sort-by-attributes:before{content:"\e155"}.glyphicon-sort-by-attributes-alt:before{content:"\e156"}.glyphicon-unchecked:before{content:"\e157"}.glyphicon-expand:before{content:"\e158"}.glyphicon-collapse-down:before{content:"\e159"}.glyphicon-collapse-up:before{content:"\e160"}.glyphicon-log-in:before{content:"\e161"}.glyphicon-flash:before{content:"\e162"}.glyphicon-log-out:before{content:"\e163"}.glyphicon-new-window:before{content:"\e164"}.glyphicon-record:before{content:"\e165"}.glyphicon-save:before{content:"\e166"}.glyphicon-open:before{content:"\e167"}.glyphicon-saved:before{content:"\e168"}.glyphicon-import:before{content:"\e169"}.glyphicon-export:before{content:"\e170"}.glyphicon-send:before{content:"\e171"}.glyphicon-floppy-disk:before{content:"\e172"}.glyphicon-floppy-saved:before{content:"\e173"}.glyphicon-floppy-remove:before{content:"\e174"}.glyphicon-floppy-save:before{content:"\e175"}.glyphicon-floppy-open:before{content:"\e176"}.glyphicon-credit-card:before{content:"\e177"}.glyphicon-transfer:before{content:"\e178"}.glyphicon-cutlery:before{content:"\e179"}.glyphicon-header:before{content:"\e180"}.glyphicon-compressed:before{content:"\e181"}.glyphicon-earphone:before{content:"\e182"}.glyphicon-phone-alt:before{content:"\e183"}.glyphicon-tower:before{content:"\e184"}.glyphicon-stats:before{content:"\e185"}.glyphicon-sd-video:before{content:"\e186"}.glyphicon-hd-video:before{content:"\e187"}.glyphicon-subtitles:before{content:"\e188"}.glyphicon-sound-stereo:before{content:"\e189"}.glyphicon-sound-dolby:before{content:"\e190"}.glyphicon-sound-5-1:before{content:"\e191"}.glyphicon-sound-6-1:before{content:"\e192"}.glyphicon-sound-7-1:before{content:"\e193"}.glyphicon-copyright-mark:before{content:"\e194"}.glyphicon-registration-mark:before{content:"\e195"}.glyphicon-cloud-download:before{content:"\e197"}.glyphicon-cloud-upload:before{content:"\e198"}.glyphicon-tree-conifer:before{content:"\e199"}.glyphicon-tree-deciduous:before{content:"\e200"}.glyphicon-cd:before{content:"\e201"}.glyphicon-save-file:before{content:"\e202"}.glyphicon-open-file:before{content:"\e203"}.glyphicon-level-up:before{content:"\e204"}.glyphicon-copy:before{content:"\e205"}.glyphicon-paste:before{content:"\e206"}.glyphicon-alert:before{content:"\e209"}.glyphicon-equalizer:before{content:"\e210"}.glyphicon-king:before{content:"\e211"}.glyphicon-queen:before{content:"\e212"}.glyphicon-pawn:before{content:"\e213"}.glyphicon-bishop:before{content:"\e214"}.glyphicon-knight:before{content:"\e215"}.glyphicon-baby-formula:before{content:"\e216"}.glyphicon-tent:before{content:"\26fa"}.glyphicon-blackboard:before{content:"\e218"}.glyphicon-bed:before{content:"\e219"}.glyphicon-apple:before{content:"\f8ff"}.glyphicon-erase:before{content:"\e221"}.glyphicon-hourglass:before{content:"\231b"}.glyphicon-lamp:before{content:"\e223"}.glyphicon-duplicate:before{content:"\e224"}.glyphicon-piggy-bank:before{content:"\e225"}.glyphicon-scissors:before{content:"\e226"}.glyphicon-bitcoin:before{content:"\e227"}.glyphicon-btc:before{content:"\e227"}.glyphicon-xbt:before{content:"\e227"}.glyphicon-yen:before{content:"\00a5"}.glyphicon-jpy:before{content:"\00a5"}.glyphicon-ruble:before{content:"\20bd"}.glyphicon-rub:before{content:"\20bd"}.glyphicon-scale:before{content:"\e230"}.glyphicon-ice-lolly:before{content:"\e231"}.glyphicon-ice-lolly-tasted:before{content:"\e232"}.glyphicon-education:before{content:"\e233"}.glyphicon-option-horizontal:before{content:"\e234"}.glyphicon-option-vertical:before{content:"\e235"}.glyphicon-menu-hamburger:before{content:"\e236"}.glyphicon-modal-window:before{content:"\e237"}.glyphicon-oil:before{content:"\e238"}.glyphicon-grain:before{content:"\e239"}.glyphicon-sunglasses:before{content:"\e240"}.glyphicon-text-size:before{content:"\e241"}.glyphicon-text-color:before{content:"\e242"}.glyphicon-text-background:before{content:"\e243"}.glyphicon-object-align-top:before{content:"\e244"}.glyphicon-object-align-bottom:before{content:"\e245"}.glyphicon-object-align-horizontal:before{content:"\e246"}.glyphicon-object-align-left:before{content:"\e247"}.glyphicon-object-align-vertical:before{content:"\e248"}.glyphicon-object-align-right:before{content:"\e249"}.glyphicon-triangle-right:before{content:"\e250"}.glyphicon-triangle-left:before{content:"\e251"}.glyphicon-triangle-bottom:before{content:"\e252"}.glyphicon-triangle-top:before{content:"\e253"}.glyphicon-console:before{content:"\e254"}.glyphicon-superscript:before{content:"\e255"}.glyphicon-subscript:before{content:"\e256"}.glyphicon-menu-left:before{content:"\e257"}.glyphicon-menu-right:before{content:"\e258"}.glyphicon-menu-down:before{content:"\e259"}.glyphicon-menu-up:before{content:"\e260"}*{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}:after,:before{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}html{font-size:10px;-webkit-tap-highlight-color:rgba(0,0,0,0)}body{font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;line-height:1.42857143;color:#333;background-color:#fff}button,input,select,textarea{font-family:inherit;font-size:inherit;line-height:inherit}a{color:#337ab7;text-decoration:none}a:focus,a:hover{color:#23527c;text-decoration:underline}a:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}figure{margin:0}img{vertical-align:middle}.carousel-inner>.item>a>img,.carousel-inner>.item>img,.img-responsive,.thumbnail a>img,.thumbnail>img{display:block;max-width:100%;height:auto}.img-rounded{border-radius:6px}.img-thumbnail{display:inline-block;max-width:100%;height:auto;padding:4px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:all .2s ease-in-out;-o-transition:all .2s ease-in-out;transition:all .2s ease-in-out}.img-circle{border-radius:50%}hr{margin-top:20px;margin-bottom:20px;border:0;border-top:1px solid #eee}.sr-only{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0,0,0,0);border:0}.sr-only-focusable:active,.sr-only-focusable:focus{position:static;width:auto;height:auto;margin:0;overflow:visible;clip:auto}[role=button]{cursor:pointer}.h1,.h2,.h3,.h4,.h5,.h6,h1,h2,h3,h4,h5,h6{font-family:inherit;font-weight:500;line-height:1.1;color:inherit}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-weight:400;line-height:1;color:#777}.h1,.h2,.h3,h1,h2,h3{margin-top:20px;margin-bottom:10px}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small{font-size:65%}.h4,.h5,.h6,h4,h5,h6{margin-top:10px;margin-bottom:10px}.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-size:75%}.h1,h1{font-size:36px}.h2,h2{font-size:30px}.h3,h3{font-size:24px}.h4,h4{font-size:18px}.h5,h5{font-size:14px}.h6,h6{font-size:12px}p{margin:0 0 10px}.lead{margin-bottom:20px;font-size:16px;font-weight:300;line-height:1.4}@media (min-width:768px){.lead{font-size:21px}}.small,small{font-size:85%}.mark,mark{padding:.2em;background-color:#fcf8e3}.text-left{text-align:left}.text-right{text-align:right}.text-center{text-align:center}.text-justify{text-align:justify}.text-nowrap{white-space:nowrap}.text-lowercase{text-transform:lowercase}.text-uppercase{text-transform:uppercase}.text-capitalize{text-transform:capitalize}.text-muted{color:#777}.text-primary{color:#337ab7}a.text-primary:focus,a.text-primary:hover{color:#286090}.text-success{color:#3c763d}a.text-success:focus,a.text-success:hover{color:#2b542c}.text-info{color:#31708f}a.text-info:focus,a.text-info:hover{color:#245269}.text-warning{color:#8a6d3b}a.text-warning:focus,a.text-warning:hover{color:#66512c}.text-danger{color:#a94442}a.text-danger:focus,a.text-danger:hover{color:#843534}.bg-primary{color:#fff;background-color:#337ab7}a.bg-primary:focus,a.bg-primary:hover{background-color:#286090}.bg-success{background-color:#dff0d8}a.bg-success:focus,a.bg-success:hover{background-color:#c1e2b3}.bg-info{background-color:#d9edf7}a.bg-info:focus,a.bg-info:hover{background-color:#afd9ee}.bg-warning{background-color:#fcf8e3}a.bg-warning:focus,a.bg-warning:hover{background-color:#f7ecb5}.bg-danger{background-color:#f2dede}a.bg-danger:focus,a.bg-danger:hover{background-color:#e4b9b9}.page-header{padding-bottom:9px;margin:40px 0 20px;border-bottom:1px solid #eee}ol,ul{margin-top:0;margin-bottom:10px}ol ol,ol ul,ul ol,ul ul{margin-bottom:0}.list-unstyled{padding-left:0;list-style:none}.list-inline{padding-left:0;margin-left:-5px;list-style:none}.list-inline>li{display:inline-block;padding-right:5px;padding-left:5px}dl{margin-top:0;margin-bottom:20px}dd,dt{line-height:1.42857143}dt{font-weight:700}dd{margin-left:0}@media (min-width:768px){.dl-horizontal dt{float:left;width:160px;overflow:hidden;clear:left;text-align:right;text-overflow:ellipsis;white-space:nowrap}.dl-horizontal dd{margin-left:180px}}abbr[data-original-title],abbr[title]{cursor:help;border-bottom:1px dotted #777}.initialism{font-size:90%;text-transform:uppercase}blockquote{padding:10px 20px;margin:0 0 20px;font-size:17.5px;border-left:5px solid #eee}blockquote ol:last-child,blockquote p:last-child,blockquote ul:last-child{margin-bottom:0}blockquote .small,blockquote footer,blockquote small{display:block;font-size:80%;line-height:1.42857143;color:#777}blockquote .small:before,blockquote footer:before,blockquote small:before{content:'\2014 \00A0'}.blockquote-reverse,blockquote.pull-right{padding-right:15px;padding-left:0;text-align:right;border-right:5px solid #eee;border-left:0}.blockquote-reverse .small:before,.blockquote-reverse footer:before,.blockquote-reverse small:before,blockquote.pull-right .small:before,blockquote.pull-right footer:before,blockquote.pull-right small:before{content:''}.blockquote-reverse .small:after,.blockquote-reverse footer:after,.blockquote-reverse small:after,blockquote.pull-right .small:after,blockquote.pull-right footer:after,blockquote.pull-right small:after{content:'\00A0 \2014'}address{margin-bottom:20px;font-style:normal;line-height:1.42857143}code,kbd,pre,samp{font-family:Menlo,Monaco,Consolas,"Courier New",monospace}code{padding:2px 4px;font-size:90%;color:#c7254e;background-color:#f9f2f4;border-radius:4px}kbd{padding:2px 4px;font-size:90%;color:#fff;background-color:#333;border-radius:3px;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.25);box-shadow:inset 0 -1px 0 rgba(0,0,0,.25)}kbd kbd{padding:0;font-size:100%;font-weight:700;-webkit-box-shadow:none;box-shadow:none}pre{display:block;padding:9.5px;margin:0 0 10px;font-size:13px;line-height:1.42857143;color:#333;word-break:break-all;word-wrap:break-word;background-color:#f5f5f5;border:1px solid #ccc;border-radius:4px}pre code{padding:0;font-size:inherit;color:inherit;white-space:pre-wrap;background-color:transparent;border-radius:0}.pre-scrollable{max-height:340px;overflow-y:scroll}.container{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}@media (min-width:768px){.container{width:750px}}@media (min-width:992px){.container{width:970px}}@media (min-width:1200px){.container{width:1170px}}.container-fluid{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}.row{margin-right:-15px;margin-left:-15px}.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9,.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9,.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9,.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{position:relative;min-height:1px;padding-right:15px;padding-left:15px}.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{float:left}.col-xs-12{width:100%}.col-xs-11{width:91.66666667%}.col-xs-10{width:83.33333333%}.col-xs-9{width:75%}.col-xs-8{width:66.66666667%}.col-xs-7{width:58.33333333%}.col-xs-6{width:50%}.col-xs-5{width:41.66666667%}.col-xs-4{width:33.33333333%}.col-xs-3{width:25%}.col-xs-2{width:16.66666667%}.col-xs-1{width:8.33333333%}.col-xs-pull-12{right:100%}.col-xs-pull-11{right:91.66666667%}.col-xs-pull-10{right:83.33333333%}.col-xs-pull-9{right:75%}.col-xs-pull-8{right:66.66666667%}.col-xs-pull-7{right:58.33333333%}.col-xs-pull-6{right:50%}.col-xs-pull-5{right:41.66666667%}.col-xs-pull-4{right:33.33333333%}.col-xs-pull-3{right:25%}.col-xs-pull-2{right:16.66666667%}.col-xs-pull-1{right:8.33333333%}.col-xs-pull-0{right:auto}.col-xs-push-12{left:100%}.col-xs-push-11{left:91.66666667%}.col-xs-push-10{left:83.33333333%}.col-xs-push-9{left:75%}.col-xs-push-8{left:66.66666667%}.col-xs-push-7{left:58.33333333%}.col-xs-push-6{left:50%}.col-xs-push-5{left:41.66666667%}.col-xs-push-4{left:33.33333333%}.col-xs-push-3{left:25%}.col-xs-push-2{left:16.66666667%}.col-xs-push-1{left:8.33333333%}.col-xs-push-0{left:auto}.col-xs-offset-12{margin-left:100%}.col-xs-offset-11{margin-left:91.66666667%}.col-xs-offset-10{margin-left:83.33333333%}.col-xs-offset-9{margin-left:75%}.col-xs-offset-8{margin-left:66.66666667%}.col-xs-offset-7{margin-left:58.33333333%}.col-xs-offset-6{margin-left:50%}.col-xs-offset-5{margin-left:41.66666667%}.col-xs-offset-4{margin-left:33.33333333%}.col-xs-offset-3{margin-left:25%}.col-xs-offset-2{margin-left:16.66666667%}.col-xs-offset-1{margin-left:8.33333333%}.col-xs-offset-0{margin-left:0}@media (min-width:768px){.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9{float:left}.col-sm-12{width:100%}.col-sm-11{width:91.66666667%}.col-sm-10{width:83.33333333%}.col-sm-9{width:75%}.col-sm-8{width:66.66666667%}.col-sm-7{width:58.33333333%}.col-sm-6{width:50%}.col-sm-5{width:41.66666667%}.col-sm-4{width:33.33333333%}.col-sm-3{width:25%}.col-sm-2{width:16.66666667%}.col-sm-1{width:8.33333333%}.col-sm-pull-12{right:100%}.col-sm-pull-11{right:91.66666667%}.col-sm-pull-10{right:83.33333333%}.col-sm-pull-9{right:75%}.col-sm-pull-8{right:66.66666667%}.col-sm-pull-7{right:58.33333333%}.col-sm-pull-6{right:50%}.col-sm-pull-5{right:41.66666667%}.col-sm-pull-4{right:33.33333333%}.col-sm-pull-3{right:25%}.col-sm-pull-2{right:16.66666667%}.col-sm-pull-1{right:8.33333333%}.col-sm-pull-0{right:auto}.col-sm-push-12{left:100%}.col-sm-push-11{left:91.66666667%}.col-sm-push-10{left:83.33333333%}.col-sm-push-9{left:75%}.col-sm-push-8{left:66.66666667%}.col-sm-push-7{left:58.33333333%}.col-sm-push-6{left:50%}.col-sm-push-5{left:41.66666667%}.col-sm-push-4{left:33.33333333%}.col-sm-push-3{left:25%}.col-sm-push-2{left:16.66666667%}.col-sm-push-1{left:8.33333333%}.col-sm-push-0{left:auto}.col-sm-offset-12{margin-left:100%}.col-sm-offset-11{margin-left:91.66666667%}.col-sm-offset-10{margin-left:83.33333333%}.col-sm-offset-9{margin-left:75%}.col-sm-offset-8{margin-left:66.66666667%}.col-sm-offset-7{margin-left:58.33333333%}.col-sm-offset-6{margin-left:50%}.col-sm-offset-5{margin-left:41.66666667%}.col-sm-offset-4{margin-left:33.33333333%}.col-sm-offset-3{margin-left:25%}.col-sm-offset-2{margin-left:16.66666667%}.col-sm-offset-1{margin-left:8.33333333%}.col-sm-offset-0{margin-left:0}}@media (min-width:992px){.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9{float:left}.col-md-12{width:100%}.col-md-11{width:91.66666667%}.col-md-10{width:83.33333333%}.col-md-9{width:75%}.col-md-8{width:66.66666667%}.col-md-7{width:58.33333333%}.col-md-6{width:50%}.col-md-5{width:41.66666667%}.col-md-4{width:33.33333333%}.col-md-3{width:25%}.col-md-2{width:16.66666667%}.col-md-1{width:8.33333333%}.col-md-pull-12{right:100%}.col-md-pull-11{right:91.66666667%}.col-md-pull-10{right:83.33333333%}.col-md-pull-9{right:75%}.col-md-pull-8{right:66.66666667%}.col-md-pull-7{right:58.33333333%}.col-md-pull-6{right:50%}.col-md-pull-5{right:41.66666667%}.col-md-pull-4{right:33.33333333%}.col-md-pull-3{right:25%}.col-md-pull-2{right:16.66666667%}.col-md-pull-1{right:8.33333333%}.col-md-pull-0{right:auto}.col-md-push-12{left:100%}.col-md-push-11{left:91.66666667%}.col-md-push-10{left:83.33333333%}.col-md-push-9{left:75%}.col-md-push-8{left:66.66666667%}.col-md-push-7{left:58.33333333%}.col-md-push-6{left:50%}.col-md-push-5{left:41.66666667%}.col-md-push-4{left:33.33333333%}.col-md-push-3{left:25%}.col-md-push-2{left:16.66666667%}.col-md-push-1{left:8.33333333%}.col-md-push-0{left:auto}.col-md-offset-12{margin-left:100%}.col-md-offset-11{margin-left:91.66666667%}.col-md-offset-10{margin-left:83.33333333%}.col-md-offset-9{margin-left:75%}.col-md-offset-8{margin-left:66.66666667%}.col-md-offset-7{margin-left:58.33333333%}.col-md-offset-6{margin-left:50%}.col-md-offset-5{margin-left:41.66666667%}.col-md-offset-4{margin-left:33.33333333%}.col-md-offset-3{margin-left:25%}.col-md-offset-2{margin-left:16.66666667%}.col-md-offset-1{margin-left:8.33333333%}.col-md-offset-0{margin-left:0}}@media (min-width:1200px){.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9{float:left}.col-lg-12{width:100%}.col-lg-11{width:91.66666667%}.col-lg-10{width:83.33333333%}.col-lg-9{width:75%}.col-lg-8{width:66.66666667%}.col-lg-7{width:58.33333333%}.col-lg-6{width:50%}.col-lg-5{width:41.66666667%}.col-lg-4{width:33.33333333%}.col-lg-3{width:25%}.col-lg-2{width:16.66666667%}.col-lg-1{width:8.33333333%}.col-lg-pull-12{right:100%}.col-lg-pull-11{right:91.66666667%}.col-lg-pull-10{right:83.33333333%}.col-lg-pull-9{right:75%}.col-lg-pull-8{right:66.66666667%}.col-lg-pull-7{right:58.33333333%}.col-lg-pull-6{right:50%}.col-lg-pull-5{right:41.66666667%}.col-lg-pull-4{right:33.33333333%}.col-lg-pull-3{right:25%}.col-lg-pull-2{right:16.66666667%}.col-lg-pull-1{right:8.33333333%}.col-lg-pull-0{right:auto}.col-lg-push-12{left:100%}.col-lg-push-11{left:91.66666667%}.col-lg-push-10{left:83.33333333%}.col-lg-push-9{left:75%}.col-lg-push-8{left:66.66666667%}.col-lg-push-7{left:58.33333333%}.col-lg-push-6{left:50%}.col-lg-push-5{left:41.66666667%}.col-lg-push-4{left:33.33333333%}.col-lg-push-3{left:25%}.col-lg-push-2{left:16.66666667%}.col-lg-push-1{left:8.33333333%}.col-lg-push-0{left:auto}.col-lg-offset-12{margin-left:100%}.col-lg-offset-11{margin-left:91.66666667%}.col-lg-offset-10{margin-left:83.33333333%}.col-lg-offset-9{margin-left:75%}.col-lg-offset-8{margin-left:66.66666667%}.col-lg-offset-7{margin-left:58.33333333%}.col-lg-offset-6{margin-left:50%}.col-lg-offset-5{margin-left:41.66666667%}.col-lg-offset-4{margin-left:33.33333333%}.col-lg-offset-3{margin-left:25%}.col-lg-offset-2{margin-left:16.66666667%}.col-lg-offset-1{margin-left:8.33333333%}.col-lg-offset-0{margin-left:0}}table{background-color:transparent}caption{padding-top:8px;padding-bottom:8px;color:#777;text-align:left}th{text-align:left}.table{width:100%;max-width:100%;margin-bottom:20px}.table>tbody>tr>td,.table>tbody>tr>th,.table>tfoot>tr>td,.table>tfoot>tr>th,.table>thead>tr>td,.table>thead>tr>th{padding:8px;line-height:1.42857143;vertical-align:top;border-top:1px solid #ddd}.table>thead>tr>th{vertical-align:bottom;border-bottom:2px solid #ddd}.table>caption+thead>tr:first-child>td,.table>caption+thead>tr:first-child>th,.table>colgroup+thead>tr:first-child>td,.table>colgroup+thead>tr:first-child>th,.table>thead:first-child>tr:first-child>td,.table>thead:first-child>tr:first-child>th{border-top:0}.table>tbody+tbody{border-top:2px solid #ddd}.table .table{background-color:#fff}.table-condensed>tbody>tr>td,.table-condensed>tbody>tr>th,.table-condensed>tfoot>tr>td,.table-condensed>tfoot>tr>th,.table-condensed>thead>tr>td,.table-condensed>thead>tr>th{padding:5px}.table-bordered{border:1px solid #ddd}.table-bordered>tbody>tr>td,.table-bordered>tbody>tr>th,.table-bordered>tfoot>tr>td,.table-bordered>tfoot>tr>th,.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border:1px solid #ddd}.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border-bottom-width:2px}.table-striped>tbody>tr:nth-of-type(odd){background-color:#f9f9f9}.table-hover>tbody>tr:hover{background-color:#f5f5f5}table col[class*=col-]{position:static;display:table-column;float:none}table td[class*=col-],table th[class*=col-]{position:static;display:table-cell;float:none}.table>tbody>tr.active>td,.table>tbody>tr.active>th,.table>tbody>tr>td.active,.table>tbody>tr>th.active,.table>tfoot>tr.active>td,.table>tfoot>tr.active>th,.table>tfoot>tr>td.active,.table>tfoot>tr>th.active,.table>thead>tr.active>td,.table>thead>tr.active>th,.table>thead>tr>td.active,.table>thead>tr>th.active{background-color:#f5f5f5}.table-hover>tbody>tr.active:hover>td,.table-hover>tbody>tr.active:hover>th,.table-hover>tbody>tr:hover>.active,.table-hover>tbody>tr>td.active:hover,.table-hover>tbody>tr>th.active:hover{background-color:#e8e8e8}.table>tbody>tr.success>td,.table>tbody>tr.success>th,.table>tbody>tr>td.success,.table>tbody>tr>th.success,.table>tfoot>tr.success>td,.table>tfoot>tr.success>th,.table>tfoot>tr>td.success,.table>tfoot>tr>th.success,.table>thead>tr.success>td,.table>thead>tr.success>th,.table>thead>tr>td.success,.table>thead>tr>th.success{background-color:#dff0d8}.table-hover>tbody>tr.success:hover>td,.table-hover>tbody>tr.success:hover>th,.table-hover>tbody>tr:hover>.success,.table-hover>tbody>tr>td.success:hover,.table-hover>tbody>tr>th.success:hover{background-color:#d0e9c6}.table>tbody>tr.info>td,.table>tbody>tr.info>th,.table>tbody>tr>td.info,.table>tbody>tr>th.info,.table>tfoot>tr.info>td,.table>tfoot>tr.info>th,.table>tfoot>tr>td.info,.table>tfoot>tr>th.info,.table>thead>tr.info>td,.table>thead>tr.info>th,.table>thead>tr>td.info,.table>thead>tr>th.info{background-color:#d9edf7}.table-hover>tbody>tr.info:hover>td,.table-hover>tbody>tr.info:hover>th,.table-hover>tbody>tr:hover>.info,.table-hover>tbody>tr>td.info:hover,.table-hover>tbody>tr>th.info:hover{background-color:#c4e3f3}.table>tbody>tr.warning>td,.table>tbody>tr.warning>th,.table>tbody>tr>td.warning,.table>tbody>tr>th.warning,.table>tfoot>tr.warning>td,.table>tfoot>tr.warning>th,.table>tfoot>tr>td.warning,.table>tfoot>tr>th.warning,.table>thead>tr.warning>td,.table>thead>tr.warning>th,.table>thead>tr>td.warning,.table>thead>tr>th.warning{background-color:#fcf8e3}.table-hover>tbody>tr.warning:hover>td,.table-hover>tbody>tr.warning:hover>th,.table-hover>tbody>tr:hover>.warning,.table-hover>tbody>tr>td.warning:hover,.table-hover>tbody>tr>th.warning:hover{background-color:#faf2cc}.table>tbody>tr.danger>td,.table>tbody>tr.danger>th,.table>tbody>tr>td.danger,.table>tbody>tr>th.danger,.table>tfoot>tr.danger>td,.table>tfoot>tr.danger>th,.table>tfoot>tr>td.danger,.table>tfoot>tr>th.danger,.table>thead>tr.danger>td,.table>thead>tr.danger>th,.table>thead>tr>td.danger,.table>thead>tr>th.danger{background-color:#f2dede}.table-hover>tbody>tr.danger:hover>td,.table-hover>tbody>tr.danger:hover>th,.table-hover>tbody>tr:hover>.danger,.table-hover>tbody>tr>td.danger:hover,.table-hover>tbody>tr>th.danger:hover{background-color:#ebcccc}.table-responsive{min-height:.01%;overflow-x:auto}@media screen and (max-width:767px){.table-responsive{width:100%;margin-bottom:15px;overflow-y:hidden;-ms-overflow-style:-ms-autohiding-scrollbar;border:1px solid #ddd}.table-responsive>.table{margin-bottom:0}.table-responsive>.table>tbody>tr>td,.table-responsive>.table>tbody>tr>th,.table-responsive>.table>tfoot>tr>td,.table-responsive>.table>tfoot>tr>th,.table-responsive>.table>thead>tr>td,.table-responsive>.table>thead>tr>th{white-space:nowrap}.table-responsive>.table-bordered{border:0}.table-responsive>.table-bordered>tbody>tr>td:first-child,.table-responsive>.table-bordered>tbody>tr>th:first-child,.table-responsive>.table-bordered>tfoot>tr>td:first-child,.table-responsive>.table-bordered>tfoot>tr>th:first-child,.table-responsive>.table-bordered>thead>tr>td:first-child,.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.table-responsive>.table-bordered>tbody>tr>td:last-child,.table-responsive>.table-bordered>tbody>tr>th:last-child,.table-responsive>.table-bordered>tfoot>tr>td:last-child,.table-responsive>.table-bordered>tfoot>tr>th:last-child,.table-responsive>.table-bordered>thead>tr>td:last-child,.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.table-responsive>.table-bordered>tbody>tr:last-child>td,.table-responsive>.table-bordered>tbody>tr:last-child>th,.table-responsive>.table-bordered>tfoot>tr:last-child>td,.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}}fieldset{min-width:0;padding:0;margin:0;border:0}legend{display:block;width:100%;padding:0;margin-bottom:20px;font-size:21px;line-height:inherit;color:#333;border:0;border-bottom:1px solid #e5e5e5}label{display:inline-block;max-width:100%;margin-bottom:5px;font-weight:700}input[type=search]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}input[type=checkbox],input[type=radio]{margin:4px 0 0;margin-top:1px\9;line-height:normal}input[type=file]{display:block}input[type=range]{display:block;width:100%}select[multiple],select[size]{height:auto}input[type=file]:focus,input[type=checkbox]:focus,input[type=radio]:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}output{display:block;padding-top:7px;font-size:14px;line-height:1.42857143;color:#555}.form-control{display:block;width:100%;height:34px;padding:6px 12px;font-size:14px;line-height:1.42857143;color:#555;background-color:#fff;background-image:none;border:1px solid #ccc;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075);-webkit-transition:border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;-o-transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s;transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s}.form-control:focus{border-color:#66afe9;outline:0;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6);box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6)}.form-control::-moz-placeholder{color:#999;opacity:1}.form-control:-ms-input-placeholder{color:#999}.form-control::-webkit-input-placeholder{color:#999}.form-control::-ms-expand{background-color:transparent;border:0}.form-control[disabled],.form-control[readonly],fieldset[disabled] .form-control{background-color:#eee;opacity:1}.form-control[disabled],fieldset[disabled] .form-control{cursor:not-allowed}textarea.form-control{height:auto}input[type=search]{-webkit-appearance:none}@media screen and (-webkit-min-device-pixel-ratio:0){input[type=date].form-control,input[type=time].form-control,input[type=datetime-local].form-control,input[type=month].form-control{line-height:34px}.input-group-sm input[type=date],.input-group-sm input[type=time],.input-group-sm input[type=datetime-local],.input-group-sm input[type=month],input[type=date].input-sm,input[type=time].input-sm,input[type=datetime-local].input-sm,input[type=month].input-sm{line-height:30px}.input-group-lg input[type=date],.input-group-lg input[type=time],.input-group-lg input[type=datetime-local],.input-group-lg input[type=month],input[type=date].input-lg,input[type=time].input-lg,input[type=datetime-local].input-lg,input[type=month].input-lg{line-height:46px}}.form-group{margin-bottom:15px}.checkbox,.radio{position:relative;display:block;margin-top:10px;margin-bottom:10px}.checkbox label,.radio label{min-height:20px;padding-left:20px;margin-bottom:0;font-weight:400;cursor:pointer}.checkbox input[type=checkbox],.checkbox-inline input[type=checkbox],.radio input[type=radio],.radio-inline input[type=radio]{position:absolute;margin-top:4px\9;margin-left:-20px}.checkbox+.checkbox,.radio+.radio{margin-top:-5px}.checkbox-inline,.radio-inline{position:relative;display:inline-block;padding-left:20px;margin-bottom:0;font-weight:400;vertical-align:middle;cursor:pointer}.checkbox-inline+.checkbox-inline,.radio-inline+.radio-inline{margin-top:0;margin-left:10px}fieldset[disabled] input[type=checkbox],fieldset[disabled] input[type=radio],input[type=checkbox].disabled,input[type=checkbox][disabled],input[type=radio].disabled,input[type=radio][disabled]{cursor:not-allowed}.checkbox-inline.disabled,.radio-inline.disabled,fieldset[disabled] .checkbox-inline,fieldset[disabled] .radio-inline{cursor:not-allowed}.checkbox.disabled label,.radio.disabled label,fieldset[disabled] .checkbox label,fieldset[disabled] .radio label{cursor:not-allowed}.form-control-static{min-height:34px;padding-top:7px;padding-bottom:7px;margin-bottom:0}.form-control-static.input-lg,.form-control-static.input-sm{padding-right:0;padding-left:0}.input-sm{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-sm{height:30px;line-height:30px}select[multiple].input-sm,textarea.input-sm{height:auto}.form-group-sm .form-control{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.form-group-sm select.form-control{height:30px;line-height:30px}.form-group-sm select[multiple].form-control,.form-group-sm textarea.form-control{height:auto}.form-group-sm .form-control-static{height:30px;min-height:32px;padding:6px 10px;font-size:12px;line-height:1.5}.input-lg{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-lg{height:46px;line-height:46px}select[multiple].input-lg,textarea.input-lg{height:auto}.form-group-lg .form-control{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.form-group-lg select.form-control{height:46px;line-height:46px}.form-group-lg select[multiple].form-control,.form-group-lg textarea.form-control{height:auto}.form-group-lg .form-control-static{height:46px;min-height:38px;padding:11px 16px;font-size:18px;line-height:1.3333333}.has-feedback{position:relative}.has-feedback .form-control{padding-right:42.5px}.form-control-feedback{position:absolute;top:0;right:0;z-index:2;display:block;width:34px;height:34px;line-height:34px;text-align:center;pointer-events:none}.form-group-lg .form-control+.form-control-feedback,.input-group-lg+.form-control-feedback,.input-lg+.form-control-feedback{width:46px;height:46px;line-height:46px}.form-group-sm .form-control+.form-control-feedback,.input-group-sm+.form-control-feedback,.input-sm+.form-control-feedback{width:30px;height:30px;line-height:30px}.has-success .checkbox,.has-success .checkbox-inline,.has-success .control-label,.has-success .help-block,.has-success .radio,.has-success .radio-inline,.has-success.checkbox label,.has-success.checkbox-inline label,.has-success.radio label,.has-success.radio-inline label{color:#3c763d}.has-success .form-control{border-color:#3c763d;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-success .form-control:focus{border-color:#2b542c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168}.has-success .input-group-addon{color:#3c763d;background-color:#dff0d8;border-color:#3c763d}.has-success .form-control-feedback{color:#3c763d}.has-warning .checkbox,.has-warning .checkbox-inline,.has-warning .control-label,.has-warning .help-block,.has-warning .radio,.has-warning .radio-inline,.has-warning.checkbox label,.has-warning.checkbox-inline label,.has-warning.radio label,.has-warning.radio-inline label{color:#8a6d3b}.has-warning .form-control{border-color:#8a6d3b;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-warning .form-control:focus{border-color:#66512c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b}.has-warning .input-group-addon{color:#8a6d3b;background-color:#fcf8e3;border-color:#8a6d3b}.has-warning .form-control-feedback{color:#8a6d3b}.has-error .checkbox,.has-error .checkbox-inline,.has-error .control-label,.has-error .help-block,.has-error .radio,.has-error .radio-inline,.has-error.checkbox label,.has-error.checkbox-inline label,.has-error.radio label,.has-error.radio-inline label{color:#a94442}.has-error .form-control{border-color:#a94442;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-error .form-control:focus{border-color:#843534;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483}.has-error .input-group-addon{color:#a94442;background-color:#f2dede;border-color:#a94442}.has-error .form-control-feedback{color:#a94442}.has-feedback label~.form-control-feedback{top:25px}.has-feedback label.sr-only~.form-control-feedback{top:0}.help-block{display:block;margin-top:5px;margin-bottom:10px;color:#737373}@media (min-width:768px){.form-inline .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.form-inline .form-control{display:inline-block;width:auto;vertical-align:middle}.form-inline .form-control-static{display:inline-block}.form-inline .input-group{display:inline-table;vertical-align:middle}.form-inline .input-group .form-control,.form-inline .input-group .input-group-addon,.form-inline .input-group .input-group-btn{width:auto}.form-inline .input-group>.form-control{width:100%}.form-inline .control-label{margin-bottom:0;vertical-align:middle}.form-inline .checkbox,.form-inline .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.form-inline .checkbox label,.form-inline .radio label{padding-left:0}.form-inline .checkbox input[type=checkbox],.form-inline .radio input[type=radio]{position:relative;margin-left:0}.form-inline .has-feedback .form-control-feedback{top:0}}.form-horizontal .checkbox,.form-horizontal .checkbox-inline,.form-horizontal .radio,.form-horizontal .radio-inline{padding-top:7px;margin-top:0;margin-bottom:0}.form-horizontal .checkbox,.form-horizontal .radio{min-height:27px}.form-horizontal .form-group{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.form-horizontal .control-label{padding-top:7px;margin-bottom:0;text-align:right}}.form-horizontal .has-feedback .form-control-feedback{right:15px}@media (min-width:768px){.form-horizontal .form-group-lg .control-label{padding-top:11px;font-size:18px}}@media (min-width:768px){.form-horizontal .form-group-sm .control-label{padding-top:6px;font-size:12px}}.btn{display:inline-block;padding:6px 12px;margin-bottom:0;font-size:14px;font-weight:400;line-height:1.42857143;text-align:center;white-space:nowrap;vertical-align:middle;-ms-touch-action:manipulation;touch-action:manipulation;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;background-image:none;border:1px solid transparent;border-radius:4px}.btn.active.focus,.btn.active:focus,.btn.focus,.btn:active.focus,.btn:active:focus,.btn:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}.btn.focus,.btn:focus,.btn:hover{color:#333;text-decoration:none}.btn.active,.btn:active{background-image:none;outline:0;-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn.disabled,.btn[disabled],fieldset[disabled] .btn{cursor:not-allowed;filter:alpha(opacity=65);-webkit-box-shadow:none;box-shadow:none;opacity:.65}a.btn.disabled,fieldset[disabled] a.btn{pointer-events:none}.btn-default{color:#333;background-color:#fff;border-color:#ccc}.btn-default.focus,.btn-default:focus{color:#333;background-color:#e6e6e6;border-color:#8c8c8c}.btn-default:hover{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active.focus,.btn-default.active:focus,.btn-default.active:hover,.btn-default:active.focus,.btn-default:active:focus,.btn-default:active:hover,.open>.dropdown-toggle.btn-default.focus,.open>.dropdown-toggle.btn-default:focus,.open>.dropdown-toggle.btn-default:hover{color:#333;background-color:#d4d4d4;border-color:#8c8c8c}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{background-image:none}.btn-default.disabled.focus,.btn-default.disabled:focus,.btn-default.disabled:hover,.btn-default[disabled].focus,.btn-default[disabled]:focus,.btn-default[disabled]:hover,fieldset[disabled] .btn-default.focus,fieldset[disabled] .btn-default:focus,fieldset[disabled] .btn-default:hover{background-color:#fff;border-color:#ccc}.btn-default .badge{color:#fff;background-color:#333}.btn-primary{color:#fff;background-color:#337ab7;border-color:#2e6da4}.btn-primary.focus,.btn-primary:focus{color:#fff;background-color:#286090;border-color:#122b40}.btn-primary:hover{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active.focus,.btn-primary.active:focus,.btn-primary.active:hover,.btn-primary:active.focus,.btn-primary:active:focus,.btn-primary:active:hover,.open>.dropdown-toggle.btn-primary.focus,.open>.dropdown-toggle.btn-primary:focus,.open>.dropdown-toggle.btn-primary:hover{color:#fff;background-color:#204d74;border-color:#122b40}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{background-image:none}.btn-primary.disabled.focus,.btn-primary.disabled:focus,.btn-primary.disabled:hover,.btn-primary[disabled].focus,.btn-primary[disabled]:focus,.btn-primary[disabled]:hover,fieldset[disabled] .btn-primary.focus,fieldset[disabled] .btn-primary:focus,fieldset[disabled] .btn-primary:hover{background-color:#337ab7;border-color:#2e6da4}.btn-primary .badge{color:#337ab7;background-color:#fff}.btn-success{color:#fff;background-color:#5cb85c;border-color:#4cae4c}.btn-success.focus,.btn-success:focus{color:#fff;background-color:#449d44;border-color:#255625}.btn-success:hover{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active.focus,.btn-success.active:focus,.btn-success.active:hover,.btn-success:active.focus,.btn-success:active:focus,.btn-success:active:hover,.open>.dropdown-toggle.btn-success.focus,.open>.dropdown-toggle.btn-success:focus,.open>.dropdown-toggle.btn-success:hover{color:#fff;background-color:#398439;border-color:#255625}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{background-image:none}.btn-success.disabled.focus,.btn-success.disabled:focus,.btn-success.disabled:hover,.btn-success[disabled].focus,.btn-success[disabled]:focus,.btn-success[disabled]:hover,fieldset[disabled] .btn-success.focus,fieldset[disabled] .btn-success:focus,fieldset[disabled] .btn-success:hover{background-color:#5cb85c;border-color:#4cae4c}.btn-success .badge{color:#5cb85c;background-color:#fff}.btn-info{color:#fff;background-color:#5bc0de;border-color:#46b8da}.btn-info.focus,.btn-info:focus{color:#fff;background-color:#31b0d5;border-color:#1b6d85}.btn-info:hover{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active.focus,.btn-info.active:focus,.btn-info.active:hover,.btn-info:active.focus,.btn-info:active:focus,.btn-info:active:hover,.open>.dropdown-toggle.btn-info.focus,.open>.dropdown-toggle.btn-info:focus,.open>.dropdown-toggle.btn-info:hover{color:#fff;background-color:#269abc;border-color:#1b6d85}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{background-image:none}.btn-info.disabled.focus,.btn-info.disabled:focus,.btn-info.disabled:hover,.btn-info[disabled].focus,.btn-info[disabled]:focus,.btn-info[disabled]:hover,fieldset[disabled] .btn-info.focus,fieldset[disabled] .btn-info:focus,fieldset[disabled] .btn-info:hover{background-color:#5bc0de;border-color:#46b8da}.btn-info .badge{color:#5bc0de;background-color:#fff}.btn-warning{color:#fff;background-color:#f0ad4e;border-color:#eea236}.btn-warning.focus,.btn-warning:focus{color:#fff;background-color:#ec971f;border-color:#985f0d}.btn-warning:hover{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active.focus,.btn-warning.active:focus,.btn-warning.active:hover,.btn-warning:active.focus,.btn-warning:active:focus,.btn-warning:active:hover,.open>.dropdown-toggle.btn-warning.focus,.open>.dropdown-toggle.btn-warning:focus,.open>.dropdown-toggle.btn-warning:hover{color:#fff;background-color:#d58512;border-color:#985f0d}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{background-image:none}.btn-warning.disabled.focus,.btn-warning.disabled:focus,.btn-warning.disabled:hover,.btn-warning[disabled].focus,.btn-warning[disabled]:focus,.btn-warning[disabled]:hover,fieldset[disabled] .btn-warning.focus,fieldset[disabled] .btn-warning:focus,fieldset[disabled] .btn-warning:hover{background-color:#f0ad4e;border-color:#eea236}.btn-warning .badge{color:#f0ad4e;background-color:#fff}.btn-danger{color:#fff;background-color:#d9534f;border-color:#d43f3a}.btn-danger.focus,.btn-danger:focus{color:#fff;background-color:#c9302c;border-color:#761c19}.btn-danger:hover{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active.focus,.btn-danger.active:focus,.btn-danger.active:hover,.btn-danger:active.focus,.btn-danger:active:focus,.btn-danger:active:hover,.open>.dropdown-toggle.btn-danger.focus,.open>.dropdown-toggle.btn-danger:focus,.open>.dropdown-toggle.btn-danger:hover{color:#fff;background-color:#ac2925;border-color:#761c19}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{background-image:none}.btn-danger.disabled.focus,.btn-danger.disabled:focus,.btn-danger.disabled:hover,.btn-danger[disabled].focus,.btn-danger[disabled]:focus,.btn-danger[disabled]:hover,fieldset[disabled] .btn-danger.focus,fieldset[disabled] .btn-danger:focus,fieldset[disabled] .btn-danger:hover{background-color:#d9534f;border-color:#d43f3a}.btn-danger .badge{color:#d9534f;background-color:#fff}.btn-link{font-weight:400;color:#337ab7;border-radius:0}.btn-link,.btn-link.active,.btn-link:active,.btn-link[disabled],fieldset[disabled] .btn-link{background-color:transparent;-webkit-box-shadow:none;box-shadow:none}.btn-link,.btn-link:active,.btn-link:focus,.btn-link:hover{border-color:transparent}.btn-link:focus,.btn-link:hover{color:#23527c;text-decoration:underline;background-color:transparent}.btn-link[disabled]:focus,.btn-link[disabled]:hover,fieldset[disabled] .btn-link:focus,fieldset[disabled] .btn-link:hover{color:#777;text-decoration:none}.btn-group-lg>.btn,.btn-lg{padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.btn-group-sm>.btn,.btn-sm{padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.btn-group-xs>.btn,.btn-xs{padding:1px 5px;font-size:12px;line-height:1.5;border-radius:3px}.btn-block{display:block;width:100%}.btn-block+.btn-block{margin-top:5px}input[type=button].btn-block,input[type=reset].btn-block,input[type=submit].btn-block{width:100%}.fade{opacity:0;-webkit-transition:opacity .15s linear;-o-transition:opacity .15s linear;transition:opacity .15s linear}.fade.in{opacity:1}.collapse{display:none}.collapse.in{display:block}tr.collapse.in{display:table-row}tbody.collapse.in{display:table-row-group}.collapsing{position:relative;height:0;overflow:hidden;-webkit-transition-timing-function:ease;-o-transition-timing-function:ease;transition-timing-function:ease;-webkit-transition-duration:.35s;-o-transition-duration:.35s;transition-duration:.35s;-webkit-transition-property:height,visibility;-o-transition-property:height,visibility;transition-property:height,visibility}.caret{display:inline-block;width:0;height:0;margin-left:2px;vertical-align:middle;border-top:4px dashed;border-top:4px solid\9;border-right:4px solid transparent;border-left:4px solid transparent}.dropdown,.dropup{position:relative}.dropdown-toggle:focus{outline:0}.dropdown-menu{position:absolute;top:100%;left:0;z-index:1000;display:none;float:left;min-width:160px;padding:5px 0;margin:2px 0 0;font-size:14px;text-align:left;list-style:none;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.15);border-radius:4px;-webkit-box-shadow:0 6px 12px rgba(0,0,0,.175);box-shadow:0 6px 12px rgba(0,0,0,.175)}.dropdown-menu.pull-right{right:0;left:auto}.dropdown-menu .divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.dropdown-menu>li>a{display:block;padding:3px 20px;clear:both;font-weight:400;line-height:1.42857143;color:#333;white-space:nowrap}.dropdown-menu>li>a:focus,.dropdown-menu>li>a:hover{color:#262626;text-decoration:none;background-color:#f5f5f5}.dropdown-menu>.active>a,.dropdown-menu>.active>a:focus,.dropdown-menu>.active>a:hover{color:#fff;text-decoration:none;background-color:#337ab7;outline:0}.dropdown-menu>.disabled>a,.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{color:#777}.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{text-decoration:none;cursor:not-allowed;background-color:transparent;background-image:none;filter:progid:DXImageTransform.Microsoft.gradient(enabled=false)}.open>.dropdown-menu{display:block}.open>a{outline:0}.dropdown-menu-right{right:0;left:auto}.dropdown-menu-left{right:auto;left:0}.dropdown-header{display:block;padding:3px 20px;font-size:12px;line-height:1.42857143;color:#777;white-space:nowrap}.dropdown-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:990}.pull-right>.dropdown-menu{right:0;left:auto}.dropup .caret,.navbar-fixed-bottom .dropdown .caret{content:"";border-top:0;border-bottom:4px dashed;border-bottom:4px solid\9}.dropup .dropdown-menu,.navbar-fixed-bottom .dropdown .dropdown-menu{top:auto;bottom:100%;margin-bottom:2px}@media (min-width:768px){.navbar-right .dropdown-menu{right:0;left:auto}.navbar-right .dropdown-menu-left{right:auto;left:0}}.btn-group,.btn-group-vertical{position:relative;display:inline-block;vertical-align:middle}.btn-group-vertical>.btn,.btn-group>.btn{position:relative;float:left}.btn-group-vertical>.btn.active,.btn-group-vertical>.btn:active,.btn-group-vertical>.btn:focus,.btn-group-vertical>.btn:hover,.btn-group>.btn.active,.btn-group>.btn:active,.btn-group>.btn:focus,.btn-group>.btn:hover{z-index:2}.btn-group .btn+.btn,.btn-group .btn+.btn-group,.btn-group .btn-group+.btn,.btn-group .btn-group+.btn-group{margin-left:-1px}.btn-toolbar{margin-left:-5px}.btn-toolbar .btn,.btn-toolbar .btn-group,.btn-toolbar .input-group{float:left}.btn-toolbar>.btn,.btn-toolbar>.btn-group,.btn-toolbar>.input-group{margin-left:5px}.btn-group>.btn:not(:first-child):not(:last-child):not(.dropdown-toggle){border-radius:0}.btn-group>.btn:first-child{margin-left:0}.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn:last-child:not(:first-child),.btn-group>.dropdown-toggle:not(:first-child){border-top-left-radius:0;border-bottom-left-radius:0}.btn-group>.btn-group{float:left}.btn-group>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-bottom-left-radius:0}.btn-group .dropdown-toggle:active,.btn-group.open .dropdown-toggle{outline:0}.btn-group>.btn+.dropdown-toggle{padding-right:8px;padding-left:8px}.btn-group>.btn-lg+.dropdown-toggle{padding-right:12px;padding-left:12px}.btn-group.open .dropdown-toggle{-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn-group.open .dropdown-toggle.btn-link{-webkit-box-shadow:none;box-shadow:none}.btn .caret{margin-left:0}.btn-lg .caret{border-width:5px 5px 0;border-bottom-width:0}.dropup .btn-lg .caret{border-width:0 5px 5px}.btn-group-vertical>.btn,.btn-group-vertical>.btn-group,.btn-group-vertical>.btn-group>.btn{display:block;float:none;width:100%;max-width:100%}.btn-group-vertical>.btn-group>.btn{float:none}.btn-group-vertical>.btn+.btn,.btn-group-vertical>.btn+.btn-group,.btn-group-vertical>.btn-group+.btn,.btn-group-vertical>.btn-group+.btn-group{margin-top:-1px;margin-left:0}.btn-group-vertical>.btn:not(:first-child):not(:last-child){border-radius:0}.btn-group-vertical>.btn:first-child:not(:last-child){border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn:last-child:not(:first-child){border-top-left-radius:0;border-top-right-radius:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}.btn-group-vertical>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group-vertical>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group-vertical>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-top-right-radius:0}.btn-group-justified{display:table;width:100%;table-layout:fixed;border-collapse:separate}.btn-group-justified>.btn,.btn-group-justified>.btn-group{display:table-cell;float:none;width:1%}.btn-group-justified>.btn-group .btn{width:100%}.btn-group-justified>.btn-group .dropdown-menu{left:auto}[data-toggle=buttons]>.btn input[type=checkbox],[data-toggle=buttons]>.btn input[type=radio],[data-toggle=buttons]>.btn-group>.btn input[type=checkbox],[data-toggle=buttons]>.btn-group>.btn input[type=radio]{position:absolute;clip:rect(0,0,0,0);pointer-events:none}.input-group{position:relative;display:table;border-collapse:separate}.input-group[class*=col-]{float:none;padding-right:0;padding-left:0}.input-group .form-control{position:relative;z-index:2;float:left;width:100%;margin-bottom:0}.input-group .form-control:focus{z-index:3}.input-group-lg>.form-control,.input-group-lg>.input-group-addon,.input-group-lg>.input-group-btn>.btn{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-group-lg>.form-control,select.input-group-lg>.input-group-addon,select.input-group-lg>.input-group-btn>.btn{height:46px;line-height:46px}select[multiple].input-group-lg>.form-control,select[multiple].input-group-lg>.input-group-addon,select[multiple].input-group-lg>.input-group-btn>.btn,textarea.input-group-lg>.form-control,textarea.input-group-lg>.input-group-addon,textarea.input-group-lg>.input-group-btn>.btn{height:auto}.input-group-sm>.form-control,.input-group-sm>.input-group-addon,.input-group-sm>.input-group-btn>.btn{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-group-sm>.form-control,select.input-group-sm>.input-group-addon,select.input-group-sm>.input-group-btn>.btn{height:30px;line-height:30px}select[multiple].input-group-sm>.form-control,select[multiple].input-group-sm>.input-group-addon,select[multiple].input-group-sm>.input-group-btn>.btn,textarea.input-group-sm>.form-control,textarea.input-group-sm>.input-group-addon,textarea.input-group-sm>.input-group-btn>.btn{height:auto}.input-group .form-control,.input-group-addon,.input-group-btn{display:table-cell}.input-group .form-control:not(:first-child):not(:last-child),.input-group-addon:not(:first-child):not(:last-child),.input-group-btn:not(:first-child):not(:last-child){border-radius:0}.input-group-addon,.input-group-btn{width:1%;white-space:nowrap;vertical-align:middle}.input-group-addon{padding:6px 12px;font-size:14px;font-weight:400;line-height:1;color:#555;text-align:center;background-color:#eee;border:1px solid #ccc;border-radius:4px}.input-group-addon.input-sm{padding:5px 10px;font-size:12px;border-radius:3px}.input-group-addon.input-lg{padding:10px 16px;font-size:18px;border-radius:6px}.input-group-addon input[type=checkbox],.input-group-addon input[type=radio]{margin-top:0}.input-group .form-control:first-child,.input-group-addon:first-child,.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group>.btn,.input-group-btn:first-child>.dropdown-toggle,.input-group-btn:last-child>.btn-group:not(:last-child)>.btn,.input-group-btn:last-child>.btn:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.input-group-addon:first-child{border-right:0}.input-group .form-control:last-child,.input-group-addon:last-child,.input-group-btn:first-child>.btn-group:not(:first-child)>.btn,.input-group-btn:first-child>.btn:not(:first-child),.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group>.btn,.input-group-btn:last-child>.dropdown-toggle{border-top-left-radius:0;border-bottom-left-radius:0}.input-group-addon:last-child{border-left:0}.input-group-btn{position:relative;font-size:0;white-space:nowrap}.input-group-btn>.btn{position:relative}.input-group-btn>.btn+.btn{margin-left:-1px}.input-group-btn>.btn:active,.input-group-btn>.btn:focus,.input-group-btn>.btn:hover{z-index:2}.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group{margin-right:-1px}.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group{z-index:2;margin-left:-1px}.nav{padding-left:0;margin-bottom:0;list-style:none}.nav>li{position:relative;display:block}.nav>li>a{position:relative;display:block;padding:10px 15px}.nav>li>a:focus,.nav>li>a:hover{text-decoration:none;background-color:#eee}.nav>li.disabled>a{color:#777}.nav>li.disabled>a:focus,.nav>li.disabled>a:hover{color:#777;text-decoration:none;cursor:not-allowed;background-color:transparent}.nav .open>a,.nav .open>a:focus,.nav .open>a:hover{background-color:#eee;border-color:#337ab7}.nav .nav-divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.nav>li>a>img{max-width:none}.nav-tabs{border-bottom:1px solid #ddd}.nav-tabs>li{float:left;margin-bottom:-1px}.nav-tabs>li>a{margin-right:2px;line-height:1.42857143;border:1px solid transparent;border-radius:4px 4px 0 0}.nav-tabs>li>a:hover{border-color:#eee #eee #ddd}.nav-tabs>li.active>a,.nav-tabs>li.active>a:focus,.nav-tabs>li.active>a:hover{color:#555;cursor:default;background-color:#fff;border:1px solid #ddd;border-bottom-color:transparent}.nav-tabs.nav-justified{width:100%;border-bottom:0}.nav-tabs.nav-justified>li{float:none}.nav-tabs.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-tabs.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-tabs.nav-justified>li{display:table-cell;width:1%}.nav-tabs.nav-justified>li>a{margin-bottom:0}}.nav-tabs.nav-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs.nav-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border-bottom-color:#fff}}.nav-pills>li{float:left}.nav-pills>li>a{border-radius:4px}.nav-pills>li+li{margin-left:2px}.nav-pills>li.active>a,.nav-pills>li.active>a:focus,.nav-pills>li.active>a:hover{color:#fff;background-color:#337ab7}.nav-stacked>li{float:none}.nav-stacked>li+li{margin-top:2px;margin-left:0}.nav-justified{width:100%}.nav-justified>li{float:none}.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-justified>li{display:table-cell;width:1%}.nav-justified>li>a{margin-bottom:0}}.nav-tabs-justified{border-bottom:0}.nav-tabs-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border-bottom-color:#fff}}.tab-content>.tab-pane{display:none}.tab-content>.active{display:block}.nav-tabs .dropdown-menu{margin-top:-1px;border-top-left-radius:0;border-top-right-radius:0}.navbar{position:relative;min-height:50px;margin-bottom:20px;border:1px solid transparent}@media (min-width:768px){.navbar{border-radius:4px}}@media (min-width:768px){.navbar-header{float:left}}.navbar-collapse{padding-right:15px;padding-left:15px;overflow-x:visible;-webkit-overflow-scrolling:touch;border-top:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1)}.navbar-collapse.in{overflow-y:auto}@media (min-width:768px){.navbar-collapse{width:auto;border-top:0;-webkit-box-shadow:none;box-shadow:none}.navbar-collapse.collapse{display:block!important;height:auto!important;padding-bottom:0;overflow:visible!important}.navbar-collapse.in{overflow-y:visible}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse,.navbar-static-top .navbar-collapse{padding-right:0;padding-left:0}}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:340px}@media (max-device-width:480px) and (orientation:landscape){.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:200px}}.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:0;margin-left:0}}.navbar-static-top{z-index:1000;border-width:0 0 1px}@media (min-width:768px){.navbar-static-top{border-radius:0}}.navbar-fixed-bottom,.navbar-fixed-top{position:fixed;right:0;left:0;z-index:1030}@media (min-width:768px){.navbar-fixed-bottom,.navbar-fixed-top{border-radius:0}}.navbar-fixed-top{top:0;border-width:0 0 1px}.navbar-fixed-bottom{bottom:0;margin-bottom:0;border-width:1px 0 0}.navbar-brand{float:left;height:50px;padding:15px 15px;font-size:18px;line-height:20px}.navbar-brand:focus,.navbar-brand:hover{text-decoration:none}.navbar-brand>img{display:block}@media (min-width:768px){.navbar>.container .navbar-brand,.navbar>.container-fluid .navbar-brand{margin-left:-15px}}.navbar-toggle{position:relative;float:right;padding:9px 10px;margin-top:8px;margin-right:15px;margin-bottom:8px;background-color:transparent;background-image:none;border:1px solid transparent;border-radius:4px}.navbar-toggle:focus{outline:0}.navbar-toggle .icon-bar{display:block;width:22px;height:2px;border-radius:1px}.navbar-toggle .icon-bar+.icon-bar{margin-top:4px}@media (min-width:768px){.navbar-toggle{display:none}}.navbar-nav{margin:7.5px -15px}.navbar-nav>li>a{padding-top:10px;padding-bottom:10px;line-height:20px}@media (max-width:767px){.navbar-nav .open .dropdown-menu{position:static;float:none;width:auto;margin-top:0;background-color:transparent;border:0;-webkit-box-shadow:none;box-shadow:none}.navbar-nav .open .dropdown-menu .dropdown-header,.navbar-nav .open .dropdown-menu>li>a{padding:5px 15px 5px 25px}.navbar-nav .open .dropdown-menu>li>a{line-height:20px}.navbar-nav .open .dropdown-menu>li>a:focus,.navbar-nav .open .dropdown-menu>li>a:hover{background-image:none}}@media (min-width:768px){.navbar-nav{float:left;margin:0}.navbar-nav>li{float:left}.navbar-nav>li>a{padding-top:15px;padding-bottom:15px}}.navbar-form{padding:10px 15px;margin-top:8px;margin-right:-15px;margin-bottom:8px;margin-left:-15px;border-top:1px solid transparent;border-bottom:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1)}@media (min-width:768px){.navbar-form .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.navbar-form .form-control{display:inline-block;width:auto;vertical-align:middle}.navbar-form .form-control-static{display:inline-block}.navbar-form .input-group{display:inline-table;vertical-align:middle}.navbar-form .input-group .form-control,.navbar-form .input-group .input-group-addon,.navbar-form .input-group .input-group-btn{width:auto}.navbar-form .input-group>.form-control{width:100%}.navbar-form .control-label{margin-bottom:0;vertical-align:middle}.navbar-form .checkbox,.navbar-form .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.navbar-form .checkbox label,.navbar-form .radio label{padding-left:0}.navbar-form .checkbox input[type=checkbox],.navbar-form .radio input[type=radio]{position:relative;margin-left:0}.navbar-form .has-feedback .form-control-feedback{top:0}}@media (max-width:767px){.navbar-form .form-group{margin-bottom:5px}.navbar-form .form-group:last-child{margin-bottom:0}}@media (min-width:768px){.navbar-form{width:auto;padding-top:0;padding-bottom:0;margin-right:0;margin-left:0;border:0;-webkit-box-shadow:none;box-shadow:none}}.navbar-nav>li>.dropdown-menu{margin-top:0;border-top-left-radius:0;border-top-right-radius:0}.navbar-fixed-bottom .navbar-nav>li>.dropdown-menu{margin-bottom:0;border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.navbar-btn{margin-top:8px;margin-bottom:8px}.navbar-btn.btn-sm{margin-top:10px;margin-bottom:10px}.navbar-btn.btn-xs{margin-top:14px;margin-bottom:14px}.navbar-text{margin-top:15px;margin-bottom:15px}@media (min-width:768px){.navbar-text{float:left;margin-right:15px;margin-left:15px}}@media (min-width:768px){.navbar-left{float:left!important}.navbar-right{float:right!important;margin-right:-15px}.navbar-right~.navbar-right{margin-right:0}}.navbar-default{background-color:#f8f8f8;border-color:#e7e7e7}.navbar-default .navbar-brand{color:#777}.navbar-default .navbar-brand:focus,.navbar-default .navbar-brand:hover{color:#5e5e5e;background-color:transparent}.navbar-default .navbar-text{color:#777}.navbar-default .navbar-nav>li>a{color:#777}.navbar-default .navbar-nav>li>a:focus,.navbar-default .navbar-nav>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav>.active>a,.navbar-default .navbar-nav>.active>a:focus,.navbar-default .navbar-nav>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav>.disabled>a,.navbar-default .navbar-nav>.disabled>a:focus,.navbar-default .navbar-nav>.disabled>a:hover{color:#ccc;background-color:transparent}.navbar-default .navbar-toggle{border-color:#ddd}.navbar-default .navbar-toggle:focus,.navbar-default .navbar-toggle:hover{background-color:#ddd}.navbar-default .navbar-toggle .icon-bar{background-color:#888}.navbar-default .navbar-collapse,.navbar-default .navbar-form{border-color:#e7e7e7}.navbar-default .navbar-nav>.open>a,.navbar-default .navbar-nav>.open>a:focus,.navbar-default .navbar-nav>.open>a:hover{color:#555;background-color:#e7e7e7}@media (max-width:767px){.navbar-default .navbar-nav .open .dropdown-menu>li>a{color:#777}.navbar-default .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav .open .dropdown-menu>.active>a,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#ccc;background-color:transparent}}.navbar-default .navbar-link{color:#777}.navbar-default .navbar-link:hover{color:#333}.navbar-default .btn-link{color:#777}.navbar-default .btn-link:focus,.navbar-default .btn-link:hover{color:#333}.navbar-default .btn-link[disabled]:focus,.navbar-default .btn-link[disabled]:hover,fieldset[disabled] .navbar-default .btn-link:focus,fieldset[disabled] .navbar-default .btn-link:hover{color:#ccc}.navbar-inverse{background-color:#222;border-color:#080808}.navbar-inverse .navbar-brand{color:#9d9d9d}.navbar-inverse .navbar-brand:focus,.navbar-inverse .navbar-brand:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-text{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a:focus,.navbar-inverse .navbar-nav>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav>.active>a,.navbar-inverse .navbar-nav>.active>a:focus,.navbar-inverse .navbar-nav>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav>.disabled>a,.navbar-inverse .navbar-nav>.disabled>a:focus,.navbar-inverse .navbar-nav>.disabled>a:hover{color:#444;background-color:transparent}.navbar-inverse .navbar-toggle{border-color:#333}.navbar-inverse .navbar-toggle:focus,.navbar-inverse .navbar-toggle:hover{background-color:#333}.navbar-inverse .navbar-toggle .icon-bar{background-color:#fff}.navbar-inverse .navbar-collapse,.navbar-inverse .navbar-form{border-color:#101010}.navbar-inverse .navbar-nav>.open>a,.navbar-inverse .navbar-nav>.open>a:focus,.navbar-inverse .navbar-nav>.open>a:hover{color:#fff;background-color:#080808}@media (max-width:767px){.navbar-inverse .navbar-nav .open .dropdown-menu>.dropdown-header{border-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu .divider{background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#444;background-color:transparent}}.navbar-inverse .navbar-link{color:#9d9d9d}.navbar-inverse .navbar-link:hover{color:#fff}.navbar-inverse .btn-link{color:#9d9d9d}.navbar-inverse .btn-link:focus,.navbar-inverse .btn-link:hover{color:#fff}.navbar-inverse .btn-link[disabled]:focus,.navbar-inverse .btn-link[disabled]:hover,fieldset[disabled] .navbar-inverse .btn-link:focus,fieldset[disabled] .navbar-inverse .btn-link:hover{color:#444}.breadcrumb{padding:8px 15px;margin-bottom:20px;list-style:none;background-color:#f5f5f5;border-radius:4px}.breadcrumb>li{display:inline-block}.breadcrumb>li+li:before{padding:0 5px;color:#ccc;content:"/\00a0"}.breadcrumb>.active{color:#777}.pagination{display:inline-block;padding-left:0;margin:20px 0;border-radius:4px}.pagination>li{display:inline}.pagination>li>a,.pagination>li>span{position:relative;float:left;padding:6px 12px;margin-left:-1px;line-height:1.42857143;color:#337ab7;text-decoration:none;background-color:#fff;border:1px solid #ddd}.pagination>li:first-child>a,.pagination>li:first-child>span{margin-left:0;border-top-left-radius:4px;border-bottom-left-radius:4px}.pagination>li:last-child>a,.pagination>li:last-child>span{border-top-right-radius:4px;border-bottom-right-radius:4px}.pagination>li>a:focus,.pagination>li>a:hover,.pagination>li>span:focus,.pagination>li>span:hover{z-index:2;color:#23527c;background-color:#eee;border-color:#ddd}.pagination>.active>a,.pagination>.active>a:focus,.pagination>.active>a:hover,.pagination>.active>span,.pagination>.active>span:focus,.pagination>.active>span:hover{z-index:3;color:#fff;cursor:default;background-color:#337ab7;border-color:#337ab7}.pagination>.disabled>a,.pagination>.disabled>a:focus,.pagination>.disabled>a:hover,.pagination>.disabled>span,.pagination>.disabled>span:focus,.pagination>.disabled>span:hover{color:#777;cursor:not-allowed;background-color:#fff;border-color:#ddd}.pagination-lg>li>a,.pagination-lg>li>span{padding:10px 16px;font-size:18px;line-height:1.3333333}.pagination-lg>li:first-child>a,.pagination-lg>li:first-child>span{border-top-left-radius:6px;border-bottom-left-radius:6px}.pagination-lg>li:last-child>a,.pagination-lg>li:last-child>span{border-top-right-radius:6px;border-bottom-right-radius:6px}.pagination-sm>li>a,.pagination-sm>li>span{padding:5px 10px;font-size:12px;line-height:1.5}.pagination-sm>li:first-child>a,.pagination-sm>li:first-child>span{border-top-left-radius:3px;border-bottom-left-radius:3px}.pagination-sm>li:last-child>a,.pagination-sm>li:last-child>span{border-top-right-radius:3px;border-bottom-right-radius:3px}.pager{padding-left:0;margin:20px 0;text-align:center;list-style:none}.pager li{display:inline}.pager li>a,.pager li>span{display:inline-block;padding:5px 14px;background-color:#fff;border:1px solid #ddd;border-radius:15px}.pager li>a:focus,.pager li>a:hover{text-decoration:none;background-color:#eee}.pager .next>a,.pager .next>span{float:right}.pager .previous>a,.pager .previous>span{float:left}.pager .disabled>a,.pager .disabled>a:focus,.pager .disabled>a:hover,.pager .disabled>span{color:#777;cursor:not-allowed;background-color:#fff}.label{display:inline;padding:.2em .6em .3em;font-size:75%;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:baseline;border-radius:.25em}a.label:focus,a.label:hover{color:#fff;text-decoration:none;cursor:pointer}.label:empty{display:none}.btn .label{position:relative;top:-1px}.label-default{background-color:#777}.label-default[href]:focus,.label-default[href]:hover{background-color:#5e5e5e}.label-primary{background-color:#337ab7}.label-primary[href]:focus,.label-primary[href]:hover{background-color:#286090}.label-success{background-color:#5cb85c}.label-success[href]:focus,.label-success[href]:hover{background-color:#449d44}.label-info{background-color:#5bc0de}.label-info[href]:focus,.label-info[href]:hover{background-color:#31b0d5}.label-warning{background-color:#f0ad4e}.label-warning[href]:focus,.label-warning[href]:hover{background-color:#ec971f}.label-danger{background-color:#d9534f}.label-danger[href]:focus,.label-danger[href]:hover{background-color:#c9302c}.badge{display:inline-block;min-width:10px;padding:3px 7px;font-size:12px;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:middle;background-color:#777;border-radius:10px}.badge:empty{display:none}.btn .badge{position:relative;top:-1px}.btn-group-xs>.btn .badge,.btn-xs .badge{top:0;padding:1px 5px}a.badge:focus,a.badge:hover{color:#fff;text-decoration:none;cursor:pointer}.list-group-item.active>.badge,.nav-pills>.active>a>.badge{color:#337ab7;background-color:#fff}.list-group-item>.badge{float:right}.list-group-item>.badge+.badge{margin-right:5px}.nav-pills>li>a>.badge{margin-left:3px}.jumbotron{padding-top:30px;padding-bottom:30px;margin-bottom:30px;color:inherit;background-color:#eee}.jumbotron .h1,.jumbotron h1{color:inherit}.jumbotron p{margin-bottom:15px;font-size:21px;font-weight:200}.jumbotron>hr{border-top-color:#d5d5d5}.container .jumbotron,.container-fluid .jumbotron{padding-right:15px;padding-left:15px;border-radius:6px}.jumbotron .container{max-width:100%}@media screen and (min-width:768px){.jumbotron{padding-top:48px;padding-bottom:48px}.container .jumbotron,.container-fluid .jumbotron{padding-right:60px;padding-left:60px}.jumbotron .h1,.jumbotron h1{font-size:63px}}.thumbnail{display:block;padding:4px;margin-bottom:20px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:border .2s ease-in-out;-o-transition:border .2s ease-in-out;transition:border .2s ease-in-out}.thumbnail a>img,.thumbnail>img{margin-right:auto;margin-left:auto}a.thumbnail.active,a.thumbnail:focus,a.thumbnail:hover{border-color:#337ab7}.thumbnail .caption{padding:9px;color:#333}.alert{padding:15px;margin-bottom:20px;border:1px solid transparent;border-radius:4px}.alert h4{margin-top:0;color:inherit}.alert .alert-link{font-weight:700}.alert>p,.alert>ul{margin-bottom:0}.alert>p+p{margin-top:5px}.alert-dismissable,.alert-dismissible{padding-right:35px}.alert-dismissable .close,.alert-dismissible .close{position:relative;top:-2px;right:-21px;color:inherit}.alert-success{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.alert-success hr{border-top-color:#c9e2b3}.alert-success .alert-link{color:#2b542c}.alert-info{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.alert-info hr{border-top-color:#a6e1ec}.alert-info .alert-link{color:#245269}.alert-warning{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.alert-warning hr{border-top-color:#f7e1b5}.alert-warning .alert-link{color:#66512c}.alert-danger{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.alert-danger hr{border-top-color:#e4b9c0}.alert-danger .alert-link{color:#843534}@-webkit-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@-o-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}.progress{height:20px;margin-bottom:20px;overflow:hidden;background-color:#f5f5f5;border-radius:4px;-webkit-box-shadow:inset 0 1px 2px rgba(0,0,0,.1);box-shadow:inset 0 1px 2px rgba(0,0,0,.1)}.progress-bar{float:left;width:0;height:100%;font-size:12px;line-height:20px;color:#fff;text-align:center;background-color:#337ab7;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);-webkit-transition:width .6s ease;-o-transition:width .6s ease;transition:width .6s ease}.progress-bar-striped,.progress-striped .progress-bar{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);-webkit-background-size:40px 40px;background-size:40px 40px}.progress-bar.active,.progress.active .progress-bar{-webkit-animation:progress-bar-stripes 2s linear infinite;-o-animation:progress-bar-stripes 2s linear infinite;animation:progress-bar-stripes 2s linear infinite}.progress-bar-success{background-color:#5cb85c}.progress-striped .progress-bar-success{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-info{background-color:#5bc0de}.progress-striped .progress-bar-info{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-warning{background-color:#f0ad4e}.progress-striped .progress-bar-warning{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-danger{background-color:#d9534f}.progress-striped .progress-bar-danger{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.media{margin-top:15px}.media:first-child{margin-top:0}.media,.media-body{overflow:hidden;zoom:1}.media-body{width:10000px}.media-object{display:block}.media-object.img-thumbnail{max-width:none}.media-right,.media>.pull-right{padding-left:10px}.media-left,.media>.pull-left{padding-right:10px}.media-body,.media-left,.media-right{display:table-cell;vertical-align:top}.media-middle{vertical-align:middle}.media-bottom{vertical-align:bottom}.media-heading{margin-top:0;margin-bottom:5px}.media-list{padding-left:0;list-style:none}.list-group{padding-left:0;margin-bottom:20px}.list-group-item{position:relative;display:block;padding:10px 15px;margin-bottom:-1px;background-color:#fff;border:1px solid #ddd}.list-group-item:first-child{border-top-left-radius:4px;border-top-right-radius:4px}.list-group-item:last-child{margin-bottom:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}a.list-group-item,button.list-group-item{color:#555}a.list-group-item .list-group-item-heading,button.list-group-item .list-group-item-heading{color:#333}a.list-group-item:focus,a.list-group-item:hover,button.list-group-item:focus,button.list-group-item:hover{color:#555;text-decoration:none;background-color:#f5f5f5}button.list-group-item{width:100%;text-align:left}.list-group-item.disabled,.list-group-item.disabled:focus,.list-group-item.disabled:hover{color:#777;cursor:not-allowed;background-color:#eee}.list-group-item.disabled .list-group-item-heading,.list-group-item.disabled:focus .list-group-item-heading,.list-group-item.disabled:hover .list-group-item-heading{color:inherit}.list-group-item.disabled .list-group-item-text,.list-group-item.disabled:focus .list-group-item-text,.list-group-item.disabled:hover .list-group-item-text{color:#777}.list-group-item.active,.list-group-item.active:focus,.list-group-item.active:hover{z-index:2;color:#fff;background-color:#337ab7;border-color:#337ab7}.list-group-item.active .list-group-item-heading,.list-group-item.active .list-group-item-heading>.small,.list-group-item.active .list-group-item-heading>small,.list-group-item.active:focus .list-group-item-heading,.list-group-item.active:focus .list-group-item-heading>.small,.list-group-item.active:focus .list-group-item-heading>small,.list-group-item.active:hover .list-group-item-heading,.list-group-item.active:hover .list-group-item-heading>.small,.list-group-item.active:hover .list-group-item-heading>small{color:inherit}.list-group-item.active .list-group-item-text,.list-group-item.active:focus .list-group-item-text,.list-group-item.active:hover .list-group-item-text{color:#c7ddef}.list-group-item-success{color:#3c763d;background-color:#dff0d8}a.list-group-item-success,button.list-group-item-success{color:#3c763d}a.list-group-item-success .list-group-item-heading,button.list-group-item-success .list-group-item-heading{color:inherit}a.list-group-item-success:focus,a.list-group-item-success:hover,button.list-group-item-success:focus,button.list-group-item-success:hover{color:#3c763d;background-color:#d0e9c6}a.list-group-item-success.active,a.list-group-item-success.active:focus,a.list-group-item-success.active:hover,button.list-group-item-success.active,button.list-group-item-success.active:focus,button.list-group-item-success.active:hover{color:#fff;background-color:#3c763d;border-color:#3c763d}.list-group-item-info{color:#31708f;background-color:#d9edf7}a.list-group-item-info,button.list-group-item-info{color:#31708f}a.list-group-item-info .list-group-item-heading,button.list-group-item-info .list-group-item-heading{color:inherit}a.list-group-item-info:focus,a.list-group-item-info:hover,button.list-group-item-info:focus,button.list-group-item-info:hover{color:#31708f;background-color:#c4e3f3}a.list-group-item-info.active,a.list-group-item-info.active:focus,a.list-group-item-info.active:hover,button.list-group-item-info.active,button.list-group-item-info.active:focus,button.list-group-item-info.active:hover{color:#fff;background-color:#31708f;border-color:#31708f}.list-group-item-warning{color:#8a6d3b;background-color:#fcf8e3}a.list-group-item-warning,button.list-group-item-warning{color:#8a6d3b}a.list-group-item-warning .list-group-item-heading,button.list-group-item-warning .list-group-item-heading{color:inherit}a.list-group-item-warning:focus,a.list-group-item-warning:hover,button.list-group-item-warning:focus,button.list-group-item-warning:hover{color:#8a6d3b;background-color:#faf2cc}a.list-group-item-warning.active,a.list-group-item-warning.active:focus,a.list-group-item-warning.active:hover,button.list-group-item-warning.active,button.list-group-item-warning.active:focus,button.list-group-item-warning.active:hover{color:#fff;background-color:#8a6d3b;border-color:#8a6d3b}.list-group-item-danger{color:#a94442;background-color:#f2dede}a.list-group-item-danger,button.list-group-item-danger{color:#a94442}a.list-group-item-danger .list-group-item-heading,button.list-group-item-danger .list-group-item-heading{color:inherit}a.list-group-item-danger:focus,a.list-group-item-danger:hover,button.list-group-item-danger:focus,button.list-group-item-danger:hover{color:#a94442;background-color:#ebcccc}a.list-group-item-danger.active,a.list-group-item-danger.active:focus,a.list-group-item-danger.active:hover,button.list-group-item-danger.active,button.list-group-item-danger.active:focus,button.list-group-item-danger.active:hover{color:#fff;background-color:#a94442;border-color:#a94442}.list-group-item-heading{margin-top:0;margin-bottom:5px}.list-group-item-text{margin-bottom:0;line-height:1.3}.panel{margin-bottom:20px;background-color:#fff;border:1px solid transparent;border-radius:4px;-webkit-box-shadow:0 1px 1px rgba(0,0,0,.05);box-shadow:0 1px 1px rgba(0,0,0,.05)}.panel-body{padding:15px}.panel-heading{padding:10px 15px;border-bottom:1px solid transparent;border-top-left-radius:3px;border-top-right-radius:3px}.panel-heading>.dropdown .dropdown-toggle{color:inherit}.panel-title{margin-top:0;margin-bottom:0;font-size:16px;color:inherit}.panel-title>.small,.panel-title>.small>a,.panel-title>a,.panel-title>small,.panel-title>small>a{color:inherit}.panel-footer{padding:10px 15px;background-color:#f5f5f5;border-top:1px solid #ddd;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.list-group,.panel>.panel-collapse>.list-group{margin-bottom:0}.panel>.list-group .list-group-item,.panel>.panel-collapse>.list-group .list-group-item{border-width:1px 0;border-radius:0}.panel>.list-group:first-child .list-group-item:first-child,.panel>.panel-collapse>.list-group:first-child .list-group-item:first-child{border-top:0;border-top-left-radius:3px;border-top-right-radius:3px}.panel>.list-group:last-child .list-group-item:last-child,.panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{border-bottom:0;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.panel-heading+.panel-collapse>.list-group .list-group-item:first-child{border-top-left-radius:0;border-top-right-radius:0}.panel-heading+.list-group .list-group-item:first-child{border-top-width:0}.list-group+.panel-footer{border-top-width:0}.panel>.panel-collapse>.table,.panel>.table,.panel>.table-responsive>.table{margin-bottom:0}.panel>.panel-collapse>.table caption,.panel>.table caption,.panel>.table-responsive>.table caption{padding-right:15px;padding-left:15px}.panel>.table-responsive:first-child>.table:first-child,.panel>.table:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child,.panel>.table:first-child>thead:first-child>tr:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table:first-child>thead:first-child>tr:first-child th:first-child{border-top-left-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table:first-child>thead:first-child>tr:first-child th:last-child{border-top-right-radius:3px}.panel>.table-responsive:last-child>.table:last-child,.panel>.table:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:first-child{border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:last-child{border-bottom-right-radius:3px}.panel>.panel-body+.table,.panel>.panel-body+.table-responsive,.panel>.table+.panel-body,.panel>.table-responsive+.panel-body{border-top:1px solid #ddd}.panel>.table>tbody:first-child>tr:first-child td,.panel>.table>tbody:first-child>tr:first-child th{border-top:0}.panel>.table-bordered,.panel>.table-responsive>.table-bordered{border:0}.panel>.table-bordered>tbody>tr>td:first-child,.panel>.table-bordered>tbody>tr>th:first-child,.panel>.table-bordered>tfoot>tr>td:first-child,.panel>.table-bordered>tfoot>tr>th:first-child,.panel>.table-bordered>thead>tr>td:first-child,.panel>.table-bordered>thead>tr>th:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:first-child,.panel>.table-responsive>.table-bordered>thead>tr>td:first-child,.panel>.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.panel>.table-bordered>tbody>tr>td:last-child,.panel>.table-bordered>tbody>tr>th:last-child,.panel>.table-bordered>tfoot>tr>td:last-child,.panel>.table-bordered>tfoot>tr>th:last-child,.panel>.table-bordered>thead>tr>td:last-child,.panel>.table-bordered>thead>tr>th:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:last-child,.panel>.table-responsive>.table-bordered>thead>tr>td:last-child,.panel>.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.panel>.table-bordered>tbody>tr:first-child>td,.panel>.table-bordered>tbody>tr:first-child>th,.panel>.table-bordered>thead>tr:first-child>td,.panel>.table-bordered>thead>tr:first-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>th,.panel>.table-responsive>.table-bordered>thead>tr:first-child>td,.panel>.table-responsive>.table-bordered>thead>tr:first-child>th{border-bottom:0}.panel>.table-bordered>tbody>tr:last-child>td,.panel>.table-bordered>tbody>tr:last-child>th,.panel>.table-bordered>tfoot>tr:last-child>td,.panel>.table-bordered>tfoot>tr:last-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>th,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>td,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}.panel>.table-responsive{margin-bottom:0;border:0}.panel-group{margin-bottom:20px}.panel-group .panel{margin-bottom:0;border-radius:4px}.panel-group .panel+.panel{margin-top:5px}.panel-group .panel-heading{border-bottom:0}.panel-group .panel-heading+.panel-collapse>.list-group,.panel-group .panel-heading+.panel-collapse>.panel-body{border-top:1px solid #ddd}.panel-group .panel-footer{border-top:0}.panel-group .panel-footer+.panel-collapse .panel-body{border-bottom:1px solid #ddd}.panel-default{border-color:#ddd}.panel-default>.panel-heading{color:#333;background-color:#f5f5f5;border-color:#ddd}.panel-default>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ddd}.panel-default>.panel-heading .badge{color:#f5f5f5;background-color:#333}.panel-default>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ddd}.panel-primary{border-color:#337ab7}.panel-primary>.panel-heading{color:#fff;background-color:#337ab7;border-color:#337ab7}.panel-primary>.panel-heading+.panel-collapse>.panel-body{border-top-color:#337ab7}.panel-primary>.panel-heading .badge{color:#337ab7;background-color:#fff}.panel-primary>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#337ab7}.panel-success{border-color:#d6e9c6}.panel-success>.panel-heading{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.panel-success>.panel-heading+.panel-collapse>.panel-body{border-top-color:#d6e9c6}.panel-success>.panel-heading .badge{color:#dff0d8;background-color:#3c763d}.panel-success>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#d6e9c6}.panel-info{border-color:#bce8f1}.panel-info>.panel-heading{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.panel-info>.panel-heading+.panel-collapse>.panel-body{border-top-color:#bce8f1}.panel-info>.panel-heading .badge{color:#d9edf7;background-color:#31708f}.panel-info>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#bce8f1}.panel-warning{border-color:#faebcc}.panel-warning>.panel-heading{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.panel-warning>.panel-heading+.panel-collapse>.panel-body{border-top-color:#faebcc}.panel-warning>.panel-heading .badge{color:#fcf8e3;background-color:#8a6d3b}.panel-warning>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#faebcc}.panel-danger{border-color:#ebccd1}.panel-danger>.panel-heading{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.panel-danger>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ebccd1}.panel-danger>.panel-heading .badge{color:#f2dede;background-color:#a94442}.panel-danger>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ebccd1}.embed-responsive{position:relative;display:block;height:0;padding:0;overflow:hidden}.embed-responsive .embed-responsive-item,.embed-responsive embed,.embed-responsive iframe,.embed-responsive object,.embed-responsive video{position:absolute;top:0;bottom:0;left:0;width:100%;height:100%;border:0}.embed-responsive-16by9{padding-bottom:56.25%}.embed-responsive-4by3{padding-bottom:75%}.well{min-height:20px;padding:19px;margin-bottom:20px;background-color:#f5f5f5;border:1px solid #e3e3e3;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.05);box-shadow:inset 0 1px 1px rgba(0,0,0,.05)}.well blockquote{border-color:#ddd;border-color:rgba(0,0,0,.15)}.well-lg{padding:24px;border-radius:6px}.well-sm{padding:9px;border-radius:3px}.close{float:right;font-size:21px;font-weight:700;line-height:1;color:#000;text-shadow:0 1px 0 #fff;filter:alpha(opacity=20);opacity:.2}.close:focus,.close:hover{color:#000;text-decoration:none;cursor:pointer;filter:alpha(opacity=50);opacity:.5}button.close{-webkit-appearance:none;padding:0;cursor:pointer;background:0 0;border:0}.modal-open{overflow:hidden}.modal{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1050;display:none;overflow:hidden;-webkit-overflow-scrolling:touch;outline:0}.modal.fade .modal-dialog{-webkit-transition:-webkit-transform .3s ease-out;-o-transition:-o-transform .3s ease-out;transition:transform .3s ease-out;-webkit-transform:translate(0,-25%);-ms-transform:translate(0,-25%);-o-transform:translate(0,-25%);transform:translate(0,-25%)}.modal.in .modal-dialog{-webkit-transform:translate(0,0);-ms-transform:translate(0,0);-o-transform:translate(0,0);transform:translate(0,0)}.modal-open .modal{overflow-x:hidden;overflow-y:auto}.modal-dialog{position:relative;width:auto;margin:10px}.modal-content{position:relative;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #999;border:1px solid rgba(0,0,0,.2);border-radius:6px;outline:0;-webkit-box-shadow:0 3px 9px rgba(0,0,0,.5);box-shadow:0 3px 9px rgba(0,0,0,.5)}.modal-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1040;background-color:#000}.modal-backdrop.fade{filter:alpha(opacity=0);opacity:0}.modal-backdrop.in{filter:alpha(opacity=50);opacity:.5}.modal-header{padding:15px;border-bottom:1px solid #e5e5e5}.modal-header .close{margin-top:-2px}.modal-title{margin:0;line-height:1.42857143}.modal-body{position:relative;padding:15px}.modal-footer{padding:15px;text-align:right;border-top:1px solid #e5e5e5}.modal-footer .btn+.btn{margin-bottom:0;margin-left:5px}.modal-footer .btn-group .btn+.btn{margin-left:-1px}.modal-footer .btn-block+.btn-block{margin-left:0}.modal-scrollbar-measure{position:absolute;top:-9999px;width:50px;height:50px;overflow:scroll}@media (min-width:768px){.modal-dialog{width:600px;margin:30px auto}.modal-content{-webkit-box-shadow:0 5px 15px rgba(0,0,0,.5);box-shadow:0 5px 15px rgba(0,0,0,.5)}.modal-sm{width:300px}}@media (min-width:992px){.modal-lg{width:900px}}.tooltip{position:absolute;z-index:1070;display:block;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:12px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;filter:alpha(opacity=0);opacity:0;line-break:auto}.tooltip.in{filter:alpha(opacity=90);opacity:.9}.tooltip.top{padding:5px 0;margin-top:-3px}.tooltip.right{padding:0 5px;margin-left:3px}.tooltip.bottom{padding:5px 0;margin-top:3px}.tooltip.left{padding:0 5px;margin-left:-3px}.tooltip-inner{max-width:200px;padding:3px 8px;color:#fff;text-align:center;background-color:#000;border-radius:4px}.tooltip-arrow{position:absolute;width:0;height:0;border-color:transparent;border-style:solid}.tooltip.top .tooltip-arrow{bottom:0;left:50%;margin-left:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-left .tooltip-arrow{right:5px;bottom:0;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-right .tooltip-arrow{bottom:0;left:5px;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.right .tooltip-arrow{top:50%;left:0;margin-top:-5px;border-width:5px 5px 5px 0;border-right-color:#000}.tooltip.left .tooltip-arrow{top:50%;right:0;margin-top:-5px;border-width:5px 0 5px 5px;border-left-color:#000}.tooltip.bottom .tooltip-arrow{top:0;left:50%;margin-left:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-left .tooltip-arrow{top:0;right:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-right .tooltip-arrow{top:0;left:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.popover{position:absolute;top:0;left:0;z-index:1060;display:none;max-width:276px;padding:1px;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.2);border-radius:6px;-webkit-box-shadow:0 5px 10px rgba(0,0,0,.2);box-shadow:0 5px 10px rgba(0,0,0,.2);line-break:auto}.popover.top{margin-top:-10px}.popover.right{margin-left:10px}.popover.bottom{margin-top:10px}.popover.left{margin-left:-10px}.popover-title{padding:8px 14px;margin:0;font-size:14px;background-color:#f7f7f7;border-bottom:1px solid #ebebeb;border-radius:5px 5px 0 0}.popover-content{padding:9px 14px}.popover>.arrow,.popover>.arrow:after{position:absolute;display:block;width:0;height:0;border-color:transparent;border-style:solid}.popover>.arrow{border-width:11px}.popover>.arrow:after{content:"";border-width:10px}.popover.top>.arrow{bottom:-11px;left:50%;margin-left:-11px;border-top-color:#999;border-top-color:rgba(0,0,0,.25);border-bottom-width:0}.popover.top>.arrow:after{bottom:1px;margin-left:-10px;content:" ";border-top-color:#fff;border-bottom-width:0}.popover.right>.arrow{top:50%;left:-11px;margin-top:-11px;border-right-color:#999;border-right-color:rgba(0,0,0,.25);border-left-width:0}.popover.right>.arrow:after{bottom:-10px;left:1px;content:" ";border-right-color:#fff;border-left-width:0}.popover.bottom>.arrow{top:-11px;left:50%;margin-left:-11px;border-top-width:0;border-bottom-color:#999;border-bottom-color:rgba(0,0,0,.25)}.popover.bottom>.arrow:after{top:1px;margin-left:-10px;content:" ";border-top-width:0;border-bottom-color:#fff}.popover.left>.arrow{top:50%;right:-11px;margin-top:-11px;border-right-width:0;border-left-color:#999;border-left-color:rgba(0,0,0,.25)}.popover.left>.arrow:after{right:1px;bottom:-10px;content:" ";border-right-width:0;border-left-color:#fff}.carousel{position:relative}.carousel-inner{position:relative;width:100%;overflow:hidden}.carousel-inner>.item{position:relative;display:none;-webkit-transition:.6s ease-in-out left;-o-transition:.6s ease-in-out left;transition:.6s ease-in-out left}.carousel-inner>.item>a>img,.carousel-inner>.item>img{line-height:1}@media all and (transform-3d),(-webkit-transform-3d){.carousel-inner>.item{-webkit-transition:-webkit-transform .6s ease-in-out;-o-transition:-o-transform .6s ease-in-out;transition:transform .6s ease-in-out;-webkit-backface-visibility:hidden;backface-visibility:hidden;-webkit-perspective:1000px;perspective:1000px}.carousel-inner>.item.active.right,.carousel-inner>.item.next{left:0;-webkit-transform:translate3d(100%,0,0);transform:translate3d(100%,0,0)}.carousel-inner>.item.active.left,.carousel-inner>.item.prev{left:0;-webkit-transform:translate3d(-100%,0,0);transform:translate3d(-100%,0,0)}.carousel-inner>.item.active,.carousel-inner>.item.next.left,.carousel-inner>.item.prev.right{left:0;-webkit-transform:translate3d(0,0,0);transform:translate3d(0,0,0)}}.carousel-inner>.active,.carousel-inner>.next,.carousel-inner>.prev{display:block}.carousel-inner>.active{left:0}.carousel-inner>.next,.carousel-inner>.prev{position:absolute;top:0;width:100%}.carousel-inner>.next{left:100%}.carousel-inner>.prev{left:-100%}.carousel-inner>.next.left,.carousel-inner>.prev.right{left:0}.carousel-inner>.active.left{left:-100%}.carousel-inner>.active.right{left:100%}.carousel-control{position:absolute;top:0;bottom:0;left:0;width:15%;font-size:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6);background-color:rgba(0,0,0,0);filter:alpha(opacity=50);opacity:.5}.carousel-control.left{background-image:-webkit-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.5)),to(rgba(0,0,0,.0001)));background-image:linear-gradient(to right,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#80000000', endColorstr='#00000000', GradientType=1);background-repeat:repeat-x}.carousel-control.right{right:0;left:auto;background-image:-webkit-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.0001)),to(rgba(0,0,0,.5)));background-image:linear-gradient(to right,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#00000000', endColorstr='#80000000', GradientType=1);background-repeat:repeat-x}.carousel-control:focus,.carousel-control:hover{color:#fff;text-decoration:none;filter:alpha(opacity=90);outline:0;opacity:.9}.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{position:absolute;top:50%;z-index:5;display:inline-block;margin-top:-10px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{left:50%;margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{right:50%;margin-right:-10px}.carousel-control .icon-next,.carousel-control .icon-prev{width:20px;height:20px;font-family:serif;line-height:1}.carousel-control .icon-prev:before{content:'\2039'}.carousel-control .icon-next:before{content:'\203a'}.carousel-indicators{position:absolute;bottom:10px;left:50%;z-index:15;width:60%;padding-left:0;margin-left:-30%;text-align:center;list-style:none}.carousel-indicators li{display:inline-block;width:10px;height:10px;margin:1px;text-indent:-999px;cursor:pointer;background-color:#000\9;background-color:rgba(0,0,0,0);border:1px solid #fff;border-radius:10px}.carousel-indicators .active{width:12px;height:12px;margin:0;background-color:#fff}.carousel-caption{position:absolute;right:15%;bottom:20px;left:15%;z-index:10;padding-top:20px;padding-bottom:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6)}.carousel-caption .btn{text-shadow:none}@media screen and (min-width:768px){.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{width:30px;height:30px;margin-top:-10px;font-size:30px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{margin-right:-10px}.carousel-caption{right:20%;left:20%;padding-bottom:30px}.carousel-indicators{bottom:20px}}.btn-group-vertical>.btn-group:after,.btn-group-vertical>.btn-group:before,.btn-toolbar:after,.btn-toolbar:before,.clearfix:after,.clearfix:before,.container-fluid:after,.container-fluid:before,.container:after,.container:before,.dl-horizontal dd:after,.dl-horizontal dd:before,.form-horizontal .form-group:after,.form-horizontal .form-group:before,.modal-footer:after,.modal-footer:before,.modal-header:after,.modal-header:before,.nav:after,.nav:before,.navbar-collapse:after,.navbar-collapse:before,.navbar-header:after,.navbar-header:before,.navbar:after,.navbar:before,.pager:after,.pager:before,.panel-body:after,.panel-body:before,.row:after,.row:before{display:table;content:" "}.btn-group-vertical>.btn-group:after,.btn-toolbar:after,.clearfix:after,.container-fluid:after,.container:after,.dl-horizontal dd:after,.form-horizontal .form-group:after,.modal-footer:after,.modal-header:after,.nav:after,.navbar-collapse:after,.navbar-header:after,.navbar:after,.pager:after,.panel-body:after,.row:after{clear:both}.center-block{display:block;margin-right:auto;margin-left:auto}.pull-right{float:right!important}.pull-left{float:left!important}.hide{display:none!important}.show{display:block!important}.invisible{visibility:hidden}.text-hide{font:0/0 a;color:transparent;text-shadow:none;background-color:transparent;border:0}.hidden{display:none!important}.affix{position:fixed}@-ms-viewport{width:device-width}.visible-lg,.visible-md,.visible-sm,.visible-xs{display:none!important}.visible-lg-block,.visible-lg-inline,.visible-lg-inline-block,.visible-md-block,.visible-md-inline,.visible-md-inline-block,.visible-sm-block,.visible-sm-inline,.visible-sm-inline-block,.visible-xs-block,.visible-xs-inline,.visible-xs-inline-block{display:none!important}@media (max-width:767px){.visible-xs{display:block!important}table.visible-xs{display:table!important}tr.visible-xs{display:table-row!important}td.visible-xs,th.visible-xs{display:table-cell!important}}@media (max-width:767px){.visible-xs-block{display:block!important}}@media (max-width:767px){.visible-xs-inline{display:inline!important}}@media (max-width:767px){.visible-xs-inline-block{display:inline-block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm{display:block!important}table.visible-sm{display:table!important}tr.visible-sm{display:table-row!important}td.visible-sm,th.visible-sm{display:table-cell!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-block{display:block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline{display:inline!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline-block{display:inline-block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md{display:block!important}table.visible-md{display:table!important}tr.visible-md{display:table-row!important}td.visible-md,th.visible-md{display:table-cell!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-block{display:block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline{display:inline!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline-block{display:inline-block!important}}@media (min-width:1200px){.visible-lg{display:block!important}table.visible-lg{display:table!important}tr.visible-lg{display:table-row!important}td.visible-lg,th.visible-lg{display:table-cell!important}}@media (min-width:1200px){.visible-lg-block{display:block!important}}@media (min-width:1200px){.visible-lg-inline{display:inline!important}}@media (min-width:1200px){.visible-lg-inline-block{display:inline-block!important}}@media (max-width:767px){.hidden-xs{display:none!important}}@media (min-width:768px) and (max-width:991px){.hidden-sm{display:none!important}}@media (min-width:992px) and (max-width:1199px){.hidden-md{display:none!important}}@media (min-width:1200px){.hidden-lg{display:none!important}}.visible-print{display:none!important}@media print{.visible-print{display:block!important}table.visible-print{display:table!important}tr.visible-print{display:table-row!important}td.visible-print,th.visible-print{display:table-cell!important}}.visible-print-block{display:none!important}@media print{.visible-print-block{display:block!important}}.visible-print-inline{display:none!important}@media print{.visible-print-inline{display:inline!important}}.visible-print-inline-block{display:none!important}@media print{.visible-print-inline-block{display:inline-block!important}}@media print{.hidden-print{display:none!important}}
        /*# sourceMappingURL=bootstrap.min.css.map */
    </style>
    <!--<link rel="stylesheet" type="text/css" media="all" href="../../resources/bootstrap-3.2.0/css/bootstrap.min.css" th:href="@{resources/bootstrap-3.2.0/css/bootstrap.min.css}"/>-->
</head>
<body>
<div id="content" class="container">
    <h2>
        Exomiser Analysis Results for
        Anonymous
    </h2>
    <ul class="nav nav-pills" id="tabs">
        <li><a href="#settings" role="tab" data-toggle="tab"><b>Analysis Settings</b></a></li>
        <li><a href="#filtering" role="tab" data-toggle="tab"><b>Filtering Summary</b></a></li>
        <li><a href="#distribution" role="tab" data-toggle="tab"><b>Variant Type Distribution</b></a></li>
        <li><a href="#prioritisation" role="tab" data-toggle="tab"><b>Prioritised Genes</b></a></li>
        
        <li><a href="#about" role="tab" data-toggle="tab"><b>About</b></a></li>
    </ul>
    <div class="panel panel-default" id="settings">
        <div class="panel-heading">
            <h3>Analysis Settings</h3>
        </div>
        <div class="panel-body"><p>Settings used for this analysis:</p>
            <pre>---
sample:
  genomeAssembly: &quot;hg19&quot;
  vcf: &quot;/Users/munaberhe/projects/genomics-portfolio/data/case01_hcm.vep.vcf.gz&quot;
  hpoIds:
  - &quot;HP:0001639&quot;
  - &quot;HP:0001671&quot;
  pedigree: {}
  age: {}
analysis:
  inheritanceModes:
    AUTOSOMAL_DOMINANT: 1.0
  frequencySources:
  - &quot;GNOMAD_E_AFR&quot;
  - &quot;GNOMAD_E_AMR&quot;
  - &quot;GNOMAD_E_EAS&quot;
  - &quot;GNOMAD_E_NFE&quot;
  - &quot;GNOMAD_E_SAS&quot;
  - &quot;GNOMAD_G_AFR&quot;
  - &quot;GNOMAD_G_AMR&quot;
  - &quot;GNOMAD_G_EAS&quot;
  - &quot;GNOMAD_G_NFE&quot;
  - &quot;GNOMAD_G_SAS&quot;
  pathogenicitySources:
  - &quot;REVEL&quot;
  - &quot;MVP&quot;
  - &quot;SPLICE_AI&quot;
  - &quot;ALPHA_MISSENSE&quot;
  steps:
  - failedVariantFilter: {}
  - variantEffectFilter:
      remove:
      - &quot;CODING_TRANSCRIPT_INTRON_VARIANT&quot;
      - &quot;FIVE_PRIME_UTR_EXON_VARIANT&quot;
      - &quot;THREE_PRIME_UTR_EXON_VARIANT&quot;
      - &quot;NON_CODING_TRANSCRIPT_INTRON_VARIANT&quot;
      - &quot;UPSTREAM_GENE_VARIANT&quot;
      - &quot;DOWNSTREAM_GENE_VARIANT&quot;
      - &quot;INTERGENIC_VARIANT&quot;
  - frequencyFilter:
      maxFrequency: 2.0
  - pathogenicityFilter:
      keepNonPathogenic: true
  - inheritanceFilter: {}
  - omimPrioritiser: {}
  - hiPhivePrioritiser:
      runParams: &quot;human, mouse, fish, ppi&quot;
outputOptions:
  outputFormats:
  - &quot;HTML&quot;
  - &quot;TSV_GENE&quot;
  - &quot;TSV_VARIANT&quot;
  - &quot;JSON&quot;
  outputDirectory: &quot;/Users/munaberhe/projects/genomics-portfolio/outputs/case01_hcm&quot;
</pre>
        </div>
    </div>

    <div class="panel panel-default" id="filtering">
        <div class="panel-heading">
            <h3>Filtering Summary</h3>
        </div>
        <div class="panel-body">
            <table class="table">
                <thead>
                <tr>
                    <th>Filter</th>
                    <th>Report</th>
                    <th>Passed filter</th>
                    <th>Failed filter</th>
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td><b>Failed previous VCF filters</b></td>
                    <td>
                        <ul>Removed variants without PASS or . in VCF FILTER field</ul>
                    </td>
                    <td>35362</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Variant effect</b></td>
                    <td>
                        <ul>Removed variants with effects of type: [CODING_TRANSCRIPT_INTRON_VARIANT, FIVE_PRIME_UTR_EXON_VARIANT, THREE_PRIME_UTR_EXON_VARIANT, NON_CODING_TRANSCRIPT_INTRON_VARIANT, UPSTREAM_GENE_VARIANT, DOWNSTREAM_GENE_VARIANT, INTERGENIC_VARIANT]</ul>
                    </td>
                    <td>20189</td>
                    <td>15173</td>
                </tr>
                <tr>
                    <td><b>Frequency</b></td>
                    <td>
                        <ul>Variants filtered for maximum allele frequency of 2.00%</ul>
                    </td>
                    <td>573</td>
                    <td>19616</td>
                </tr>
                <tr>
                    <td><b>Pathogenicity</b></td>
                    <td>
                        <ul>Retained all non-pathogenic variants of all types. Scoring was applied, but the filter passed all variants.</ul>
                    </td>
                    <td>573</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Inheritance</b></td>
                    <td>
                        <ul>Variants filtered for compatibility with AUTOSOMAL_DOMINANT inheritance.</ul>
                    </td>
                    <td>414</td>
                    <td>159</td>
                </tr>
                </tbody>
            </table>
        </div>
    </div>
    <div class="panel panel-default" id="distribution">
        <div class="panel-heading">
            <h3>Variant Type Distribution</h3>
        </div>
        <div class="panel-body">
            <table class="table-striped table-condensed">
                <thead>
                <tr>
                    <th>Variant Type</th>
                    
                        <th>NA19722</th>
                    
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td>FRAMESHIFT_ELONGATION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_TRUNCATION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INTERNAL_FEATURE_ELONGATION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FEATURE_TRUNCATION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>MNV</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>STOP_GAINED</td>
                    
                        <td>5</td>
                    
                </tr>
                <tr>
                    <td>STOP_LOST</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>START_LOST</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_ACCEPTOR_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_DONOR_VARIANT</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>MISSENSE_VARIANT</td>
                    
                        <td>249</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_INSERTION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_INSERTION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_DELETION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_DELETION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_REGION_VARIANT</td>
                    
                        <td>23</td>
                    
                </tr>
                <tr>
                    <td>STOP_RETAINED_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INITIATOR_CODON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SYNONYMOUS_VARIANT</td>
                    
                        <td>131</td>
                    
                </tr>
                <tr>
                    <td>CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>3</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>UPSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DOWNSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INTERGENIC_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>REGULATORY_REGION_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                </tbody>
            </table>
        </div>
    </div>

    

    <div class="panel panel-default" id="prioritisation">
        <div class="panel-heading">
            <h3>Prioritised Genes</h3>
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CUL3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000036257">CUL3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.860</b>
                    <data>(p=7.9E-3)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.651</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.908</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.651 to <a href="http://www.omim.org/entry/619239">Neurodevelopmental disorder with or without autism or seizures</a> associated with CUL3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001642, Pulmonic stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Phenotypic similarity 0.462 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CUL3">CUL3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000231, hypertension</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000231, hypertension</dd></dl><dl><dt>Proximity score 0.598 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUL3%0DKBTBD13&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KBTBD13</a> and phenotypic similarity 0.837 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=171439">Childhood-onset nemaline myopathy</a> associated with KBTBD13.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/614496">OMIM:614496</a> Pseudohypoaldosteronism, type IIE - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/619239">OMIM:619239</a> Neurodevelopmental disorder with or without autism or seizures - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.860</b>
                                    <data>(p=7.9E-3)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.651</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.908</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.651 to OMIM:619239 Neurodevelopmental disorder with or without autism or seizures</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-225362559-A-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs143436770">rs143436770</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.908</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264414.4">CUL3:ENST00000264414.4:c.1618T&gt;G:p.(Leu540Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344951.4">CUL3:ENST00000344951.4:c.1420T&gt;G:p.(Leu474Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409096.1">CUL3:ENST00000409096.1:c.1546T&gt;G:p.(Leu516Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409777.1">CUL3:ENST00000409777.1:c.1546T&gt;G:p.(Leu516Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9087225</dd>
                                    
                                    <dd>REVEL: 0.608</dd>
                                    <dd>MVP: 0.909</dd>
                                    <dd>ALPHA_MISSENSE: 0.759</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="STAB1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000010327">STAB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.748</b>
                    <data>(p=1.5E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.244 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=STAB1">STAB1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000263, thoracic duct hypoplastic, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000263, thoracic duct hypoplastic, abnormal</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=STAB1%0DCCN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCN1</a> and phenotypic similarity 0.777 to mouse mutant of CCN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000281, abnormal interventricular septum morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0004225, patent cardiac foramen ovale</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.748</b>
                                    <data>(p=1.5E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>1.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                            <b>SNV</b>
                            <b>3-52536743-G-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201713438">rs201713438</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000321725.6">STAB1:ENST00000321725.6:c.583G&gt;T:p.(Glu195*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.33</dd>
                                    
                                    <dd>SPLICE_AI: 0.330</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC4A7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000033867">SLC4A7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.729</b>
                    <data>(p=1.9E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.988</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.476 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SLC4A7">SLC4A7</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000230, abnormal systemic arterial blood pressure</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000230, abnormal systemic arterial blood pressure</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC4A7%0DSLC26A7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC26A7</a> and phenotypic similarity 0.729 to mouse mutant of SLC26A7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003503, decreased activity of thyroid gland</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.729</b>
                                    <data>(p=1.9E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.988</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-27462259-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs189574936">rs189574936</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.988</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295736.5">SLC4A7:ENST00000295736.5:c.1417G&gt;A:p.(Gly473Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000388777.4">SLC4A7:ENST00000388777.4:c.67G&gt;A:p.(Gly23Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000425128.2">SLC4A7:ENST00000425128.2:c.1393G&gt;A:p.(Gly465Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428386.1">SLC4A7:ENST00000428386.1:c.1045G&gt;A:p.(Gly349Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000435667.2">SLC4A7:ENST00000435667.2:c.1072G&gt;A:p.(Gly358Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000437179.1">SLC4A7:ENST00000437179.1:c.1060G&gt;A:p.(Gly354Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000440156.1">SLC4A7:ENST00000440156.1:c.1405G&gt;A:p.(Gly469Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445684.1">SLC4A7:ENST00000445684.1:c.1405G&gt;A:p.(Gly469Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000446700.1">SLC4A7:ENST00000446700.1:c.1393G&gt;A:p.(Gly465Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454389.1">SLC4A7:ENST00000454389.1:c.1444G&gt;A:p.(Gly482Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000455077.1">SLC4A7:ENST00000455077.1:c.1060G&gt;A:p.(Gly354Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9886</dd>
                                    
                                    <dd>REVEL: 0.825</dd>
                                    <dd>MVP: 0.683</dd>
                                    <dd>ALPHA_MISSENSE: 0.989</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DAAM1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100592">DAAM1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.721</b>
                    <data>(p=2.1E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.705</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.752</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.705 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=DAAM1">DAAM1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0008788, abnormal fetal cardiomyocyte morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DAAM1%0DDVL2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DVL2</a> and phenotypic similarity 0.731 to mouse mutant of DVL2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000284, double outlet right ventricle</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.721</b>
                                    <data>(p=2.1E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.705</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.752</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>14-59827588-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs181920728">rs181920728</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3049921">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.752</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000351081.1">DAAM1:ENST00000351081.1:c.2724+8T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360909.3">DAAM1:ENST00000360909.3:c.2694+8T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395125.1">DAAM1:ENST00000395125.1:c.2724+8T&gt;C:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0749%</dd>
                                    <dd>gnomAD_E_AMR: 0.0672%</dd>
                                    <dd>gnomAD_E_NFE: 0.3654%</dd>
                                    <dd>gnomAD_E_SAS: 0.1416%</dd>
                                    <dd>gnomAD_G_AFR: 0.1050%</dd>
                                    <dd>gnomAD_G_NFE: 0.2982%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SCN1A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000144285">SCN1A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.712</b>
                    <data>(p=2.3E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.530</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.945</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.431 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=293181">Malignant migrating focal seizures of infancy</a> associated with SCN1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0031834, Aortopulmonary collateral arteries</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0031834, Aortopulmonary collateral arteries</dd></dl><dl><dt>Phenotypic similarity 0.493 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SCN1A">SCN1A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001544, abnormal cardiovascular system physiology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001544, abnormal cardiovascular system physiology</dd></dl><dl><dt>Proximity score 0.530 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SCN1A%0DTBC1D24&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TBC1D24</a> and phenotypic similarity 0.653 to <a href="http://www.omim.org/entry/220500">DOORS syndrome</a> associated with TBC1D24.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/604403">OMIM:604403</a> Febrile seizures, familial, 3A - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/607208">OMIM:607208</a> Dravet syndrome - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/609634">OMIM:609634</a> Migraine, familial hemiplegic, 3 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/619317">OMIM:619317</a> Developmental and epileptic encephalopathy 6B, non-Dravet - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1942">ORPHA:1942</a> Myoclonic-astatic epilepsy - unknown</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2382">ORPHA:2382</a> Lennox-Gastaut syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=293181">ORPHA:293181</a> Malignant migrating focal seizures of infancy - autosomal dominant/recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=33069">ORPHA:33069</a> Dravet syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=36387">ORPHA:36387</a> Generalized epilepsy with febrile seizures-plus - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=569">ORPHA:569</a> Familial or sporadic hemiplegic migraine - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.712</b>
                                    <data>(p=2.3E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.530</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.945</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.431 to ORPHA:293181 Malignant migrating focal seizures of infancy</dt>
                                
                                    <dt>Phenotypic similarity 0.331 to ORPHA:569 Familial or sporadic hemiplegic migraine</dt>
                                
                                    <dt>Phenotypic similarity 0.331 to OMIM:609634 Migraine, familial hemiplegic, 3</dt>
                                
                                    <dt>Phenotypic similarity 0.318 to OMIM:607208 Dravet syndrome</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-166850811-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200263247">rs200263247</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM1_Supporting]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/572354">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.945</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000303395.4">SCN1A:ENST00000303395.4:c.4697G&gt;A:p.(Ser1566Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375405.3">SCN1A:ENST00000375405.3:c.4664G&gt;A:p.(Ser1555Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409050.1">SCN1A:ENST00000409050.1:c.4613G&gt;A:p.(Ser1538Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423058.2">SCN1A:ENST00000423058.2:c.4697G&gt;A:p.(Ser1566Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96169114</dd>
                                    
                                    <dd>REVEL: 0.565</dd>
                                    <dd>MVP: 0.962</dd>
                                    <dd>ALPHA_MISSENSE: 0.283</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0087%</dd>
                                    <dd>gnomAD_G_AMR: 0.1182%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CPB1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000153002">CPB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.686</b>
                    <data>(p=2.8E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.510</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.955</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CPB1%0DCPE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CPE</a> and phenotypic similarity 0.607 to <a href="http://www.omim.org/entry/619326">BDV syndrome</a> associated with CPE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CPB1%0DCPE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CPE</a> and phenotypic similarity 0.356 to mouse mutant of CPE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010463, aorta stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010463, aorta stenosis</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.686</b>
                                    <data>(p=2.8E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.510</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.955</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                            <b>SNV</b>
                            <b>3-148559694-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs141911824">rs141911824</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.955</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000282957.4">CPB1:ENST00000282957.4:c.559C&gt;T:p.(Gln187*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000491148.1">CPB1:ENST00000491148.1:c.559C&gt;T:p.(Gln187*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.2892%</dd>
                                    <dd>gnomAD_E_AMR: 0.0231%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.2295%</dd>
                                    <dd>gnomAD_G_AMR: 0.1182%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TKTL2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000151005">TKTL2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.680</b>
                    <data>(p=2.9E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.508</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.954</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TKTL2%0DG6PC3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to G6PC3</a> and phenotypic similarity 0.617 to <a href="http://www.omim.org/entry/612541">Dursun syndrome</a> associated with G6PC3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0010774, Cor triatriatum</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001684, Secundum atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.680</b>
                                    <data>(p=2.9E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.508</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.954</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-164393253-T-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202081893">rs202081893</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2357625">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.954</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000280605.3">TKTL2:ENST00000280605.3:c.1634A&gt;T:p.(Asp545Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9590869</dd>
                                    
                                    <dd>REVEL: 0.865</dd>
                                    <dd>MVP: 0.959</dd>
                                    <dd>ALPHA_MISSENSE: 0.603</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0376%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AKAP12" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000131016">AKAP12</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.666</b>
                    <data>(p=3.2E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.643</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.794</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.643 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=AKAP12">AKAP12</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000278, abnormal myocardial fiber morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0006113, abnormal heart septum morphology</dd></dl><dl><dt>Phenotypic similarity 0.345 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=AKAP12">AKAP12</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0004649, heart primordium morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0004649, heart primordium morphology, abnormal</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AKAP12%0DMMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MMP2</a> and phenotypic similarity 0.626 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=371428">Multicentric osteolysis-nodulosis-arthropathy spectrum</a> associated with MMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001719, Double outlet right ventricle</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AKAP12%0DMMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MMP2</a> and phenotypic similarity 0.512 to mouse mutant of MMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AKAP12%0DMMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MMP2</a> and phenotypic similarity 0.251 to fish mutant of MMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0013882, interrenal  angiogenic sprout decreased length, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0013882, interrenal  angiogenic sprout decreased length, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.666</b>
                                    <data>(p=3.2E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.643</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.794</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>6-151627046-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201662069">rs201662069</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.794</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000253332.1">AKAP12:ENST00000253332.1:c.319+8G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000402676.2">AKAP12:ENST00000402676.2:c.319+8G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0561%</dd>
                                    <dd>gnomAD_E_EAS: 0.0062%</dd>
                                    <dd>gnomAD_E_NFE: 0.0103%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PARP8" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000151883">PARP8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.647</b>
                    <data>(p=3.6E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.455</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.998</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.455 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PARP8">PARP8</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002833, increased heart weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002833, increased heart weight</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.647</b>
                                    <data>(p=3.6E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.455</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.998</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-50117060-T-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs140755482">rs140755482</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.998</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000281631.5">PARP8:ENST00000281631.5:c.1682T&gt;A:p.(Leu561Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000505554.1">PARP8:ENST00000505554.1:c.1619T&gt;A:p.(Leu540Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000505697.2">PARP8:ENST00000505697.2:c.1682T&gt;A:p.(Leu561Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000503750.2">PARP8:ENST00000503750.2:c.1669-1109T&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000514067.2">PARP8:ENST00000514067.2:c.1669-1109T&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000514342.2">PARP8:ENST00000514342.2:c.928-1109T&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9982</dd>
                                    
                                    <dd>REVEL: 0.472</dd>
                                    <dd>MVP: 0.364</dd>
                                    <dd>ALPHA_MISSENSE: 0.998</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TBC1D1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000065882">TBC1D1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.638</b>
                    <data>(p=3.8E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.940</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBC1D1%0DPRKAG2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAG2</a> and phenotypic similarity 0.975 to <a href="http://www.omim.org/entry/600858">Cardiomyopathy, hypertrophic 6</a> associated with PRKAG2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0031318, Myofiber disarray</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBC1D1%0DPRKAG2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAG2</a> and phenotypic similarity 0.490 to mouse mutant of PRKAG2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004115, abnormal sinoatrial node morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0006142, abnormal sinoatrial node conduction</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.638</b>
                                    <data>(p=3.8E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.940</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-38119753-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200460442">rs200460442</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2341252">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.940</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261439.4">TBC1D1:ENST00000261439.4:c.2902C&gt;T:p.(Pro968Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000508802.1">TBC1D1:ENST00000508802.1:c.3084+2178C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9444</dd>
                                    
                                    <dd>REVEL: 0.395</dd>
                                    <dd>MVP: 0.558</dd>
                                    <dd>ALPHA_MISSENSE: 0.944</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0376%</dd>
                                    <dd>gnomAD_E_NFE: 0.0070%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CYP39A1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000146233">CYP39A1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.627</b>
                    <data>(p=4.0E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.934</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CYP39A1%0DHSD17B7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HSD17B7</a> and phenotypic similarity 0.600 to mouse mutant of HSD17B7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010656, thick myocardium</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000282, abnormal interatrial septum morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.627</b>
                                    <data>(p=4.0E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.934</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-46563779-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147961800">rs147961800</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.934</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000275016.2">CYP39A1:ENST00000275016.2:c.1010G&gt;A:p.(Arg337His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.95729244</dd>
                                    
                                    <dd>REVEL: 0.917</dd>
                                    <dd>MVP: 0.957</dd>
                                    <dd>ALPHA_MISSENSE: 0.547</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0247%</dd>
                                    <dd>gnomAD_E_AMR: 0.1078%</dd>
                                    <dd>gnomAD_E_NFE: 0.1632%</dd>
                                    <dd>gnomAD_E_SAS: 0.0263%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                    <dd>gnomAD_G_NFE: 0.1038%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TRRAP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196367">TRRAP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.625</b>
                    <data>(p=4.0E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.635</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.784</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.635 to <a href="http://www.omim.org/entry/618454">Developmental delay with or without dysmorphic facies and autism</a> associated with TRRAP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.514 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRRAP%0DNPAT&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NPAT</a> and phenotypic similarity 0.646 to mouse mutant of NPAT.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010418, perimembraneous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010418, perimembraneous ventricular septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/618454">OMIM:618454</a> Developmental delay with or without dysmorphic facies and autism - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/618778">OMIM:618778</a> ?Deafness, autosomal dominant 75 (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.625</b>
                                    <data>(p=4.0E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.635</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.784</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.635 to OMIM:618454 Developmental delay with or without dysmorphic facies and autism</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>7-98547888-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202055793">rs202055793</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2050508">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:1, LB:1)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.784</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355540.3">TRRAP:ENST00000355540.3:c.5259+3G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359863.4">TRRAP:ENST00000359863.4:c.5313+3G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000446306.3">TRRAP:ENST00000446306.3:c.5256+3G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1417%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PTPDC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000158079">PTPDC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.620</b>
                    <data>(p=4.1E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.934</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PTPDC1%0DCPLANE2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CPLANE2</a> and phenotypic similarity 0.685 to mouse mutant of CPLANE2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000644, dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010406, common atrium</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.620</b>
                                    <data>(p=4.1E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.934</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-96857621-G-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs61743388">rs61743388</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.934</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000288976.3">PTPDC1:ENST00000288976.3:c.633G&gt;T:p.(Trp211Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375360.3">PTPDC1:ENST00000375360.3:c.477G&gt;T:p.(Trp159Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9845</dd>
                                    
                                    <dd>REVEL: 0.720</dd>
                                    <dd>MVP: 0.659</dd>
                                    <dd>ALPHA_MISSENSE: 0.984</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.3240%</dd>
                                    <dd>gnomAD_E_NFE: 0.0144%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>9-96870198-C-T [0|1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs61734082">rs61734082</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.232</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000288976.3">PTPDC1:ENST00000288976.3:c.2393C&gt;T:p.(Thr798Ile)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375360.3">PTPDC1:ENST00000375360.3:c.2237C&gt;T:p.(Thr746Ile)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            <dd>Best Score: 0.24621569</dd>
                            
                            <dd>REVEL: 0.026</dd>
                            <dd>MVP: 0.246</dd>
                            <dd>ALPHA_MISSENSE: 0.134</dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>gnomAD_E_AFR: 0.0206%</dd>
                            <dd>gnomAD_E_AMR: 0.3454%</dd>
                            <dd>gnomAD_E_NFE: 0.1707%</dd>
                            <dd>gnomAD_G_AFR: 0.0230%</dd>
                            <dd>gnomAD_G_AMR: 0.1179%</dd>
                            <dd>gnomAD_G_NFE: 0.0909%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ERBB3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000065361">ERBB3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.560</b>
                    <data>(p=5.2E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.538</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.864</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.839 to <a href="http://www.omim.org/entry/607598">?Lethal congenital contractural syndrome 2</a> associated with ERBB3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Phenotypic similarity 0.538 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ERBB3">ERBB3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005329, abnormal myocardium layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ERBB3%0DEYA4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EYA4</a> and phenotypic similarity 0.823 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=217622">Sensorineural deafness with dilated cardiomyopathy</a> associated with EYA4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ERBB3%0DEYA4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EYA4</a> and phenotypic similarity 0.323 to fish mutant of EYA4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/133180">OMIM:133180</a> ?Erythroleukemia, familial, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/243180">OMIM:243180</a> Visceral neuropathy, familial, 1, autosomal recessive - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/607598">OMIM:607598</a> ?Lethal congenital contractural syndrome 2 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=388">ORPHA:388</a> Hirschsprung disease - autosomal dominant/recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.560</b>
                                    <data>(p=5.2E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.538</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.864</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-56493677-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs56259600">rs56259600</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1163537">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.864</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000267101.3">ERBB3:ENST00000267101.3:c.2993A&gt;G:p.(Lys998Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000415288.2">ERBB3:ENST00000415288.2:c.2816A&gt;G:p.(Lys939Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000450146.2">ERBB3:ENST00000450146.2:c.1064A&gt;G:p.(Lys355Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000549832.1">ERBB3:ENST00000549832.1:c.353A&gt;G:p.(Lys118Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000553131.1">ERBB3:ENST00000553131.1:c.716A&gt;G:p.(Lys239Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.91234386</dd>
                                    
                                    <dd>REVEL: 0.304</dd>
                                    <dd>MVP: 0.912</dd>
                                    <dd>SPLICE_AI: 0.430</dd>
                                    <dd>ALPHA_MISSENSE: 0.083</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0752%</dd>
                                    <dd>gnomAD_E_NFE: 0.0431%</dd>
                                    <dd>gnomAD_E_SAS: 0.3299%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.0195%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DDX42" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198231">DDX42</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.518</b>
                    <data>(p=6.0E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.510</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.878</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DDX42%0DTP53BP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TP53BP2</a> and phenotypic similarity 0.769 to mouse mutant of TP53BP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010402, ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.518</b>
                                    <data>(p=6.0E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.510</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.878</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-61892969-A-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142666932">rs142666932</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2341651">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.878</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359353.5">DDX42:ENST00000359353.5:c.1592A&gt;T:p.(Lys531Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389924.2">DDX42:ENST00000389924.2:c.1949A&gt;T:p.(Lys650Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457800.2">DDX42:ENST00000457800.2:c.1949A&gt;T:p.(Lys650Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000578681.1">DDX42:ENST00000578681.1:c.1949A&gt;T:p.(Lys650Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583590.1">DDX42:ENST00000583590.1:c.1949A&gt;T:p.(Lys650Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8817</dd>
                                    
                                    <dd>REVEL: 0.235</dd>
                                    <dd>MVP: 0.468</dd>
                                    <dd>ALPHA_MISSENSE: 0.882</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                    <dd>gnomAD_E_NFE: 0.0317%</dd>
                                    <dd>gnomAD_E_SAS: 0.0163%</dd>
                                    <dd>gnomAD_G_NFE: 0.0324%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DCDC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170959">DCDC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.504</b>
                    <data>(p=6.2E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.882</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DCDC1%0DABCA5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ABCA5</a> and phenotypic similarity 0.753 to mouse mutant of ABCA5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003503, decreased activity of thyroid gland</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002795, dilated cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.504</b>
                                    <data>(p=6.2E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.882</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-31312205-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201039841">rs201039841</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2387120">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.882</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000452803.1">DCDC1:ENST00000452803.1:c.949A&gt;G:p.(Thr317Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000597505.1">DCDC1:ENST00000597505.1:c.949A&gt;G:p.(Thr317Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.887031</dd>
                                    
                                    <dd>REVEL: 0.109</dd>
                                    <dd>MVP: 0.887</dd>
                                    <dd>ALPHA_MISSENSE: 0.112</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0373%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC26A4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000091137">SLC26A4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.489</b>
                    <data>(p=6.5E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.873</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.338 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=705">Pendred syndrome</a> associated with SLC26A4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000821, Hypothyroidism</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC26A4%0DNKX2-1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NKX2-1</a> and phenotypic similarity 0.727 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=209905">Brain-lung-thyroid syndrome</a> associated with NKX2-1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001671, Abnormal cardiac septum morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC26A4%0DNKX2-1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NKX2-1</a> and phenotypic similarity 0.428 to mouse mutant of NKX2-1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002295, abnormal pulmonary circulation</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002295, abnormal pulmonary circulation</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/274600">OMIM:274600</a> Pendred syndrome - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/600791">OMIM:600791</a> Deafness, autosomal recessive 4, with enlarged vestibular aqueduct - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=705">ORPHA:705</a> Pendred syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=95713">ORPHA:95713</a> Athyreosis - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=95720">ORPHA:95720</a> Thyroid hypoplasia - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.489</b>
                                    <data>(p=6.5E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.873</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.338 to ORPHA:95713 Athyreosis</dt>
                                
                                    <dt>Phenotypic similarity 0.338 to ORPHA:95720 Thyroid hypoplasia</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-107303865-G-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200139879">rs200139879</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM1, PM5_Supporting]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2050509">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.873</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265715.3">SLC26A4:ENST00000265715.3:c.289G&gt;T:p.(Val97Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8786026</dd>
                                    
                                    <dd>REVEL: 0.435</dd>
                                    <dd>MVP: 0.879</dd>
                                    <dd>ALPHA_MISSENSE: 0.746</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0463%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MAP3K3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198909">MAP3K3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.488</b>
                    <data>(p=6.5E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.537</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.834</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.527 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MAP3K3">MAP3K3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005329, abnormal myocardium layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.537 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAP3K3%0DCPLANE2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CPLANE2</a> and phenotypic similarity 0.685 to mouse mutant of CPLANE2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000644, dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010406, common atrium</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.488</b>
                                    <data>(p=6.5E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.537</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.834</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-61767045-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201458544">rs201458544</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.834</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361357.3">MAP3K3:ENST00000361357.3:c.1106C&gt;T:p.(Ala369Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361733.3">MAP3K3:ENST00000361733.3:c.1013C&gt;T:p.(Ala338Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000577395.1">MAP3K3:ENST00000577395.1:c.1001C&gt;T:p.(Ala334Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579585.1">MAP3K3:ENST00000579585.1:c.1106C&gt;T:p.(Ala369Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000584573.1">MAP3K3:ENST00000584573.1:c.1094C&gt;T:p.(Ala365Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8742115</dd>
                                    
                                    <dd>REVEL: 0.272</dd>
                                    <dd>MVP: 0.874</dd>
                                    <dd>ALPHA_MISSENSE: 0.094</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.2901%</dd>
                                    <dd>gnomAD_E_NFE: 0.0035%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="HK3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000160883">HK3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.474</b>
                    <data>(p=6.8E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.510</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.858</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=HK3%0DPMM2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PMM2</a> and phenotypic similarity 0.833 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79318">PMM2-CDG</a> associated with PMM2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001698, Pericardial effusion</dd></dl><dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=HK3%0DPMM2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PMM2</a> and phenotypic similarity 0.715 to mouse mutant of PMM2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005329, abnormal myocardium layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010725, thin interventricular septum</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.474</b>
                                    <data>(p=6.8E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.510</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.858</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-176318150-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199663371">rs199663371</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/717587">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.858</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000292432.5">HK3:ENST00000292432.5:c.302C&gt;T:p.(Ala101Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98564416</dd>
                                    
                                    <dd>REVEL: 0.673</dd>
                                    <dd>MVP: 0.986</dd>
                                    <dd>ALPHA_MISSENSE: 0.174</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0308%</dd>
                                    <dd>gnomAD_E_AMR: 0.6707%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.5896%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="UBQLN3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000175520">UBQLN3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.452</b>
                    <data>(p=7.2E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.860</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=UBQLN3%0DCDKN1C&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDKN1C</a> and phenotypic similarity 0.817 to <a href="http://www.omim.org/entry/130650">Beckwith-Wiedemann syndrome</a> associated with CDKN1C.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001640, Cardiomegaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=UBQLN3%0DCDKN1C&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDKN1C</a> and phenotypic similarity 0.479 to mouse mutant of CDKN1C.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002842, increased systemic arterial blood pressure</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002842, increased systemic arterial blood pressure</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.452</b>
                                    <data>(p=7.2E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.860</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-5530137-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201719446">rs201719446</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.860</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311659.4">UBQLN3:ENST00000311659.4:c.652C&gt;T:p.(Arg218Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380259.2">UBQLN3:ENST00000380259.2:c.-565+81918C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.87479776</dd>
                                    
                                    <dd>REVEL: 0.483</dd>
                                    <dd>MVP: 0.875</dd>
                                    <dd>ALPHA_MISSENSE: 0.185</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0318%</dd>
                                    <dd>gnomAD_E_EAS: 0.0109%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SETD1A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000099381">SETD1A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.437</b>
                    <data>(p=7.4E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.626</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.711</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.626 to <a href="http://www.omim.org/entry/618832">Epilepsy, early-onset, 2, with or without developmental delay</a> associated with SETD1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001655, Patent foramen ovale</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001655, Patent foramen ovale</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SETD1A%0DKDM5B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KDM5B</a> and phenotypic similarity 0.661 to <a href="http://www.omim.org/entry/618109">Intellectual developmental disorder, autosomal recessive 65</a> associated with KDM5B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/618832">OMIM:618832</a> Epilepsy, early-onset, 2, with or without developmental delay - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/619056">OMIM:619056</a> Neurodevelopmental disorder with speech impairment and dysmorphic facies - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.437</b>
                                    <data>(p=7.4E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.626</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.711</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.626 to OMIM:618832 Epilepsy, early-onset, 2, with or without developmental delay</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-30976694-C-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199846877">rs199846877</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP4, BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2184689">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.711</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262519.8">SETD1A:ENST00000262519.8:c.1631C&gt;G:p.(Ala544Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7113944</dd>
                                    
                                    <dd>REVEL: 0.235</dd>
                                    <dd>MVP: 0.711</dd>
                                    <dd>ALPHA_MISSENSE: 0.121</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0087%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TMUB1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000164897">TMUB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.425</b>
                    <data>(p=7.6E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.848</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TMUB1%0DIL6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IL6</a> and phenotypic similarity 0.409 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=85414">Systemic-onset juvenile idiopathic arthritis</a> associated with IL6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001701, Pericarditis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001701, Pericarditis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TMUB1%0DIL6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IL6</a> and phenotypic similarity 0.607 to mouse mutant of IL6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001544, abnormal cardiovascular system physiology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000281, abnormal interventricular septum morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.425</b>
                                    <data>(p=7.6E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.848</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-150778923-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202144001">rs202144001</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.848</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000297533.4">TMUB1:ENST00000297533.4:c.454G&gt;A:p.(Asp152Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392818.3">TMUB1:ENST00000392818.3:c.454G&gt;A:p.(Asp152Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000462940.1">TMUB1:ENST00000462940.1:c.454G&gt;A:p.(Asp152Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000476627.1">TMUB1:ENST00000476627.1:c.454G&gt;A:p.(Asp152Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000482202.1">TMUB1:ENST00000482202.1:c.454G&gt;A:p.(Asp152Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.85259104</dd>
                                    
                                    <dd>REVEL: 0.335</dd>
                                    <dd>MVP: 0.853</dd>
                                    <dd>ALPHA_MISSENSE: 0.336</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0404%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DGKZ" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000149091">DGKZ</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.414</b>
                    <data>(p=7.9E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.837</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DGKZ%0DAGPAT2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AGPAT2</a> and phenotypic similarity 0.899 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=528">Congenital generalized lipodystrophy</a> associated with AGPAT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.414</b>
                                    <data>(p=7.9E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.837</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-46401108-C-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs76583617">rs76583617</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/431729">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:1, LB:1, B:1)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.837</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000318201.8">DGKZ:ENST00000318201.8:c.2594C&gt;G:p.(Thr865Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343674.6">DGKZ:ENST00000343674.6:c.2711C&gt;G:p.(Thr904Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395574.3">DGKZ:ENST00000395574.3:c.2561C&gt;G:p.(Thr854Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421244.2">DGKZ:ENST00000421244.2:c.2663C&gt;G:p.(Thr888Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454345.1">DGKZ:ENST00000454345.1:c.3227C&gt;G:p.(Thr1076Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456247.2">DGKZ:ENST00000456247.2:c.2660C&gt;G:p.(Thr887Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000527911.1">DGKZ:ENST00000527911.1:c.2678C&gt;G:p.(Thr893Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000528615.1">DGKZ:ENST00000528615.1:c.1997C&gt;G:p.(Thr666Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000532868.2">DGKZ:ENST00000532868.2:c.2675C&gt;G:p.(Thr892Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543978.1">DGKZ:ENST00000543978.1:c.719C&gt;G:p.(Thr240Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9544401</dd>
                                    
                                    <dd>REVEL: 0.566</dd>
                                    <dd>MVP: 0.954</dd>
                                    <dd>ALPHA_MISSENSE: 0.590</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0941%</dd>
                                    <dd>gnomAD_E_AMR: 0.2521%</dd>
                                    <dd>gnomAD_E_NFE: 0.6184%</dd>
                                    <dd>gnomAD_E_SAS: 0.0101%</dd>
                                    <dd>gnomAD_G_AFR: 0.1263%</dd>
                                    <dd>gnomAD_G_AMR: 0.4717%</dd>
                                    <dd>gnomAD_G_NFE: 0.6483%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CYP2C19" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165841">CYP2C19</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.412</b>
                    <data>(p=7.9E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.554</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.781</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.554 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CYP2C19%0DVEZF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to VEZF1</a> and phenotypic similarity 0.839 to <a href="http://www.omim.org/entry/620247">?Cardiomyopathy, dilated, 1OO</a> associated with VEZF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.554 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CYP2C19%0DVEZF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to VEZF1</a> and phenotypic similarity 0.479 to mouse mutant of VEZF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001914, hemorrhage</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.412</b>
                                    <data>(p=7.9E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.554</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.781</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>10-96540422-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200936950">rs200936950</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.781</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371321.3">CYP2C19:ENST00000371321.3:c.642+6G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1626%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GUCY1B1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000061918">GUCY1B1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.407</b>
                    <data>(p=8.0E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.839</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.481 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GUCY1B1">GUCY1B1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000230, abnormal systemic arterial blood pressure</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000230, abnormal systemic arterial blood pressure</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GUCY1B1%0DPOLR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POLR1A</a> and phenotypic similarity 0.777 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1200">Burn-McKeown syndrome</a> associated with POLR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001671, Abnormal cardiac septum morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GUCY1B1%0DPOLR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POLR1A</a> and phenotypic similarity 0.323 to fish mutant of POLR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.407</b>
                                    <data>(p=8.0E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.839</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-156717580-A-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199558345">rs199558345</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.839</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264424.8">GUCY1B1:ENST00000264424.8:c.895A&gt;T:p.(Thr299Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000502959.1">GUCY1B1:ENST00000502959.1:c.961A&gt;T:p.(Thr321Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000503520.1">GUCY1B1:ENST00000503520.1:c.895A&gt;T:p.(Thr299Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000505154.1">GUCY1B1:ENST00000505154.1:c.691A&gt;T:p.(Thr231Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000505764.1">GUCY1B1:ENST00000505764.1:c.835A&gt;T:p.(Thr279Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000507146.1">GUCY1B1:ENST00000507146.1:c.691A&gt;T:p.(Thr231Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513437.1">GUCY1B1:ENST00000513437.1:c.691A&gt;T:p.(Thr231Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8400401</dd>
                                    
                                    <dd>REVEL: 0.221</dd>
                                    <dd>MVP: 0.840</dd>
                                    <dd>ALPHA_MISSENSE: 0.097</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0116%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="HLTF" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000071794">HLTF</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.393</b>
                    <data>(p=8.2E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.833</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=HLTF%0DRNF220&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RNF220</a> and phenotypic similarity 0.768 to <a href="http://www.omim.org/entry/619688">Leukodystrophy, hypomyelinating, 23, with ataxia, deafness, liver dysfunction, and dilated cardiomyopathy</a> associated with RNF220.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.393</b>
                                    <data>(p=8.2E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.833</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-148757909-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs61750365">rs61750365</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3056755">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.833</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000310053.5">HLTF:ENST00000310053.5:c.2411T&gt;C:p.(Ile804Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392912.2">HLTF:ENST00000392912.2:c.2411T&gt;C:p.(Ile804Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000465259.1">HLTF:ENST00000465259.1:c.2408T&gt;C:p.(Ile803Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000494055.1">HLTF:ENST00000494055.1:c.2411T&gt;C:p.(Ile804Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9021784</dd>
                                    
                                    <dd>REVEL: 0.563</dd>
                                    <dd>MVP: 0.902</dd>
                                    <dd>ALPHA_MISSENSE: 0.815</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0556%</dd>
                                    <dd>gnomAD_E_AMR: 0.2298%</dd>
                                    <dd>gnomAD_E_NFE: 0.4478%</dd>
                                    <dd>gnomAD_E_SAS: 0.0463%</dd>
                                    <dd>gnomAD_G_AFR: 0.0459%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.3436%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="HSPBP1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000133265">HSPBP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.372</b>
                    <data>(p=8.6E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.824</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=HSPBP1%0DBAG5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BAG5</a> and phenotypic similarity 0.823 to <a href="http://www.omim.org/entry/619747">Cardiomyopathy, dilated, 2F</a> associated with BAG5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=HSPBP1%0DBAG5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BAG5</a> and phenotypic similarity 0.504 to mouse mutant of BAG5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000278, abnormal myocardial fiber morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002753, dilated heart left ventricle</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.372</b>
                                    <data>(p=8.6E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.824</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-55776753-C-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199998150">rs199998150</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/749881">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.824</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000255631.5">HSPBP1:ENST00000255631.5:c.898G&gt;T:p.(Val300Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000433386.2">HSPBP1:ENST00000433386.2:c.898G&gt;T:p.(Val300Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000587922.1">HSPBP1:ENST00000587922.1:c.898G&gt;T:p.(Val300Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376343.3">HSPBP1:ENST00000376343.3:c.610-18G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8672</dd>
                                    
                                    <dd>REVEL: 0.241</dd>
                                    <dd>MVP: 0.392</dd>
                                    <dd>SPLICE_AI: 0.280</dd>
                                    <dd>ALPHA_MISSENSE: 0.867</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.3155%</dd>
                                    <dd>gnomAD_G_AMR: 0.1182%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TRAF3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000131323">TRAF3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.365</b>
                    <data>(p=8.8E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.521</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.797</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.521 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRAF3%0DTNFRSF11A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TNFRSF11A</a> and phenotypic similarity 0.651 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1782">Dysosteosclerosis</a> associated with TNFRSF11A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/614849">OMIM:614849</a> ?Encephalopathy, acute, infection-induced (herpes-specific), susceptibility to, 5 (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1930">ORPHA:1930</a> Herpes simplex virus encephalitis (susceptibility)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.365</b>
                                    <data>(p=8.8E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.521</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.797</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>14-103342072-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142069424">rs142069424</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/758872">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.797</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000347662.4">TRAF3:ENST00000347662.4:c.402+7A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000351691.5">TRAF3:ENST00000351691.5:c.402+7A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392745.2">TRAF3:ENST00000392745.2:c.402+7A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539721.1">TRAF3:ENST00000539721.1:c.402+7A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000560371.1">TRAF3:ENST00000560371.1:c.402+7A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0260%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC35D2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130958">SLC35D2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.362</b>
                    <data>(p=8.8E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.811</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC35D2%0DEXT2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EXT2</a> and phenotypic similarity 0.643 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=466926">Seizures-scoliosis-macrocephaly syndrome</a> associated with EXT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0030680, Abnormal cardiovascular system morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC35D2%0DEXT2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EXT2</a> and phenotypic similarity 0.323 to fish mutant of EXT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.362</b>
                                    <data>(p=8.8E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.811</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-99122481-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200009581">rs200009581</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3164358">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.811</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000253270.7">SLC35D2:ENST00000253270.7:c.302A&gt;G:p.(Tyr101Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375257.1">SLC35D2:ENST00000375257.1:c.302A&gt;G:p.(Tyr101Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375259.4">SLC35D2:ENST00000375259.4:c.302A&gt;G:p.(Tyr101Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8127</dd>
                                    
                                    <dd>REVEL: 0.483</dd>
                                    <dd>MVP: 0.711</dd>
                                    <dd>ALPHA_MISSENSE: 0.813</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0116%</dd>
                                    <dd>gnomAD_E_NFE: 0.0026%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TUBA3D" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000075886">TUBA3D</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.360</b>
                    <data>(p=8.9E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.816</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TUBA3D%0DGJA1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GJA1</a> and phenotypic similarity 0.616 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2248">Hypoplastic left heart syndrome</a> associated with GJA1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0004383, Hypoplastic left heart</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TUBA3D%0DGJA1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GJA1</a> and phenotypic similarity 0.696 to mouse mutant of GJA1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005329, abnormal myocardium layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0004225, patent cardiac foramen ovale</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/617928">OMIM:617928</a> Keratoconus 9 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.360</b>
                                    <data>(p=8.9E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.816</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-132237732-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs145612794">rs145612794</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.816</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000321253.6">TUBA3D:ENST00000321253.6:c.466C&gt;T:p.(Arg156Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8993</dd>
                                    
                                    <dd>REVEL: 0.692</dd>
                                    <dd>MVP: 0.516</dd>
                                    <dd>ALPHA_MISSENSE: 0.899</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0804%</dd>
                                    <dd>gnomAD_E_AMR: 0.0261%</dd>
                                    <dd>gnomAD_E_EAS: 0.0327%</dd>
                                    <dd>gnomAD_E_NFE: 0.1041%</dd>
                                    <dd>gnomAD_E_SAS: 0.0721%</dd>
                                    <dd>gnomAD_G_AFR: 0.1527%</dd>
                                    <dd>gnomAD_G_EAS: 0.5208%</dd>
                                    <dd>gnomAD_G_NFE: 0.1211%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC39A3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000141873">SLC39A3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.356</b>
                    <data>(p=9.0E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.516</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.799</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.516 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC39A3%0DSLC39A8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC39A8</a> and phenotypic similarity 0.714 to mouse mutant of SLC39A8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002189, abnormal myocardial trabeculae morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.356</b>
                                    <data>(p=9.0E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.516</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.799</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>19-2732747-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199727394">rs199727394</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.799</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545664.1">SLC39A3:ENST00000545664.1:c.941+6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000586572.1">SLC39A3:ENST00000586572.1:c.210+4299C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000269740.4">SLC39A3:ENST00000269740.4:c.*2C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.49</dd>
                                    
                                    <dd>SPLICE_AI: 0.490</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0114%</dd>
                                    <dd>gnomAD_E_NFE: 0.0044%</dd>
                                    <dd>gnomAD_E_SAS: 0.0093%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NDUFB1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183648">NDUFB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.341</b>
                    <data>(p=9.3E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.803</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NDUFB1%0DNDUFAF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NDUFAF1</a> and phenotypic similarity 0.962 to <a href="http://www.omim.org/entry/618234">Mitochondrial complex I deficiency, nuclear type 11</a> associated with NDUFAF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.341</b>
                                    <data>(p=9.3E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.803</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-92583974-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201547910">rs201547910</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.803</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329559.3">NDUFB1:ENST00000329559.3:c.149A&gt;G:p.(Asn50Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000553514.1">NDUFB1:ENST00000553514.1:c.8A&gt;G:p.(Asn3Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000555441.1">NDUFB1:ENST00000555441.1:c.8A&gt;G:p.(Asn3Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000605997.1">NDUFB1:ENST00000605997.1:c.8A&gt;G:p.(Asn3Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8034723</dd>
                                    
                                    <dd>REVEL: 0.235</dd>
                                    <dd>MVP: 0.803</dd>
                                    <dd>ALPHA_MISSENSE: 0.102</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                    <dd>gnomAD_E_NFE: 0.0026%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CHRNB3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000147432">CHRNB3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.337</b>
                    <data>(p=9.3E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.807</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CHRNB3%0DAPOE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to APOE</a> and phenotypic similarity 0.466 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=412">Dysbetalipoproteinemia</a> associated with APOE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0005181, Premature coronary artery atherosclerosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0005181, Premature coronary artery atherosclerosis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CHRNB3%0DAPOE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to APOE</a> and phenotypic similarity 0.642 to mouse mutant of APOE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0031384, impaired blood-brain barrier function</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001625, cardiac hypertrophy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.337</b>
                                    <data>(p=9.3E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.807</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-42587301-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs75384358">rs75384358</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.807</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000289957.2">CHRNB3:ENST00000289957.2:c.851A&gt;G:p.(Glu284Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.95395106</dd>
                                    
                                    <dd>REVEL: 0.838</dd>
                                    <dd>MVP: 0.954</dd>
                                    <dd>ALPHA_MISSENSE: 0.792</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.7584%</dd>
                                    <dd>gnomAD_G_AMR: 0.4717%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FGF22" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000070388">FGF22</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.325</b>
                    <data>(p=9.6E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.799</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FGF22%0DFGFR3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FGFR3</a> and phenotypic similarity 0.643 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93274">Thanatophoric dysplasia type 2</a> associated with FGFR3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.325</b>
                                    <data>(p=9.6E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.799</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>19-643402-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200852866">rs200852866</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.799</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000166133.3">FGF22:ENST00000166133.3:c.196-8G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000215530.5">FGF22:ENST00000215530.5:c.319-8G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000586042.2">FGF22:ENST00000586042.2:c.319-31G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0059%</dd>
                                    <dd>gnomAD_E_NFE: 0.0058%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="HAUS8" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000131351">HAUS8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.325</b>
                    <data>(p=9.6E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.801</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=HAUS8%0DSF3B2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SF3B2</a> and phenotypic similarity 0.607 to <a href="http://www.omim.org/entry/164210">Craniofacial microsomia</a> associated with SF3B2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.325</b>
                                    <data>(p=9.6E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.801</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-17160939-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs61734359">rs61734359</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.801</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000253669.5">HAUS8:ENST00000253669.5:c.977C&gt;T:p.(Ala326Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000448593.2">HAUS8:ENST00000448593.2:c.974C&gt;T:p.(Ala325Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000593360.1">HAUS8:ENST00000593360.1:c.794C&gt;T:p.(Ala265Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8205794</dd>
                                    
                                    <dd>REVEL: 0.286</dd>
                                    <dd>MVP: 0.821</dd>
                                    <dd>ALPHA_MISSENSE: 0.193</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1610%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MYO19" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000278259">MYO19</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.320</b>
                    <data>(p=9.7E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.799</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MYO19%0DYY1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to YY1</a> and phenotypic similarity 0.607 to <a href="http://www.omim.org/entry/617557">Gabriele-de Vries syndrome</a> associated with YY1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001655, Patent foramen ovale</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001655, Patent foramen ovale</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MYO19%0DYY1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to YY1</a> and phenotypic similarity 0.467 to fish mutant of YY1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000107, heart morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001777, atrium aplastic, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.320</b>
                                    <data>(p=9.7E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.799</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>17-34871839-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201822341">rs201822341</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.799</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000268852.9">MYO19:ENST00000268852.9:c.415-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000431794.3">MYO19:ENST00000431794.3:c.415-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544606.1">MYO19:ENST00000544606.1:c.13-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000586007.1">MYO19:ENST00000586007.1:c.415-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0117%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ACO1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000122729">ACO1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.304</b>
                    <data>(p=1.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.514</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.775</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.514 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACO1%0DSLC40A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC40A1</a> and phenotypic similarity 0.837 to <a href="http://www.omim.org/entry/606069">Hemochromatosis, type 4</a> associated with SLC40A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl><dt>Proximity score 0.514 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACO1%0DSLC40A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC40A1</a> and phenotypic similarity 0.378 to mouse mutant of SLC40A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0020367, increased heart iron level</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0020367, increased heart iron level</dd></dl><dl><dt>Proximity score 0.514 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACO1%0DSLC40A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC40A1</a> and phenotypic similarity 0.277 to fish mutant of SLC40A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000512, dorsal longitudinal anastomotic vessel structure, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000512, dorsal longitudinal anastomotic vessel structure, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.304</b>
                                    <data>(p=1.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.514</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.775</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-32436303-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202244128">rs202244128</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.775</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000309951.6">ACO1:ENST00000309951.6:c.2155A&gt;G:p.(Met719Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379923.1">ACO1:ENST00000379923.1:c.2155A&gt;G:p.(Met719Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541043.1">ACO1:ENST00000541043.1:c.1858A&gt;G:p.(Met620Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7773548</dd>
                                    
                                    <dd>REVEL: 0.704</dd>
                                    <dd>MVP: 0.777</dd>
                                    <dd>ALPHA_MISSENSE: 0.732</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0202%</dd>
                                    <dd>gnomAD_E_NFE: 0.0062%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TRAPPC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170043">TRAPPC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.300</b>
                    <data>(p=1.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.516</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.771</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.516 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TRAPPC1">TRAPPC1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002188, small heart</dd></dl><dl><dt>Proximity score 0.515 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRAPPC1%0DTRAPPC10&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRAPPC10</a> and phenotypic similarity 0.761 to mouse mutant of TRAPPC10.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010412, atrioventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.300</b>
                                    <data>(p=1.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.516</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.771</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>17-7835043-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs76144653">rs76144653</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.771</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000303731.4">TRAPPC1:ENST00000303731.4:c.99+6G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540486.1">TRAPPC1:ENST00000540486.1:c.99+6G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.1070%</dd>
                                    <dd>gnomAD_E_NFE: 0.2274%</dd>
                                    <dd>gnomAD_E_SAS: 0.1241%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.1688%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ARL14" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000179674">ARL14</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.291</b>
                    <data>(p=1.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.784</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARL14%0DNSD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NSD1</a> and phenotypic similarity 0.659 to <a href="http://www.omim.org/entry/117550">Sotos syndrome</a> associated with NSD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.291</b>
                                    <data>(p=1.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.784</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-160395207-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146619115">rs146619115</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3_Strong]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.784</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000320767.2">ARL14:ENST00000320767.2:c.73G&gt;A:p.(Gly25Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9739815</dd>
                                    
                                    <dd>REVEL: 0.960</dd>
                                    <dd>MVP: 0.974</dd>
                                    <dd>ALPHA_MISSENSE: 0.955</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.8927%</dd>
                                    <dd>gnomAD_E_AMR: 0.0609%</dd>
                                    <dd>gnomAD_E_NFE: 0.0026%</dd>
                                    <dd>gnomAD_G_AFR: 0.7812%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ALPI" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163295">ALPI</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.287</b>
                    <data>(p=1.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.781</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ALPI%0DHACD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HACD1</a> and phenotypic similarity 0.650 to <a href="http://www.omim.org/entry/619967">Congenital myopathy 11</a> associated with HACD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ALPI%0DHACD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HACD1</a> and phenotypic similarity 0.417 to mouse mutant of HACD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001614, abnormal blood vessel morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001614, abnormal blood vessel morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.287</b>
                                    <data>(p=1.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.781</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>2-233322701-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201251145">rs201251145</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3034146">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.781</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295463.3">ALPI:ENST00000295463.3:c.857-7C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.1594%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.1334%</dd>
                                    <dd>gnomAD_E_SAS: 0.1046%</dd>
                                    <dd>gnomAD_G_AFR: 0.0460%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.1234%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FAAP20" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162585">FAAP20</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.280</b>
                    <data>(p=1.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.771</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FAAP20%0DFANCB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FANCB</a> and phenotypic similarity 0.844 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with FANCB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.280</b>
                                    <data>(p=1.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.771</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>1-2125347-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202236326">rs202236326</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.771</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378545.3">FAAP20:ENST00000378545.3:c.510G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378546.4">FAAP20:ENST00000378546.4:c.201G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400919.3">FAAP20:ENST00000400919.3:c.-877G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3848</dd>
                                    
                                    <dd>REVEL: 0.019</dd>
                                    <dd>MVP: 0.231</dd>
                                    <dd>ALPHA_MISSENSE: 0.385</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1176%</dd>
                                    <dd>gnomAD_E_NFE: 0.0350%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.0195%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="STK11IP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000144589">STK11IP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.280</b>
                    <data>(p=1.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.777</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.466 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=STK11IP">STK11IP</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004486, decreased response of heart to induced stress</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0004486, decreased response of heart to induced stress</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=STK11IP%0DSMAD4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMAD4</a> and phenotypic similarity 0.735 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2588">Myhre syndrome</a> associated with SMAD4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001671, Abnormal cardiac septum morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=STK11IP%0DSMAD4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMAD4</a> and phenotypic similarity 0.476 to fish mutant of SMAD4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000486, cardiac ventricle structure, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000486, cardiac ventricle structure, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.280</b>
                                    <data>(p=1.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.777</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-220471207-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199837177">rs199837177</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.777</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295641.10">STK11IP:ENST00000295641.10:c.988G&gt;A:p.(Asp330Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456909.1">STK11IP:ENST00000456909.1:c.955G&gt;A:p.(Asp319Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7861</dd>
                                    
                                    <dd>REVEL: 0.230</dd>
                                    <dd>MVP: 0.105</dd>
                                    <dd>SPLICE_AI: 0.300</dd>
                                    <dd>ALPHA_MISSENSE: 0.786</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0791%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TNS3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136205">TNS3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.272</b>
                    <data>(p=1.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.773</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.444 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TNS3">TNS3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005140, decreased cardiac muscle contractility</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0005140, decreased cardiac muscle contractility</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TNS3%0DADGRA2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ADGRA2</a> and phenotypic similarity 0.679 to mouse mutant of ADGRA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0031384, impaired blood-brain barrier function</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001614, abnormal blood vessel morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TNS3%0DADGRA2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ADGRA2</a> and phenotypic similarity 0.274 to fish mutant of ADGRA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0102131, blood vessel brain morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0102131, blood vessel brain morphology, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.272</b>
                                    <data>(p=1.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.773</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-47342713-G-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs188962919">rs188962919</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/713684">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.773</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311160.9">TNS3:ENST00000311160.9:c.3292C&gt;G:p.(Leu1098Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355730.3">TNS3:ENST00000355730.3:c.2572C&gt;G:p.(Leu858Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000398879.1">TNS3:ENST00000398879.1:c.3292C&gt;G:p.(Leu1098Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.91704035</dd>
                                    
                                    <dd>REVEL: 0.477</dd>
                                    <dd>MVP: 0.917</dd>
                                    <dd>ALPHA_MISSENSE: 0.524</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1179%</dd>
                                    <dd>gnomAD_E_AMR: 0.2406%</dd>
                                    <dd>gnomAD_E_NFE: 0.7172%</dd>
                                    <dd>gnomAD_E_SAS: 0.0752%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_AMR: 0.4717%</dd>
                                    <dd>gnomAD_G_NFE: 0.7722%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ITFG1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000129636">ITFG1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.267</b>
                    <data>(p=1.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.771</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ITFG1%0DCDC45&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDC45</a> and phenotypic similarity 0.737 to <a href="http://www.omim.org/entry/617063">Meier-Gorlin syndrome 7</a> associated with CDC45.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001674, Complete atrioventricular canal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001674, Complete atrioventricular canal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.267</b>
                                    <data>(p=1.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.771</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>16-47294448-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200576597">rs200576597</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.771</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000320640.6">ITFG1:ENST00000320640.6:c.1221+8C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544001.2">ITFG1:ENST00000544001.2:c.882+8C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0781%</dd>
                                    <dd>gnomAD_G_AMR: 0.2370%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="UNC79" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000133958">UNC79</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.253</b>
                    <data>(p=1.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.761</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=UNC79%0DSLC26A7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC26A7</a> and phenotypic similarity 0.729 to mouse mutant of SLC26A7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003503, decreased activity of thyroid gland</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.253</b>
                                    <data>(p=1.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.761</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-94007022-G-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs139796738">rs139796738</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3041972">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.761</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000256339.4">UNC79:ENST00000256339.4:c.838G&gt;C:p.(Glu280Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393151.2">UNC79:ENST00000393151.2:c.1369G&gt;C:p.(Glu457Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000553484.1">UNC79:ENST00000553484.1:c.1369G&gt;C:p.(Glu457Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000555664.1">UNC79:ENST00000555664.1:c.1369G&gt;C:p.(Glu457Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8151</dd>
                                    
                                    <dd>REVEL: 0.257</dd>
                                    <dd>MVP: 0.129</dd>
                                    <dd>ALPHA_MISSENSE: 0.815</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.3997%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PIK3CD" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171608">PIK3CD</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.251</b>
                    <data>(p=1.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.514</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.746</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.455 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=221139">Combined immunodeficiency with facio-oculo-skeletal anomalies</a> associated with PIK3CD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0031014, Arteria lusoria</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0031014, Arteria lusoria</dd></dl><dl><dt>Proximity score 0.514 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PIK3CD%0DPIK3R2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PIK3R2</a> and phenotypic similarity 0.773 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=83473">Megalencephaly-polymicrogyria-postaxial polydactyly-hydrocephalus syndrome</a> associated with PIK3R2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001671, Abnormal cardiac septum morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/613328">OMIM:613328</a> ?Roifman-Chitayat syndrome, digenic (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/615513">OMIM:615513</a> Immunodeficiency 14A, autosomal dominant - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/619281">OMIM:619281</a> Immunodeficiency 14B, autosomal recessive - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=221139">ORPHA:221139</a> Combined immunodeficiency with facio-oculo-skeletal anomalies - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=397596">ORPHA:397596</a> Activated PI3K-delta syndrome - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.251</b>
                                    <data>(p=1.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.514</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.746</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>1-9781304-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs151278626">rs151278626</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/474024">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.746</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361110.2">PIK3CD:ENST00000361110.2:c.1881G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377346.4">PIK3CD:ENST00000377346.4:c.1809G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536656.1">PIK3CD:ENST00000536656.1:c.1881G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543390.1">PIK3CD:ENST00000543390.1:c.810G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0309%</dd>
                                    <dd>gnomAD_E_AMR: 0.0405%</dd>
                                    <dd>gnomAD_E_NFE: 0.4030%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.0804%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.3829%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PRR21" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=">PRR21</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.246</b>
                    <data>(p=1.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.641</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.599</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.641 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PRR21%0DPRR12&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRR12</a> and phenotypic similarity 0.626 to <a href="http://www.omim.org/entry/619539">Neuroocular syndrome</a> associated with PRR12.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001655, Patent foramen ovale</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001655, Patent foramen ovale</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.246</b>
                                    <data>(p=1.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.641</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.599</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-240982171-A-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs78953006">rs78953006</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.599</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000408934.1">PRR21:ENST00000408934.1:c.229T&gt;G:p.(Ser77Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0102%</dd>
                                    <dd>gnomAD_E_AMR: 0.0053%</dd>
                                    <dd>gnomAD_E_NFE: 0.0026%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SCIN" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000006747">SCIN</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.234</b>
                    <data>(p=1.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.749</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SCIN%0DACTC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTC1</a> and phenotypic similarity 0.909 to <a href="http://www.omim.org/entry/612098">Cardiomyopathy, hypertrophic, 11</a> associated with ACTC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SCIN%0DACTC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTC1</a> and phenotypic similarity 0.556 to mouse mutant of ACTC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002190, disorganized myocardium</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002190, disorganized myocardium</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SCIN%0DACTC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTC1</a> and phenotypic similarity 0.473 to fish mutant of ACTC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0014931, endocardial cushion agenesis, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000860, cardiac ventricle dilated, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.234</b>
                                    <data>(p=1.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.749</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-12668760-T-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs188068227">rs188068227</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.749</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000297029.5">SCIN:ENST00000297029.5:c.1232T&gt;A:p.(Val411Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445618.2">SCIN:ENST00000445618.2:c.491T&gt;A:p.(Val164Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000519209.1">SCIN:ENST00000519209.1:c.491T&gt;A:p.(Val164Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8703815</dd>
                                    
                                    <dd>REVEL: 0.361</dd>
                                    <dd>MVP: 0.870</dd>
                                    <dd>ALPHA_MISSENSE: 0.742</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0194%</dd>
                                    <dd>gnomAD_E_AMR: 0.5881%</dd>
                                    <dd>gnomAD_E_EAS: 0.0447%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                    <dd>gnomAD_G_AMR: 0.7075%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC4A5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188687">SLC4A5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.226</b>
                    <data>(p=1.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.745</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.474 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SLC4A5">SLC4A5</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0006143, increased systemic arterial diastolic blood pressure</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0006143, increased systemic arterial diastolic blood pressure</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RP11-287D1.3%0DALMS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ALMS1</a> and phenotypic similarity 0.745 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=64">Alström syndrome</a> associated with ALMS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001685, Myocardial fibrosis</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RP11-287D1.3%0DALMS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ALMS1</a> and phenotypic similarity 0.511 to fish mutant of ALMS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0005415, cardiac ventricle morphology, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.226</b>
                                    <data>(p=1.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.745</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-74481817-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200346496">rs200346496</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.745</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346834.4">SLC4A5:ENST00000346834.4:c.1042C&gt;G:p.(Leu348Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357822.5">SLC4A5:ENST00000357822.5:c.1042C&gt;G:p.(Leu348Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358683.4">SLC4A5:ENST00000358683.4:c.850C&gt;G:p.(Leu284Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359484.4">SLC4A5:ENST00000359484.4:c.850C&gt;G:p.(Leu284Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377632.1">SLC4A5:ENST00000377632.1:c.1042C&gt;G:p.(Leu348Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377634.4">SLC4A5:ENST00000377634.4:c.1042C&gt;G:p.(Leu348Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394019.2">SLC4A5:ENST00000394019.2:c.1042C&gt;G:p.(Leu348Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423644.1">SLC4A5:ENST00000423644.1:c.1042C&gt;G:p.(Leu348Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000451608.2">SLC4A5:ENST00000451608.2:c.*1630C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.745</dd>
                                    
                                    <dd>REVEL: 0.745</dd>
                                    <dd>MVP: 0.639</dd>
                                    <dd>ALPHA_MISSENSE: 0.576</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LRP1B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000168702">LRP1B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.218</b>
                    <data>(p=1.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.741</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRP1B%0DPTPN11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTPN11</a> and phenotypic similarity 0.908 to <a href="http://www.omim.org/entry/151100">LEOPARD syndrome 1</a> associated with PTPN11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001674, Complete atrioventricular canal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRP1B%0DPTPN11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTPN11</a> and phenotypic similarity 0.748 to mouse mutant of PTPN11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002652, thin myocardium</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRP1B%0DPTPN11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTPN11</a> and phenotypic similarity 0.426 to fish mutant of PTPN11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001192, heart edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.218</b>
                                    <data>(p=1.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.741</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-141773306-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs143197572">rs143197572</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.741</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389484.3">LRP1B:ENST00000389484.3:c.2149G&gt;A:p.(Asp717Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7434985</dd>
                                    
                                    <dd>REVEL: 0.486</dd>
                                    <dd>MVP: 0.743</dd>
                                    <dd>ALPHA_MISSENSE: 0.558</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0232%</dd>
                                    <dd>gnomAD_E_EAS: 0.0055%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CLCA1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000016490">CLCA1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.206</b>
                    <data>(p=1.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.734</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CLCA1%0DPHYH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PHYH</a> and phenotypic similarity 0.817 to <a href="http://www.omim.org/entry/266500">Refsum disease</a> associated with PHYH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001640, Cardiomegaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.206</b>
                                    <data>(p=1.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.734</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-86964434-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201131437">rs201131437</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.734</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000234701.3">CLCA1:ENST00000234701.3:c.2293G&gt;A:p.(Gly765Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394711.1">CLCA1:ENST00000394711.1:c.2293G&gt;A:p.(Gly765Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.73631704</dd>
                                    
                                    <dd>REVEL: 0.190</dd>
                                    <dd>MVP: 0.736</dd>
                                    <dd>ALPHA_MISSENSE: 0.133</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0231%</dd>
                                    <dd>gnomAD_E_NFE: 0.0035%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ADGRF1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000153292">ADGRF1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.203</b>
                    <data>(p=1.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.491</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.743</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.491 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ADGRF1">ADGRF1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.203</b>
                                    <data>(p=1.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.491</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.743</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-46977304-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs151335777">rs151335777</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3085156">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.743</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000283297.5">ADGRF1:ENST00000283297.5:c.1276C&gt;T:p.(Arg426Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371253.2">ADGRF1:ENST00000371253.2:c.1867C&gt;T:p.(Arg623Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7557344</dd>
                                    
                                    <dd>REVEL: 0.545</dd>
                                    <dd>MVP: 0.756</dd>
                                    <dd>ALPHA_MISSENSE: 0.153</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0290%</dd>
                                    <dd>gnomAD_E_EAS: 0.0055%</dd>
                                    <dd>gnomAD_E_NFE: 0.0044%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PPL" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000118898">PPL</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.197</b>
                    <data>(p=1.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.512</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.714</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.512 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PPL%0DKRT8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KRT8</a> and phenotypic similarity 0.775 to mouse mutant of KRT8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0006372, impaired placental function</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001634, internal hemorrhage</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.197</b>
                                    <data>(p=1.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.512</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.714</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-4935517-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200407505">rs200407505</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.714</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000345988.2">PPL:ENST00000345988.2:c.3139C&gt;T:p.(Arg1047Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000590782.2">PPL:ENST00000590782.2:c.3133C&gt;T:p.(Arg1045Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7148875</dd>
                                    
                                    <dd>REVEL: 0.082</dd>
                                    <dd>MVP: 0.715</dd>
                                    <dd>ALPHA_MISSENSE: 0.106</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0087%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ATP8B1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000081923">ATP8B1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.191</b>
                    <data>(p=1.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.716</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.338 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=69665">Intrahepatic cholestasis of pregnancy</a> associated with ATP8B1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000821, Hypothyroidism</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Phenotypic similarity 0.452 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ATP8B1">ATP8B1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005333, decreased heart rate</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002834, decreased heart weight</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ATP8B1%0DATP11A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ATP11A</a> and phenotypic similarity 0.338 to <a href="http://www.omim.org/entry/619851">?Leukodystrophy, hypomyelinating, 24</a> associated with ATP11A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000821, Hypothyroidism</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ATP8B1%0DATP11A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ATP11A</a> and phenotypic similarity 0.624 to mouse mutant of ATP11A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004057, thin myocardium compact layer</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010420, muscular ventricular septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/147480">OMIM:147480</a> Cholestasis, intrahepatic, of pregnancy, 1 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/211600">OMIM:211600</a> Cholestasis, progressive familial intrahepatic 1 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/243300">OMIM:243300</a> Cholestasis, benign recurrent intrahepatic - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=69665">ORPHA:69665</a> Intrahepatic cholestasis of pregnancy (susceptibility)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.191</b>
                                    <data>(p=1.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.716</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.338 to ORPHA:69665 Intrahepatic cholestasis of pregnancy</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>18-55359082-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs34315917">rs34315917</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/259815">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:1, LB:3, B:4)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.716</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000283684.4">ATP8B1:ENST00000283684.4:c.1177A&gt;G:p.(Ile393Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536015.1">ATP8B1:ENST00000536015.1:c.1177A&gt;G:p.(Ile393Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8323948</dd>
                                    
                                    <dd>REVEL: 0.434</dd>
                                    <dd>MVP: 0.832</dd>
                                    <dd>ALPHA_MISSENSE: 0.101</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0984%</dd>
                                    <dd>gnomAD_E_AMR: 0.4972%</dd>
                                    <dd>gnomAD_E_NFE: 0.6147%</dd>
                                    <dd>gnomAD_E_SAS: 0.0784%</dd>
                                    <dd>gnomAD_G_AFR: 0.1149%</dd>
                                    <dd>gnomAD_G_AMR: 0.7075%</dd>
                                    <dd>gnomAD_G_NFE: 0.5638%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CCR3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183625">CCR3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.190</b>
                    <data>(p=1.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.512</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.710</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.512 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CCR3">CCR3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CCR3%0DHRH3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HRH3</a> and phenotypic similarity 0.781 to mouse mutant of HRH3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0031384, impaired blood-brain barrier function</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0031384, impaired blood-brain barrier function</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.190</b>
                                    <data>(p=1.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.512</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.710</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-46307377-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142318962">rs142318962</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.710</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357422.2">CCR3:ENST00000357422.2:c.728T&gt;C:p.(Val243Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395940.2">CCR3:ENST00000395940.2:c.728T&gt;C:p.(Val243Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395942.2">CCR3:ENST00000395942.2:c.728T&gt;C:p.(Val243Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541018.1">CCR3:ENST00000541018.1:c.728T&gt;C:p.(Val243Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545097.1">CCR3:ENST00000545097.1:c.791T&gt;C:p.(Val264Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.71010035</dd>
                                    
                                    <dd>REVEL: 0.107</dd>
                                    <dd>MVP: 0.710</dd>
                                    <dd>ALPHA_MISSENSE: 0.230</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TMEM87B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000153214">TMEM87B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.185</b>
                    <data>(p=1.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.431</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.798</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.431 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=TMEM87B">TMEM87B</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0002485, cardiac ventricle hypoplastic, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0002485, cardiac ventricle hypoplastic, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.185</b>
                                    <data>(p=1.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.431</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.798</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>2-112834862-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146130208">rs146130208</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.798</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000283206.4">TMEM87B:ENST00000283206.4:c.592+3A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8</dd>
                                    
                                    <dd>SPLICE_AI: 0.800</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0221%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="EMID1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000186998">EMID1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.184</b>
                    <data>(p=1.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.719</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EMID1%0DTNFRSF21&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TNFRSF21</a> and phenotypic similarity 0.720 to mouse mutant of TNFRSF21.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0031384, impaired blood-brain barrier function</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001915, intracranial hemorrhage</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EMID1%0DTNFRSF21&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TNFRSF21</a> and phenotypic similarity 0.250 to fish mutant of TNFRSF21.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0002187, brain vasculature has fewer parts of type central artery, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0002187, brain vasculature has fewer parts of type central artery, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.184</b>
                                    <data>(p=1.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.719</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>22-29654833-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs61736091">rs61736091</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.719</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000334018.6">EMID1:ENST00000334018.6:c.1232C&gt;T:p.(Pro411Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000404755.3">EMID1:ENST00000404755.3:c.1169C&gt;T:p.(Pro390Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000404820.3">EMID1:ENST00000404820.3:c.1238C&gt;T:p.(Pro413Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7965322</dd>
                                    
                                    <dd>REVEL: 0.150</dd>
                                    <dd>MVP: 0.797</dd>
                                    <dd>ALPHA_MISSENSE: 0.100</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.5436%</dd>
                                    <dd>gnomAD_E_AMR: 0.0581%</dd>
                                    <dd>gnomAD_E_NFE: 0.0092%</dd>
                                    <dd>gnomAD_G_AFR: 0.4706%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MAP9" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000164114">MAP9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.182</b>
                    <data>(p=1.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.717</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAP9%0DARSL&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ARSL</a> and phenotypic similarity 0.632 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79345">Brachytelephalangic chondrodysplasia punctata</a> associated with ARSL.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.182</b>
                                    <data>(p=1.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.717</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>4-156273742-A-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199795369">rs199795369</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.717</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311277.4">MAP9:ENST00000311277.4:c.1821+6T&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000515654.1">MAP9:ENST00000515654.1:c.1749+6T&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.5677%</dd>
                                    <dd>gnomAD_E_AMR: 0.0068%</dd>
                                    <dd>gnomAD_E_EAS: 0.0059%</dd>
                                    <dd>gnomAD_E_NFE: 0.0075%</dd>
                                    <dd>gnomAD_E_SAS: 0.0037%</dd>
                                    <dd>gnomAD_G_AFR: 0.4250%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PPHLN1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000134283">PPHLN1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.181</b>
                    <data>(p=1.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.716</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PPHLN1%0DKRT8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KRT8</a> and phenotypic similarity 0.775 to mouse mutant of KRT8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0006372, impaired placental function</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001634, internal hemorrhage</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.181</b>
                                    <data>(p=1.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.716</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-42835294-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202148947">rs202148947</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.716</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358314.7">PPHLN1:ENST00000358314.7:c.1087T&gt;C:p.(Phe363Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395580.3">PPHLN1:ENST00000395580.3:c.1108T&gt;C:p.(Phe370Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449194.2">PPHLN1:ENST00000449194.2:c.1030T&gt;C:p.(Phe344Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000549190.1">PPHLN1:ENST00000549190.1:c.1141T&gt;C:p.(Phe381Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000552761.1">PPHLN1:ENST00000552761.1:c.943T&gt;C:p.(Phe315Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000256678.8">PPHLN1:ENST00000256678.8:c.664+63T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317560.9">PPHLN1:ENST00000317560.9:c.823+63T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000337898.6">PPHLN1:ENST00000337898.6:c.859+63T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395568.2">PPHLN1:ENST00000395568.2:c.1024+63T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000432191.2">PPHLN1:ENST00000432191.2:c.859+63T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7171</dd>
                                    
                                    <dd>REVEL: 0.082</dd>
                                    <dd>MVP: 0.605</dd>
                                    <dd>ALPHA_MISSENSE: 0.717</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0098%</dd>
                                    <dd>gnomAD_E_EAS: 0.0056%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="POLQ" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000051341">POLQ</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.163</b>
                    <data>(p=1.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.702</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=POLQ%0DDDX59&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DDX59</a> and phenotypic similarity 0.631 to <a href="http://www.omim.org/entry/174300">Orofaciodigital syndrome V</a> associated with DDX59.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.163</b>
                                    <data>(p=1.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.702</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-121206542-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200881849">rs200881849</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1746255">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.702</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264233.5">POLQ:ENST00000264233.5:c.5236A&gt;G:p.(Lys1746Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.70573074</dd>
                                    
                                    <dd>REVEL: 0.156</dd>
                                    <dd>MVP: 0.706</dd>
                                    <dd>ALPHA_MISSENSE: 0.099</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0347%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="APC" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000134982">APC</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.162</b>
                    <data>(p=1.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.515</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.684</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.338 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3258">Cenani-Lenz syndrome</a> associated with APC.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000821, Hypothyroidism</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Phenotypic similarity 0.484 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=APC">APC</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0012700, abnormal endocardial heart tube morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0012700, abnormal endocardial heart tube morphology</dd></dl><dl><dt>Phenotypic similarity 0.507 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=APC">APC</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0005538, endocardium hyperplastic, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000378, cardiac jelly increased volume, abnormal</dd></dl><dl><dt>Proximity score 0.515 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=APC%0DAMER1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AMER1</a> and phenotypic similarity 0.653 to <a href="http://www.omim.org/entry/300373">Osteopathia striata with cranial sclerosis</a> associated with AMER1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.515 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=APC%0DAMER1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AMER1</a> and phenotypic similarity 0.487 to mouse mutant of AMER1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000274, enlarged heart</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000274, enlarged heart</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/114500">OMIM:114500</a> Colorectal cancer, somatic - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/114550">OMIM:114550</a> Hepatoblastoma, somatic - somatic</dd>
<dd><a href="http://www.omim.org/entry/135290">OMIM:135290</a> Desmoid disease, hereditary - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/175100">OMIM:175100</a> Brain tumor-polyposis syndrome 2 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/613659">OMIM:613659</a> Gastric cancer, somatic - somatic</dd>
<dd><a href="http://www.omim.org/entry/619182">OMIM:619182</a> Gastric adenocarcinoma and proximal polyposis of the stomach - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=247806">ORPHA:247806</a> APC-related attenuated familial adenomatous polyposis - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3258">ORPHA:3258</a> Cenani-Lenz syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79665">ORPHA:79665</a> Gardner syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=873">ORPHA:873</a> Desmoid tumor (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=99818">ORPHA:99818</a> Turcot syndrome with polyposis - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.162</b>
                                    <data>(p=1.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.515</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.684</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-112177036-G-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201988789">rs201988789</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1237153">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.684</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000257430.4">APC:ENST00000257430.4:c.5745G&gt;C:p.(Lys1915Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457016.1">APC:ENST00000457016.1:c.5745G&gt;C:p.(Lys1915Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000508376.2">APC:ENST00000508376.2:c.5745G&gt;C:p.(Lys1915Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000520401.1">APC:ENST00000520401.1:c.230+12367G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.88408935</dd>
                                    
                                    <dd>REVEL: 0.375</dd>
                                    <dd>MVP: 0.884</dd>
                                    <dd>ALPHA_MISSENSE: 0.379</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.9823%</dd>
                                    <dd>gnomAD_E_AMR: 0.0087%</dd>
                                    <dd>gnomAD_E_EAS: 0.0440%</dd>
                                    <dd>gnomAD_E_NFE: 0.0797%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SCN7A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136546">SCN7A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.161</b>
                    <data>(p=1.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.700</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SCN7A%0DCAMK2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAMK2A</a> and phenotypic similarity 0.651 to <a href="http://www.omim.org/entry/617798">Intellectual developmental disorder, autosomal dominant 53</a> associated with CAMK2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SCN7A%0DCAMK2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAMK2A</a> and phenotypic similarity 0.295 to fish mutant of CAMK2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0005428, heart tube inverted, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0005428, heart tube inverted, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.161</b>
                                    <data>(p=1.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.700</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-167266360-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs141677829">rs141677829</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.700</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409855.1">SCN7A:ENST00000409855.1:c.3797T&gt;C:p.(Ile1266Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7116497</dd>
                                    
                                    <dd>REVEL: 0.325</dd>
                                    <dd>MVP: 0.712</dd>
                                    <dd>ALPHA_MISSENSE: 0.086</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0217%</dd>
                                    <dd>gnomAD_G_AMR: 0.1185%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TMPRSS7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000176040">TMPRSS7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.148</b>
                    <data>(p=1.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.691</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TMPRSS7%0DFBXL4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FBXL4</a> and phenotypic similarity 0.889 to <a href="http://www.omim.org/entry/615471">Mitochondrial DNA depletion syndrome 13 (encephalomyopathic type)</a> associated with FBXL4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.148</b>
                                    <data>(p=1.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.691</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-111794246-T-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200656460">rs200656460</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.691</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000419127.1">TMPRSS7:ENST00000419127.1:c.1484T&gt;G:p.(Val495Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000452346.2">TMPRSS7:ENST00000452346.2:c.1862T&gt;G:p.(Val621Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.694</dd>
                                    
                                    <dd>REVEL: 0.694</dd>
                                    <dd>MVP: 0.477</dd>
                                    <dd>ALPHA_MISSENSE: 0.304</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0350%</dd>
                                    <dd>gnomAD_E_NFE: 0.0073%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RUVBL2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183207">RUVBL2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.145</b>
                    <data>(p=1.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.508</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.679</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.481 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=RUVBL2">RUVBL2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0002174, cardiac ventricle non-functional, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0003048, cardiac ventricle hyperplastic, abnormal</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RUVBL2%0DKANSL1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KANSL1</a> and phenotypic similarity 0.743 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=363958">17q21.31 microdeletion syndrome</a> associated with KANSL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.145</b>
                                    <data>(p=1.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.508</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.679</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-49514319-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201529731">rs201529731</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.679</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000413176.2">RUVBL2:ENST00000413176.2:c.716G&gt;A:p.(Arg239His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000595090.1">RUVBL2:ENST00000595090.1:c.851G&gt;A:p.(Arg284His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000601968.1">RUVBL2:ENST00000601968.1:c.716G&gt;A:p.(Arg239His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6789599</dd>
                                    
                                    <dd>MVP: 0.679</dd>
                                    <dd>ALPHA_MISSENSE: 0.652</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BOC" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000144857">BOC</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.144</b>
                    <data>(p=1.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.685</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BOC%0DEMD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EMD</a> and phenotypic similarity 0.962 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98863">X-linked Emery-Dreifuss muscular dystrophy</a> associated with EMD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BOC%0DEMD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EMD</a> and phenotypic similarity 0.512 to mouse mutant of EMD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000278, abnormal myocardial fiber morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000278, abnormal myocardial fiber morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.144</b>
                                    <data>(p=1.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.685</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-112993456-A-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs141826592">rs141826592</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.685</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000273395.4">BOC:ENST00000273395.4:c.1469A&gt;T:p.(Glu490Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355385.3">BOC:ENST00000355385.3:c.1469A&gt;T:p.(Glu490Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000495514.1">BOC:ENST00000495514.1:c.1469A&gt;T:p.(Glu490Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8437331</dd>
                                    
                                    <dd>REVEL: 0.357</dd>
                                    <dd>MVP: 0.844</dd>
                                    <dd>ALPHA_MISSENSE: 0.581</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0438%</dd>
                                    <dd>gnomAD_E_AMR: 0.8709%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                    <dd>gnomAD_G_AMR: 0.4717%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ACLY" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000131473">ACLY</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.142</b>
                    <data>(p=1.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.511</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.673</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.511 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACLY%0DCOX10&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COX10</a> and phenotypic similarity 0.837 to <a href="http://www.omim.org/entry/619046">Mitochondrial complex IV deficiency, nuclear type 3</a> associated with COX10.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.142</b>
                                    <data>(p=1.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.511</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.673</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-40042392-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148168367">rs148168367</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/731742">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.673</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000352035.2">ACLY:ENST00000352035.2:c.2045A&gt;G:p.(Tyr682Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000353196.1">ACLY:ENST00000353196.1:c.2015A&gt;G:p.(Tyr672Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393896.2">ACLY:ENST00000393896.2:c.2015A&gt;G:p.(Tyr672Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537919.1">ACLY:ENST00000537919.1:c.1232A&gt;G:p.(Tyr411Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000590151.1">ACLY:ENST00000590151.1:c.2045A&gt;G:p.(Tyr682Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.73007023</dd>
                                    
                                    <dd>REVEL: 0.556</dd>
                                    <dd>MVP: 0.730</dd>
                                    <dd>ALPHA_MISSENSE: 0.083</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                    <dd>gnomAD_E_NFE: 0.1742%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.4538%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KIAA0391" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000258790">KIAA0391</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.137</b>
                    <data>(p=1.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.510</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.669</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PSMA6%0DCPE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CPE</a> and phenotypic similarity 0.607 to <a href="http://www.omim.org/entry/619326">BDV syndrome</a> associated with CPE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PSMA6%0DCPE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CPE</a> and phenotypic similarity 0.356 to mouse mutant of CPE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010463, aorta stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010463, aorta stenosis</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.137</b>
                                    <data>(p=1.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.510</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.669</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-35593137-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs45626032">rs45626032</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3043006">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.669</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000557565.1">KIAA0391:ENST00000557565.1:c.686C&gt;T:p.(Ala229Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.77952904</dd>
                                    
                                    <dd>REVEL: 0.367</dd>
                                    <dd>MVP: 0.780</dd>
                                    <dd>ALPHA_MISSENSE: 0.382</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0437%</dd>
                                    <dd>gnomAD_E_AMR: 0.1359%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.1629%</dd>
                                    <dd>gnomAD_E_SAS: 0.7154%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.1620%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ARV1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000173409">ARV1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.135</b>
                    <data>(p=1.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.243</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.970</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.298 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=442835">Non-specific early-onset epileptic encephalopathy</a> associated with ARV1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001257, Spasticity</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Phenotypic similarity 0.486 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ARV1">ARV1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001633, poor circulation</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001633, poor circulation</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617020">OMIM:617020</a> Developmental and epileptic encephalopathy 38 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=442835">ORPHA:442835</a> Non-specific early-onset epileptic encephalopathy - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.135</b>
                                    <data>(p=1.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.243</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.970</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-231124099-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs144095931">rs144095931</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.970</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000310256.2">ARV1:ENST00000310256.2:c.208G&gt;A:p.(Glu70Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366658.2">ARV1:ENST00000366658.2:c.175-1757G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9707</dd>
                                    
                                    <dd>REVEL: 0.709</dd>
                                    <dd>MVP: 0.820</dd>
                                    <dd>ALPHA_MISSENSE: 0.971</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0053%</dd>
                                    <dd>gnomAD_E_SAS: 0.0034%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ATP7B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000123191">ATP7B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.131</b>
                    <data>(p=1.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.958</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.376 to <a href="http://www.omim.org/entry/277900">Wilson disease</a> associated with ATP7B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000124, Renal tubular dysfunction</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0006580, Portal fibrosis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ATP7B%0DSLC40A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC40A1</a> and phenotypic similarity 0.837 to <a href="http://www.omim.org/entry/606069">Hemochromatosis, type 4</a> associated with SLC40A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ATP7B%0DSLC40A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC40A1</a> and phenotypic similarity 0.378 to mouse mutant of SLC40A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0020367, increased heart iron level</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0020367, increased heart iron level</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ATP7B%0DSLC40A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC40A1</a> and phenotypic similarity 0.277 to fish mutant of SLC40A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000512, dorsal longitudinal anastomotic vessel structure, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000512, dorsal longitudinal anastomotic vessel structure, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/277900">OMIM:277900</a> Wilson disease - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=905">ORPHA:905</a> Wilson disease - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.131</b>
                                    <data>(p=1.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.958</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>13-52509155-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs181250704">rs181250704</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/157957">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:15, LB:2)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.958</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000242839.4">ATP7B:ENST00000242839.4:c.4135C&gt;T:p.(Pro1379Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344297.5">ATP7B:ENST00000344297.5:c.3514C&gt;T:p.(Pro1172Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400366.3">ATP7B:ENST00000400366.3:c.3802C&gt;T:p.(Pro1268Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400370.3">ATP7B:ENST00000400370.3:c.2845C&gt;T:p.(Pro949Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000417240.2">ATP7B:ENST00000417240.2:c.1768C&gt;T:p.(Pro590Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000418097.2">ATP7B:ENST00000418097.2:c.3940C&gt;T:p.(Pro1314Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000448424.2">ATP7B:ENST00000448424.2:c.3901C&gt;T:p.(Pro1301Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9857301</dd>
                                    
                                    <dd>REVEL: 0.879</dd>
                                    <dd>MVP: 0.986</dd>
                                    <dd>ALPHA_MISSENSE: 0.359</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0453%</dd>
                                    <dd>gnomAD_E_AMR: 0.1420%</dd>
                                    <dd>gnomAD_E_NFE: 0.1894%</dd>
                                    <dd>gnomAD_G_AFR: 0.0460%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.1755%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KCNRG" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198553">KCNRG</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.124</b>
                    <data>(p=1.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.668</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KCNRG%0DJAG2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to JAG2</a> and phenotypic similarity 0.837 to <a href="http://www.omim.org/entry/619566">Muscular dystrophy, limb-girdle, autosomal recessive 27</a> associated with JAG2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.124</b>
                                    <data>(p=1.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.668</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>13-50589952-C-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200397876">rs200397876</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.668</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000312942.1">KCNRG:ENST00000312942.1:c.323C&gt;G:p.(Ala108Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360473.4">KCNRG:ENST00000360473.4:c.323C&gt;G:p.(Ala108Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378182.3">KCNRG:ENST00000378182.3:c.*2652C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.66870433</dd>
                                    
                                    <dd>REVEL: 0.060</dd>
                                    <dd>MVP: 0.669</dd>
                                    <dd>ALPHA_MISSENSE: 0.159</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0087%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF200" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000010539">ZNF200</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.123</b>
                    <data>(p=1.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.667</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF200%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.782 to <a href="http://www.omim.org/entry/608068">Neutrophilic dermatosis, acute febrile</a> associated with MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.123</b>
                                    <data>(p=1.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.667</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-3283644-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147207388">rs147207388</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.667</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396868.3">ZNF200:ENST00000396868.3:c.112G&gt;A:p.(Gly38Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396870.4">ZNF200:ENST00000396870.4:c.112G&gt;A:p.(Gly38Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396871.4">ZNF200:ENST00000396871.4:c.112G&gt;A:p.(Gly38Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000414144.2">ZNF200:ENST00000414144.2:c.112G&gt;A:p.(Gly38Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000431561.3">ZNF200:ENST00000431561.3:c.112G&gt;A:p.(Gly38Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000575948.1">ZNF200:ENST00000575948.1:c.112G&gt;A:p.(Gly38Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6675</dd>
                                    
                                    <dd>REVEL: 0.019</dd>
                                    <dd>MVP: 0.219</dd>
                                    <dd>ALPHA_MISSENSE: 0.668</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DPP10" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000175497">DPP10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.122</b>
                    <data>(p=1.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.666</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DPP10%0DIRX5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRX5</a> and phenotypic similarity 0.737 to <a href="http://www.omim.org/entry/611174">Hamamy syndrome</a> associated with IRX5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001674, Complete atrioventricular canal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001674, Complete atrioventricular canal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.122</b>
                                    <data>(p=1.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.666</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>2-116539979-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs141057562">rs141057562</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/788214">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.666</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000310323.8">DPP10:ENST00000310323.8:c.1510+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393147.2">DPP10:ENST00000393147.2:c.1543+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409163.1">DPP10:ENST00000409163.1:c.1381+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000410059.1">DPP10:ENST00000410059.1:c.1531+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.74</dd>
                                    
                                    <dd>SPLICE_AI: 0.740</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1171%</dd>
                                    <dd>gnomAD_E_AMR: 0.2233%</dd>
                                    <dd>gnomAD_E_NFE: 0.7625%</dd>
                                    <dd>gnomAD_E_SAS: 0.1014%</dd>
                                    <dd>gnomAD_G_AFR: 0.1378%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.8048%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DHX36" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000174953">DHX36</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.116</b>
                    <data>(p=1.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.660</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DHX36%0DWBP11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to WBP11</a> and phenotypic similarity 0.643 to <a href="http://www.omim.org/entry/619227">Vertebral, cardiac, tracheoesophageal, renal, and limb defects</a> associated with WBP11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0011623, Muscular ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011623, Muscular ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.116</b>
                                    <data>(p=1.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.660</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-154011573-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs143919087">rs143919087</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.660</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308361.6">DHX36:ENST00000308361.6:c.1634G&gt;A:p.(Arg545Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329463.5">DHX36:ENST00000329463.5:c.1592G&gt;A:p.(Arg531Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000496811.1">DHX36:ENST00000496811.1:c.1634G&gt;A:p.(Arg545Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544526.1">DHX36:ENST00000544526.1:c.1592G&gt;A:p.(Arg531Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6867</dd>
                                    
                                    <dd>REVEL: 0.381</dd>
                                    <dd>MVP: 0.554</dd>
                                    <dd>ALPHA_MISSENSE: 0.687</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.2101%</dd>
                                    <dd>gnomAD_E_AMR: 0.0290%</dd>
                                    <dd>gnomAD_E_NFE: 0.0062%</dd>
                                    <dd>gnomAD_G_AFR: 0.2528%</dd>
                                    <dd>gnomAD_G_AMR: 0.1185%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GLDN" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000186417">GLDN</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.111</b>
                    <data>(p=1.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.937</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GLDN%0DSPTB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SPTB</a> and phenotypic similarity 0.823 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=822">Hereditary spherocytosis</a> associated with SPTB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001723, Restrictive cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001723, Restrictive cardiomyopathy</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GLDN%0DSPTB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SPTB</a> and phenotypic similarity 0.547 to mouse mutant of SPTB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005329, abnormal myocardium layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000274, enlarged heart</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GLDN%0DSPTB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SPTB</a> and phenotypic similarity 0.481 to fish mutant of SPTB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000724, atrium dilated, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000724, atrium dilated, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617194">OMIM:617194</a> Lethal congenital contracture syndrome 11 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.111</b>
                                    <data>(p=1.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.937</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>15-51669682-G-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200838713">rs200838713</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.937</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335449.6">GLDN:ENST00000335449.6:c.400G&gt;T:p.(Gly134Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396399.2">GLDN:ENST00000396399.2:c.28G&gt;T:p.(Gly10Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.937365</dd>
                                    
                                    <dd>REVEL: 0.661</dd>
                                    <dd>MVP: 0.937</dd>
                                    <dd>ALPHA_MISSENSE: 0.848</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ANKRD2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165887">ANKRD2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.108</b>
                    <data>(p=1.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.645</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ANKRD2%0DTCAP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TCAP</a> and phenotypic similarity 0.924 to <a href="http://www.omim.org/entry/607487">Cardiomyopathy, hypertrophic, 25</a> associated with TCAP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001712, Left ventricular hypertrophy</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ANKRD2%0DTCAP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TCAP</a> and phenotypic similarity 0.521 to mouse mutant of TCAP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0006138, congestive heart failure</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003141, cardiac fibrosis</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ANKRD2%0DTCAP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TCAP</a> and phenotypic similarity 0.422 to fish mutant of TCAP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001192, heart edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.108</b>
                                    <data>(p=1.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.645</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-99338262-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs36020819">rs36020819</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/774685">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.645</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000298808.5">ANKRD2:ENST00000298808.5:c.437C&gt;T:p.(Pro146Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000307518.5">ANKRD2:ENST00000307518.5:c.437C&gt;T:p.(Pro146Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370655.1">ANKRD2:ENST00000370655.1:c.356C&gt;T:p.(Pro119Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000455090.1">ANKRD2:ENST00000455090.1:c.356C&gt;T:p.(Pro119Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.806232</dd>
                                    
                                    <dd>REVEL: 0.184</dd>
                                    <dd>MVP: 0.806</dd>
                                    <dd>ALPHA_MISSENSE: 0.140</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0575%</dd>
                                    <dd>gnomAD_E_AMR: 0.1801%</dd>
                                    <dd>gnomAD_E_NFE: 0.6948%</dd>
                                    <dd>gnomAD_E_SAS: 0.0227%</dd>
                                    <dd>gnomAD_G_AFR: 0.1494%</dd>
                                    <dd>gnomAD_G_NFE: 0.9083%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="POMGNT2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000144647">POMGNT2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.096</b>
                    <data>(p=1.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.285</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.880</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.324 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=899">Walker-Warburg syndrome</a> associated with POMGNT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0007731, Chorioretinal dysplasia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0007731, Chorioretinal dysplasia</dd></dl><dl><dt>Proximity score 0.570 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=POMGNT2%0DPOMK&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POMK</a> and phenotypic similarity 0.786 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=370959">Congenital muscular dystrophy with cerebellar involvement</a> associated with POMK.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/614830">OMIM:614830</a> Muscular dystrophy-dystroglycanopathy (congenital with brain and eye anomalies), type A, 8 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/618135">OMIM:618135</a> Muscular dystrophy-dystroglycanopathy (limb-girdle) type C, 8 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=899">ORPHA:899</a> Walker-Warburg syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.096</b>
                                    <data>(p=1.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.285</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.880</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-43121633-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201446071">rs201446071</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2072374">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.880</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344697.2">POMGNT2:ENST00000344697.2:c.1291G&gt;A:p.(Val431Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441964.1">POMGNT2:ENST00000441964.1:c.1291G&gt;A:p.(Val431Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.880917</dd>
                                    
                                    <dd>REVEL: 0.555</dd>
                                    <dd>MVP: 0.881</dd>
                                    <dd>ALPHA_MISSENSE: 0.207</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                    <dd>gnomAD_E_NFE: 0.0035%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC10A2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000125255">SLC10A2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.088</b>
                    <data>(p=1.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.265</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.892</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.530 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SLC10A2">SLC10A2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010579, increased heart left ventricle size</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010579, increased heart left ventricle size</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC10A2%0DHNF1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HNF1A</a> and phenotypic similarity 0.681 to mouse mutant of HNF1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003325, decreased liver function</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/613291">OMIM:613291</a> ?Bile acid malabsorption, primary, 1 (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.088</b>
                                    <data>(p=1.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.265</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.892</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>13-103701788-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs145541774">rs145541774</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2177058">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.892</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245312.3">SLC10A2:ENST00000245312.3:c.770C&gt;T:p.(Thr257Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.90000665</dd>
                                    
                                    <dd>REVEL: 0.803</dd>
                                    <dd>MVP: 0.900</dd>
                                    <dd>ALPHA_MISSENSE: 0.841</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0492%</dd>
                                    <dd>gnomAD_E_AMR: 0.0116%</dd>
                                    <dd>gnomAD_E_EAS: 0.0218%</dd>
                                    <dd>gnomAD_E_NFE: 0.0114%</dd>
                                    <dd>gnomAD_G_AFR: 0.0459%</dd>
                                    <dd>gnomAD_G_EAS: 0.0643%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="USH2A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000042781">USH2A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.072</b>
                    <data>(p=1.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.885</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.334 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=791">Retinitis pigmentosa</a> associated with USH2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000135, Hypogonadism</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=USH2A%0DB2M&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to B2M</a> and phenotypic similarity 0.673 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=314652">Variant ABeta2M amyloidosis</a> associated with B2M.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0030843, Cardiac amyloidosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0030843, Cardiac amyloidosis</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/276901">OMIM:276901</a> Usher syndrome, type 2A - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/613809">OMIM:613809</a> Retinitis pigmentosa 39 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=231178">ORPHA:231178</a> Usher syndrome type 2 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=791">ORPHA:791</a> Retinitis pigmentosa - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.072</b>
                                    <data>(p=1.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.885</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-216462627-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146824138">rs146824138</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/48481">(REVIEWED_BY_EXPERT_PANEL)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.885</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000307340.3">USH2A:ENST00000307340.3:c.1966G&gt;A:p.(Asp656Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366942.3">USH2A:ENST00000366942.3:c.1966G&gt;A:p.(Asp656Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366943.2">USH2A:ENST00000366943.2:c.1966G&gt;A:p.(Asp656Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.90143436</dd>
                                    
                                    <dd>REVEL: 0.073</dd>
                                    <dd>MVP: 0.901</dd>
                                    <dd>ALPHA_MISSENSE: 0.094</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0308%</dd>
                                    <dd>gnomAD_E_AMR: 0.0173%</dd>
                                    <dd>gnomAD_E_NFE: 0.1109%</dd>
                                    <dd>gnomAD_G_AFR: 0.0345%</dd>
                                    <dd>gnomAD_G_NFE: 0.1232%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RAD51D" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185379">RAD51D</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.071</b>
                    <data>(p=1.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.482</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.622</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.482 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RAD51D">RAD51D</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001787, pericardial edema</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001787, pericardial edema</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=145">ORPHA:145</a> Hereditary breast and/or ovarian cancer syndrome - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.071</b>
                                    <data>(p=1.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.482</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.622</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-33434093-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201141245">rs201141245</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/185178">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:9, LB:3)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.622</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000345365.6">RAD51D:ENST00000345365.6:c.394G&gt;A:p.(Val132Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394589.4">RAD51D:ENST00000394589.4:c.394G&gt;A:p.(Val132Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000460118.2">RAD51D:ENST00000460118.2:c.37G&gt;A:p.(Val13Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000590016.1">RAD51D:ENST00000590016.1:c.454G&gt;A:p.(Val152Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335858.7">RAD51D:ENST00000335858.7:c.145-593G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360276.3">RAD51D:ENST00000360276.3:c.345+292G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000593039.1">RAD51D:ENST00000593039.1:c.4-593G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6243043</dd>
                                    
                                    <dd>REVEL: 0.229</dd>
                                    <dd>MVP: 0.624</dd>
                                    <dd>ALPHA_MISSENSE: 0.110</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0308%</dd>
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                    <dd>gnomAD_E_NFE: 0.0053%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DNAAF10" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000243667">DNAAF10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.068</b>
                    <data>(p=1.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.596</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAAF10%0DRNF113A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RNF113A</a> and phenotypic similarity 0.755 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=33364">Trichothiodystrophy</a> associated with RNF113A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAAF10%0DRNF113A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RNF113A</a> and phenotypic similarity 0.323 to fish mutant of RNF113A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.068</b>
                                    <data>(p=1.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.596</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-68374629-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200352366">rs200352366</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.596</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295121.6">DNAAF10:ENST00000295121.6:c.254C&gt;T:p.(Thr85Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409164.1">DNAAF10:ENST00000409164.1:c.254C&gt;T:p.(Thr85Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000406245.2">DNAAF10:ENST00000406245.2:c.-50C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000406334.3">DNAAF10:ENST00000406334.3:c.*271C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.597</dd>
                                    
                                    <dd>REVEL: 0.469</dd>
                                    <dd>MVP: 0.448</dd>
                                    <dd>ALPHA_MISSENSE: 0.597</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0088%</dd>
                                    <dd>gnomAD_E_EAS: 0.0110%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PRAMEF7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204510">PRAMEF7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.067</b>
                    <data>(p=1.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.516</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.576</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.516 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PRAMEF7">PRAMEF7</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002188, small heart</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.067</b>
                                    <data>(p=1.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.516</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.576</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-12980122-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1641459392">rs1641459392</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.576</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361079.2">PRAMEF7:ENST00000361079.2:c.1314G&gt;C:p.(Lys438Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.2561%</dd>
                                    <dd>gnomAD_E_AMR: 0.1200%</dd>
                                    <dd>gnomAD_E_EAS: 0.1677%</dd>
                                    <dd>gnomAD_E_NFE: 0.1105%</dd>
                                    <dd>gnomAD_E_SAS: 0.0886%</dd>
                                    <dd>gnomAD_G_AFR: 0.1320%</dd>
                                    <dd>gnomAD_G_EAS: 0.1821%</dd>
                                    <dd>gnomAD_G_NFE: 0.1645%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>1-12980127-C-T [1|0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1832597">rs1832597</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.574</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361079.2">PRAMEF7:ENST00000361079.2:c.1319C&gt;T:p.(Pro440Leu)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>gnomAD_E_AFR: 0.2793%</dd>
                            <dd>gnomAD_E_AMR: 0.0890%</dd>
                            <dd>gnomAD_E_EAS: 0.0186%</dd>
                            <dd>gnomAD_E_NFE: 0.0631%</dd>
                            <dd>gnomAD_E_SAS: 0.0295%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF34" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196378">ZNF34</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.066</b>
                    <data>(p=1.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.593</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF34%0DRNASEH2B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RNASEH2B</a> and phenotypic similarity 0.770 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=51">Aicardi-Goutières syndrome</a> associated with RNASEH2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001640, Cardiomegaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF34%0DRNASEH2B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RNASEH2B</a> and phenotypic similarity 0.479 to mouse mutant of RNASEH2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001914, hemorrhage</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.066</b>
                                    <data>(p=1.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.593</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-145999498-T-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199914624">rs199914624</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.593</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343459.4">ZNF34:ENST00000343459.4:c.836A&gt;T:p.(Tyr279Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000429371.2">ZNF34:ENST00000429371.2:c.773A&gt;T:p.(Tyr258Phe)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.62151486</dd>
                                    
                                    <dd>REVEL: 0.066</dd>
                                    <dd>MVP: 0.622</dd>
                                    <dd>ALPHA_MISSENSE: 0.274</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.2921%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TBX5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000089225">TBX5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.064</b>
                    <data>(p=1.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.751</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.306</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.751 to <a href="http://www.omim.org/entry/142900">Holt-Oram syndrome</a> associated with TBX5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0011664, Left ventricular noncompaction cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001674, Complete atrioventricular canal defect</dd></dl><dl><dt>Phenotypic similarity 0.719 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TBX5">TBX5</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010418, perimembraneous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Phenotypic similarity 0.502 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=TBX5">TBX5</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0010672, myocardium disorganized, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0005415, cardiac ventricle morphology, abnormal</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBX5%0DGJA5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GJA5</a> and phenotypic similarity 0.823 to <a href="http://www.omim.org/entry/108770">Atrial standstill, digenic (GJA5/SCN5A)</a> associated with GJA5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0200127, Atrial cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001706, Endocardial fibroelastosis</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBX5%0DGJA5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GJA5</a> and phenotypic similarity 0.719 to mouse mutant of GJA5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBX5%0DGJA5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GJA5</a> and phenotypic similarity 0.230 to fish mutant of GJA5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0016958, intersegmental vein increased branchiness, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0016958, intersegmental vein increased branchiness, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/142900">OMIM:142900</a> Holt-Oram syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=101016">ORPHA:101016</a> Romano-Ward syndrome - autosomal dominant/recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=392">ORPHA:392</a> Holt-Oram syndrome - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.064</b>
                                    <data>(p=1.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.751</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.306</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.751 to OMIM:142900 Holt-Oram syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.690 to ORPHA:392 Holt-Oram syndrome</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-114793513-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201152128">rs201152128</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PP4, BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1578780">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:1, LB:1)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.306</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000310346.4">TBX5:ENST00000310346.4:c.1381G&gt;A:p.(Val461Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349716.5">TBX5:ENST00000349716.5:c.1231G&gt;A:p.(Val411Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405440.2">TBX5:ENST00000405440.2:c.1381G&gt;A:p.(Val461Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3072795</dd>
                                    
                                    <dd>REVEL: 0.092</dd>
                                    <dd>MVP: 0.307</dd>
                                    <dd>ALPHA_MISSENSE: 0.079</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0249%</dd>
                                    <dd>gnomAD_E_AMR: 0.0319%</dd>
                                    <dd>gnomAD_E_EAS: 0.0164%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0098%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PRKCE" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171132">PRKCE</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.064</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.530</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.555</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.455 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PRKCE">PRKCE</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003037, increased myocardial infarct size</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003037, increased myocardial infarct size</dd></dl><dl><dt>Phenotypic similarity 0.323 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=PRKCE">PRKCE</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl><dt>Proximity score 0.530 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PRKCE%0DKCNJ8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KCNJ8</a> and phenotypic similarity 0.901 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1517">Cantú syndrome</a> associated with KCNJ8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001654, Abnormal heart valve morphology</dd></dl><dl><dt>Proximity score 0.530 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PRKCE%0DKCNJ8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KCNJ8</a> and phenotypic similarity 0.483 to mouse mutant of KCNJ8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001636, irregular heartbeat</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001636, irregular heartbeat</dd></dl><dl><dt>Proximity score 0.530 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PRKCE%0DKCNJ8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KCNJ8</a> and phenotypic similarity 0.423 to fish mutant of KCNJ8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001207, cardiac ventricle increased size, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001207, cardiac ventricle increased size, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.064</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.530</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.555</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-46234762-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs77951226">rs77951226</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.555</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000306156.3">PRKCE:ENST00000306156.3:c.1225A&gt;G:p.(Asn409Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394874.1">PRKCE:ENST00000394874.1:c.394A&gt;G:p.(Asn132Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5714597</dd>
                                    
                                    <dd>REVEL: 0.090</dd>
                                    <dd>MVP: 0.571</dd>
                                    <dd>ALPHA_MISSENSE: 0.364</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0198%</dd>
                                    <dd>gnomAD_E_AMR: 0.1278%</dd>
                                    <dd>gnomAD_E_NFE: 0.1878%</dd>
                                    <dd>gnomAD_G_NFE: 0.1620%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NLRP14" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000158077">NLRP14</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.064</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.511</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.576</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.511 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NLRP14%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.782 to <a href="http://www.omim.org/entry/608068">Neutrophilic dermatosis, acute febrile</a> associated with MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.064</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.511</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.576</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-7064929-T-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs144045277">rs144045277</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3200595">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.576</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000299481.4">NLRP14:ENST00000299481.4:c.1672T&gt;A:p.(Ser558Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.579726</dd>
                                    
                                    <dd>REVEL: 0.114</dd>
                                    <dd>MVP: 0.580</dd>
                                    <dd>ALPHA_MISSENSE: 0.084</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0434%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PKHD1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170927">PKHD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.062</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.255</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.862</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.313 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PKHD1">PKHD1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0011427, increased mesangial cell number</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0011427, increased mesangial cell number</dd></dl><dl><dt>Proximity score 0.511 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PKHD1%0DHNF1B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HNF1B</a> and phenotypic similarity 0.342 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93111">HNF1B-related autosomal dominant tubulointerstitial kidney disease</a> associated with HNF1B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000821, Hypothyroidism</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.511 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PKHD1%0DHNF1B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HNF1B</a> and phenotypic similarity 0.665 to fish mutant of HNF1B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0002050, atrioventricular node closed, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0002050, atrioventricular node closed, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/263200">OMIM:263200</a> Polycystic kidney disease 4, with or without hepatic disease - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=53035">ORPHA:53035</a> Caroli disease - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=731">ORPHA:731</a> Autosomal recessive polycystic kidney disease - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.062</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.255</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.862</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-51889483-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs45517932">rs45517932</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/262404">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:2, LB:7)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.862</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340994.4">PKHD1:ENST00000340994.4:c.5125C&gt;T:p.(Leu1709Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371117.3">PKHD1:ENST00000371117.3:c.5125C&gt;T:p.(Leu1709Phe)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9517779</dd>
                                    
                                    <dd>REVEL: 0.614</dd>
                                    <dd>MVP: 0.952</dd>
                                    <dd>ALPHA_MISSENSE: 0.118</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1107%</dd>
                                    <dd>gnomAD_E_AMR: 0.1446%</dd>
                                    <dd>gnomAD_E_NFE: 0.5279%</dd>
                                    <dd>gnomAD_E_SAS: 0.0327%</dd>
                                    <dd>gnomAD_G_AFR: 0.0689%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.5185%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SPDL1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000040275">SPDL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.061</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.580</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPDL1%0DCENPE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CENPE</a> and phenotypic similarity 0.823 to <a href="http://www.omim.org/entry/616051">?Microcephaly 13, primary, autosomal recessive</a> associated with CENPE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001723, Restrictive cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001723, Restrictive cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.061</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.580</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-169021233-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142049273">rs142049273</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.580</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265295.4">SPDL1:ENST00000265295.4:c.616G&gt;A:p.(Asp206Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5974831</dd>
                                    
                                    <dd>REVEL: 0.139</dd>
                                    <dd>MVP: 0.597</dd>
                                    <dd>SPLICE_AI: 0.210</dd>
                                    <dd>ALPHA_MISSENSE: 0.078</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1915%</dd>
                                    <dd>gnomAD_E_AMR: 0.0723%</dd>
                                    <dd>gnomAD_E_NFE: 0.0299%</dd>
                                    <dd>gnomAD_G_AFR: 0.1606%</dd>
                                    <dd>gnomAD_G_NFE: 0.0389%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="JAML" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000160593">JAML</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.061</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.582</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=JAML%0DFAT4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FAT4</a> and phenotypic similarity 0.592 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=314679">Cerebrofacioarticular syndrome</a> associated with FAT4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001627, Abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001627, Abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=JAML%0DFAT4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FAT4</a> and phenotypic similarity 0.653 to mouse mutant of FAT4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000282, abnormal interatrial septum morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000282, abnormal interatrial septum morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.061</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.582</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-118065068-C-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201241187">rs201241187</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.582</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000292067.7">JAML:ENST00000292067.7:c.1127G&gt;C:p.(Gly376Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356289.5">JAML:ENST00000356289.5:c.1157G&gt;C:p.(Gly386Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000526620.1">JAML:ENST00000526620.1:c.1040G&gt;C:p.(Gly347Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000533261.1">JAML:ENST00000533261.1:c.1124G&gt;C:p.(Gly375Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.58292437</dd>
                                    
                                    <dd>REVEL: 0.256</dd>
                                    <dd>MVP: 0.583</dd>
                                    <dd>ALPHA_MISSENSE: 0.100</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ERCC4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000175595">ERCC4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.060</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.258</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.856</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.844 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with ERCC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.515 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ERCC4%0DSLX4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLX4</a> and phenotypic similarity 0.844 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with SLX4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.515 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ERCC4%0DSLX4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLX4</a> and phenotypic similarity 0.303 to mouse mutant of SLX4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002792, abnormal retina vasculature morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002792, abnormal retina vasculature morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/278760">OMIM:278760</a> Xeroderma pigmentosum, type F/Cockayne syndrome - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/610965">OMIM:610965</a> XFE progeroid syndrome - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/615272">OMIM:615272</a> Fanconi anemia, complementation group Q - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=220295">ORPHA:220295</a> Xeroderma pigmentosum-Cockayne syndrome complex - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">ORPHA:84</a> Fanconi anemia - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=90321">ORPHA:90321</a> Cockayne syndrome type 1 - unknown</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=910">ORPHA:910</a> Xeroderma pigmentosum - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.060</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.258</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.856</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-14031628-A-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs189463122">rs189463122</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1957359">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.856</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311895.7">ERCC4:ENST00000311895.7:c.1817A&gt;T:p.(Tyr606Phe)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.85677147</dd>
                                    
                                    <dd>REVEL: 0.312</dd>
                                    <dd>MVP: 0.857</dd>
                                    <dd>SPLICE_AI: 0.270</dd>
                                    <dd>ALPHA_MISSENSE: 0.261</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CAND2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000144712">CAND2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.058</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.577</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CAND2%0DLMOD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LMOD2</a> and phenotypic similarity 0.823 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=154">Familial isolated dilated cardiomyopathy</a> associated with LMOD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CAND2%0DLMOD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LMOD2</a> and phenotypic similarity 0.558 to mouse mutant of LMOD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000278, abnormal myocardial fiber morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000280, thin ventricular wall</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.058</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.577</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-12845013-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs62620196">rs62620196</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.577</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295989.5">CAND2:ENST00000295989.5:c.95C&gt;T:p.(Ser32Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456430.2">CAND2:ENST00000456430.2:c.95C&gt;T:p.(Ser32Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.70563716</dd>
                                    
                                    <dd>REVEL: 0.026</dd>
                                    <dd>MVP: 0.706</dd>
                                    <dd>ALPHA_MISSENSE: 0.099</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1545%</dd>
                                    <dd>gnomAD_E_AMR: 0.3674%</dd>
                                    <dd>gnomAD_E_NFE: 0.8553%</dd>
                                    <dd>gnomAD_E_SAS: 0.3223%</dd>
                                    <dd>gnomAD_G_AFR: 0.2296%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.8435%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PAFAH2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000158006">PAFAH2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.055</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.570</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PAFAH2%0DPLD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLD1</a> and phenotypic similarity 0.623 to <a href="http://www.omim.org/entry/212093">Cardiac valvular dysplasia 1</a> associated with PLD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001642, Pulmonic stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PAFAH2%0DPLD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLD1</a> and phenotypic similarity 0.455 to mouse mutant of PLD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002833, increased heart weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002833, increased heart weight</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PAFAH2%0DPLD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLD1</a> and phenotypic similarity 0.277 to fish mutant of PLD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001301, intersegmental vessel morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001301, intersegmental vessel morphology, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.055</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.570</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-26310555-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201037372">rs201037372</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.570</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374282.3">PAFAH2:ENST00000374282.3:c.434A&gt;G:p.(Tyr145Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374284.1">PAFAH2:ENST00000374284.1:c.434A&gt;G:p.(Tyr145Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6551373</dd>
                                    
                                    <dd>REVEL: 0.227</dd>
                                    <dd>MVP: 0.655</dd>
                                    <dd>ALPHA_MISSENSE: 0.070</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.6707%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_AMR: 0.3546%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SGIP1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000118473">SGIP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.054</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.296</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.296 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SGIP1">SGIP1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010097, abnormal retina blood vessel morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010097, abnormal retina blood vessel morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.054</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.296</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.800</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>1-67155865-C-T [0|1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000237247.6">SGIP1:ENST00000237247.6:c.1537-8C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371035.3">SGIP1:ENST00000371035.3:c.814-8C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371036.3">SGIP1:ENST00000371036.3:c.844-8C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371037.4">SGIP1:ENST00000371037.4:c.1444-8C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371039.1">SGIP1:ENST00000371039.1:c.847-8C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LLGL2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000073350">LLGL2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.054</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.567</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.372 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LLGL2">LLGL2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0008803, abnormal placental labyrinth vasculature morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0008803, abnormal placental labyrinth vasculature morphology</dd></dl><dl><dt>Phenotypic similarity 0.422 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=LLGL2">LLGL2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0002904, heart symmetry, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0002904, heart symmetry, abnormal</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LLGL2%0DPRKCZ&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKCZ</a> and phenotypic similarity 0.740 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1606">1p36 deletion syndrome</a> associated with PRKCZ.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LLGL2%0DPRKCZ&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKCZ</a> and phenotypic similarity 0.272 to fish mutant of PRKCZ.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0008757, intersegmental vessel has extra parts of type filopodium endothelial tip cell, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0008757, intersegmental vessel has extra parts of type filopodium endothelial tip cell, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.054</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.567</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-73567801-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148020757">rs148020757</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/769435">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.567</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000167462.5">LLGL2:ENST00000167462.5:c.2230G&gt;A:p.(Ala744Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392550.3">LLGL2:ENST00000392550.3:c.2230G&gt;A:p.(Ala744Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000577200.1">LLGL2:ENST00000577200.1:c.2230G&gt;A:p.(Ala744Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7286756</dd>
                                    
                                    <dd>REVEL: 0.266</dd>
                                    <dd>MVP: 0.729</dd>
                                    <dd>ALPHA_MISSENSE: 0.297</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1306%</dd>
                                    <dd>gnomAD_E_AMR: 0.4601%</dd>
                                    <dd>gnomAD_E_NFE: 0.9723%</dd>
                                    <dd>gnomAD_E_SAS: 0.0719%</dd>
                                    <dd>gnomAD_G_AFR: 0.1263%</dd>
                                    <dd>gnomAD_G_AMR: 0.9456%</dd>
                                    <dd>gnomAD_G_NFE: 0.9336%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NPC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000141458">NPC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.050</b>
                    <data>(p=2.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.842</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.400 to <a href="http://www.omim.org/entry/257220">Niemann-Pick disease, type C1</a> associated with NPC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0003651, Foam cells</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0003651, Foam cells</dd></dl><dl><dt>Phenotypic similarity 0.382 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NPC1">NPC1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004007, abnormal lung vasculature morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0004007, abnormal lung vasculature morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NPC1%0DLAMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LAMP2</a> and phenotypic similarity 0.964 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=34587">Glycogen storage disease due to LAMP-2 deficiency</a> associated with LAMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NPC1%0DLAMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LAMP2</a> and phenotypic similarity 0.534 to mouse mutant of LAMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001544, abnormal cardiovascular system physiology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0021157, increased heart left ventricle wall thickness</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NPC1%0DLAMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LAMP2</a> and phenotypic similarity 0.522 to fish mutant of LAMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0138138, trabecular layer has extra parts of type cardiac muscle cell, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0103127, atrium increased area, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/257220">OMIM:257220</a> Niemann-Pick disease, type C1 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.050</b>
                                    <data>(p=2.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.842</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>18-21140381-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201956601">rs201956601</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1148181">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:1, LB:1)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.842</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000269228.5">NPC1:ENST00000269228.5:c.695A&gt;G:p.(Asp232Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.84587735</dd>
                                    
                                    <dd>REVEL: 0.324</dd>
                                    <dd>MVP: 0.846</dd>
                                    <dd>ALPHA_MISSENSE: 0.134</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0318%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AHNAK" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000124942">AHNAK</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.046</b>
                    <data>(p=2.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.544</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AHNAK%0DPYGM&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PYGM</a> and phenotypic similarity 0.962 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=368">Glycogen storage disease due to muscle glycogen phosphorylase deficiency</a> associated with PYGM.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AHNAK%0DPYGM&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PYGM</a> and phenotypic similarity 0.455 to mouse mutant of PYGM.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010506, prolonged RR interval</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010506, prolonged RR interval</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AHNAK%0DPYGM&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PYGM</a> and phenotypic similarity 0.323 to fish mutant of PYGM.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.046</b>
                                    <data>(p=2.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.544</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-62287497-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148186265">rs148186265</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2353102">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.544</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378024.4">AHNAK:ENST00000378024.4:c.14392C&gt;T:p.(Pro4798Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000257247.7">AHNAK:ENST00000257247.7:c.342+14978C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530124.1">AHNAK:ENST00000530124.1:c.342+14978C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.54678005</dd>
                                    
                                    <dd>REVEL: 0.217</dd>
                                    <dd>MVP: 0.547</dd>
                                    <dd>ALPHA_MISSENSE: 0.349</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                    <dd>gnomAD_E_NFE: 0.0396%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_NFE: 0.0390%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CUL9" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000112659">CUL9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.045</b>
                    <data>(p=2.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.549</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUL9%0DMID1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MID1</a> and phenotypic similarity 0.651 to <a href="http://www.omim.org/entry/300000">Opitz GBBB syndrome</a> associated with MID1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUL9%0DMID1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MID1</a> and phenotypic similarity 0.455 to mouse mutant of MID1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002833, increased heart weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002833, increased heart weight</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.045</b>
                                    <data>(p=2.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.549</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-43152062-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200369406">rs200369406</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.549</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000252050.4">CUL9:ENST00000252050.4:c.14G&gt;A:p.(Arg5Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354495.3">CUL9:ENST00000354495.3:c.14G&gt;A:p.(Arg5Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372647.2">CUL9:ENST00000372647.2:c.14G&gt;A:p.(Arg5Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.55053127</dd>
                                    
                                    <dd>REVEL: 0.058</dd>
                                    <dd>MVP: 0.551</dd>
                                    <dd>ALPHA_MISSENSE: 0.074</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0174%</dd>
                                    <dd>gnomAD_E_NFE: 0.0012%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TF" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000091513">TF</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.042</b>
                    <data>(p=2.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.271</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.801</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.338 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1195">Congenital atransferrinemia</a> associated with TF.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000821, Hypothyroidism</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.541 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TF%0DTFR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TFR2</a> and phenotypic similarity 0.788 to <a href="http://www.omim.org/entry/604250">Hemochromatosis, type 3</a> associated with TFR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl><dt>Proximity score 0.541 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TF%0DTFR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TFR2</a> and phenotypic similarity 0.512 to mouse mutant of TFR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/209300">OMIM:209300</a> Atransferrinemia - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1195">ORPHA:1195</a> Congenital atransferrinemia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.042</b>
                                    <data>(p=2.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.271</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.801</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-133496032-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs121918677">rs121918677</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/12616">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.801</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264998.3">TF:ENST00000264998.3:c.1631G&gt;A:p.(Gly544Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000402696.3">TF:ENST00000402696.3:c.2012G&gt;A:p.(Gly671Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8640936</dd>
                                    
                                    <dd>REVEL: 0.386</dd>
                                    <dd>MVP: 0.864</dd>
                                    <dd>ALPHA_MISSENSE: 0.774</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1292%</dd>
                                    <dd>gnomAD_E_AMR: 0.1822%</dd>
                                    <dd>gnomAD_E_NFE: 0.4292%</dd>
                                    <dd>gnomAD_E_SAS: 0.0425%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.4150%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="VPREB3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000128218">VPREB3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.042</b>
                    <data>(p=2.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.540</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=VPREB3%0DCPE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CPE</a> and phenotypic similarity 0.607 to <a href="http://www.omim.org/entry/619326">BDV syndrome</a> associated with CPE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=VPREB3%0DCPE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CPE</a> and phenotypic similarity 0.356 to mouse mutant of CPE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010463, aorta stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010463, aorta stenosis</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.042</b>
                                    <data>(p=2.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.540</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>22-24095221-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs140601102">rs140601102</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.540</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000248948.3">VPREB3:ENST00000248948.3:c.214C&gt;T:p.(Arg72Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000398465.3">VPREB3:ENST00000398465.3:c.166C&gt;T:p.(Arg56Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.58862275</dd>
                                    
                                    <dd>REVEL: 0.149</dd>
                                    <dd>MVP: 0.589</dd>
                                    <dd>ALPHA_MISSENSE: 0.269</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1047%</dd>
                                    <dd>gnomAD_E_AMR: 0.2111%</dd>
                                    <dd>gnomAD_E_NFE: 0.4804%</dd>
                                    <dd>gnomAD_E_SAS: 0.0523%</dd>
                                    <dd>gnomAD_G_AFR: 0.1034%</dd>
                                    <dd>gnomAD_G_NFE: 0.4538%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PAK5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000101349">PAK5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.038</b>
                    <data>(p=2.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.521</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PAK5%0DMYLK2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MYLK2</a> and phenotypic similarity 0.924 to <a href="http://www.omim.org/entry/192600">Cardiomyopathy, hypertrophic, 1, digenic</a> associated with MYLK2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001670, Asymmetric septal hypertrophy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001682, Subvalvular aortic stenosis</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.038</b>
                                    <data>(p=2.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.521</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>20-9520135-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202003893">rs202003893</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.521</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000353224.5">PAK5:ENST00000353224.5:c.2134C&gt;G:p.(Leu712Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378423.1">PAK5:ENST00000378423.1:c.2134C&gt;G:p.(Leu712Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378429.3">PAK5:ENST00000378429.3:c.2134C&gt;G:p.(Leu712Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5574</dd>
                                    
                                    <dd>REVEL: 0.447</dd>
                                    <dd>MVP: 0.459</dd>
                                    <dd>ALPHA_MISSENSE: 0.557</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.3903%</dd>
                                    <dd>gnomAD_E_SAS: 0.0098%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PPP3CA" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000138814">PPP3CA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.038</b>
                    <data>(p=2.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.526</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.499</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.298 to <a href="http://www.omim.org/entry/617711">Developmental and epileptic encephalopathy 91</a> associated with PPP3CA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001257, Spasticity</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.526 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PPP3CA%0DMYOZ2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MYOZ2</a> and phenotypic similarity 0.949 to <a href="http://www.omim.org/entry/613838">Cardiomyopathy, hypertrophic, 16</a> associated with MYOZ2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001712, Left ventricular hypertrophy</dd></dl><dl><dt>Proximity score 0.526 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PPP3CA%0DMYOZ2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MYOZ2</a> and phenotypic similarity 0.525 to mouse mutant of MYOZ2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000278, abnormal myocardial fiber morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003140, dilated heart atrium</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/617711">OMIM:617711</a> Developmental and epileptic encephalopathy 91 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/618265">OMIM:618265</a> Arthrogryposis, cleft palate, craniosynostosis, and impaired intellectual development - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=442835">ORPHA:442835</a> Non-specific early-onset epileptic encephalopathy - autosomal dominant/recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.038</b>
                                    <data>(p=2.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.526</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.499</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.298 to OMIM:617711 Developmental and epileptic encephalopathy 91</dt>
                                
                                    <dt>Phenotypic similarity 0.298 to ORPHA:442835 Non-specific early-onset epileptic encephalopathy</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-102019648-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142836504">rs142836504</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1543291">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.499</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000323055.6">PPP3CA:ENST00000323055.6:c.518G&gt;A:p.(Arg173His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394853.4">PPP3CA:ENST00000394853.4:c.518G&gt;A:p.(Arg173His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394854.3">PPP3CA:ENST00000394854.3:c.518G&gt;A:p.(Arg173His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000507176.1">PPP3CA:ENST00000507176.1:c.224G&gt;A:p.(Arg75His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000523694.2">PPP3CA:ENST00000523694.2:c.317G&gt;A:p.(Arg106His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000512215.1">PPP3CA:ENST00000512215.1:c.260-35134G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.51231754</dd>
                                    
                                    <dd>REVEL: 0.126</dd>
                                    <dd>MVP: 0.512</dd>
                                    <dd>ALPHA_MISSENSE: 0.106</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1730%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BRCA2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000139618">BRCA2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.038</b>
                    <data>(p=2.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.519</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.507</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.844 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with BRCA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Phenotypic similarity 0.422 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=BRCA2">BRCA2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001192, heart edematous, abnormal</dd></dl><dl><dt>Proximity score 0.519 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BRCA2%0DXRCC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to XRCC2</a> and phenotypic similarity 0.844 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with XRCC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/114480">OMIM:114480</a> Breast cancer, male, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/155255">OMIM:155255</a> Medulloblastoma (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/176807">OMIM:176807</a> Prostate cancer (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/194070">OMIM:194070</a> Wilms tumor - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/605724">OMIM:605724</a> Fanconi anemia, complementation group D1 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/612555">OMIM:612555</a> Breast-ovarian cancer, familial, 2 (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/613029">OMIM:613029</a> Glioblastoma 3 (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/613347">OMIM:613347</a> Pancreatic cancer 2 (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1333">ORPHA:1333</a> Familial pancreatic carcinoma (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=145">ORPHA:145</a> Hereditary breast and/or ovarian cancer syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=654">ORPHA:654</a> Nephroblastoma - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=70567">ORPHA:70567</a> Cholangiocarcinoma (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">ORPHA:84</a> Fanconi anemia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.038</b>
                                    <data>(p=2.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.519</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.507</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>13-32937521-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs28897749">rs28897749</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/41565">(REVIEWED_BY_EXPERT_PANEL)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.507</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380152.3">BRCA2:ENST00000380152.3:c.8182G&gt;A:p.(Val2728Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544455.1">BRCA2:ENST00000544455.1:c.8182G&gt;A:p.(Val2728Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5429407</dd>
                                    
                                    <dd>REVEL: 0.462</dd>
                                    <dd>MVP: 0.543</dd>
                                    <dd>ALPHA_MISSENSE: 0.066</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0677%</dd>
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                    <dd>gnomAD_E_NFE: 0.4020%</dd>
                                    <dd>gnomAD_G_AFR: 0.0459%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.3499%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PPM1J" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000155367">PPM1J</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.037</b>
                    <data>(p=2.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.526</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PPM1J%0DCTU2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CTU2</a> and phenotypic similarity 0.646 to <a href="http://www.omim.org/entry/618142">Microcephaly, facial dysmorphism, renal agenesis, and ambiguous genitalia syndrome</a> associated with CTU2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0010954, Hypoplastic right heart</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.037</b>
                                    <data>(p=2.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.526</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-113252868-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148123745">rs148123745</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.526</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000309276.6">PPM1J:ENST00000309276.6:c.1435C&gt;T:p.(Arg479Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359994.4">PPM1J:ENST00000359994.4:c.817C&gt;T:p.(Arg273Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000464951.1">PPM1J:ENST00000464951.1:c.817C&gt;T:p.(Arg273Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000606505.1">PPM1J:ENST00000606505.1:c.414+214C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.567967</dd>
                                    
                                    <dd>REVEL: 0.337</dd>
                                    <dd>MVP: 0.568</dd>
                                    <dd>ALPHA_MISSENSE: 0.234</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.4345%</dd>
                                    <dd>gnomAD_E_NFE: 0.0028%</dd>
                                    <dd>gnomAD_G_AMR: 0.3546%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ADAM15" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143537">ADAM15</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.036</b>
                    <data>(p=2.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.352</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.691</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.352 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ADAM15">ADAM15</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003434, decreased susceptibility to induced choroidal neovascularization</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003434, decreased susceptibility to induced choroidal neovascularization</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.036</b>
                                    <data>(p=2.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.352</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.691</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-155030376-T-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201599586">rs201599586</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.691</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000271836.6">ADAM15:ENST00000271836.6:c.1466T&gt;G:p.(Leu489Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355956.2">ADAM15:ENST00000355956.2:c.1466T&gt;G:p.(Leu489Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356955.2">ADAM15:ENST00000356955.2:c.1466T&gt;G:p.(Leu489Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359280.4">ADAM15:ENST00000359280.4:c.1466T&gt;G:p.(Leu489Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360674.4">ADAM15:ENST00000360674.4:c.1466T&gt;G:p.(Leu489Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368410.2">ADAM15:ENST00000368410.2:c.584T&gt;G:p.(Leu195Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368412.3">ADAM15:ENST00000368412.3:c.1466T&gt;G:p.(Leu489Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368413.1">ADAM15:ENST00000368413.1:c.584T&gt;G:p.(Leu195Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000447332.3">ADAM15:ENST00000447332.3:c.1418T&gt;G:p.(Leu473Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449910.2">ADAM15:ENST00000449910.2:c.1466T&gt;G:p.(Leu489Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000531455.1">ADAM15:ENST00000531455.1:c.1496T&gt;G:p.(Leu499Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6969535</dd>
                                    
                                    <dd>REVEL: 0.365</dd>
                                    <dd>MVP: 0.697</dd>
                                    <dd>ALPHA_MISSENSE: 0.508</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0636%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SGCG" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000102683">SGCG</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.035</b>
                    <data>(p=2.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.278</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.772</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.594 to <a href="http://www.omim.org/entry/253700">Muscular dystrophy, limb-girdle, autosomal recessive 5</a> associated with SGCG.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0005133, Right ventricular dilatation</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0005133, Right ventricular dilatation</dd></dl><dl><dt>Phenotypic similarity 0.556 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SGCG">SGCG</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005294, abnormal heart ventricle morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002953, thick ventricular wall</dd></dl><dl><dt>Proximity score 0.517 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SGCG%0DFKRP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FKRP</a> and phenotypic similarity 0.823 to <a href="http://www.omim.org/entry/607155">Muscular dystrophy-dystroglycanopathy (limb-girdle), type C, 5</a> associated with FKRP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dl><dt>Proximity score 0.517 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SGCG%0DFKRP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FKRP</a> and phenotypic similarity 0.303 to mouse mutant of FKRP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002792, abnormal retina vasculature morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002792, abnormal retina vasculature morphology</dd></dl><dl><dt>Proximity score 0.517 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SGCG%0DFKRP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FKRP</a> and phenotypic similarity 0.304 to fish mutant of FKRP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/253700">OMIM:253700</a> Muscular dystrophy, limb-girdle, autosomal recessive 5 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=353">ORPHA:353</a> Gamma-sarcoglycan-related limb-girdle muscular dystrophy R5 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.035</b>
                                    <data>(p=2.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.278</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.772</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>13-23824775-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201683351">rs201683351</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.772</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000218867.3">SGCG:ENST00000218867.3:c.304T&gt;C:p.(Ser102Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537476.1">SGCG:ENST00000537476.1:c.304T&gt;C:p.(Ser102Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545013.1">SGCG:ENST00000545013.1:c.304T&gt;C:p.(Ser102Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.77404445</dd>
                                    
                                    <dd>REVEL: 0.319</dd>
                                    <dd>MVP: 0.774</dd>
                                    <dd>ALPHA_MISSENSE: 0.078</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                    <dd>gnomAD_E_EAS: 0.0163%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DEPTOR" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000155792">DEPTOR</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.033</b>
                    <data>(p=2.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.515</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.496</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.515 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DEPTOR%0DRRAGD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RRAGD</a> and phenotypic similarity 0.823 to <a href="http://www.omim.org/entry/620152">Hypomagnesemia 7, renal, with or without dilated cardiomyopathy</a> associated with RRAGD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.033</b>
                                    <data>(p=2.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.515</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.496</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-120886112-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs145341556">rs145341556</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.496</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000286234.5">DEPTOR:ENST00000286234.5:c.26G&gt;A:p.(Ser9Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000523492.1">DEPTOR:ENST00000523492.1:c.26G&gt;A:p.(Ser9Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.56832147</dd>
                                    
                                    <dd>REVEL: 0.049</dd>
                                    <dd>MVP: 0.568</dd>
                                    <dd>ALPHA_MISSENSE: 0.129</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.6608%</dd>
                                    <dd>gnomAD_E_AMR: 0.0348%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_G_AFR: 0.5282%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DYTN" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000232125">DYTN</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.033</b>
                    <data>(p=2.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.570</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.433</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.460 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=DYTN">DYTN</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002834, decreased heart weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002834, decreased heart weight</dd></dl><dl><dt>Proximity score 0.570 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DYTN%0DFASTKD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FASTKD2</a> and phenotypic similarity 0.962 to <a href="http://www.omim.org/entry/618855">Combined oxidative phosphorylation deficiency 44</a> associated with FASTKD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.033</b>
                                    <data>(p=2.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.570</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.433</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-207564886-G-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200369556">rs200369556</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.433</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000452335.2">DYTN:ENST00000452335.2:c.538C&gt;A:p.(Arg180Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.434</dd>
                                    
                                    <dd>REVEL: 0.434</dd>
                                    <dd>MVP: 0.333</dd>
                                    <dd>ALPHA_MISSENSE: 0.254</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SNAPC4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165684">SNAPC4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.032</b>
                    <data>(p=2.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.792</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.312 to <a href="http://www.omim.org/entry/620515">Neurodevelopmental disorder with motor regression, progressive spastic paraplegia, and oromotor dysfunction</a> associated with SNAPC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001257, Spasticity</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Phenotypic similarity 0.473 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=SNAPC4">SNAPC4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0004122, liver and biliary system decreased functionality, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SNAPC4%0DPITX2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PITX2</a> and phenotypic similarity 0.508 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=782">Axenfeld-Rieger syndrome</a> associated with PITX2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0030680, Abnormal cardiovascular system morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0030680, Abnormal cardiovascular system morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SNAPC4%0DPITX2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PITX2</a> and phenotypic similarity 0.716 to mouse mutant of PITX2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000284, double outlet right ventricle</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SNAPC4%0DPITX2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PITX2</a> and phenotypic similarity 0.304 to fish mutant of PITX2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/620515">OMIM:620515</a> Neurodevelopmental disorder with motor regression, progressive spastic paraplegia, and oromotor dysfunction - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.032</b>
                                    <data>(p=2.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.792</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>9-139289747-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201492253">rs201492253</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.792</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000298532.2">SNAPC4:ENST00000298532.2:c.471+3G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0464%</dd>
                                    <dd>gnomAD_E_NFE: 0.0516%</dd>
                                    <dd>gnomAD_E_SAS: 0.0590%</dd>
                                    <dd>gnomAD_G_NFE: 0.0713%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TSPAN32" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000064201">TSPAN32</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.032</b>
                    <data>(p=2.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.512</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.496</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.512 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TSPAN32">TSPAN32</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TSPAN32%0DCDKN1C&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDKN1C</a> and phenotypic similarity 0.817 to <a href="http://www.omim.org/entry/130650">Beckwith-Wiedemann syndrome</a> associated with CDKN1C.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001640, Cardiomegaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TSPAN32%0DCDKN1C&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDKN1C</a> and phenotypic similarity 0.479 to mouse mutant of CDKN1C.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002842, increased systemic arterial blood pressure</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002842, increased systemic arterial blood pressure</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.032</b>
                                    <data>(p=2.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.512</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.496</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-2334957-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs2234302">rs2234302</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.496</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000182290.4">TSPAN32:ENST00000182290.4:c.428G&gt;A:p.(Arg143Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381121.3">TSPAN32:ENST00000381121.3:c.428G&gt;A:p.(Arg143Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000451520.2">TSPAN32:ENST00000451520.2:c.395G&gt;A:p.(Arg132Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5152599</dd>
                                    
                                    <dd>REVEL: 0.077</dd>
                                    <dd>MVP: 0.515</dd>
                                    <dd>ALPHA_MISSENSE: 0.087</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0209%</dd>
                                    <dd>gnomAD_E_AMR: 0.0513%</dd>
                                    <dd>gnomAD_E_EAS: 0.0114%</dd>
                                    <dd>gnomAD_E_NFE: 0.0462%</dd>
                                    <dd>gnomAD_E_SAS: 0.2454%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TUBGCP3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000126216">TUBGCP3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.031</b>
                    <data>(p=2.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.501</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TUBGCP3%0DSDCCAG8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SDCCAG8</a> and phenotypic similarity 0.744 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=110">Bardet-Biedl syndrome</a> associated with SDCCAG8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001627, Abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TUBGCP3%0DSDCCAG8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SDCCAG8</a> and phenotypic similarity 0.360 to mouse mutant of SDCCAG8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000820, abnormal choroid plexus morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000820, abnormal choroid plexus morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.031</b>
                                    <data>(p=2.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.501</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>13-113176817-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200753524">rs200753524</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.501</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261965.3">TUBGCP3:ENST00000261965.3:c.1562A&gt;G:p.(Asp521Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375669.3">TUBGCP3:ENST00000375669.3:c.1562A&gt;G:p.(Asp521Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.50282043</dd>
                                    
                                    <dd>REVEL: 0.191</dd>
                                    <dd>MVP: 0.503</dd>
                                    <dd>ALPHA_MISSENSE: 0.087</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                    <dd>gnomAD_E_NFE: 0.0194%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_NFE: 0.0194%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="EPAS1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000116016">EPAS1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.030</b>
                    <data>(p=2.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.550</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.444</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.550 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=EPAS1">EPAS1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010755, abnormal heart right ventricle pressure</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002953, thick ventricular wall</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EPAS1%0DLIN28A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LIN28A</a> and phenotypic similarity 0.769 to mouse mutant of LIN28A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010402, ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/611783">OMIM:611783</a> Erythrocytosis, familial, 4 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.030</b>
                                    <data>(p=2.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.550</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.444</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-46603747-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs61757375">rs61757375</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/336260">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:1, LB:2, B:2)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.444</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263734.3">EPAS1:ENST00000263734.3:c.1104G&gt;A:p.(Met368Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.45595098</dd>
                                    
                                    <dd>REVEL: 0.099</dd>
                                    <dd>MVP: 0.456</dd>
                                    <dd>ALPHA_MISSENSE: 0.224</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0554%</dd>
                                    <dd>gnomAD_E_AMR: 0.0520%</dd>
                                    <dd>gnomAD_E_NFE: 0.1283%</dd>
                                    <dd>gnomAD_E_SAS: 0.0523%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_NFE: 0.1751%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BUD31" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000106245">BUD31</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.028</b>
                    <data>(p=2.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.518</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.474</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.518 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BUD31%0DCWC27&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CWC27</a> and phenotypic similarity 0.651 to <a href="http://www.omim.org/entry/250410">Retinitis pigmentosa with or without skeletal anomalies</a> associated with CWC27.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.028</b>
                                    <data>(p=2.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.518</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.474</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-99013772-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150153594">rs150153594</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.474</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000222969.5">BUD31:ENST00000222969.5:c.106C&gt;T:p.(Pro36Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000403633.2">BUD31:ENST00000403633.2:c.106C&gt;T:p.(Pro36Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000431419.1">BUD31:ENST00000431419.1:c.19C&gt;T:p.(Pro7Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456893.1">BUD31:ENST00000456893.1:c.95-1280C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.47464362</dd>
                                    
                                    <dd>REVEL: 0.145</dd>
                                    <dd>MVP: 0.475</dd>
                                    <dd>ALPHA_MISSENSE: 0.150</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DSG1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000134760">DSG1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.027</b>
                    <data>(p=2.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.510</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.480</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.637 to <a href="http://www.omim.org/entry/615508">Erythroderma, congenital, with palmoplantar keratoderma, hypotrichosis, and hyper IgE</a> associated with DSG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001642, Pulmonic stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011625, Multiple muscular ventricular septal defects</dd></dl><dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DSG1%0DPKP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKP2</a> and phenotypic similarity 0.722 to <a href="http://www.omim.org/entry/609040">Arrhythmogenic right ventricular dysplasia 9</a> associated with PKP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0011663, Right ventricular cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0034364, Fibrofatty replacement of right ventricular myocardium</dd></dl><dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DSG1%0DPKP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKP2</a> and phenotypic similarity 0.538 to mouse mutant of PKP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005329, abnormal myocardium layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DSG1%0DPKP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKP2</a> and phenotypic similarity 0.501 to fish mutant of PKP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0005415, cardiac ventricle morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0005415, cardiac ventricle morphology, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/148700">OMIM:148700</a> Keratosis palmoplantaris striata I, AD - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/615508">OMIM:615508</a> Erythroderma, congenital, with palmoplantar keratoderma, hypotrichosis, and hyper IgE - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=50942">ORPHA:50942</a> Striate palmoplantar keratoderma - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.027</b>
                                    <data>(p=2.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.510</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.480</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>18-28934374-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs78995670">rs78995670</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/775466">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.480</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000257192.4">DSG1:ENST00000257192.4:c.2215A&gt;G:p.(Ile739Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000462981.2">DSG1:ENST00000462981.2:c.292A&gt;G:p.(Ile98Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5831155</dd>
                                    
                                    <dd>REVEL: 0.417</dd>
                                    <dd>MVP: 0.583</dd>
                                    <dd>ALPHA_MISSENSE: 0.323</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1476%</dd>
                                    <dd>gnomAD_E_AMR: 0.0867%</dd>
                                    <dd>gnomAD_E_EAS: 0.0109%</dd>
                                    <dd>gnomAD_E_NFE: 0.7112%</dd>
                                    <dd>gnomAD_E_SAS: 0.2123%</dd>
                                    <dd>gnomAD_G_AFR: 0.1608%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.8359%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC16A8" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100156">SLC16A8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.024</b>
                    <data>(p=2.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.478</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC16A8%0DMMP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MMP1</a> and phenotypic similarity 0.823 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79408">Autosomal recessive generalized dystrophic epidermolysis bullosa, severe form</a> associated with MMP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.024</b>
                                    <data>(p=2.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.478</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>22-38477156-A-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199982500">rs199982500</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.478</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000320521.5">SLC16A8:ENST00000320521.5:c.889T&gt;G:p.(Phe297Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.48649457</dd>
                                    
                                    <dd>REVEL: 0.330</dd>
                                    <dd>MVP: 0.486</dd>
                                    <dd>ALPHA_MISSENSE: 0.192</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0125%</dd>
                                    <dd>gnomAD_G_AMR: 0.1182%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DYNC1LI1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000144635">DYNC1LI1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.022</b>
                    <data>(p=2.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.508</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.458</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.508 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=DYNC1LI1">DYNC1LI1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002188, small heart</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002188, small heart</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DYNC1LI1%0DODAD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ODAD1</a> and phenotypic similarity 0.619 to <a href="http://www.omim.org/entry/615067">Ciliary dyskinesia, primary, 20</a> associated with ODAD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001651, Dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.022</b>
                                    <data>(p=2.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.508</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.458</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-32586374-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199851895">rs199851895</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.458</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000273130.4">DYNC1LI1:ENST00000273130.4:c.562C&gt;G:p.(Gln188Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000432458.2">DYNC1LI1:ENST00000432458.2:c.221-3676C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.4584867</dd>
                                    
                                    <dd>REVEL: 0.074</dd>
                                    <dd>MVP: 0.458</dd>
                                    <dd>ALPHA_MISSENSE: 0.078</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TBC1D4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136111">TBC1D4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.021</b>
                    <data>(p=2.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.462</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.455 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TBC1D4">TBC1D4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002833, increased heart weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002833, increased heart weight</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBC1D4%0DATIC&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ATIC</a> and phenotypic similarity 0.643 to <a href="http://www.omim.org/entry/608688">AICA-ribosiduria due to ATIC deficiency</a> associated with ATIC.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001684, Secundum atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001684, Secundum atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.021</b>
                                    <data>(p=2.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.462</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>13-75936328-C-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200840673">rs200840673</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.462</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377625.2">TBC1D4:ENST00000377625.2:c.914G&gt;C:p.(Gly305Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377636.3">TBC1D4:ENST00000377636.3:c.914G&gt;C:p.(Gly305Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000431480.2">TBC1D4:ENST00000431480.2:c.914G&gt;C:p.(Gly305Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000425511.1">TBC1D4:ENST00000425511.1:c.-1436G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5014334</dd>
                                    
                                    <dd>REVEL: 0.037</dd>
                                    <dd>MVP: 0.501</dd>
                                    <dd>ALPHA_MISSENSE: 0.085</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.4462%</dd>
                                    <dd>gnomAD_E_AMR: 0.0116%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.4598%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LPIN2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000101577">LPIN2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.021</b>
                    <data>(p=2.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.275</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.718</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.550 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LPIN2%0DZBTB14&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ZBTB14</a> and phenotypic similarity 0.708 to mouse mutant of ZBTB14.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010418, perimembraneous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010418, perimembraneous ventricular septal defect</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/609628">OMIM:609628</a> Majeed syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=77297">ORPHA:77297</a> Majeed syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.021</b>
                                    <data>(p=2.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.275</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.718</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>18-2937867-C-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs80338805">rs80338805</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/21520">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:1, LB:6, B:1)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.718</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261596.4">LPIN2:ENST00000261596.4:c.991G&gt;T:p.(Ala331Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8054305</dd>
                                    
                                    <dd>REVEL: 0.419</dd>
                                    <dd>MVP: 0.805</dd>
                                    <dd>ALPHA_MISSENSE: 0.074</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0431%</dd>
                                    <dd>gnomAD_E_AMR: 0.4221%</dd>
                                    <dd>gnomAD_E_NFE: 0.1913%</dd>
                                    <dd>gnomAD_E_SAS: 0.0131%</dd>
                                    <dd>gnomAD_G_AFR: 0.0345%</dd>
                                    <dd>gnomAD_G_AMR: 0.5910%</dd>
                                    <dd>gnomAD_G_NFE: 0.1882%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SIVA1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000184990">SIVA1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.019</b>
                    <data>(p=2.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.452</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SIVA1%0DABL1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ABL1</a> and phenotypic similarity 0.627 to <a href="http://www.omim.org/entry/617602">Congenital heart defects and skeletal malformations syndrome</a> associated with ABL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SIVA1%0DABL1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ABL1</a> and phenotypic similarity 0.663 to mouse mutant of ABL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004056, abnormal myocardium compact layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010724, thick interventricular septum</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.019</b>
                                    <data>(p=2.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.452</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-105223001-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200119861">rs200119861</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.452</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329967.6">SIVA1:ENST00000329967.6:c.374C&gt;T:p.(Ala125Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000347067.5">SIVA1:ENST00000347067.5:c.179C&gt;T:p.(Ala60Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.45283487</dd>
                                    
                                    <dd>REVEL: 0.070</dd>
                                    <dd>MVP: 0.453</dd>
                                    <dd>ALPHA_MISSENSE: 0.084</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0116%</dd>
                                    <dd>gnomAD_E_NFE: 0.0053%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PPAN" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130810">PPAN</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.019</b>
                    <data>(p=2.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.448</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PPAN%0DLARP7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LARP7</a> and phenotypic similarity 0.651 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=319671">Alazami syndrome</a> associated with LARP7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PPAN%0DLARP7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LARP7</a> and phenotypic similarity 0.474 to fish mutant of LARP7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000724, atrium dilated, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000724, atrium dilated, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.019</b>
                                    <data>(p=2.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.448</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-10224579-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201039858">rs201039858</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.448</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556468.1">PPAN:ENST00000556468.1:c.1550A&gt;G:p.(Tyr517Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.45608968</dd>
                                    
                                    <dd>REVEL: 0.178</dd>
                                    <dd>MVP: 0.456</dd>
                                    <dd>ALPHA_MISSENSE: 0.389</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0870%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PAPOLA" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000090060">PAPOLA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.017</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.437</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PAPOLA%0DHNRNPA2B1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HNRNPA2B1</a> and phenotypic similarity 0.787 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=52430">Inclusion body myopathy with Paget disease of bone and frontotemporal dementia</a> associated with HNRNPA2B1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.017</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.437</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-97022649-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200844555">rs200844555</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.437</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000216277.8">PAPOLA:ENST00000216277.8:c.1903T&gt;C:p.(Ser635Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392990.2">PAPOLA:ENST00000392990.2:c.1903T&gt;C:p.(Ser635Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.43734792</dd>
                                    
                                    <dd>REVEL: 0.102</dd>
                                    <dd>MVP: 0.437</dd>
                                    <dd>ALPHA_MISSENSE: 0.068</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MARCHF10" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000173838">MARCHF10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.017</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.438</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MARCHF10%0DAFG2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AFG2A</a> and phenotypic similarity 0.618 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=457351">Microcephaly-intellectual disability-sensorineural hearing loss-epilepsy-abnormal muscle tone syndrome</a> associated with AFG2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.017</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.438</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-60802337-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201329810">rs201329810</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.438</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311269.5">MARCHF10:ENST00000311269.5:c.2066A&gt;G:p.(Glu689Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456609.2">MARCHF10:ENST00000456609.2:c.2066A&gt;G:p.(Glu689Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544856.2">MARCHF10:ENST00000544856.2:c.2063A&gt;G:p.(Glu688Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583600.1">MARCHF10:ENST00000583600.1:c.2180A&gt;G:p.(Glu727Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.43827805</dd>
                                    
                                    <dd>REVEL: 0.254</dd>
                                    <dd>MVP: 0.438</dd>
                                    <dd>ALPHA_MISSENSE: 0.255</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0090%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PPARGC1A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000109819">PPARGC1A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.017</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.429</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.311 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=803">Amyotrophic lateral sclerosis</a> associated with PPARGC1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001257, Spasticity</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Phenotypic similarity 0.493 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PPARGC1A">PPARGC1A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002128, abnormal blood circulation</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002128, abnormal blood circulation</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PPARGC1A%0DCITED2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CITED2</a> and phenotypic similarity 0.643 to <a href="http://www.omim.org/entry/614431">Ventricular septal defect 2</a> associated with CITED2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0011682, Perimembranous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011682, Perimembranous ventricular septal defect</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PPARGC1A%0DCITED2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CITED2</a> and phenotypic similarity 0.747 to mouse mutant of CITED2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000284, double outlet right ventricle</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PPARGC1A%0DCITED2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CITED2</a> and phenotypic similarity 0.738 to fish mutant of CITED2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0005049, cardiovascular system morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0005049, cardiovascular system morphology, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=803">ORPHA:803</a> Amyotrophic lateral sclerosis (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.017</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.429</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.311 to ORPHA:803 Amyotrophic lateral sclerosis</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-23797521-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201337076">rs201337076</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.429</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264867.2">PPARGC1A:ENST00000264867.2:c.2321C&gt;G:p.(Ala774Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.43508825</dd>
                                    
                                    <dd>REVEL: 0.205</dd>
                                    <dd>MVP: 0.435</dd>
                                    <dd>ALPHA_MISSENSE: 0.295</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0997%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ACSF3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000176715">ACSF3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/614265">OMIM:614265</a> Combined malonic and methylmalonic aciduria - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=289504">ORPHA:289504</a> Combined malonic and methylmalonic acidemia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>1.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-89180844-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150487794">rs150487794</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                <span class="label label-warning">LIKELY_PATHOGENIC</span>
                                
                                
                                
                                <b>[PM5_Supporting, PP3_Strong, PP5_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-danger">PATHOGENIC_OR_LIKELY_PATHOGENIC</span>
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/31136">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                <span class="label label-danger">WHITELIST VARIANT</span>
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317447.4">ACSF3:ENST00000317447.4:c.1075G&gt;A:p.(Glu359Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378345.4">ACSF3:ENST00000378345.4:c.280G&gt;A:p.(Glu94Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000406948.3">ACSF3:ENST00000406948.3:c.1075G&gt;A:p.(Glu359Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9659</dd>
                                    
                                    <dd>REVEL: 0.936</dd>
                                    <dd>MVP: 0.777</dd>
                                    <dd>ALPHA_MISSENSE: 0.966</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0308%</dd>
                                    <dd>gnomAD_E_AMR: 0.0810%</dd>
                                    <dd>gnomAD_E_NFE: 0.1074%</dd>
                                    <dd>gnomAD_E_SAS: 0.0098%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_NFE: 0.0908%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="HFE" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000010704">HFE</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.781 to <a href="http://www.omim.org/entry/235200">Hemochromatosis, type 1</a> associated with HFE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001640, Cardiomegaly</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/235200">OMIM:235200</a> Hemochromatosis, type 1 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=465508">ORPHA:465508</a> Symptomatic form of HFE-related hemochromatosis - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=586">ORPHA:586</a> Cystic fibrosis (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>1.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-26091179-C-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1799945">rs1799945</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP5_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-danger">PATHOGENIC_OR_LIKELY_PATHOGENIC</span>
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/10">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                <span class="label label-danger">WHITELIST VARIANT</span>
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000309234.6">HFE:ENST00000309234.6:c.187C&gt;G:p.(His63Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317896.7">HFE:ENST00000317896.7:c.187C&gt;G:p.(His63Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336625.8">HFE:ENST00000336625.8:c.187C&gt;G:p.(His63Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357618.5">HFE:ENST00000357618.5:c.187C&gt;G:p.(His63Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397022.3">HFE:ENST00000397022.3:c.118C&gt;G:p.(His40Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000461397.1">HFE:ENST00000461397.1:c.187C&gt;G:p.(His63Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000470149.1">HFE:ENST00000470149.1:c.187C&gt;G:p.(His63Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349999.4">HFE:ENST00000349999.4:c.77-363C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000352392.4">HFE:ENST00000352392.4:c.77-2168C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000353147.5">HFE:ENST00000353147.5:c.77-1734C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000488199.1">HFE:ENST00000488199.1:c.77-363C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5719</dd>
                                    
                                    <dd>REVEL: 0.315</dd>
                                    <dd>ALPHA_MISSENSE: 0.572</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 2.7436%</dd>
                                    <dd>gnomAD_E_AMR: 10.0428%</dd>
                                    <dd>gnomAD_E_EAS: 3.4685%</dd>
                                    <dd>gnomAD_E_NFE: 14.4141%</dd>
                                    <dd>gnomAD_E_SAS: 8.0252%</dd>
                                    <dd>gnomAD_G_AFR: 2.6999%</dd>
                                    <dd>gnomAD_G_AMR: 9.6927%</dd>
                                    <dd>gnomAD_G_EAS: 2.6958%</dd>
                                    <dd>gnomAD_G_NFE: 14.5222%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NLRP7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167634">NLRP7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/231090">OMIM:231090</a> Hydatidiform mole, recurrent, 1 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>1.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-55447681-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs104895512">rs104895512</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP5_Strong, BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                <span class="label label-danger">PATHOGENIC</span>
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/97750">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                <span class="label label-danger">WHITELIST VARIANT</span>
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000328092.5">NLRP7:ENST00000328092.5:c.2164C&gt;G:p.(Leu722Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340844.2">NLRP7:ENST00000340844.2:c.2248C&gt;G:p.(Leu750Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000446217.1">NLRP7:ENST00000446217.1:c.2332C&gt;G:p.(Leu778Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000448121.2">NLRP7:ENST00000448121.2:c.2164C&gt;G:p.(Leu722Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000588756.1">NLRP7:ENST00000588756.1:c.2248C&gt;G:p.(Leu750Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000590030.1">NLRP7:ENST00000590030.1:c.2248C&gt;G:p.(Leu750Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000592784.1">NLRP7:ENST00000592784.1:c.2248C&gt;G:p.(Leu750Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.40304038</dd>
                                    
                                    <dd>REVEL: 0.245</dd>
                                    <dd>MVP: 0.403</dd>
                                    <dd>ALPHA_MISSENSE: 0.100</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.3817%</dd>
                                    <dd>gnomAD_G_AMR: 0.3538%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="POP1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000104356">POP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.713</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=POP1%0DRNASEH1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RNASEH1</a> and phenotypic similarity 0.786 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=329336">Adult-onset chronic progressive external ophthalmoplegia with mitochondrial myopathy</a> associated with RNASEH1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617396">OMIM:617396</a> Anauxetic dysplasia 2 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.713</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>8-99149189-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs182805220">rs182805220</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/783025">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.713</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349693.3">POP1:ENST00000349693.3:c.1362+7G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000401707.2">POP1:ENST00000401707.2:c.1362+7G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0308%</dd>
                                    <dd>gnomAD_E_AMR: 0.3444%</dd>
                                    <dd>gnomAD_E_NFE: 0.2927%</dd>
                                    <dd>gnomAD_G_AFR: 0.0459%</dd>
                                    <dd>gnomAD_G_AMR: 0.5896%</dd>
                                    <dd>gnomAD_G_NFE: 0.2657%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DELEC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000173077">DELEC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.430</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DELEC1%0DNR1D2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NR1D2</a> and phenotypic similarity 0.746 to mouse mutant of NR1D2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010404, ostium primum atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.430</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-118163577-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147129385">rs147129385</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.430</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374016.1">DELEC1:ENST00000374016.1:c.193G&gt;C:p.(Ala65Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.47875518</dd>
                                    
                                    <dd>MVP: 0.479</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.5605%</dd>
                                    <dd>gnomAD_E_AMR: 0.0261%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_AFR: 0.3671%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SPINT2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167642">SPINT2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b>
                    <data>(p=2.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.709</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPINT2%0DGATA6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GATA6</a> and phenotypic similarity 0.728 to <a href="http://www.omim.org/entry/614474">Atrioventricular septal defect 5</a> associated with GATA6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0006695, Atrioventricular canal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0006695, Atrioventricular canal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPINT2%0DGATA6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GATA6</a> and phenotypic similarity 0.446 to fish mutant of GATA6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000681, swim bladder non-functional, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000684, heart hypoplastic, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/270420">OMIM:270420</a> Diarrhea 3, secretory sodium, congenital, syndromic - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.015</b>
                                    <data>(p=2.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.709</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>19-38781218-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs111990687">rs111990687</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/774527">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.709</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000301244.7">SPINT2:ENST00000301244.7:c.592+3A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454580.3">SPINT2:ENST00000454580.3:c.421+3A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000587090.1">SPINT2:ENST00000587090.1:c.442+3A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000591889.1">SPINT2:ENST00000591889.1:c.225+3A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1110%</dd>
                                    <dd>gnomAD_E_AMR: 0.4226%</dd>
                                    <dd>gnomAD_E_NFE: 0.5554%</dd>
                                    <dd>gnomAD_E_SAS: 0.1440%</dd>
                                    <dd>gnomAD_G_AFR: 0.1837%</dd>
                                    <dd>gnomAD_G_AMR: 0.3538%</dd>
                                    <dd>gnomAD_G_NFE: 0.6097%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ODC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000115758">ODC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b>
                    <data>(p=2.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.424</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.405 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=544488">Global developmental delay-alopecia-macrocephaly-facial dysmorphism-structural brain anomalies syndrome</a> associated with ODC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0012520, Dilation of Virchow-Robin spaces</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0012520, Dilation of Virchow-Robin spaces</dd></dl><dl><dt>Phenotypic similarity 0.455 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ODC1">ODC1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002833, increased heart weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002833, increased heart weight</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ODC1%0DPSME4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PSME4</a> and phenotypic similarity 0.670 to mouse mutant of PSME4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000644, dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010420, muscular ventricular septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/619075">OMIM:619075</a> Bachmann-Bupp syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=544488">ORPHA:544488</a> Global developmental delay-alopecia-macrocephaly-facial dysmorphism-structural brain anomalies syndrome - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.015</b>
                                    <data>(p=2.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.424</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.405 to ORPHA:544488 Global developmental delay-alopecia-macrocephaly-facial dysmorphism-structural brain anomalies syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.404 to OMIM:619075 Bachmann-Bupp syndrome</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-10583846-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200691250">rs200691250</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.424</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000234111.4">ODC1:ENST00000234111.4:c.566T&gt;C:p.(Ile189Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405333.1">ODC1:ENST00000405333.1:c.566T&gt;C:p.(Ile189Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.424</dd>
                                    
                                    <dd>REVEL: 0.424</dd>
                                    <dd>MVP: 0.274</dd>
                                    <dd>ALPHA_MISSENSE: 0.212</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0026%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PLXNB2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196576">PLXNB2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b>
                    <data>(p=2.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.423</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.360 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PLXNB2">PLXNB2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000820, abnormal choroid plexus morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000820, abnormal choroid plexus morphology</dd></dl><dl><dt>Phenotypic similarity 0.299 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=PLXNB2">PLXNB2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0004430, axial vasculature has fewer parts of type intersegmental vessel, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0004430, axial vasculature has fewer parts of type intersegmental vessel, abnormal</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLXNB2%0DPLXND1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLXND1</a> and phenotypic similarity 0.656 to <a href="http://www.omim.org/entry/620294">Congenital heart defects, multiple types, 9</a> associated with PLXND1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0011579, Unbalanced atrioventricular canal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011579, Unbalanced atrioventricular canal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLXNB2%0DPLXND1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLXND1</a> and phenotypic similarity 0.766 to mouse mutant of PLXND1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010499, abnormal ventricle myocardium morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLXNB2%0DPLXND1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLXND1</a> and phenotypic similarity 0.296 to fish mutant of PLXND1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0004216, vascular sprouts disorganized, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0004216, vascular sprouts disorganized, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.015</b>
                                    <data>(p=2.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.423</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>22-50728478-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201664331">rs201664331</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.423</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359337.4">PLXNB2:ENST00000359337.4:c.536A&gt;G:p.(Asn179Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449103.1">PLXNB2:ENST00000449103.1:c.536A&gt;G:p.(Asn179Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.42378289</dd>
                                    
                                    <dd>REVEL: 0.088</dd>
                                    <dd>MVP: 0.424</dd>
                                    <dd>ALPHA_MISSENSE: 0.252</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0203%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>22-50721244-C-T [1|0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs200925999">rs200925999</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.099</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359337.4">PLXNB2:ENST00000359337.4:c.2883G&gt;A:p.(=)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449103.1">PLXNB2:ENST00000449103.1:c.2883G&gt;A:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>gnomAD_E_AFR: 0.0197%</dd>
                            <dd>gnomAD_E_AMR: 0.0319%</dd>
                            <dd>gnomAD_E_NFE: 0.0822%</dd>
                            <dd>gnomAD_G_AFR: 0.0231%</dd>
                            <dd>gnomAD_G_NFE: 0.0715%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TBX18" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000112837">TBX18</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b>
                    <data>(p=2.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.534</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.385</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.534 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TBX18">TBX18</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004056, abnormal myocardium compact layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010551, abnormal coronary vessel morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBX18%0DGATA4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GATA4</a> and phenotypic similarity 0.725 to <a href="http://www.omim.org/entry/607941">Atrial septal defect 2</a> associated with GATA4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001651, Dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0006695, Atrioventricular canal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBX18%0DGATA4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GATA4</a> and phenotypic similarity 0.731 to mouse mutant of GATA4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBX18%0DGATA4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GATA4</a> and phenotypic similarity 0.403 to fish mutant of GATA4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000490, heart malformed, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000490, heart malformed, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/143400">OMIM:143400</a> Congenital anomalies of kidney and urinary tract 2 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.015</b>
                                    <data>(p=2.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.534</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.385</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-85472367-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200869232">rs200869232</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2177028">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:1, LB:1)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.385</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369663.5">TBX18:ENST00000369663.5:c.392C&gt;T:p.(Thr131Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000606784.1">TBX18:ENST00000606784.1:c.-83C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.39423698</dd>
                                    
                                    <dd>REVEL: 0.170</dd>
                                    <dd>MVP: 0.394</dd>
                                    <dd>ALPHA_MISSENSE: 0.087</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1549%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CCL23" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000274736">CCL23</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b>
                    <data>(p=2.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.512</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.405</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.512 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CCL23">CCL23</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CCL23%0DMMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MMP2</a> and phenotypic similarity 0.626 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=371428">Multicentric osteolysis-nodulosis-arthropathy spectrum</a> associated with MMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001719, Double outlet right ventricle</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CCL23%0DMMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MMP2</a> and phenotypic similarity 0.512 to mouse mutant of MMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CCL23%0DMMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MMP2</a> and phenotypic similarity 0.251 to fish mutant of MMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0013882, interrenal  angiogenic sprout decreased length, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0013882, interrenal  angiogenic sprout decreased length, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.014</b>
                                    <data>(p=2.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.512</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.405</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-34340846-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs182352077">rs182352077</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.405</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000293280.2">CCL23:ENST00000293280.2:c.240C&gt;G:p.(Ile80Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000591423.1">CCL23:ENST00000591423.1:c.189C&gt;G:p.(Ile63Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.40556094</dd>
                                    
                                    <dd>MVP: 0.406</dd>
                                    <dd>ALPHA_MISSENSE: 0.276</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0116%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GALNS" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000141012">GALNS</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b>
                    <data>(p=2.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.254</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.693</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.636 to <a href="http://www.omim.org/entry/253000">Mucopolysaccharidosis IVA</a> associated with GALNS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001654, Abnormal heart valve morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001654, Abnormal heart valve morphology</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GALNS%0DIDUA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IDUA</a> and phenotypic similarity 0.835 to <a href="http://www.omim.org/entry/607014">Mucopolysaccharidosis Ih</a> associated with IDUA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001706, Endocardial fibroelastosis</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GALNS%0DIDUA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IDUA</a> and phenotypic similarity 0.640 to mouse mutant of IDUA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003974, abnormal endocardium morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010724, thick interventricular septum</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/253000">OMIM:253000</a> Mucopolysaccharidosis IVA - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.014</b>
                                    <data>(p=2.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.254</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.693</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-88893122-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150734270">rs150734270</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/193597">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:5, LB:2)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.693</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000268695.5">GALNS:ENST00000268695.5:c.1127G&gt;A:p.(Arg376Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000542788.1">GALNS:ENST00000542788.1:c.902G&gt;A:p.(Arg301Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.74629366</dd>
                                    
                                    <dd>REVEL: 0.502</dd>
                                    <dd>MVP: 0.746</dd>
                                    <dd>ALPHA_MISSENSE: 0.061</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0254%</dd>
                                    <dd>gnomAD_E_AMR: 0.4246%</dd>
                                    <dd>gnomAD_E_EAS: 0.1204%</dd>
                                    <dd>gnomAD_E_NFE: 0.2186%</dd>
                                    <dd>gnomAD_G_AFR: 0.0921%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_EAS: 0.1282%</dd>
                                    <dd>gnomAD_G_NFE: 0.1623%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DNAH5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000039139">DNAH5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b>
                    <data>(p=2.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.344</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.589</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.582 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with DNAH5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001627, Abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011535, Abnormal atrial arrangement</dd></dl><dl><dt>Phenotypic similarity 0.689 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=DNAH5">DNAH5</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.516 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAH5%0DCCDC39&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCDC39</a> and phenotypic similarity 0.582 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with CCDC39.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001627, Abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011535, Abnormal atrial arrangement</dd></dl><dl><dt>Proximity score 0.516 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAH5%0DCCDC39&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCDC39</a> and phenotypic similarity 0.707 to mouse mutant of CCDC39.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000644, dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010403, atrial septal defect</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/608644">OMIM:608644</a> Ciliary dyskinesia, primary, 3, with or without situs inversus - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">ORPHA:244</a> Primary ciliary dyskinesia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.013</b>
                                    <data>(p=2.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.344</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.589</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-13830838-C-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs143894811">rs143894811</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.589</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265104.4">DNAH5:ENST00000265104.4:c.5929G&gt;T:p.(Ala1977Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.589461</dd>
                                    
                                    <dd>REVEL: 0.150</dd>
                                    <dd>MVP: 0.589</dd>
                                    <dd>ALPHA_MISSENSE: 0.103</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>5-13913913-G-T [0|1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs189145014">rs189145014</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    <div class="col-sm-12">
                        <b>ClinVar: </b>
                        
                        
                            
                            
                            
                            
                            
                            <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                            <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/351210">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                            <span>(VUS:1, LB:1)</span>
                        
                    </div>
                    <div class="col-sm-6">
                        <b>Variant score: 0.532</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265104.4">DNAH5:ENST00000265104.4:c.1475C&gt;A:p.(Thr492Lys)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            <dd>Best Score: 0.5400164</dd>
                            
                            <dd>REVEL: 0.218</dd>
                            <dd>MVP: 0.540</dd>
                            <dd>ALPHA_MISSENSE: 0.147</dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>gnomAD_E_AMR: 0.0058%</dd>
                            <dd>gnomAD_E_SAS: 0.0980%</dd>
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>5-13882897-C-G [1|0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs200249508">rs200249508</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    <div class="col-sm-12">
                        <b>ClinVar: </b>
                        
                        
                            
                            
                            
                            
                            
                            <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                            <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/414356">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                            <span>(VUS:2, LB:1)</span>
                        
                    </div>
                    <div class="col-sm-6">
                        <b>Variant score: 0.322</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265104.4">DNAH5:ENST00000265104.4:c.3202G&gt;C:p.(Ala1068Pro)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            <dd>Best Score: 0.35418397</dd>
                            
                            <dd>REVEL: 0.057</dd>
                            <dd>MVP: 0.354</dd>
                            <dd>ALPHA_MISSENSE: 0.143</dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>gnomAD_E_AMR: 0.5149%</dd>
                            <dd>gnomAD_G_AMR: 0.2364%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="EGR3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000179388">EGR3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b>
                    <data>(p=2.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.403</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EGR3%0DNFATC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NFATC1</a> and phenotypic similarity 0.679 to mouse mutant of NFATC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001544, abnormal cardiovascular system physiology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.013</b>
                                    <data>(p=2.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.403</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-22548526-C-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199865032">rs199865032</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.403</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317216.2">EGR3:ENST00000317216.2:c.624G&gt;T:p.(Glu208Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000522910.1">EGR3:ENST00000522910.1:c.510G&gt;T:p.(Glu170Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000519492.1">EGR3:ENST00000519492.1:c.*461G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.40397477</dd>
                                    
                                    <dd>REVEL: 0.096</dd>
                                    <dd>MVP: 0.404</dd>
                                    <dd>ALPHA_MISSENSE: 0.229</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PNPLA1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000180316">PNPLA1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b>
                    <data>(p=2.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.256</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.677</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.511 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PNPLA1%0DABHD5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ABHD5</a> and phenotypic similarity 0.837 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98907">Neutral lipid storage disease with ichthyosis</a> associated with ABHD5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/615024">OMIM:615024</a> Ichthyosis, congenital, autosomal recessive 10 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79394">ORPHA:79394</a> Congenital ichthyosiform erythroderma - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.012</b>
                                    <data>(p=2.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.256</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.677</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-36263171-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs45524833">rs45524833</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/356564">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.677</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000312917.5">PNPLA1:ENST00000312917.5:c.487G&gt;A:p.(Glu163Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000388715.3">PNPLA1:ENST00000388715.3:c.460G&gt;A:p.(Glu154Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394571.2">PNPLA1:ENST00000394571.2:c.745G&gt;A:p.(Glu249Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.77371377</dd>
                                    
                                    <dd>REVEL: 0.109</dd>
                                    <dd>MVP: 0.774</dd>
                                    <dd>ALPHA_MISSENSE: 0.273</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.2399%</dd>
                                    <dd>gnomAD_E_AMR: 0.6533%</dd>
                                    <dd>gnomAD_E_NFE: 0.3217%</dd>
                                    <dd>gnomAD_E_SAS: 0.2646%</dd>
                                    <dd>gnomAD_G_AFR: 0.3215%</dd>
                                    <dd>gnomAD_G_AMR: 0.4717%</dd>
                                    <dd>gnomAD_G_NFE: 0.2528%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="C7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000112936">C7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b>
                    <data>(p=2.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.680</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=C7%0DLY6E&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LY6E</a> and phenotypic similarity 0.621 to mouse mutant of LY6E.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005294, abnormal heart ventricle morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000281, abnormal interventricular septum morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/610102">OMIM:610102</a> C7 deficiency - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.012</b>
                                    <data>(p=2.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.680</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-40931237-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs78327832">rs78327832</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1084122">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.680</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313164.9">C7:ENST00000313164.9:c.134C&gt;T:p.(Thr45Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7555125</dd>
                                    
                                    <dd>REVEL: 0.192</dd>
                                    <dd>MVP: 0.756</dd>
                                    <dd>ALPHA_MISSENSE: 0.135</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0129%</dd>
                                    <dd>gnomAD_E_AMR: 0.5518%</dd>
                                    <dd>gnomAD_E_EAS: 0.1113%</dd>
                                    <dd>gnomAD_E_NFE: 0.0115%</dd>
                                    <dd>gnomAD_E_SAS: 0.0360%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.0194%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MED1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000125686">MED1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b>
                    <data>(p=2.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.759</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.097</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.759 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MED1">MED1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0006372, impaired placental function</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000280, thin ventricular wall</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MED1%0DANKRD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANKRD1</a> and phenotypic similarity 0.823 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=154">Familial isolated dilated cardiomyopathy</a> associated with ANKRD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MED1%0DANKRD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANKRD1</a> and phenotypic similarity 0.489 to mouse mutant of ANKRD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0012688, abnormal heart ventricles weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0012688, abnormal heart ventricles weight</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.011</b>
                                    <data>(p=2.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.759</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.097</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-37566398-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199502446">rs199502446</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.097</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000300651.6">MED1:ENST00000300651.6:c.2076A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394287.3">MED1:ENST00000394287.3:c.1640+436A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                    <dd>gnomAD_E_NFE: 0.0502%</dd>
                                    <dd>gnomAD_G_NFE: 0.1879%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NOM1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000146909">NOM1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b>
                    <data>(p=3.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.384</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NOM1%0DCCDC174&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCDC174</a> and phenotypic similarity 0.651 to <a href="http://www.omim.org/entry/616816">Hypotonia, infantile, with psychomotor retardation</a> associated with CCDC174.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.010</b>
                                    <data>(p=3.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.384</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-156742868-C-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199647631">rs199647631</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3201134">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.384</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000275820.3">NOM1:ENST00000275820.3:c.437C&gt;A:p.(Pro146Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.39783898</dd>
                                    
                                    <dd>REVEL: 0.051</dd>
                                    <dd>MVP: 0.398</dd>
                                    <dd>ALPHA_MISSENSE: 0.094</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0156%</dd>
                                    <dd>gnomAD_E_AMR: 0.1493%</dd>
                                    <dd>gnomAD_E_NFE: 0.0517%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.0325%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SPHKAP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000153820">SPHKAP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b>
                    <data>(p=3.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.383</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPHKAP%0DFLNA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FLNA</a> and phenotypic similarity 0.832 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=88630">Terminal osseous dysplasia-pigmentary defects syndrome</a> associated with FLNA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001723, Restrictive cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001655, Patent foramen ovale</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPHKAP%0DFLNA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FLNA</a> and phenotypic similarity 0.747 to mouse mutant of FLNA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000284, double outlet right ventricle</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.010</b>
                                    <data>(p=3.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.383</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-228858331-C-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs183902648">rs183902648</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.383</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392056.3">SPHKAP:ENST00000392056.3:c.4640G&gt;T:p.(Gly1547Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344657.5">SPHKAP:ENST00000344657.5:c.4634+1894G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3834</dd>
                                    
                                    <dd>REVEL: 0.256</dd>
                                    <dd>MVP: 0.354</dd>
                                    <dd>ALPHA_MISSENSE: 0.383</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TRUB2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167112">TRUB2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b>
                    <data>(p=3.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.378</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRUB2%0DSBDS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SBDS</a> and phenotypic similarity 0.739 to <a href="http://www.omim.org/entry/260400">Shwachman-Diamond syndrome 1</a> associated with SBDS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001700, Myocardial necrosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001700, Myocardial necrosis</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.010</b>
                                    <data>(p=3.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.378</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-131072058-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs34262454">rs34262454</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3183548">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.378</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372890.4">TRUB2:ENST00000372890.4:c.767G&gt;A:p.(Arg256Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000546104.1">TRUB2:ENST00000546104.1:c.599G&gt;A:p.(Arg200Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.385</dd>
                                    
                                    <dd>REVEL: 0.385</dd>
                                    <dd>MVP: 0.274</dd>
                                    <dd>ALPHA_MISSENSE: 0.158</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1292%</dd>
                                    <dd>gnomAD_E_AMR: 0.0231%</dd>
                                    <dd>gnomAD_E_EAS: 0.0109%</dd>
                                    <dd>gnomAD_G_AFR: 0.1147%</dd>
                                    <dd>gnomAD_G_EAS: 0.0642%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GBP7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000213512">GBP7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b>
                    <data>(p=3.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.943</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.010</b>
                                    <data>(p=3.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.943</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                            <b>SNV</b>
                            <b>1-89616127-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs138285876">rs138285876</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.943</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000294671.2">GBP7:ENST00000294671.2:c.757C&gt;T:p.(Arg253*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1822%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AMR: 0.3538%</dd>
                                    <dd>gnomAD_G_NFE: 0.0194%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="HIKESHI" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000149196">HIKESHI</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b>
                    <data>(p=3.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.940</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.313 to <a href="http://www.omim.org/entry/616881">Leukodystrophy, hypomyelinating, 13</a> associated with HIKESHI.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001257, Spasticity</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/616881">OMIM:616881</a> Leukodystrophy, hypomyelinating, 13 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.009</b>
                                    <data>(p=3.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.940</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-86048477-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201042989">rs201042989</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1029600">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.940</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000278483.3">HIKESHI:ENST00000278483.3:c.325G&gt;C:p.(Ala109Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000533986.1">HIKESHI:ENST00000533986.1:c.325G&gt;C:p.(Ala109Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9491</dd>
                                    
                                    <dd>REVEL: 0.611</dd>
                                    <dd>MVP: 0.857</dd>
                                    <dd>ALPHA_MISSENSE: 0.949</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0665%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AURKA" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000087586">AURKA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b>
                    <data>(p=3.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.509</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.363</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.509 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AURKA%0DKIF20A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KIF20A</a> and phenotypic similarity 0.882 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=75249">Familial isolated restrictive cardiomyopathy</a> associated with KIF20A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0031329, Interstitial cardiac fibrosis</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AURKA%0DKIF20A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KIF20A</a> and phenotypic similarity 0.513 to fish mutant of KIF20A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000233, heart decreased functionality, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0006958, bulbus arteriosus decreased size, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/114500">OMIM:114500</a> Colon cancer, susceptibility to (susceptibility)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.009</b>
                                    <data>(p=3.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.509</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.363</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>20-54961321-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs2230743">rs2230743</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3033554">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.363</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000312783.6">AURKA:ENST00000312783.6:c.311C&gt;T:p.(Ser104Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000347343.2">AURKA:ENST00000347343.2:c.311C&gt;T:p.(Ser104Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371356.2">AURKA:ENST00000371356.2:c.311C&gt;T:p.(Ser104Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395907.1">AURKA:ENST00000395907.1:c.311C&gt;T:p.(Ser104Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395909.4">AURKA:ENST00000395909.4:c.311C&gt;T:p.(Ser104Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395911.1">AURKA:ENST00000395911.1:c.311C&gt;T:p.(Ser104Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395913.3">AURKA:ENST00000395913.3:c.311C&gt;T:p.(Ser104Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395914.1">AURKA:ENST00000395914.1:c.311C&gt;T:p.(Ser104Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395915.3">AURKA:ENST00000395915.3:c.311C&gt;T:p.(Ser104Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.39769113</dd>
                                    
                                    <dd>REVEL: 0.079</dd>
                                    <dd>MVP: 0.398</dd>
                                    <dd>SPLICE_AI: 0.170</dd>
                                    <dd>ALPHA_MISSENSE: 0.098</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.4921%</dd>
                                    <dd>gnomAD_E_AMR: 0.4828%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.3327%</dd>
                                    <dd>gnomAD_G_AMR: 0.3538%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="UGGT2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000102595">UGGT2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b>
                    <data>(p=3.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.366</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=UGGT2%0DSLC35A2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC35A2</a> and phenotypic similarity 0.651 to <a href="http://www.omim.org/entry/300896">Congenital disorder of glycosylation, type IIm</a> associated with SLC35A2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.009</b>
                                    <data>(p=3.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.366</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>13-96600271-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs189009329">rs189009329</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.366</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376747.3">UGGT2:ENST00000376747.3:c.1520A&gt;G:p.(Tyr507Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.36665913</dd>
                                    
                                    <dd>REVEL: 0.172</dd>
                                    <dd>MVP: 0.367</dd>
                                    <dd>ALPHA_MISSENSE: 0.103</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                    <dd>gnomAD_E_SAS: 0.0197%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NBEAL1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000144426">NBEAL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b>
                    <data>(p=3.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.934</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.009</b>
                                    <data>(p=3.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.934</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-204016305-T-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201891714">rs201891714</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.934</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449802.1">NBEAL1:ENST00000449802.1:c.5493T&gt;G:p.(Asp1831Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9353</dd>
                                    
                                    <dd>REVEL: 0.216</dd>
                                    <dd>MVP: 0.547</dd>
                                    <dd>ALPHA_MISSENSE: 0.935</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PZP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000126838">PZP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b>
                    <data>(p=3.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.923</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.008</b>
                                    <data>(p=3.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.923</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                            <b>SNV</b>
                            <b>12-9310383-C-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs140851445">rs140851445</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.923</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261336.2">PZP:ENST00000261336.2:c.3349G&gt;T:p.(Glu1117*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381997.2">PZP:ENST00000381997.2:c.2707G&gt;T:p.(Glu903*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.4</dd>
                                    
                                    <dd>SPLICE_AI: 0.400</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.4484%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CHIA" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000134216">CHIA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b>
                    <data>(p=3.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.342</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CHIA%0DSLC32A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC32A1</a> and phenotypic similarity 0.618 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1934">Early infantile epileptic encephalopathy</a> associated with SLC32A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.008</b>
                                    <data>(p=3.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.342</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-111862903-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs138227985">rs138227985</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.342</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343320.6">CHIA:ENST00000343320.6:c.1246G&gt;A:p.(Gly416Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000353665.6">CHIA:ENST00000353665.6:c.763G&gt;A:p.(Gly255Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369740.1">CHIA:ENST00000369740.1:c.1246G&gt;A:p.(Gly416Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430615.1">CHIA:ENST00000430615.1:c.922G&gt;A:p.(Gly308Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000451398.2">CHIA:ENST00000451398.2:c.763G&gt;A:p.(Gly255Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483391.1">CHIA:ENST00000483391.1:c.763G&gt;A:p.(Gly255Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.36016384</dd>
                                    
                                    <dd>REVEL: 0.086</dd>
                                    <dd>MVP: 0.360</dd>
                                    <dd>ALPHA_MISSENSE: 0.181</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.3138%</dd>
                                    <dd>gnomAD_E_AMR: 0.0636%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_AFR: 0.2984%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TRPM5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000070985">TRPM5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b>
                    <data>(p=3.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.347</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRPM5%0DJPH2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to JPH2</a> and phenotypic similarity 0.925 to <a href="http://www.omim.org/entry/613873">Cardiomyopathy, hypertrophic, 17</a> associated with JPH2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001685, Myocardial fibrosis</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRPM5%0DJPH2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to JPH2</a> and phenotypic similarity 0.519 to mouse mutant of JPH2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000278, abnormal myocardial fiber morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001544, abnormal cardiovascular system physiology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.007</b>
                                    <data>(p=3.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.347</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-2441470-G-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200126412">rs200126412</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.347</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000155858.6">TRPM5:ENST00000155858.6:c.631C&gt;G:p.(Gln211Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000452833.1">TRPM5:ENST00000452833.1:c.637C&gt;G:p.(Gln213Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000528453.1">TRPM5:ENST00000528453.1:c.631C&gt;G:p.(Gln211Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000533060.1">TRPM5:ENST00000533060.1:c.631C&gt;G:p.(Gln211Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.353</dd>
                                    
                                    <dd>REVEL: 0.353</dd>
                                    <dd>MVP: 0.227</dd>
                                    <dd>ALPHA_MISSENSE: 0.186</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0232%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ADAM29" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000168594">ADAM29</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b>
                    <data>(p=3.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.904</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.007</b>
                                    <data>(p=3.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.904</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">STOP_LOST</span>
                            <b>SNV</b>
                            <b>4-175899137-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150490039">rs150490039</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.904</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359240.3">ADAM29:ENST00000359240.3:c.2461T&gt;C:p.(*821Glnext*61)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000404450.4">ADAM29:ENST00000404450.4:c.2461T&gt;C:p.(*821Glnext*61)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445694.1">ADAM29:ENST00000445694.1:c.2461T&gt;C:p.(*821Glnext*61)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000514159.1">ADAM29:ENST00000514159.1:c.2461T&gt;C:p.(*821Glnext*-821)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0746%</dd>
                                    <dd>gnomAD_E_AMR: 0.0987%</dd>
                                    <dd>gnomAD_E_EAS: 0.1014%</dd>
                                    <dd>gnomAD_E_NFE: 0.0219%</dd>
                                    <dd>gnomAD_E_SAS: 0.0557%</dd>
                                    <dd>gnomAD_G_AFR: 0.3042%</dd>
                                    <dd>gnomAD_G_AMR: 0.2092%</dd>
                                    <dd>gnomAD_G_EAS: 0.5362%</dd>
                                    <dd>gnomAD_G_NFE: 0.2157%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="COQ9" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000088682">COQ9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b>
                    <data>(p=3.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.269</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.569</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.525 to <a href="http://www.omim.org/entry/614654">Coenzyme Q10 deficiency, primary, 5</a> associated with COQ9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001712, Left ventricular hypertrophy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001712, Left ventricular hypertrophy</dd></dl><dl><dt>Phenotypic similarity 0.538 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=COQ9">COQ9</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003141, cardiac fibrosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003141, cardiac fibrosis</dd></dl><dl><dt>Proximity score 0.529 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=COQ9%0DCOQ4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COQ4</a> and phenotypic similarity 0.903 to <a href="http://www.omim.org/entry/616276">Coenzyme Q10 deficiency, primary, 7</a> associated with COQ4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/614654">OMIM:614654</a> Coenzyme Q10 deficiency, primary, 5 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.005</b>
                                    <data>(p=3.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.269</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.569</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-57492176-C-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs78846023">rs78846023</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PP3, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/136988">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.569</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262507.6">COQ9:ENST00000262507.6:c.625C&gt;G:p.(Leu209Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000567072.1">COQ9:ENST00000567072.1:c.520C&gt;G:p.(Leu174Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000567933.1">COQ9:ENST00000567933.1:c.379-1301C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.674</dd>
                                    
                                    <dd>REVEL: 0.674</dd>
                                    <dd>MVP: 0.485</dd>
                                    <dd>ALPHA_MISSENSE: 0.232</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.5765%</dd>
                                    <dd>gnomAD_E_AMR: 0.0350%</dd>
                                    <dd>gnomAD_E_EAS: 0.7657%</dd>
                                    <dd>gnomAD_E_NFE: 0.0063%</dd>
                                    <dd>gnomAD_E_SAS: 0.1751%</dd>
                                    <dd>gnomAD_G_AFR: 0.4933%</dd>
                                    <dd>gnomAD_G_EAS: 0.6410%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TDRD5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162782">TDRD5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b>
                    <data>(p=3.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.305</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TDRD5%0DBICC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BICC1</a> and phenotypic similarity 0.499 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=730">Autosomal dominant polycystic kidney disease</a> associated with BICC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001634, Mitral valve prolapse</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001634, Mitral valve prolapse</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TDRD5%0DBICC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BICC1</a> and phenotypic similarity 0.769 to mouse mutant of BICC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010402, ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.005</b>
                                    <data>(p=3.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.305</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-179609576-T-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs183321034">rs183321034</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.305</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000294848.8">TDRD5:ENST00000294848.8:c.1796T&gt;A:p.(Val599Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367614.1">TDRD5:ENST00000367614.1:c.1796T&gt;A:p.(Val599Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000444136.1">TDRD5:ENST00000444136.1:c.1796T&gt;A:p.(Val599Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3151</dd>
                                    
                                    <dd>REVEL: 0.102</dd>
                                    <dd>MVP: 0.043</dd>
                                    <dd>ALPHA_MISSENSE: 0.315</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.2052%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GADD45B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000099860">GADD45B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b>
                    <data>(p=3.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.295</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GADD45B%0DIL12B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IL12B</a> and phenotypic similarity 0.847 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3287">Takayasu arteritis</a> associated with IL12B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001654, Abnormal heart valve morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.005</b>
                                    <data>(p=3.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.295</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-2476388-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs140611154">rs140611154</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.295</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000215631.4">GADD45B:ENST00000215631.4:c.32A&gt;G:p.(Asn11Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000587345.1">GADD45B:ENST00000587345.1:c.32A&gt;G:p.(Asn11Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.30637732</dd>
                                    
                                    <dd>REVEL: 0.037</dd>
                                    <dd>MVP: 0.306</dd>
                                    <dd>ALPHA_MISSENSE: 0.079</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.2429%</dd>
                                    <dd>gnomAD_E_NFE: 0.0698%</dd>
                                    <dd>gnomAD_G_AMR: 0.2364%</dd>
                                    <dd>gnomAD_G_NFE: 0.0389%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CSF1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000184371">CSF1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b>
                    <data>(p=3.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.293</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.402 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CSF1">CSF1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005341, decreased susceptibility to atherosclerosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0005341, decreased susceptibility to atherosclerosis</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CSF1%0DNF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NF1</a> and phenotypic similarity 0.931 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=638">Neurofibromatosis-Noonan syndrome</a> associated with NF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001642, Pulmonic stenosis</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CSF1%0DNF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NF1</a> and phenotypic similarity 0.677 to mouse mutant of NF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002190, disorganized myocardium</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CSF1%0DNF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NF1</a> and phenotypic similarity 0.458 to fish mutant of NF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0003926, atrioventricular valve decreased functionality, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.005</b>
                                    <data>(p=3.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.293</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-110465935-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200796309">rs200796309</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Strong]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.293</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329608.6">CSF1:ENST00000329608.6:c.692G&gt;A:p.(Gly231Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344188.5">CSF1:ENST00000344188.5:c.692G&gt;A:p.(Gly231Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369801.1">CSF1:ENST00000369801.1:c.692G&gt;A:p.(Gly231Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369802.3">CSF1:ENST00000369802.3:c.692G&gt;A:p.(Gly231Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000420111.2">CSF1:ENST00000420111.2:c.545-747G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.30107844</dd>
                                    
                                    <dd>REVEL: 0.016</dd>
                                    <dd>MVP: 0.301</dd>
                                    <dd>SPLICE_AI: 0.160</dd>
                                    <dd>ALPHA_MISSENSE: 0.090</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1777%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NUP210" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000132182">NUP210</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b>
                    <data>(p=3.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.290</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.435 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=NUP210">NUP210</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001192, heart edematous, abnormal</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NUP210%0DNEK9&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NEK9</a> and phenotypic similarity 0.660 to <a href="http://www.omim.org/entry/614262">?Arthrogryposis, Perthes disease, and upward gaze palsy</a> associated with NEK9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001642, Pulmonic stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NUP210%0DNEK9&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NEK9</a> and phenotypic similarity 0.423 to mouse mutant of NEK9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0030975, placenta hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0030975, placenta hemorrhage</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.005</b>
                                    <data>(p=3.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.290</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-13381419-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs114096350">rs114096350</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/711803">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.290</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000254508.5">NUP210:ENST00000254508.5:c.3406G&gt;A:p.(Gly1136Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.36874348</dd>
                                    
                                    <dd>REVEL: 0.022</dd>
                                    <dd>MVP: 0.369</dd>
                                    <dd>ALPHA_MISSENSE: 0.077</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1354%</dd>
                                    <dd>gnomAD_E_AMR: 0.4483%</dd>
                                    <dd>gnomAD_E_NFE: 0.7834%</dd>
                                    <dd>gnomAD_E_SAS: 0.1437%</dd>
                                    <dd>gnomAD_G_AFR: 0.1377%</dd>
                                    <dd>gnomAD_G_AMR: 0.9434%</dd>
                                    <dd>gnomAD_G_NFE: 0.7321%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AFAP1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196526">AFAP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b>
                    <data>(p=3.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.294</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AFAP1%0DSHROOM3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SHROOM3</a> and phenotypic similarity 0.707 to mouse mutant of SHROOM3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0008788, abnormal fetal cardiomyocyte morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AFAP1%0DSHROOM3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SHROOM3</a> and phenotypic similarity 0.323 to fish mutant of SHROOM3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.005</b>
                                    <data>(p=3.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.294</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-7770645-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199953617">rs199953617</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.294</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358461.2">AFAP1:ENST00000358461.2:c.2092G&gt;A:p.(Ala698Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360265.4">AFAP1:ENST00000360265.4:c.2092G&gt;A:p.(Ala698Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382543.3">AFAP1:ENST00000382543.3:c.2344G&gt;A:p.(Ala782Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000420658.1">AFAP1:ENST00000420658.1:c.2344G&gt;A:p.(Ala782Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.35565886</dd>
                                    
                                    <dd>REVEL: 0.242</dd>
                                    <dd>MVP: 0.356</dd>
                                    <dd>ALPHA_MISSENSE: 0.206</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.4373%</dd>
                                    <dd>gnomAD_E_EAS: 0.0109%</dd>
                                    <dd>gnomAD_E_NFE: 0.0062%</dd>
                                    <dd>gnomAD_G_AMR: 0.8255%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SFI1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198089">SFI1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=3.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.857</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=3.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.857</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>22-31985498-G-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200928606">rs200928606</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.857</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400288.2">SFI1:ENST00000400288.2:c.1479G&gt;T:p.(Trp493Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400289.1">SFI1:ENST00000400289.1:c.1233G&gt;T:p.(Trp411Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000414585.1">SFI1:ENST00000414585.1:c.1020G&gt;T:p.(Trp340Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000432498.1">SFI1:ENST00000432498.1:c.1386G&gt;T:p.(Trp462Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000443011.1">SFI1:ENST00000443011.1:c.1020G&gt;T:p.(Trp340Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000443326.1">SFI1:ENST00000443326.1:c.1233G&gt;T:p.(Trp411Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540643.1">SFI1:ENST00000540643.1:c.1314G&gt;T:p.(Trp438Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.858</dd>
                                    
                                    <dd>REVEL: 0.473</dd>
                                    <dd>MVP: 0.347</dd>
                                    <dd>ALPHA_MISSENSE: 0.858</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0124%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GRM8" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000179603">GRM8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=3.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.856</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=3.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.856</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-126173481-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs182088200">rs182088200</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2270025">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.856</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339582.2">GRM8:ENST00000339582.2:c.1955G&gt;A:p.(Arg652Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358373.3">GRM8:ENST00000358373.3:c.1955G&gt;A:p.(Arg652Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000444921.2">GRM8:ENST00000444921.2:c.1955G&gt;A:p.(Arg652Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.856</dd>
                                    
                                    <dd>REVEL: 0.856</dd>
                                    <dd>MVP: 0.783</dd>
                                    <dd>ALPHA_MISSENSE: 0.745</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PACS2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000179364">PACS2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=4.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.667</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.667 to <a href="http://www.omim.org/entry/618067">Developmental and epileptic encephalopathy 66</a> associated with PACS2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001651, Dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PACS2%0DPKD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKD2</a> and phenotypic similarity 0.571 to <a href="http://www.omim.org/entry/613095">Polycystic kidney disease 2</a> associated with PKD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001651, Dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001651, Dextrocardia</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PACS2%0DPKD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKD2</a> and phenotypic similarity 0.732 to mouse mutant of PKD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PACS2%0DPKD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKD2</a> and phenotypic similarity 0.461 to fish mutant of PKD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000107, heart morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001722, atrioventricular valve morphology, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/618067">OMIM:618067</a> Developmental and epileptic encephalopathy 66 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=442835">ORPHA:442835</a> Non-specific early-onset epileptic encephalopathy - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=4.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.667</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.667 to OMIM:618067 Developmental and epileptic encephalopathy 66</dt>
                                
                                    <dt>Phenotypic similarity 0.298 to ORPHA:442835 Non-specific early-onset epileptic encephalopathy</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>14-105859593-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs141292203">rs141292203</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PP4, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1588343">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000325438.8">PACS2:ENST00000325438.8:c.2472C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430725.2">PACS2:ENST00000430725.2:c.2247C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000447393.1">PACS2:ENST00000447393.1:c.2484C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000458164.2">PACS2:ENST00000458164.2:c.2517C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000547217.1">PACS2:ENST00000547217.1:c.2382C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000551743.1">PACS2:ENST00000551743.1:c.1014C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000551801.1">PACS2:ENST00000551801.1:c.75C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                    <dd>gnomAD_E_NFE: 0.1424%</dd>
                                    <dd>gnomAD_E_SAS: 0.0588%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_NFE: 0.0649%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GNAS" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000087460">GNAS</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=4.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.538</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.243</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.390 to <a href="http://www.omim.org/entry/612462">Pseudohypoparathyroidism Ic</a> associated with GNAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000821, Hypothyroidism</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0006960, Choroid plexus calcification</dd></dl><dl><dt>Phenotypic similarity 0.538 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GNAS">GNAS</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005406, abnormal heart size</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0005406, abnormal heart size</dd></dl><dl><dt>Proximity score 0.517 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GNAS%0DRAMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAMP2</a> and phenotypic similarity 0.706 to mouse mutant of RAMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0031384, impaired blood-brain barrier function</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0006144, increased systemic arterial systolic blood pressure</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/103580">OMIM:103580</a> Pseudohypoparathyroidism Ia - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/166350">OMIM:166350</a> Osseous heteroplasia, progressive - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/174800">OMIM:174800</a> McCune-Albright syndrome, somatic, mosaic - somatic</dd>
<dd><a href="http://www.omim.org/entry/219080">OMIM:219080</a> ACTH-independent macronodular adrenal hyperplasia - somatic</dd>
<dd><a href="http://www.omim.org/entry/603233">OMIM:603233</a> Pseudohypoparathyroidism Ib - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/612462">OMIM:612462</a> Pseudohypoparathyroidism Ic - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/612463">OMIM:612463</a> Pseudopseudohypoparathyroidism - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=189427">ORPHA:189427</a> Cushing syndrome due to bilateral macronodular adrenocortical disease (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2762">ORPHA:2762</a> Progressive osseous heteroplasia - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=562">ORPHA:562</a> McCune-Albright syndrome (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79443">ORPHA:79443</a> Pseudohypoparathyroidism type 1A - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79444">ORPHA:79444</a> Pseudohypoparathyroidism type 1C - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79445">ORPHA:79445</a> Pseudopseudohypoparathyroidism - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=94089">ORPHA:94089</a> Pseudohypoparathyroidism type 1B - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=4.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.538</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.243</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.390 to OMIM:612462 Pseudohypoparathyroidism Ic</dt>
                                
                                    <dt>Phenotypic similarity 0.390 to OMIM:103580 Pseudohypoparathyroidism Ia</dt>
                                
                                    <dt>Phenotypic similarity 0.379 to ORPHA:79444 Pseudohypoparathyroidism type 1C</dt>
                                
                                    <dt>Phenotypic similarity 0.379 to ORPHA:79443 Pseudohypoparathyroidism type 1A</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>20-57430020-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200585625">rs200585625</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2223471">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.243</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000306120.3">GNAS:ENST00000306120.3:c.1510A&gt;G:p.(Lys504Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371099.2">GNAS:ENST00000371099.2:c.1700A&gt;G:p.(Glu567Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371100.4">GNAS:ENST00000371100.4:c.1700A&gt;G:p.(Glu567Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371102.4">GNAS:ENST00000371102.4:c.1700A&gt;G:p.(Glu567Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313949.7">GNAS:ENST00000313949.7:c.*42+14079A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371075.3">GNAS:ENST00000371075.3:c.*42+14079A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371098.2">GNAS:ENST00000371098.2:c.*42+14079A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.2473</dd>
                                    
                                    <dd>REVEL: 0.018</dd>
                                    <dd>MVP: 0.093</dd>
                                    <dd>ALPHA_MISSENSE: 0.247</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1173%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>20-57415536-C-T [1|0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs200256285">rs200256285</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    <div class="col-sm-12">
                        <b>ClinVar: </b>
                        
                        
                            
                            <span class="label label-info">LIKELY_BENIGN</span>
                            
                            
                            
                            
                            <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3048537">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                            
                        
                    </div>
                    <div class="col-sm-6">
                        <b>Variant score: 0.100</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313949.7">GNAS:ENST00000313949.7:c.375C&gt;T:p.(=)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371075.3">GNAS:ENST00000371075.3:c.375C&gt;T:p.(=)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371098.2">GNAS:ENST00000371098.2:c.375C&gt;T:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>gnomAD_E_AMR: 0.0318%</dd>
                            <dd>gnomAD_E_NFE: 0.0009%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="STON1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000243244">STON1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=4.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.280</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=STON1%0DSYT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SYT1</a> and phenotypic similarity 0.651 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=522077">Infantile hypotonia-oculomotor anomalies-hyperkinetic movements-developmental delay syndrome</a> associated with SYT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=STON1%0DSYT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SYT1</a> and phenotypic similarity 0.679 to mouse mutant of SYT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010418, perimembraneous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010418, perimembraneous ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=4.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.280</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-48809450-G-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146130623">rs146130623</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.280</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000309835.3">STON1:ENST00000309835.3:c.1678G&gt;T:p.(Gly560Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000404752.1">STON1:ENST00000404752.1:c.1678G&gt;T:p.(Gly560Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000406226.1">STON1:ENST00000406226.1:c.1678G&gt;T:p.(Gly560Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.32167277</dd>
                                    
                                    <dd>REVEL: 0.047</dd>
                                    <dd>MVP: 0.322</dd>
                                    <dd>ALPHA_MISSENSE: 0.122</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.6521%</dd>
                                    <dd>gnomAD_E_AMR: 0.0318%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_G_AFR: 0.6772%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="STON1-GTF2A1L" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000068781">STON1-GTF2A1L</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=4.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.280</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=STON1-GTF2A1L%0DSYT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SYT1</a> and phenotypic similarity 0.651 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=522077">Infantile hypotonia-oculomotor anomalies-hyperkinetic movements-developmental delay syndrome</a> associated with SYT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=STON1-GTF2A1L%0DSYT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SYT1</a> and phenotypic similarity 0.679 to mouse mutant of SYT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010418, perimembraneous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010418, perimembraneous ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=4.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.280</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-48809450-G-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146130623">rs146130623</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.280</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000309827.2">STON1-GTF2A1L:ENST00000309827.2:c.1678G&gt;T:p.(Gly560Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394751.3">STON1-GTF2A1L:ENST00000394751.3:c.1678G&gt;T:p.(Gly560Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394754.1">STON1-GTF2A1L:ENST00000394754.1:c.1678G&gt;T:p.(Gly560Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000402114.2">STON1-GTF2A1L:ENST00000402114.2:c.1678G&gt;T:p.(Gly560Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405008.1">STON1-GTF2A1L:ENST00000405008.1:c.1678G&gt;T:p.(Gly560Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.32167277</dd>
                                    
                                    <dd>REVEL: 0.047</dd>
                                    <dd>MVP: 0.322</dd>
                                    <dd>ALPHA_MISSENSE: 0.122</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.6521%</dd>
                                    <dd>gnomAD_E_AMR: 0.0318%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_G_AFR: 0.6772%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TDRD9" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000156414">TDRD9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=4.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.560</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TDRD9%0DBICC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BICC1</a> and phenotypic similarity 0.499 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=730">Autosomal dominant polycystic kidney disease</a> associated with BICC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001634, Mitral valve prolapse</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001634, Mitral valve prolapse</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TDRD9%0DBICC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BICC1</a> and phenotypic similarity 0.769 to mouse mutant of BICC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010402, ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618110">OMIM:618110</a> ?Spermatogenic failure 30 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=399805">ORPHA:399805</a> Male infertility with azoospermia or oligozoospermia due to single gene mutation - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=4.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.560</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-104470661-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200657917">rs200657917</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.560</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339063.5">TDRD9:ENST00000339063.5:c.1570C&gt;T:p.(Pro524Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409874.4">TDRD9:ENST00000409874.4:c.1570C&gt;T:p.(Pro524Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5653</dd>
                                    
                                    <dd>REVEL: 0.448</dd>
                                    <dd>MVP: 0.468</dd>
                                    <dd>SPLICE_AI: 0.270</dd>
                                    <dd>ALPHA_MISSENSE: 0.565</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0705%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CPPED1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000103381">CPPED1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=4.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.276</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CPPED1%0DIL6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IL6</a> and phenotypic similarity 0.409 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=85414">Systemic-onset juvenile idiopathic arthritis</a> associated with IL6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001701, Pericarditis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001701, Pericarditis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CPPED1%0DIL6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IL6</a> and phenotypic similarity 0.607 to mouse mutant of IL6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001544, abnormal cardiovascular system physiology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000281, abnormal interventricular septum morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=4.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.276</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-12798778-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs149102231">rs149102231</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.276</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381774.4">CPPED1:ENST00000381774.4:c.418G&gt;A:p.(Glu140Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261660.4">CPPED1:ENST00000261660.4:c.290-39901G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000433677.2">CPPED1:ENST00000433677.2:c.290-39806G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.33416107</dd>
                                    
                                    <dd>REVEL: 0.173</dd>
                                    <dd>MVP: 0.334</dd>
                                    <dd>ALPHA_MISSENSE: 0.088</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.2456%</dd>
                                    <dd>gnomAD_E_AMR: 0.8225%</dd>
                                    <dd>gnomAD_E_NFE: 0.7257%</dd>
                                    <dd>gnomAD_E_SAS: 0.2941%</dd>
                                    <dd>gnomAD_G_AFR: 0.2069%</dd>
                                    <dd>gnomAD_G_AMR: 0.3538%</dd>
                                    <dd>gnomAD_G_NFE: 0.5835%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GBP4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162654">GBP4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=4.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.839</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=4.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.839</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                            <b>SNV</b>
                            <b>1-89655720-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs41296177">rs41296177</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.839</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355754.6">GBP4:ENST00000355754.6:c.1197+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98</dd>
                                    
                                    <dd>SPLICE_AI: 0.980</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0246%</dd>
                                    <dd>gnomAD_E_AMR: 0.2305%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.5540%</dd>
                                    <dd>gnomAD_E_SAS: 0.2337%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_AMR: 0.4717%</dd>
                                    <dd>gnomAD_G_NFE: 0.7845%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DCAF4L2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000176566">DCAF4L2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.831</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.831</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-88886024-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201620366">rs201620366</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.831</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000319675.3">DCAF4L2:ENST00000319675.3:c.176T&gt;C:p.(Val59Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8320689</dd>
                                    
                                    <dd>REVEL: 0.133</dd>
                                    <dd>MVP: 0.832</dd>
                                    <dd>ALPHA_MISSENSE: 0.617</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0116%</dd>
                                    <dd>gnomAD_E_NFE: 0.0026%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZBTB24" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000112365">ZBTB24</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.151</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.659</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.302 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ZBTB24">ZBTB24</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002792, abnormal retina vasculature morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002792, abnormal retina vasculature morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/614069">OMIM:614069</a> Immunodeficiency-centromeric instability-facial anomalies syndrome 2 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2268">ORPHA:2268</a> ICF syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.151</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.659</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-109803084-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147441359">rs147441359</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/252685">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:1, B:5)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.659</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000230122.3">ZBTB24:ENST00000230122.3:c.146G&gt;A:p.(Arg49Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8371345</dd>
                                    
                                    <dd>REVEL: 0.273</dd>
                                    <dd>MVP: 0.837</dd>
                                    <dd>ALPHA_MISSENSE: 0.251</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1661%</dd>
                                    <dd>gnomAD_E_AMR: 0.7950%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.9089%</dd>
                                    <dd>gnomAD_E_SAS: 0.2221%</dd>
                                    <dd>gnomAD_G_AFR: 0.1606%</dd>
                                    <dd>gnomAD_G_AMR: 0.9434%</dd>
                                    <dd>gnomAD_G_NFE: 0.7909%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TMEM131" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000075568">TMEM131</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.486</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.279</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.486 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TMEM131">TMEM131</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001914, hemorrhage</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.486</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.279</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-98378573-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201313610">rs201313610</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2369085">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.279</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000186436.5">TMEM131:ENST00000186436.5:c.4814C&gt;T:p.(Pro1605Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.28</dd>
                                    
                                    <dd>REVEL: 0.104</dd>
                                    <dd>MVP: 0.082</dd>
                                    <dd>SPLICE_AI: 0.280</dd>
                                    <dd>ALPHA_MISSENSE: 0.097</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0087%</dd>
                                    <dd>gnomAD_E_NFE: 0.0133%</dd>
                                    <dd>gnomAD_E_SAS: 0.0066%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BTN3A3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000111801">BTN3A3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.826</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.826</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                            <b>SNV</b>
                            <b>6-26446011-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs138065289">rs138065289</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.826</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000244519.2">BTN3A3:ENST00000244519.2:c.513G&gt;A:p.(Trp171*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339789.4">BTN3A3:ENST00000339789.4:c.387G&gt;A:p.(Trp129*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361232.3">BTN3A3:ENST00000361232.3:c.387G&gt;A:p.(Trp129*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0369%</dd>
                                    <dd>gnomAD_E_AMR: 0.3990%</dd>
                                    <dd>gnomAD_E_EAS: 0.1957%</dd>
                                    <dd>gnomAD_E_NFE: 0.6822%</dd>
                                    <dd>gnomAD_E_SAS: 0.6729%</dd>
                                    <dd>gnomAD_G_AFR: 0.0689%</dd>
                                    <dd>gnomAD_G_AMR: 0.8255%</dd>
                                    <dd>gnomAD_G_EAS: 0.3205%</dd>
                                    <dd>gnomAD_G_NFE: 0.5774%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FSD1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105255">FSD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.260</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.316 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=FSD1">FSD1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FSD1%0DTMEM94&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TMEM94</a> and phenotypic similarity 0.629 to <a href="http://www.omim.org/entry/618316">Intellectual developmental disorder with cardiac defects and dysmorphic facies</a> associated with TMEM94.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0011571, Parachute mitral valve</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FSD1%0DTMEM94&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TMEM94</a> and phenotypic similarity 0.508 to mouse mutant of TMEM94.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001634, internal hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002188, small heart</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.260</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-4310552-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200331149">rs200331149</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.260</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000221856.6">FSD1:ENST00000221856.6:c.449C&gt;T:p.(Ala150Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000597590.1">FSD1:ENST00000597590.1:c.449C&gt;T:p.(Ala150Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.26472834</dd>
                                    
                                    <dd>REVEL: 0.244</dd>
                                    <dd>MVP: 0.265</dd>
                                    <dd>ALPHA_MISSENSE: 0.121</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.1273%</dd>
                                    <dd>gnomAD_E_NFE: 0.0326%</dd>
                                    <dd>gnomAD_G_NFE: 0.0194%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NACA" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196531">NACA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.623</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.115</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.623 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NACA">NACA</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005329, abnormal myocardium layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000281, abnormal interventricular septum morphology</dd></dl><dl><dt>Phenotypic similarity 0.387 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=NACA">NACA</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001192, heart edematous, abnormal</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NACA%0DIRX5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRX5</a> and phenotypic similarity 0.737 to <a href="http://www.omim.org/entry/611174">Hamamy syndrome</a> associated with IRX5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001674, Complete atrioventricular canal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001674, Complete atrioventricular canal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.623</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.115</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-57114842-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148087516">rs148087516</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.115</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454682.1">NACA:ENST00000454682.1:c.472C&gt;T:p.(Leu158Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000550952.1">NACA:ENST00000550952.1:c.472C&gt;T:p.(Leu158Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356769.3">NACA:ENST00000356769.3:c.70+3394C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393891.4">NACA:ENST00000393891.4:c.70+3394C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000546392.1">NACA:ENST00000546392.1:c.70+3394C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000552540.1">NACA:ENST00000552540.1:c.70+3394C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000548563.1">NACA:ENST00000548563.1:c.-184+3394C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1387</dd>
                                    
                                    <dd>REVEL: 0.090</dd>
                                    <dd>MVP: 0.118</dd>
                                    <dd>ALPHA_MISSENSE: 0.139</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.5278%</dd>
                                    <dd>gnomAD_G_AMR: 0.8255%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FUZ" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000010361">FUZ</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.653</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.077</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.508 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3027">Caudal regression syndrome</a> associated with FUZ.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0030680, Abnormal cardiovascular system morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0030680, Abnormal cardiovascular system morphology</dd></dl><dl><dt>Phenotypic similarity 0.598 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FUZ">FUZ</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010412, atrioventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010412, atrioventricular septal defect</dd></dl><dl><dt>Proximity score 0.653 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FUZ%0DINTU&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to INTU</a> and phenotypic similarity 0.706 to <a href="http://www.omim.org/entry/617925">?Short-rib thoracic dysplasia 20 with polydactyly</a> associated with INTU.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001674, Complete atrioventricular canal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001674, Complete atrioventricular canal defect</dd></dl><dl><dt>Proximity score 0.653 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FUZ%0DINTU&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to INTU</a> and phenotypic similarity 0.485 to mouse mutant of INTU.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000269, abnormal heart looping</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001787, pericardial edema</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/182940">OMIM:182940</a> Neural tube defects, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3027">ORPHA:3027</a> Caudal regression syndrome - polygenic</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.653</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.077</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-50311997-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201118677">rs201118677</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.077</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313777.4">FUZ:ENST00000313777.4:c.870C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445575.2">FUZ:ENST00000445575.2:c.870C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000528094.1">FUZ:ENST00000528094.1:c.762C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000533418.1">FUZ:ENST00000533418.1:c.720C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.9796%</dd>
                                    <dd>gnomAD_E_AMR: 0.0517%</dd>
                                    <dd>gnomAD_E_EAS: 0.1358%</dd>
                                    <dd>gnomAD_E_NFE: 0.1552%</dd>
                                    <dd>gnomAD_E_SAS: 0.0344%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.0649%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PLB1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163803">PLB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.248</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLB1%0DPLD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLD1</a> and phenotypic similarity 0.623 to <a href="http://www.omim.org/entry/212093">Cardiac valvular dysplasia 1</a> associated with PLD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001642, Pulmonic stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLB1%0DPLD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLD1</a> and phenotypic similarity 0.455 to mouse mutant of PLD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002833, increased heart weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002833, increased heart weight</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLB1%0DPLD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLD1</a> and phenotypic similarity 0.277 to fish mutant of PLD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001301, intersegmental vessel morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001301, intersegmental vessel morphology, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.248</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-28836901-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs115147925">rs115147925</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2650779">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.248</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000327757.5">PLB1:ENST00000327757.5:c.3133G&gt;A:p.(Glu1045Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000422425.2">PLB1:ENST00000422425.2:c.3100G&gt;A:p.(Glu1034Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541605.1">PLB1:ENST00000541605.1:c.28G&gt;A:p.(Glu10Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.2704478</dd>
                                    
                                    <dd>REVEL: 0.108</dd>
                                    <dd>MVP: 0.270</dd>
                                    <dd>SPLICE_AI: 0.140</dd>
                                    <dd>ALPHA_MISSENSE: 0.095</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0862%</dd>
                                    <dd>gnomAD_E_AMR: 0.1852%</dd>
                                    <dd>gnomAD_E_NFE: 0.4776%</dd>
                                    <dd>gnomAD_E_SAS: 0.0719%</dd>
                                    <dd>gnomAD_G_AFR: 0.0924%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.4392%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="OR4N2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000176294">OR4N2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.247</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR4N2%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.782 to <a href="http://www.omim.org/entry/608068">Neutrophilic dermatosis, acute febrile</a> associated with MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.247</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-20296397-G-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200261876">rs200261876</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Strong]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.247</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000315947.1">OR4N2:ENST00000315947.1:c.790G&gt;T:p.(Ala264Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568211.1">OR4N2:ENST00000568211.1:c.*251G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.24732235</dd>
                                    
                                    <dd>REVEL: 0.011</dd>
                                    <dd>MVP: 0.247</dd>
                                    <dd>SPLICE_AI: 0.130</dd>
                                    <dd>ALPHA_MISSENSE: 0.083</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SNCAIP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000064692">SNCAIP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.238</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SNCAIP%0DSIAH1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SIAH1</a> and phenotypic similarity 0.626 to <a href="http://www.omim.org/entry/619314">Buratti-Harel syndrome</a> associated with SIAH1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.238</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-121759040-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs144384727">rs144384727</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/350486">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.238</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261367.7">SNCAIP:ENST00000261367.7:c.749A&gt;G:p.(Asn250Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261368.8">SNCAIP:ENST00000261368.8:c.608A&gt;G:p.(Asn203Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379533.2">SNCAIP:ENST00000379533.2:c.749A&gt;G:p.(Asn250Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379536.2">SNCAIP:ENST00000379536.2:c.608A&gt;G:p.(Asn203Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000503116.2">SNCAIP:ENST00000503116.2:c.749A&gt;G:p.(Asn250Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379538.3">SNCAIP:ENST00000379538.3:c.85-8624A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000414317.2">SNCAIP:ENST00000414317.2:c.103-17284A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000504884.2">SNCAIP:ENST00000504884.2:c.131-2007A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000542191.1">SNCAIP:ENST00000542191.1:c.-199-8624A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.24282551</dd>
                                    
                                    <dd>REVEL: 0.031</dd>
                                    <dd>MVP: 0.243</dd>
                                    <dd>ALPHA_MISSENSE: 0.056</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0665%</dd>
                                    <dd>gnomAD_E_NFE: 0.1438%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.1491%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TULP4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130338">TULP4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.628</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.096</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.628 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TULP4">TULP4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010418, perimembraneous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010420, muscular ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.628</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.096</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>6-158873293-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs41266333">rs41266333</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2657086">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.096</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367094.2">TULP4:ENST00000367094.2:c.852G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367097.3">TULP4:ENST00000367097.3:c.852G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0677%</dd>
                                    <dd>gnomAD_E_AMR: 0.1330%</dd>
                                    <dd>gnomAD_E_NFE: 0.2351%</dd>
                                    <dd>gnomAD_E_SAS: 0.0360%</dd>
                                    <dd>gnomAD_G_AFR: 0.0688%</dd>
                                    <dd>gnomAD_G_NFE: 0.2138%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BMP2K" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000138756">BMP2K</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.240</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BMP2K%0DGNPTAB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNPTAB</a> and phenotypic similarity 0.929 to <a href="http://www.omim.org/entry/252500">Mucolipidosis II alpha/beta</a> associated with GNPTAB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001640, Cardiomegaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BMP2K%0DGNPTAB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNPTAB</a> and phenotypic similarity 0.507 to mouse mutant of GNPTAB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002188, small heart</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BMP2K%0DGNPTAB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNPTAB</a> and phenotypic similarity 0.323 to fish mutant of GNPTAB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.240</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-79832486-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs189080275">rs189080275</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.240</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335016.5">BMP2K:ENST00000335016.5:c.2785G&gt;A:p.(Val929Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295462.3">BMP2K:ENST00000295462.3:c.*1228+3878C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.25679356</dd>
                                    
                                    <dd>REVEL: 0.059</dd>
                                    <dd>MVP: 0.257</dd>
                                    <dd>ALPHA_MISSENSE: 0.075</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.3975%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="XPO6" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169180">XPO6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.239</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=XPO6%0DVIPAS39&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to VIPAS39</a> and phenotypic similarity 0.614 to <a href="http://www.omim.org/entry/613404">Arthrogryposis, renal dysfunction, and cholestasis 2</a> associated with VIPAS39.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.239</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-28115967-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200543236">rs200543236</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.239</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000304658.5">XPO6:ENST00000304658.5:c.2846A&gt;G:p.(Asn949Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000565698.1">XPO6:ENST00000565698.1:c.2804A&gt;G:p.(Asn935Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.242</dd>
                                    
                                    <dd>REVEL: 0.242</dd>
                                    <dd>MVP: 0.117</dd>
                                    <dd>ALPHA_MISSENSE: 0.183</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1014%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PPP6R1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105063">PPP6R1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.236</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PPP6R1%0DIGBP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IGBP1</a> and phenotypic similarity 0.643 to <a href="http://www.omim.org/entry/300472">?Corpus callosum, agenesis of, with impaired intellectual development, ocular coloboma and micrognathia</a> associated with IGBP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.236</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-55752662-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200841391">rs200841391</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.236</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000412770.2">PPP6R1:ENST00000412770.2:c.1106A&gt;G:p.(Asn369Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000587283.1">PPP6R1:ENST00000587283.1:c.1106A&gt;G:p.(Asn369Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.25167963</dd>
                                    
                                    <dd>REVEL: 0.218</dd>
                                    <dd>MVP: 0.252</dd>
                                    <dd>ALPHA_MISSENSE: 0.069</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1116%</dd>
                                    <dd>gnomAD_E_AMR: 0.3404%</dd>
                                    <dd>gnomAD_E_EAS: 0.2683%</dd>
                                    <dd>gnomAD_E_NFE: 0.1353%</dd>
                                    <dd>gnomAD_E_SAS: 0.0197%</dd>
                                    <dd>gnomAD_G_AFR: 0.1378%</dd>
                                    <dd>gnomAD_G_EAS: 0.3851%</dd>
                                    <dd>gnomAD_G_NFE: 0.1103%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>19-55758397-G-A [0|1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs200237875">rs200237875</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.098</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000412770.2">PPP6R1:ENST00000412770.2:c.75C&gt;T:p.(=)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000587283.1">PPP6R1:ENST00000587283.1:c.75C&gt;T:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>gnomAD_E_AFR: 0.0517%</dd>
                            <dd>gnomAD_E_AMR: 0.0638%</dd>
                            <dd>gnomAD_E_NFE: 0.0800%</dd>
                            <dd>gnomAD_G_AFR: 0.0345%</dd>
                            <dd>gnomAD_G_AMR: 0.1182%</dd>
                            <dd>gnomAD_G_NFE: 0.0778%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FOXN1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000109101">FOXN1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.513</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.214</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.402 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FOXN1">FOXN1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004125, abnormal venule morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0004125, abnormal venule morphology</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FOXN1%0DDSC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DSC2</a> and phenotypic similarity 0.733 to <a href="http://www.omim.org/entry/610476">Arrhythmogenic right ventricular dysplasia 11 with mild palmoplantar keratoderma and woolly hair</a> associated with DSC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0011663, Right ventricular cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011663, Right ventricular cardiomyopathy</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FOXN1%0DDSC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DSC2</a> and phenotypic similarity 0.520 to mouse mutant of DSC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003141, cardiac fibrosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003141, cardiac fibrosis</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FOXN1%0DDSC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DSC2</a> and phenotypic similarity 0.323 to fish mutant of DSC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/601705">OMIM:601705</a> T-cell immunodeficiency, congenital alopecia, and nail dystrophy - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/618806">OMIM:618806</a> T-cell lymphopenia, infantile, with or without nail dystrophy, autosomal dominant - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=169095">ORPHA:169095</a> Severe combined immunodeficiency due to FOXN1 deficiency - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=169095">ORPHA:169095</a> Severe combined immunodeficiency due to FOXN1 deficiency - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.513</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.214</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-26851890-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142223881">rs142223881</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2046338">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.214</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000226247.2">FOXN1:ENST00000226247.2:c.493G&gt;A:p.(Val165Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579795.1">FOXN1:ENST00000579795.1:c.493G&gt;A:p.(Val165Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.21523975</dd>
                                    
                                    <dd>REVEL: 0.054</dd>
                                    <dd>MVP: 0.215</dd>
                                    <dd>ALPHA_MISSENSE: 0.085</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0260%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="INPP4A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000040933">INPP4A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.516</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.211</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.516 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=INPP4A%0DFIG4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FIG4</a> and phenotypic similarity 0.809 to <a href="http://www.omim.org/entry/216340">Yunis-Varon syndrome</a> associated with FIG4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.516 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=INPP4A%0DFIG4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FIG4</a> and phenotypic similarity 0.496 to mouse mutant of FIG4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003105, abnormal heart atrium morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003105, abnormal heart atrium morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.516</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.211</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-99170912-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150565121">rs150565121</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/708516">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.211</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000074304.5">INPP4A:ENST00000074304.5:c.1541G&gt;A:p.(Ser514Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409016.4">INPP4A:ENST00000409016.4:c.1541G&gt;A:p.(Ser514Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409540.3">INPP4A:ENST00000409540.3:c.1541G&gt;A:p.(Ser514Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409851.3">INPP4A:ENST00000409851.3:c.1526G&gt;A:p.(Ser509Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000523221.1">INPP4A:ENST00000523221.1:c.1541G&gt;A:p.(Ser514Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545415.1">INPP4A:ENST00000545415.1:c.1541G&gt;A:p.(Ser514Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409463.1">INPP4A:ENST00000409463.1:c.232-11268G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.2286</dd>
                                    
                                    <dd>REVEL: 0.029</dd>
                                    <dd>MVP: 0.139</dd>
                                    <dd>ALPHA_MISSENSE: 0.229</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0711%</dd>
                                    <dd>gnomAD_E_AMR: 0.2696%</dd>
                                    <dd>gnomAD_E_NFE: 0.4485%</dd>
                                    <dd>gnomAD_E_SAS: 0.1572%</dd>
                                    <dd>gnomAD_G_AFR: 0.0804%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.3954%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="C9orf50" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000179058">C9orf50</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.793</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.793</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>9-132375536-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201110691">rs201110691</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.793</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372478.4">C9orf50:ENST00000372478.4:c.1044-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372486.1">C9orf50:ENST00000372486.1:c.-55+4079G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0129%</dd>
                                    <dd>gnomAD_E_AMR: 0.0540%</dd>
                                    <dd>gnomAD_E_NFE: 0.0605%</dd>
                                    <dd>gnomAD_G_NFE: 0.0260%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KATNB1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000140854">KATNB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.509</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.659 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=89844">Lissencephaly syndrome, Norman-Roberts type</a> associated with KATNB1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KATNB1%0DCITED2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CITED2</a> and phenotypic similarity 0.643 to <a href="http://www.omim.org/entry/614431">Ventricular septal defect 2</a> associated with CITED2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0011682, Perimembranous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011682, Perimembranous ventricular septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KATNB1%0DCITED2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CITED2</a> and phenotypic similarity 0.747 to mouse mutant of CITED2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000284, double outlet right ventricle</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KATNB1%0DCITED2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CITED2</a> and phenotypic similarity 0.738 to fish mutant of CITED2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0005049, cardiovascular system morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0005049, cardiovascular system morphology, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/616212">OMIM:616212</a> Lissencephaly 6, with microcephaly - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=89844">ORPHA:89844</a> Lissencephaly syndrome, Norman-Roberts type - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.509</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-57778351-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs145567409">rs145567409</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/435541">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.509</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379661.3">KATNB1:ENST00000379661.3:c.217A&gt;G:p.(Thr73Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5878412</dd>
                                    
                                    <dd>REVEL: 0.068</dd>
                                    <dd>MVP: 0.588</dd>
                                    <dd>ALPHA_MISSENSE: 0.114</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.6894%</dd>
                                    <dd>gnomAD_E_AMR: 0.0202%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.6674%</dd>
                                    <dd>gnomAD_G_AMR: 0.1182%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TTC13" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143643">TTC13</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.508</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.211</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TTC13%0DPKP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKP2</a> and phenotypic similarity 0.722 to <a href="http://www.omim.org/entry/609040">Arrhythmogenic right ventricular dysplasia 9</a> associated with PKP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0011663, Right ventricular cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0034364, Fibrofatty replacement of right ventricular myocardium</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TTC13%0DPKP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKP2</a> and phenotypic similarity 0.538 to mouse mutant of PKP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005329, abnormal myocardium layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TTC13%0DPKP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKP2</a> and phenotypic similarity 0.501 to fish mutant of PKP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0005415, cardiac ventricle morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0005415, cardiac ventricle morphology, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.508</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.211</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-231061302-C-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201187353">rs201187353</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.211</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366661.4">TTC13:ENST00000366661.4:c.1549G&gt;C:p.(Glu517Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366662.4">TTC13:ENST00000366662.4:c.1390G&gt;C:p.(Glu464Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000414259.1">TTC13:ENST00000414259.1:c.1390G&gt;C:p.(Glu464Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.21174355</dd>
                                    
                                    <dd>REVEL: 0.123</dd>
                                    <dd>MVP: 0.212</dd>
                                    <dd>ALPHA_MISSENSE: 0.139</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0116%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TEKT4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163060">TEKT4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.220</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TEKT4%0DCCDC39&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCDC39</a> and phenotypic similarity 0.582 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with CCDC39.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001627, Abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011535, Abnormal atrial arrangement</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TEKT4%0DCCDC39&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCDC39</a> and phenotypic similarity 0.707 to mouse mutant of CCDC39.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000644, dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010403, atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.220</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-95540540-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs76696553">rs76696553</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.220</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295201.4">TEKT4:ENST00000295201.4:c.733C&gt;T:p.(Arg245Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568768.1">TEKT4:ENST00000568768.1:n.644-1246G&gt;A:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.222</dd>
                                    
                                    <dd>REVEL: 0.222</dd>
                                    <dd>MVP: 0.095</dd>
                                    <dd>ALPHA_MISSENSE: 0.024</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0757%</dd>
                                    <dd>gnomAD_E_AMR: 0.0032%</dd>
                                    <dd>gnomAD_E_EAS: 0.0127%</dd>
                                    <dd>gnomAD_E_NFE: 0.0164%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="EHHADH" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000113790">EHHADH</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.521</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.196</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.278 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3337">Primary Fanconi renotubular syndrome</a> associated with EHHADH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0002049, Proximal renal tubular acidosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Phenotypic similarity 0.455 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=EHHADH">EHHADH</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010506, prolonged RR interval</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010506, prolonged RR interval</dd></dl><dl><dt>Proximity score 0.521 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EHHADH%0DACAD9&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACAD9</a> and phenotypic similarity 0.878 to <a href="http://www.omim.org/entry/611126">Mitochondrial complex I deficiency, nuclear type 20</a> associated with ACAD9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/615605">OMIM:615605</a> ?Fanconi renotubular syndrome 3 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3337">ORPHA:3337</a> Primary Fanconi renotubular syndrome - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.521</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.196</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.278 to ORPHA:3337 Primary Fanconi renotubular syndrome</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-184911105-G-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs186736134">rs186736134</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/593437">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.196</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000231887.3">EHHADH:ENST00000231887.3:c.1081C&gt;A:p.(Pro361Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456310.1">EHHADH:ENST00000456310.1:c.793C&gt;A:p.(Pro265Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.20294946</dd>
                                    
                                    <dd>REVEL: 0.127</dd>
                                    <dd>MVP: 0.203</dd>
                                    <dd>ALPHA_MISSENSE: 0.109</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.2335%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="UMODL1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000177398">UMODL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.783</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.783</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>21-43557759-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs3819144">rs3819144</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.783</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400424.2">UMODL1:ENST00000400424.2:c.*21+8C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400427.1">UMODL1:ENST00000400427.1:c.*21+8C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000408910.2">UMODL1:ENST00000408910.2:c.*21+8C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000408989.2">UMODL1:ENST00000408989.2:c.*21+8C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0496%</dd>
                                    <dd>gnomAD_E_AMR: 0.0142%</dd>
                                    <dd>gnomAD_E_EAS: 0.0369%</dd>
                                    <dd>gnomAD_E_NFE: 0.0213%</dd>
                                    <dd>gnomAD_E_SAS: 0.0225%</dd>
                                    <dd>gnomAD_G_AFR: 0.0258%</dd>
                                    <dd>gnomAD_G_AMR: 0.1453%</dd>
                                    <dd>gnomAD_G_EAS: 0.0829%</dd>
                                    <dd>gnomAD_G_NFE: 0.0450%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">THREE_PRIME_UTR_INTRON_VARIANT</span>
                        <b>SNV</b>
                        <b>21-43557778-C-T [0|1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs117512190">rs117512190</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.000</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400424.2">UMODL1:ENST00000400424.2:c.*21+27C&gt;T:p.(=)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400427.1">UMODL1:ENST00000400427.1:c.*21+27C&gt;T:p.(=)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000408910.2">UMODL1:ENST00000408910.2:c.*21+27C&gt;T:p.(=)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000408989.2">UMODL1:ENST00000408989.2:c.*21+27C&gt;T:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>gnomAD_E_AFR: 0.0687%</dd>
                            <dd>gnomAD_E_AMR: 0.0919%</dd>
                            <dd>gnomAD_E_EAS: 0.0811%</dd>
                            <dd>gnomAD_E_NFE: 0.1235%</dd>
                            <dd>gnomAD_E_SAS: 0.0814%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KRT24" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167916">KRT24</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.518</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.195</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.518 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KRT24%0DMEOX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEOX1</a> and phenotypic similarity 0.651 to <a href="http://www.omim.org/entry/214300">Klippel-Feil syndrome 2</a> associated with MEOX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.518 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KRT24%0DMEOX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEOX1</a> and phenotypic similarity 0.255 to fish mutant of MEOX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0004056, posterior cardinal vein increased diameter, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0004056, posterior cardinal vein increased diameter, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.518</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.195</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-38856487-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150081982">rs150081982</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.195</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264651.2">KRT24:ENST00000264651.2:c.1004G&gt;A:p.(Arg335Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.19670166</dd>
                                    
                                    <dd>REVEL: 0.196</dd>
                                    <dd>MVP: 0.197</dd>
                                    <dd>ALPHA_MISSENSE: 0.057</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0087%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_SAS: 0.0098%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_EAS: 0.0642%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TBL3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183751">TBL3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.519</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.192</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.519 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBL3%0DELAC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ELAC2</a> and phenotypic similarity 0.962 to <a href="http://www.omim.org/entry/615440">Combined oxidative phosphorylation deficiency 17</a> associated with ELAC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.519</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.192</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-2024069-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200942398">rs200942398</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.192</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568546.1">TBL3:ENST00000568546.1:c.54G&gt;C:p.(Glu18Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1925</dd>
                                    
                                    <dd>MVP: 0.191</dd>
                                    <dd>ALPHA_MISSENSE: 0.192</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0186%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LRRIQ4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188306">LRRIQ4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.209</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRRIQ4%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.603 to <a href="http://www.omim.org/entry/255960">Myxoma, intracardiac</a> associated with PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0006689, Bacterial endocarditis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0006691, Pulmonic valve myxoma</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRRIQ4%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.473 to mouse mutant of PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003929, decreased heart rate variability</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003929, decreased heart rate variability</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.209</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-169555273-G-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200900869">rs200900869</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.209</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340806.6">LRRIQ4:ENST00000340806.6:c.1537G&gt;T:p.(Ala513Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.2166248</dd>
                                    
                                    <dd>REVEL: 0.045</dd>
                                    <dd>MVP: 0.217</dd>
                                    <dd>ALPHA_MISSENSE: 0.117</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1671%</dd>
                                    <dd>gnomAD_E_AMR: 0.0900%</dd>
                                    <dd>gnomAD_E_NFE: 0.0338%</dd>
                                    <dd>gnomAD_E_SAS: 0.0074%</dd>
                                    <dd>gnomAD_G_AFR: 0.2297%</dd>
                                    <dd>gnomAD_G_NFE: 0.0324%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CCDC134" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100147">CCDC134</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.491</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.455 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CCDC134">CCDC134</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001915, intracranial hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001625, cardiac hypertrophy</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CCDC134%0DTRRAP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRRAP</a> and phenotypic similarity 0.635 to <a href="http://www.omim.org/entry/618454">Developmental delay with or without dysmorphic facies and autism</a> associated with TRRAP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/619795">OMIM:619795</a> Osteogenesis imperfecta, type XXII - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.491</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>22-42206280-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201589404">rs201589404</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2466880">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.491</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000255784.5">CCDC134:ENST00000255784.5:c.295G&gt;A:p.(Glu99Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000402061.3">CCDC134:ENST00000402061.3:c.225+276G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.49902385</dd>
                                    
                                    <dd>REVEL: 0.127</dd>
                                    <dd>MVP: 0.499</dd>
                                    <dd>ALPHA_MISSENSE: 0.176</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0318%</dd>
                                    <dd>gnomAD_E_NFE: 0.0026%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PALB2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000083093">PALB2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.200</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.844 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with PALB2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Phenotypic similarity 0.386 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PALB2">PALB2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003229, abnormal vitelline vasculature morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003229, abnormal vitelline vasculature morphology</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PALB2%0DBRIP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BRIP1</a> and phenotypic similarity 0.844 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with BRIP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/610832">OMIM:610832</a> Fanconi anemia, complementation group N - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/620442">OMIM:620442</a> Breast-ovarian cancer, familial, susceptibility to, 5 (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1333">ORPHA:1333</a> Familial pancreatic carcinoma (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=145">ORPHA:145</a> Hereditary breast and/or ovarian cancer syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">ORPHA:84</a> Fanconi anemia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.200</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-23641448-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200875161">rs200875161</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                <b>[BP4_VeryStrong, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/142310">(REVIEWED_BY_EXPERT_PANEL)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.200</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261584.4">PALB2:ENST00000261584.4:c.2027T&gt;C:p.(Ile676Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.2238471</dd>
                                    
                                    <dd>REVEL: 0.003</dd>
                                    <dd>MVP: 0.224</dd>
                                    <dd>ALPHA_MISSENSE: 0.079</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.5812%</dd>
                                    <dd>gnomAD_G_AMR: 0.4717%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PEAR1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187800">PEAR1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.204</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PEAR1%0DNTRK3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NTRK3</a> and phenotypic similarity 0.686 to mouse mutant of NTRK3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000285, abnormal heart valve morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.204</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-156876492-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs61744071">rs61744071</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.204</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000292357.7">PEAR1:ENST00000292357.7:c.464C&gt;T:p.(Ser155Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000338302.3">PEAR1:ENST00000338302.3:c.464C&gt;T:p.(Ser155Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.2292643</dd>
                                    
                                    <dd>REVEL: 0.077</dd>
                                    <dd>MVP: 0.229</dd>
                                    <dd>ALPHA_MISSENSE: 0.095</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0123%</dd>
                                    <dd>gnomAD_E_AMR: 0.2198%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.1635%</dd>
                                    <dd>gnomAD_E_SAS: 0.3201%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.5896%</dd>
                                    <dd>gnomAD_G_NFE: 0.1428%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC26A9" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000174502">SLC26A9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.486</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.479 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SLC26A9">SLC26A9</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001914, hemorrhage</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC26A9%0DCLCN3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CLCN3</a> and phenotypic similarity 0.626 to <a href="http://www.omim.org/entry/619512">Neurodevelopmental disorder with hypotonia and brain abnormalities</a> associated with CLCN3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=586">ORPHA:586</a> Cystic fibrosis (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.486</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-205901026-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146704092">rs146704092</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2639857">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.486</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340781.4">SLC26A9:ENST00000340781.4:c.514G&gt;A:p.(Val172Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367134.2">SLC26A9:ENST00000367134.2:c.514G&gt;A:p.(Val172Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367135.3">SLC26A9:ENST00000367135.3:c.514G&gt;A:p.(Val172Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.61710936</dd>
                                    
                                    <dd>REVEL: 0.397</dd>
                                    <dd>MVP: 0.617</dd>
                                    <dd>ALPHA_MISSENSE: 0.292</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1477%</dd>
                                    <dd>gnomAD_E_AMR: 0.6627%</dd>
                                    <dd>gnomAD_E_NFE: 0.9261%</dd>
                                    <dd>gnomAD_E_SAS: 0.3660%</dd>
                                    <dd>gnomAD_G_AFR: 0.0459%</dd>
                                    <dd>gnomAD_G_AMR: 0.9434%</dd>
                                    <dd>gnomAD_G_NFE: 0.7844%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="N4BP2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000078177">N4BP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.193</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=N4BP2%0DRELB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RELB</a> and phenotypic similarity 0.681 to mouse mutant of RELB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003325, decreased liver function</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.193</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-40122453-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs78417585">rs78417585</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.193</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261435.6">N4BP2:ENST00000261435.6:c.2722C&gt;T:p.(Pro908Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.24732235</dd>
                                    
                                    <dd>REVEL: 0.081</dd>
                                    <dd>MVP: 0.247</dd>
                                    <dd>ALPHA_MISSENSE: 0.089</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1918%</dd>
                                    <dd>gnomAD_E_AMR: 0.4547%</dd>
                                    <dd>gnomAD_E_EAS: 0.0218%</dd>
                                    <dd>gnomAD_E_NFE: 0.9622%</dd>
                                    <dd>gnomAD_E_SAS: 0.7355%</dd>
                                    <dd>gnomAD_G_AFR: 0.1837%</dd>
                                    <dd>gnomAD_G_AMR: 0.3538%</dd>
                                    <dd>gnomAD_G_NFE: 0.7842%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ICE1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000164151">ICE1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.190</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ICE1%0DCPE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CPE</a> and phenotypic similarity 0.607 to <a href="http://www.omim.org/entry/619326">BDV syndrome</a> associated with CPE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ICE1%0DCPE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CPE</a> and phenotypic similarity 0.356 to mouse mutant of CPE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010463, aorta stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010463, aorta stenosis</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.190</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-5462905-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142087337">rs142087337</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.190</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000296564.7">ICE1:ENST00000296564.7:c.3458C&gt;T:p.(Thr1153Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.192</dd>
                                    
                                    <dd>REVEL: 0.192</dd>
                                    <dd>MVP: 0.082</dd>
                                    <dd>ALPHA_MISSENSE: 0.163</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0904%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_SAS: 0.0263%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LHFPL5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197753">LHFPL5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.754</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/610265">OMIM:610265</a> Deafness, autosomal recessive 67 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.754</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-35773490-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs149941106">rs149941106</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/228800">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:2, LB:2)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.754</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360215.1">LHFPL5:ENST00000360215.1:c.43C&gt;T:p.(His15Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373853.1">LHFPL5:ENST00000373853.1:c.43C&gt;T:p.(His15Tyr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.782</dd>
                                    
                                    <dd>REVEL: 0.782</dd>
                                    <dd>MVP: 0.630</dd>
                                    <dd>ALPHA_MISSENSE: 0.423</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1415%</dd>
                                    <dd>gnomAD_E_AMR: 0.2024%</dd>
                                    <dd>gnomAD_E_NFE: 0.0554%</dd>
                                    <dd>gnomAD_E_SAS: 0.0229%</dd>
                                    <dd>gnomAD_G_AFR: 0.2297%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.0324%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RPSA" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000168028">RPSA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=5.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.575</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.575 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RPSA">RPSA</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001636, irregular heartbeat</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0005648, heart right ventricle degeneration</dd></dl><dl><dt>Phenotypic similarity 0.323 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=RPSA">RPSA</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/271400">OMIM:271400</a> Asplenia, isolated congenital - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=5.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.575</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>3-39452430-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200785492">rs200785492</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2075687">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000301821.6">RPSA:ENST00000301821.6:c.438G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000443003.1">RPSA:ENST00000443003.1:c.453G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0064%</dd>
                                    <dd>gnomAD_E_AMR: 0.0348%</dd>
                                    <dd>gnomAD_E_NFE: 0.0107%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ADAMTSL5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185761">ADAMTSL5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=5.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.750</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=5.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.750</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-1507270-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs149051157">rs149051157</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.750</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000330475.4">ADAMTSL5:ENST00000330475.4:c.823G&gt;A:p.(Gly275Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395467.2">ADAMTSL5:ENST00000395467.2:c.130G&gt;A:p.(Gly44Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000413997.2">ADAMTSL5:ENST00000413997.2:c.853G&gt;A:p.(Gly285Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8984</dd>
                                    
                                    <dd>REVEL: 0.448</dd>
                                    <dd>MVP: 0.722</dd>
                                    <dd>ALPHA_MISSENSE: 0.898</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0445%</dd>
                                    <dd>gnomAD_E_AMR: 0.1073%</dd>
                                    <dd>gnomAD_E_EAS: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.2386%</dd>
                                    <dd>gnomAD_E_SAS: 0.7965%</dd>
                                    <dd>gnomAD_G_AFR: 0.0460%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.1624%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MTHFSD" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000103248">MTHFSD</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=5.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.183</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MTHFSD%0DSARDH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SARDH</a> and phenotypic similarity 0.885 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3129">Sarcosinemia</a> associated with SARDH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001642, Pulmonic stenosis</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=5.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.183</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-86588318-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200273062">rs200273062</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.183</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322911.6">MTHFSD:ENST00000322911.6:c.53G&gt;A:p.(Gly18Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360900.6">MTHFSD:ENST00000360900.6:c.56G&gt;A:p.(Gly19Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381214.5">MTHFSD:ENST00000381214.5:c.56G&gt;A:p.(Gly19Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543303.2">MTHFSD:ENST00000543303.2:c.53G&gt;A:p.(Gly18Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000546093.1">MTHFSD:ENST00000546093.1:c.-511G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.206</dd>
                                    
                                    <dd>REVEL: 0.206</dd>
                                    <dd>MVP: 0.201</dd>
                                    <dd>ALPHA_MISSENSE: 0.066</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0839%</dd>
                                    <dd>gnomAD_E_AMR: 0.1506%</dd>
                                    <dd>gnomAD_E_NFE: 0.5473%</dd>
                                    <dd>gnomAD_E_SAS: 0.0457%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_AMR: 0.1182%</dd>
                                    <dd>gnomAD_G_NFE: 0.6097%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ESYT1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000139641">ESYT1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=5.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.741</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=5.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.741</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-56531012-T-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs138187966">rs138187966</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3090635">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.741</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000267113.4">ESYT1:ENST00000267113.4:c.1823T&gt;G:p.(Leu608Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394048.5">ESYT1:ENST00000394048.5:c.1793T&gt;G:p.(Leu598Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541590.1">ESYT1:ENST00000541590.1:c.1823T&gt;G:p.(Leu608Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.76834035</dd>
                                    
                                    <dd>REVEL: 0.634</dd>
                                    <dd>MVP: 0.768</dd>
                                    <dd>ALPHA_MISSENSE: 0.177</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0636%</dd>
                                    <dd>gnomAD_E_NFE: 0.0237%</dd>
                                    <dd>gnomAD_E_SAS: 0.0294%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.2364%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PTPRN2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000155093">PTPRN2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.168</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PTPRN2%0DINSR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to INSR</a> and phenotypic similarity 0.900 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=508">Leprechaunism</a> associated with INSR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PTPRN2%0DINSR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to INSR</a> and phenotypic similarity 0.323 to fish mutant of INSR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.168</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-158109537-A-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs138046898">rs138046898</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/727931">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.168</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389413.3">PTPRN2:ENST00000389413.3:c.251T&gt;G:p.(Val84Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389416.4">PTPRN2:ENST00000389416.4:c.200T&gt;G:p.(Val67Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389418.4">PTPRN2:ENST00000389418.4:c.251T&gt;G:p.(Val84Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000404321.2">PTPRN2:ENST00000404321.2:c.320T&gt;G:p.(Val107Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409483.1">PTPRN2:ENST00000409483.1:c.164-111572T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.20208623</dd>
                                    
                                    <dd>REVEL: 0.034</dd>
                                    <dd>MVP: 0.202</dd>
                                    <dd>ALPHA_MISSENSE: 0.117</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.6961%</dd>
                                    <dd>gnomAD_E_AMR: 0.0234%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.8040%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>7-157985171-C-T [1|0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs140977880">rs140977880</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.150</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389413.3">PTPRN2:ENST00000389413.3:c.397G&gt;A:p.(Val133Ile)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389416.4">PTPRN2:ENST00000389416.4:c.346G&gt;A:p.(Val116Ile)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389418.4">PTPRN2:ENST00000389418.4:c.397G&gt;A:p.(Val133Ile)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000404321.2">PTPRN2:ENST00000404321.2:c.466G&gt;A:p.(Val156Ile)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409483.1">PTPRN2:ENST00000409483.1:c.283G&gt;A:p.(Val95Ile)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            <dd>Best Score: 0.17723896</dd>
                            
                            <dd>REVEL: 0.014</dd>
                            <dd>MVP: 0.177</dd>
                            <dd>ALPHA_MISSENSE: 0.090</dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>gnomAD_E_AFR: 0.7586%</dd>
                            <dd>gnomAD_E_AMR: 0.0645%</dd>
                            <dd>gnomAD_E_NFE: 0.0019%</dd>
                            <dd>gnomAD_E_SAS: 0.0109%</dd>
                            <dd>gnomAD_G_AFR: 0.6543%</dd>
                            <dd>gnomAD_G_AMR: 0.1179%</dd>
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>7-158109572-G-A [0|1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs146018380">rs146018380</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    <div class="col-sm-12">
                        <b>ClinVar: </b>
                        
                        <span class="label label-success">BENIGN</span>
                            
                            
                            
                            
                            
                            
                            <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/727932">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                            
                        
                    </div>
                    <div class="col-sm-6">
                        <b>Variant score: 0.083</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389413.3">PTPRN2:ENST00000389413.3:c.216C&gt;T:p.(=)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389416.4">PTPRN2:ENST00000389416.4:c.165C&gt;T:p.(=)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389418.4">PTPRN2:ENST00000389418.4:c.216C&gt;T:p.(=)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000404321.2">PTPRN2:ENST00000404321.2:c.285C&gt;T:p.(=)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409483.1">PTPRN2:ENST00000409483.1:c.164-111607C&gt;T:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>gnomAD_E_AFR: 0.7121%</dd>
                            <dd>gnomAD_E_AMR: 0.0319%</dd>
                            <dd>gnomAD_E_SAS: 0.0065%</dd>
                            <dd>gnomAD_G_AFR: 0.8039%</dd>
                            <dd>gnomAD_G_AMR: 0.1179%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PLA2G3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100078">PLA2G3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.165</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLA2G3%0DPLD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLD1</a> and phenotypic similarity 0.623 to <a href="http://www.omim.org/entry/212093">Cardiac valvular dysplasia 1</a> associated with PLD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001642, Pulmonic stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLA2G3%0DPLD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLD1</a> and phenotypic similarity 0.455 to mouse mutant of PLD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002833, increased heart weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002833, increased heart weight</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLA2G3%0DPLD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLD1</a> and phenotypic similarity 0.277 to fish mutant of PLD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001301, intersegmental vessel morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001301, intersegmental vessel morphology, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.165</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>22-31531875-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150521769">rs150521769</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.165</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000215885.3">PLA2G3:ENST00000215885.3:c.1364G&gt;A:p.(Arg455Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1701658</dd>
                                    
                                    <dd>REVEL: 0.049</dd>
                                    <dd>MVP: 0.170</dd>
                                    <dd>ALPHA_MISSENSE: 0.074</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1846%</dd>
                                    <dd>gnomAD_E_AMR: 0.0752%</dd>
                                    <dd>gnomAD_E_NFE: 0.0233%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.2067%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.0389%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KCTD12" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178695">KCTD12</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.730</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.730</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>13-77459383-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142368706">rs142368706</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.730</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317765.2">KCTD12:ENST00000317765.2:c.901G&gt;A:p.(Ala301Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377474.2">KCTD12:ENST00000377474.2:c.901G&gt;A:p.(Ala301Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7307637</dd>
                                    
                                    <dd>REVEL: 0.156</dd>
                                    <dd>MVP: 0.731</dd>
                                    <dd>ALPHA_MISSENSE: 0.475</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0123%</dd>
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TSC2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000103197">TSC2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.562</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.087</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.530 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=538">Lymphangioleiomyomatosis</a> associated with TSC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0011852, Chylopericardium</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011852, Chylopericardium</dd></dl><dl><dt>Phenotypic similarity 0.562 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TSC2">TSC2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003974, abnormal endocardium morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010656, thick myocardium</dd></dl><dl><dt>Phenotypic similarity 0.312 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=TSC2">TSC2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0003465, blood vessel increased diameter, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0003465, blood vessel increased diameter, abnormal</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TSC2%0DPKD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKD1</a> and phenotypic similarity 0.501 to <a href="http://www.omim.org/entry/173900">Polycystic kidney disease 1</a> associated with PKD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001634, Mitral valve prolapse</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001634, Mitral valve prolapse</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TSC2%0DPKD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKD1</a> and phenotypic similarity 0.708 to mouse mutant of PKD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002189, abnormal myocardial trabeculae morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TSC2%0DPKD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKD1</a> and phenotypic similarity 0.413 to fish mutant of PKD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001192, heart edematous, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/606690">OMIM:606690</a> Lymphangioleiomyomatosis, somatic - somatic</dd>
<dd><a href="http://www.omim.org/entry/607341">OMIM:607341</a> ?Focal cortical dysplasia, type II, somatic (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/613254">OMIM:613254</a> Tuberous sclerosis-2 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=538">ORPHA:538</a> Lymphangioleiomyomatosis (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=805">ORPHA:805</a> Tuberous sclerosis complex - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.562</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.087</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.487 to OMIM:613254 Tuberous sclerosis-2</dt>
                                
                                    <dt>Phenotypic similarity 0.477 to ORPHA:805 Tuberous sclerosis complex</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>16-2106725-C-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs45473698">rs45473698</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/49382">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.087</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000219476.3">TSC2:ENST00000219476.3:c.729C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000350773.4">TSC2:ENST00000350773.4:c.729C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000353929.4">TSC2:ENST00000353929.4:c.729C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382538.6">TSC2:ENST00000382538.6:c.582C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000401874.2">TSC2:ENST00000401874.2:c.729C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439673.2">TSC2:ENST00000439673.2:c.618C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568454.1">TSC2:ENST00000568454.1:c.762C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0615%</dd>
                                    <dd>gnomAD_E_AMR: 0.1706%</dd>
                                    <dd>gnomAD_E_NFE: 0.3543%</dd>
                                    <dd>gnomAD_E_SAS: 0.1862%</dd>
                                    <dd>gnomAD_G_AFR: 0.0689%</dd>
                                    <dd>gnomAD_G_NFE: 0.6741%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SDK2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000069188">SDK2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.722</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.722</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-71334790-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147236031">rs147236031</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3052066">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.722</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000388726.3">SDK2:ENST00000388726.3:c.6398G&gt;A:p.(Arg2133His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392650.3">SDK2:ENST00000392650.3:c.6455G&gt;A:p.(Arg2152His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.78639805</dd>
                                    
                                    <dd>REVEL: 0.239</dd>
                                    <dd>MVP: 0.786</dd>
                                    <dd>ALPHA_MISSENSE: 0.479</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0726%</dd>
                                    <dd>gnomAD_E_AMR: 0.2459%</dd>
                                    <dd>gnomAD_E_NFE: 0.0972%</dd>
                                    <dd>gnomAD_G_AFR: 0.0694%</dd>
                                    <dd>gnomAD_G_AMR: 0.4728%</dd>
                                    <dd>gnomAD_G_NFE: 0.0783%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ADCY10" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143199">ADCY10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.154</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.479 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ADCY10">ADCY10</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002626, increased heart rate</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002626, increased heart rate</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADCY10%0DGNB5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNB5</a> and phenotypic similarity 0.626 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=542306">GNB5-related intellectual disability-cardiac arrhythmia syndrome</a> associated with GNB5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001655, Patent foramen ovale</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001655, Patent foramen ovale</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADCY10%0DGNB5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNB5</a> and phenotypic similarity 0.486 to mouse mutant of GNB5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0006142, abnormal sinoatrial node conduction</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0006142, abnormal sinoatrial node conduction</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/143870">OMIM:143870</a> Hypercalciuria, absorptive, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2197">ORPHA:2197</a> Idiopathic hypercalciuria - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.154</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-167814913-G-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150156383">rs150156383</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/719941">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.154</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367848.1">ADCY10:ENST00000367848.1:c.2619C&gt;A:p.(Asp873Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367851.4">ADCY10:ENST00000367851.4:c.2895C&gt;A:p.(Asp965Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545172.1">ADCY10:ENST00000545172.1:c.2436C&gt;A:p.(Asp812Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.16767937</dd>
                                    
                                    <dd>REVEL: 0.068</dd>
                                    <dd>MVP: 0.168</dd>
                                    <dd>ALPHA_MISSENSE: 0.161</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0615%</dd>
                                    <dd>gnomAD_E_AMR: 0.2689%</dd>
                                    <dd>gnomAD_E_NFE: 0.2711%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_AMR: 0.4717%</dd>
                                    <dd>gnomAD_G_NFE: 0.2073%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GPRIN1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169258">GPRIN1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.153</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GPRIN1%0DGLI1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GLI1</a> and phenotypic similarity 0.723 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=289">Ellis Van Creveld syndrome</a> associated with GLI1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001651, Dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0006695, Atrioventricular canal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.153</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-176025757-T-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146535631">rs146535631</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.153</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000303991.4">GPRIN1:ENST00000303991.4:c.1079A&gt;T:p.(Asn360Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1555</dd>
                                    
                                    <dd>REVEL: 0.056</dd>
                                    <dd>MVP: 0.068</dd>
                                    <dd>ALPHA_MISSENSE: 0.155</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                    <dd>gnomAD_E_NFE: 0.0097%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RTP5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188011">RTP5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.147</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RTP5%0DFBXW11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FBXW11</a> and phenotypic similarity 0.631 to <a href="http://www.omim.org/entry/618914">Neurodevelopmental, jaw, eye, and digital syndrome</a> associated with FBXW11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001642, Pulmonic stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001655, Patent foramen ovale</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.147</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-242814228-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202220121">rs202220121</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2352033">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.147</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343216.3">RTP5:ENST00000343216.3:c.521C&gt;T:p.(Thr174Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.15261227</dd>
                                    
                                    <dd>REVEL: 0.113</dd>
                                    <dd>MVP: 0.153</dd>
                                    <dd>ALPHA_MISSENSE: 0.087</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1280%</dd>
                                    <dd>gnomAD_E_EAS: 0.0292%</dd>
                                    <dd>gnomAD_E_NFE: 0.0079%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ADGRV1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000164199">ADGRV1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.134</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADGRV1%0DSLC26A7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC26A7</a> and phenotypic similarity 0.729 to mouse mutant of SLC26A7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003503, decreased activity of thyroid gland</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/604352">OMIM:604352</a> ?Febrile seizures, familial, 4 (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/605472">OMIM:605472</a> Usher syndrome, type 2C - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=231178">ORPHA:231178</a> Usher syndrome type 2 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=36387">ORPHA:36387</a> Generalized epilepsy with febrile seizures-plus - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.134</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-89969880-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs72782753">rs72782753</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/46331">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:1, LB:3, B:4)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.134</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405460.2">ADGRV1:ENST00000405460.2:c.4939A&gt;G:p.(Ile1647Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.16250381</dd>
                                    
                                    <dd>REVEL: 0.074</dd>
                                    <dd>MVP: 0.163</dd>
                                    <dd>ALPHA_MISSENSE: 0.095</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1103%</dd>
                                    <dd>gnomAD_E_AMR: 0.5120%</dd>
                                    <dd>gnomAD_E_NFE: 0.7004%</dd>
                                    <dd>gnomAD_E_SAS: 0.2338%</dd>
                                    <dd>gnomAD_G_AFR: 0.1263%</dd>
                                    <dd>gnomAD_G_AMR: 0.8255%</dd>
                                    <dd>gnomAD_G_NFE: 0.8293%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ALDH9A1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143149">ALDH9A1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.128</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ALDH9A1%0DHADH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HADH</a> and phenotypic similarity 0.909 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=71212">Hyperinsulinism due to short chain 3-hydroxylacyl-CoA dehydrogenase deficiency</a> associated with HADH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.128</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-165667762-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201651041">rs201651041</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PM2_Supporting, BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.128</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354775.4">ALDH9A1:ENST00000354775.4:c.34C&gt;T:p.(Pro12Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.12839243</dd>
                                    
                                    <dd>REVEL: 0.077</dd>
                                    <dd>MVP: 0.128</dd>
                                    <dd>ALPHA_MISSENSE: 0.088</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="APOL3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000128284">APOL3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.530</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.530 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=APOL3">APOL3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010579, increased heart left ventricle size</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010579, increased heart left ventricle size</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.530</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>22-36537587-G-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs190228713">rs190228713</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349314.2">APOL3:ENST00000349314.2:c.870C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361710.2">APOL3:ENST00000361710.2:c.270C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397287.2">APOL3:ENST00000397287.2:c.270C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397293.2">APOL3:ENST00000397293.2:c.657C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000424878.2">APOL3:ENST00000424878.2:c.270C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0549%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LMNTD1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000152936">LMNTD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.132</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LMNTD1%0DCCR1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCR1</a> and phenotypic similarity 0.661 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=117">Behçet disease</a> associated with CCR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001637, Abnormal myocardium morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001637, Abnormal myocardium morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LMNTD1%0DCCR1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCR1</a> and phenotypic similarity 0.479 to mouse mutant of CCR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000343, abnormal response to cardiac infarction</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000343, abnormal response to cardiac infarction</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.132</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-25679729-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs149733853">rs149733853</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.132</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000282881.6">LMNTD1:ENST00000282881.6:c.589G&gt;A:p.(Val197Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000413632.2">LMNTD1:ENST00000413632.2:c.652G&gt;A:p.(Val218Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445693.1">LMNTD1:ENST00000445693.1:c.400G&gt;A:p.(Val134Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000458174.2">LMNTD1:ENST00000458174.2:c.652G&gt;A:p.(Val218Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539744.1">LMNTD1:ENST00000539744.1:c.298G&gt;A:p.(Val100Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.149</dd>
                                    
                                    <dd>REVEL: 0.149</dd>
                                    <dd>MVP: 0.092</dd>
                                    <dd>ALPHA_MISSENSE: 0.081</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1049%</dd>
                                    <dd>gnomAD_E_AMR: 0.5094%</dd>
                                    <dd>gnomAD_E_NFE: 0.6172%</dd>
                                    <dd>gnomAD_E_SAS: 0.0336%</dd>
                                    <dd>gnomAD_G_AFR: 0.1149%</dd>
                                    <dd>gnomAD_G_AMR: 0.3538%</dd>
                                    <dd>gnomAD_G_NFE: 0.5252%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NUP133" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000069248">NUP133</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.411</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.318 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2065">Galloway-Mowat syndrome</a> associated with NUP133.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0004374, Hemiplegia/hemiparesis</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Phenotypic similarity 0.482 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NUP133">NUP133</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001787, pericardial edema</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001787, pericardial edema</dd></dl><dl><dt>Phenotypic similarity 0.323 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=NUP133">NUP133</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NUP133%0DNEK9&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NEK9</a> and phenotypic similarity 0.660 to <a href="http://www.omim.org/entry/614262">?Arthrogryposis, Perthes disease, and upward gaze palsy</a> associated with NEK9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001642, Pulmonic stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NUP133%0DNEK9&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NEK9</a> and phenotypic similarity 0.423 to mouse mutant of NEK9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0030975, placenta hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0030975, placenta hemorrhage</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618177">OMIM:618177</a> Nephrotic syndrome, type 18 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/618349">OMIM:618349</a> ?Galloway-Mowat syndrome 8 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2065">ORPHA:2065</a> Galloway-Mowat syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=656">ORPHA:656</a> Genetic steroid-resistant nephrotic syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.411</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-229613399-G-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142046957">rs142046957</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2713692">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.411</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261396.3">NUP133:ENST00000261396.3:c.1701C&gt;A:p.(Asp567Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537506.1">NUP133:ENST00000537506.1:c.1653C&gt;A:p.(Asp551Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.4185946</dd>
                                    
                                    <dd>REVEL: 0.117</dd>
                                    <dd>MVP: 0.419</dd>
                                    <dd>ALPHA_MISSENSE: 0.219</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0185%</dd>
                                    <dd>gnomAD_E_AMR: 0.0289%</dd>
                                    <dd>gnomAD_E_NFE: 0.0457%</dd>
                                    <dd>gnomAD_G_AMR: 0.1185%</dd>
                                    <dd>gnomAD_G_NFE: 0.0713%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LARGE2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165905">LARGE2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.523</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.101</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.303 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LARGE2">LARGE2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002792, abnormal retina vasculature morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002792, abnormal retina vasculature morphology</dd></dl><dl><dt>Phenotypic similarity 0.323 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=LARGE2">LARGE2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl><dt>Proximity score 0.523 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LARGE2%0DPOMT2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POMT2</a> and phenotypic similarity 0.823 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=206559">POMT2-related limb-girdle muscular dystrophy R14</a> associated with POMT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dl><dt>Proximity score 0.523 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LARGE2%0DPOMT2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POMT2</a> and phenotypic similarity 0.323 to fish mutant of POMT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.523</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.101</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-45949903-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs188560105">rs188560105</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.101</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000325468.5">LARGE2:ENST00000325468.5:c.1930A&gt;G:p.(Ile644Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000401752.1">LARGE2:ENST00000401752.1:c.1930A&gt;G:p.(Ile644Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529052.1">LARGE2:ENST00000529052.1:c.1837A&gt;G:p.(Ile613Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000531526.1">LARGE2:ENST00000531526.1:c.1930A&gt;G:p.(Ile644Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536139.1">LARGE2:ENST00000536139.1:c.1837A&gt;G:p.(Ile613Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.11750665</dd>
                                    
                                    <dd>REVEL: 0.051</dd>
                                    <dd>MVP: 0.118</dd>
                                    <dd>ALPHA_MISSENSE: 0.072</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.3879%</dd>
                                    <dd>gnomAD_E_NFE: 0.0247%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.7075%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ATP2C2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000064270">ATP2C2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.525</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.095</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.525 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ATP2C2%0DCMIP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CMIP</a> and phenotypic similarity 0.612 to mouse mutant of CMIP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0013975, abnormal coronary sinus connection</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010420, muscular ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.525</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.095</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>16-84482279-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs62640930">rs62640930</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2646920">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.095</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262429.4">ATP2C2:ENST00000262429.4:c.1644G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000416219.2">ATP2C2:ENST00000416219.2:c.1644G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0841%</dd>
                                    <dd>gnomAD_E_AMR: 0.2169%</dd>
                                    <dd>gnomAD_E_NFE: 0.3313%</dd>
                                    <dd>gnomAD_E_SAS: 0.1019%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                    <dd>gnomAD_G_NFE: 0.2916%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC39A1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143570">SLC39A1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.522</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.522 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC39A1%0DSLC30A10&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC30A10</a> and phenotypic similarity 0.892 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=309854">Cirrhosis-dystonia-polycythemia-hypermanganesemia syndrome</a> associated with SLC30A10.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dl><dt>Proximity score 0.522 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC39A1%0DSLC30A10&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC30A10</a> and phenotypic similarity 0.729 to mouse mutant of SLC30A10.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003503, decreased activity of thyroid gland</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.522</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-153933063-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148164856">rs148164856</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000310483.6">SLC39A1:ENST00000310483.6:c.486G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356205.4">SLC39A1:ENST00000356205.4:c.486G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368621.1">SLC39A1:ENST00000368621.1:c.486G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368623.3">SLC39A1:ENST00000368623.3:c.486G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537590.1">SLC39A1:ENST00000537590.1:c.180G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0231%</dd>
                                    <dd>gnomAD_E_NFE: 0.0062%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CCDC88C" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000015133">CCDC88C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.123</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.282 to <a href="http://www.omim.org/entry/616053">?Spinocerebellar ataxia 40</a> associated with CCDC88C.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0002313, Spastic paraparesis</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CCDC88C%0DDAAM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DAAM1</a> and phenotypic similarity 0.705 to mouse mutant of DAAM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0008788, abnormal fetal cardiomyocyte morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/236600">OMIM:236600</a> Hydrocephalus, congenital, 1 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/616053">OMIM:616053</a> ?Spinocerebellar ataxia 40 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=423275">ORPHA:423275</a> Spinocerebellar ataxia type 40 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.123</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.282 to OMIM:616053 ?Spinocerebellar ataxia 40</dt>
                                
                                    <dt>Phenotypic similarity 0.282 to ORPHA:423275 Spinocerebellar ataxia type 40</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-91755563-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs189215037">rs189215037</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/777875">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.123</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389857.6">CCDC88C:ENST00000389857.6:c.4327G&gt;A:p.(Ala1443Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.12839243</dd>
                                    
                                    <dd>REVEL: 0.053</dd>
                                    <dd>MVP: 0.128</dd>
                                    <dd>ALPHA_MISSENSE: 0.068</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0194%</dd>
                                    <dd>gnomAD_E_AMR: 0.2636%</dd>
                                    <dd>gnomAD_E_NFE: 0.1969%</dd>
                                    <dd>gnomAD_E_SAS: 0.1176%</dd>
                                    <dd>gnomAD_G_AFR: 0.0459%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.1686%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="POLR1G" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000117877">POLR1G</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.116</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=POLR1G%0DTAF1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TAF1A</a> and phenotypic similarity 0.823 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=154">Familial isolated dilated cardiomyopathy</a> associated with TAF1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=POLR1G%0DTAF1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TAF1A</a> and phenotypic similarity 0.477 to fish mutant of TAF1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000233, heart decreased functionality, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.116</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-45912599-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201278532">rs201278532</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Strong]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.116</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000309424.3">POLR1G:ENST00000309424.3:c.1373G&gt;A:p.(Gly458Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589804.1">POLR1G:ENST00000589804.1:c.1379G&gt;A:p.(Gly460Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000300853.3">POLR1G:ENST00000300853.3:c.*334C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423698.2">POLR1G:ENST00000423698.2:c.*334C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1164</dd>
                                    
                                    <dd>REVEL: 0.015</dd>
                                    <dd>MVP: 0.107</dd>
                                    <dd>ALPHA_MISSENSE: 0.116</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZC3H11A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000058673">ZC3H11A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.519</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.519 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ZC3H11A">ZC3H11A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000277, abnormal heart shape</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZC3H11A%0DMPLKIP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MPLKIP</a> and phenotypic similarity 0.755 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=33364">Trichothiodystrophy</a> associated with MPLKIP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZC3H11A%0DMPLKIP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MPLKIP</a> and phenotypic similarity 0.508 to mouse mutant of MPLKIP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.519</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-203798978-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201157723">rs201157723</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000332127.4">ZC3H11A:ENST00000332127.4:c.573A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367210.1">ZC3H11A:ENST00000367210.1:c.573A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367212.3">ZC3H11A:ENST00000367212.3:c.573A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367214.1">ZC3H11A:ENST00000367214.1:c.573A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545588.1">ZC3H11A:ENST00000545588.1:c.573A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.1070%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SCAP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000114650">SCAP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.519</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.519 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SCAP%0DMBTPS2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MBTPS2</a> and phenotypic similarity 0.651 to <a href="http://www.omim.org/entry/308205">IFAP syndrome with or without BRESHECK syndrome</a> associated with MBTPS2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.519</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>3-47467590-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201263232">rs201263232</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265565.5">SCAP:ENST00000265565.5:c.807G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441517.2">SCAP:ENST00000441517.2:c.42G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545718.1">SCAP:ENST00000545718.1:c.-139-2055G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0261%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FPGT" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000254685">FPGT</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.518</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.518 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FPGT%0DTNNI3K&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TNNI3K</a> and phenotypic similarity 0.823 to <a href="http://www.omim.org/entry/616117">Cardiac conduction disease with or without dilated cardiomyopathy</a> associated with TNNI3K.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dl><dt>Proximity score 0.518 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FPGT%0DTNNI3K&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TNNI3K</a> and phenotypic similarity 0.485 to mouse mutant of TNNI3K.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005333, decreased heart rate</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002753, dilated heart left ventricle</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.518</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-74670325-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs186422789">rs186422789</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370898.3">FPGT:ENST00000370898.3:c.633G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000609362.1">FPGT:ENST00000609362.1:c.594G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370891.2">FPGT:ENST00000370891.2:c.343+3231G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370894.5">FPGT:ENST00000370894.5:c.344-567G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000534056.1">FPGT:ENST00000534056.1:c.409+185G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370893.1">FPGT:ENST00000370893.1:c.343+3231G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370895.1">FPGT:ENST00000370895.1:c.343+3231G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370899.3">FPGT:ENST00000370899.3:c.343+3231G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000557284.2">FPGT:ENST00000557284.2:c.382+3231G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0203%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FBXW10" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171931">FBXW10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.119</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FBXW10%0DLARS2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LARS2</a> and phenotypic similarity 0.604 to <a href="http://www.omim.org/entry/617021">Hydrops, lactic acidosis, and sideroblastic anemia</a> associated with LARS2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.119</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-18659367-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201169182">rs201169182</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.119</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000301938.4">FBXW10:ENST00000301938.4:c.1132A&gt;G:p.(Asn378Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308799.4">FBXW10:ENST00000308799.4:c.1219A&gt;G:p.(Asn407Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395665.4">FBXW10:ENST00000395665.4:c.1132A&gt;G:p.(Asn378Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395667.1">FBXW10:ENST00000395667.1:c.1132A&gt;G:p.(Asn378Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1201</dd>
                                    
                                    <dd>REVEL: 0.095</dd>
                                    <dd>MVP: 0.048</dd>
                                    <dd>ALPHA_MISSENSE: 0.120</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0727%</dd>
                                    <dd>gnomAD_G_NFE: 0.0338%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CFTR" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000001626">CFTR</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.517</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.597 to <a href="http://www.omim.org/entry/219700">Cystic fibrosis</a> associated with CFTR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001648, Cor pulmonale</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001648, Cor pulmonale</dd></dl><dl><dt>Proximity score 0.517 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CFTR%0DCLCN3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CLCN3</a> and phenotypic similarity 0.626 to <a href="http://www.omim.org/entry/619512">Neurodevelopmental disorder with hypotonia and brain abnormalities</a> associated with CLCN3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/167800">OMIM:167800</a> Pancreatitis, hereditary (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/211400">OMIM:211400</a> Bronchiectasis with or without elevated sweat chloride 1, modifier of (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/219700">OMIM:219700</a> Cystic fibrosis - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/277180">OMIM:277180</a> Congenital bilateral absence of vas deferens - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=399805">ORPHA:399805</a> Male infertility with azoospermia or oligozoospermia due to single gene mutation (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=48">ORPHA:48</a> Congenital bilateral absence of vas deferens - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=586">ORPHA:586</a> Cystic fibrosis - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=60033">ORPHA:60033</a> Idiopathic bronchiectasis - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=676">ORPHA:676</a> Hereditary chronic pancreatitis - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.517</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>7-117188739-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs62469440">rs62469440</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PM2_Supporting, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/193588">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000003084.6">CFTR:ENST00000003084.6:c.1254T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454343.1">CFTR:ENST00000454343.1:c.1209+6577T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NFIL3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165030">NFIL3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.115</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.369 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NFIL3">NFIL3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005542, cornea vascularization</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0005542, cornea vascularization</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NFIL3%0DNR1D2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NR1D2</a> and phenotypic similarity 0.746 to mouse mutant of NR1D2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010404, ostium primum atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.115</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-94172507-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142259863">rs142259863</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1693511">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.115</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000297689.3">NFIL3:ENST00000297689.3:c.510G&gt;A:p.(Met170Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.122</dd>
                                    
                                    <dd>REVEL: 0.122</dd>
                                    <dd>MVP: 0.043</dd>
                                    <dd>ALPHA_MISSENSE: 0.102</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0677%</dd>
                                    <dd>gnomAD_E_AMR: 0.2286%</dd>
                                    <dd>gnomAD_E_NFE: 0.3658%</dd>
                                    <dd>gnomAD_E_SAS: 0.0261%</dd>
                                    <dd>gnomAD_G_AFR: 0.1149%</dd>
                                    <dd>gnomAD_G_AMR: 0.3538%</dd>
                                    <dd>gnomAD_G_NFE: 0.3435%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RANBP2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000153201">RANBP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.514</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.323 to <a href="http://www.omim.org/entry/608033">Encephalopathy, acute, infection-induced, 3, susceptibility to</a> associated with RANBP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0002445, Tetraplegia</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.514 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RANBP2%0DMATR3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MATR3</a> and phenotypic similarity 0.311 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=803">Amyotrophic lateral sclerosis</a> associated with MATR3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001257, Spasticity</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.514 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RANBP2%0DMATR3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MATR3</a> and phenotypic similarity 0.685 to mouse mutant of MATR3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010418, perimembraneous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/608033">OMIM:608033</a> Encephalopathy, acute, infection-induced, 3, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=88619">ORPHA:88619</a> Familial acute necrotizing encephalopathy - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.514</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.323 to OMIM:608033 Encephalopathy, acute, infection-induced, 3, susceptibility to</dt>
                                
                                    <dt>Phenotypic similarity 0.308 to ORPHA:88619 Familial acute necrotizing encephalopathy</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-109381927-A-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200526519">rs200526519</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000283195.6">RANBP2:ENST00000283195.6:c.4932A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_NFE: 0.0045%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SPATA46" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171722">SPATA46</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.519</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.093</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.519 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SPATA46">SPATA46</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000277, abnormal heart shape</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.519</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.093</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-162344186-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150773743">rs150773743</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.093</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367935.5">SPATA46:ENST00000367935.5:c.438G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000431696.1">SPATA46:ENST00000431696.1:c.316+7192C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0923%</dd>
                                    <dd>gnomAD_E_AMR: 0.4077%</dd>
                                    <dd>gnomAD_E_NFE: 0.1556%</dd>
                                    <dd>gnomAD_E_SAS: 0.0784%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                    <dd>gnomAD_G_AMR: 0.1182%</dd>
                                    <dd>gnomAD_G_NFE: 0.0972%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PCNT" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000160299">PCNT</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.354</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.280</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.603 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2637">Microcephalic osteodysplastic primordial dwarfism type II</a> associated with PCNT.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Phenotypic similarity 0.708 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PCNT">PCNT</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PCNT%0DSDCCAG8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SDCCAG8</a> and phenotypic similarity 0.744 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=110">Bardet-Biedl syndrome</a> associated with SDCCAG8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001627, Abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PCNT%0DSDCCAG8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SDCCAG8</a> and phenotypic similarity 0.360 to mouse mutant of SDCCAG8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000820, abnormal choroid plexus morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000820, abnormal choroid plexus morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/210720">OMIM:210720</a> Microcephalic osteodysplastic primordial dwarfism, type II - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2637">ORPHA:2637</a> Microcephalic osteodysplastic primordial dwarfism type II - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=808">ORPHA:808</a> Seckel syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.354</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.280</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>21-47771436-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199807929">rs199807929</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2075235">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.280</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359568.5">PCNT:ENST00000359568.5:c.1438A&gt;G:p.(Ser480Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.28909883</dd>
                                    
                                    <dd>REVEL: 0.059</dd>
                                    <dd>MVP: 0.289</dd>
                                    <dd>ALPHA_MISSENSE: 0.089</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.2052%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MAML1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000161021">MAML1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.513</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAML1%0DSPEN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SPEN</a> and phenotypic similarity 0.740 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1606">1p36 deletion syndrome</a> associated with SPEN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAML1%0DSPEN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SPEN</a> and phenotypic similarity 0.496 to mouse mutant of SPEN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000267, abnormal heart development</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000267, abnormal heart development</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.513</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>5-179201806-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199901618">rs199901618</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000292599.3">MAML1:ENST00000292599.3:c.2979C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SMO" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000128602">SMO</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.522</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.089</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.508 to <a href="http://www.omim.org/entry/241800">Pallister-Hall-like syndrome</a> associated with SMO.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0030680, Abnormal cardiovascular system morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0030680, Abnormal cardiovascular system morphology</dd></dl><dl><dt>Phenotypic similarity 0.496 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SMO">SMO</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000288, abnormal pericardium morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000288, abnormal pericardium morphology</dd></dl><dl><dt>Phenotypic similarity 0.413 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=SMO">SMO</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0020215, otolith non-functional, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000490, heart malformed, abnormal</dd></dl><dl><dt>Proximity score 0.522 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SMO%0DRAB23&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAB23</a> and phenotypic similarity 0.634 to <a href="http://www.omim.org/entry/201000">Carpenter syndrome</a> associated with RAB23.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001642, Pulmonic stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/241800">OMIM:241800</a> Pallister-Hall-like syndrome - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/601707">OMIM:601707</a> Curry-Jones syndrome, somatic mosaic - somatic</dd>
<dd><a href="http://www.omim.org/entry/605462">OMIM:605462</a> Basal cell carcinoma, somatic - unknown</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1553">ORPHA:1553</a> Curry-Jones syndrome (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2495">ORPHA:2495</a> Meningioma (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=388">ORPHA:388</a> Hirschsprung disease - autosomal dominant/recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.522</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.089</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>7-128848652-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200510336">rs200510336</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/778354">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.089</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000249373.3">SMO:ENST00000249373.3:c.1317G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.5552%</dd>
                                    <dd>gnomAD_G_AMR: 0.5896%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GDPD4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178795">GDPD4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.680</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.680</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-76944123-T-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs116543089">rs116543089</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.680</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000315938.4">GDPD4:ENST00000315938.4:c.1336A&gt;T:p.(Thr446Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376217.2">GDPD4:ENST00000376217.2:c.1336A&gt;T:p.(Thr446Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8420337</dd>
                                    
                                    <dd>REVEL: 0.320</dd>
                                    <dd>MVP: 0.842</dd>
                                    <dd>ALPHA_MISSENSE: 0.462</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.7937%</dd>
                                    <dd>gnomAD_E_AMR: 0.0579%</dd>
                                    <dd>gnomAD_E_SAS: 0.0131%</dd>
                                    <dd>gnomAD_G_AFR: 0.8851%</dd>
                                    <dd>gnomAD_G_AMR: 0.1182%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ITSN2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198399">ITSN2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.512</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.512 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ITSN2">ITSN2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ITSN2%0DDNM2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DNM2</a> and phenotypic similarity 0.346 to <a href="http://www.omim.org/entry/615368">Lethal congenital contracture syndrome 5</a> associated with DNM2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0100309, Subdural hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0100309, Subdural hemorrhage</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ITSN2%0DDNM2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DNM2</a> and phenotypic similarity 0.719 to mouse mutant of DNM2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010418, perimembraneous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ITSN2%0DDNM2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DNM2</a> and phenotypic similarity 0.479 to fish mutant of DNM2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000233, heart decreased functionality, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0007930, myocardium degenerate, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.512</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-24440847-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146263778">rs146263778</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355123.4">ITSN2:ENST00000355123.4:c.3735C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361999.3">ITSN2:ENST00000361999.3:c.3654C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PIPOX" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000179761">PIPOX</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.514</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.473 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PIPOX">PIPOX</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010393, shortened QRS complex duration</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010393, shortened QRS complex duration</dd></dl><dl><dt>Proximity score 0.514 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PIPOX%0DSARDH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SARDH</a> and phenotypic similarity 0.885 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3129">Sarcosinemia</a> associated with SARDH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001642, Pulmonic stenosis</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.514</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-27382878-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs143566431">rs143566431</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000323372.4">PIPOX:ENST00000323372.4:c.1014C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0185%</dd>
                                    <dd>gnomAD_E_AMR: 0.0376%</dd>
                                    <dd>gnomAD_E_NFE: 0.1489%</dd>
                                    <dd>gnomAD_E_SAS: 0.0163%</dd>
                                    <dd>gnomAD_G_AFR: 0.0459%</dd>
                                    <dd>gnomAD_G_NFE: 0.1555%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SPTLC2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100596">SPTLC2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.511</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.511 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPTLC2%0DPSPH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PSPH</a> and phenotypic similarity 0.617 to mouse mutant of PSPH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010418, perimembraneous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010420, muscular ventricular septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/613640">OMIM:613640</a> Neuropathy, hereditary sensory and autonomic, type IC - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=36386">ORPHA:36386</a> Hereditary sensory and autonomic neuropathy type 1 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.511</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>14-78018550-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201717420">rs201717420</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000216484.2">SPTLC2:ENST00000216484.2:c.1192C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLCO4A1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000101187">SLCO4A1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.538</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.069</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.538 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLCO4A1%0DHPN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HPN</a> and phenotypic similarity 0.729 to mouse mutant of HPN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003503, decreased activity of thyroid gland</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.538</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.069</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>20-61287865-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199951467">rs199951467</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.069</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000217159.1">SLCO4A1:ENST00000217159.1:c.59T&gt;C:p.(Met20Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370507.1">SLCO4A1:ENST00000370507.1:c.59T&gt;C:p.(Met20Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.07168673</dd>
                                    
                                    <dd>REVEL: 0.021</dd>
                                    <dd>MVP: 0.072</dd>
                                    <dd>ALPHA_MISSENSE: 0.057</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1330%</dd>
                                    <dd>gnomAD_E_SAS: 0.0038%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BSN" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000164061">BSN</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.110</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BSN%0DRAB3GAP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAB3GAP2</a> and phenotypic similarity 0.753 to <a href="http://www.omim.org/entry/212720">Martsolf syndrome 1</a> associated with RAB3GAP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BSN%0DRAB3GAP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAB3GAP2</a> and phenotypic similarity 0.477 to mouse mutant of RAB3GAP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005333, decreased heart rate</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0005333, decreased heart rate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.110</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-49699447-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199632264">rs199632264</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.110</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000296452.4">BSN:ENST00000296452.4:c.10169C&gt;T:p.(Pro3390Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.11039205</dd>
                                    
                                    <dd>REVEL: 0.084</dd>
                                    <dd>MVP: 0.110</dd>
                                    <dd>ALPHA_MISSENSE: 0.075</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TOLLIP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000078902">TOLLIP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.514</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.092</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.323 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=TOLLIP">TOLLIP</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl><dt>Proximity score 0.514 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TOLLIP%0DRBPMS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RBPMS</a> and phenotypic similarity 0.705 to mouse mutant of RBPMS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002189, abnormal myocardial trabeculae morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.514</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.092</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>11-1317004-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs145957069">rs145957069</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.092</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317204.6">TOLLIP:ENST00000317204.6:c.54G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000525159.1">TOLLIP:ENST00000525159.1:c.54G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000527938.1">TOLLIP:ENST00000527938.1:c.54G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263646.7">TOLLIP:ENST00000263646.7:c.13-43G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000542915.1">TOLLIP:ENST00000542915.1:c.34-5365G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000527886.1">TOLLIP:ENST00000527886.1:c.-154G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1016%</dd>
                                    <dd>gnomAD_E_AMR: 0.1924%</dd>
                                    <dd>gnomAD_E_NFE: 0.0775%</dd>
                                    <dd>gnomAD_E_SAS: 0.0228%</dd>
                                    <dd>gnomAD_G_AFR: 0.0460%</dd>
                                    <dd>gnomAD_G_AMR: 0.4717%</dd>
                                    <dd>gnomAD_G_NFE: 0.0194%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PGR" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000082175">PGR</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.264</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.374</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.433 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PGR">PGR</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004883, abnormal vascular wound healing</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0004883, abnormal vascular wound healing</dd></dl><dl><dt>Phenotypic similarity 0.352 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=PGR">PGR</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0017610, oocyte stage V non-functional, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.529 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PGR%0DSTRA6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to STRA6</a> and phenotypic similarity 0.609 to <a href="http://www.omim.org/entry/601186">Microphthalmia, syndromic 9</a> associated with STRA6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001642, Pulmonic stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.529 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PGR%0DSTRA6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to STRA6</a> and phenotypic similarity 0.320 to mouse mutant of STRA6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005098, abnormal optic choroid morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0005098, abnormal optic choroid morphology</dd></dl><dl><dt>Proximity score 0.529 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PGR%0DSTRA6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to STRA6</a> and phenotypic similarity 0.468 to fish mutant of STRA6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000107, heart morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000724, atrium dilated, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/264080">OMIM:264080</a> ?Progesterone resistance (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.264</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.374</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-100999140-A-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200322178">rs200322178</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3211904">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.374</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263463.5">PGR:ENST00000263463.5:c.662T&gt;A:p.(Val221Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000325455.5">PGR:ENST00000325455.5:c.662T&gt;A:p.(Val221Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000534013.1">PGR:ENST00000534013.1:c.-146+1019T&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3812713</dd>
                                    
                                    <dd>REVEL: 0.053</dd>
                                    <dd>MVP: 0.381</dd>
                                    <dd>ALPHA_MISSENSE: 0.157</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1050%</dd>
                                    <dd>gnomAD_E_AMR: 0.0350%</dd>
                                    <dd>gnomAD_E_NFE: 0.0048%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.1272%</dd>
                                    <dd>gnomAD_G_NFE: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KCNK3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171303">KCNK3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KCNK3%0DGYG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GYG1</a> and phenotypic similarity 0.796 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=263297">Glycogen storage disease with severe cardiomyopathy due to glycogenin deficiency</a> associated with GYG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0005144, Ventricular septal hypertrophy</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KCNK3%0DGYG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GYG1</a> and phenotypic similarity 0.567 to mouse mutant of GYG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002953, thick ventricular wall</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002953, thick ventricular wall</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/615344">OMIM:615344</a> Pulmonary hypertension, primary, 4 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-26950560-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199957242">rs199957242</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1598272">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302909.3">KCNK3:ENST00000302909.3:c.309G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0124%</dd>
                                    <dd>gnomAD_E_SAS: 0.0186%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NFX1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000086102">NFX1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NFX1%0DFKBP1B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FKBP1B</a> and phenotypic similarity 0.661 to mouse mutant of FKBP1B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000231, hypertension</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010724, thick interventricular septum</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>9-33318977-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs144117681">rs144117681</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000318524.6">NFX1:ENST00000318524.6:c.1758G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379521.4">NFX1:ENST00000379521.4:c.1758G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379540.3">NFX1:ENST00000379540.3:c.1758G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0231%</dd>
                                    <dd>gnomAD_E_NFE: 0.0114%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ANGPT2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000091879">ANGPT2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.514</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.090</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.449 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79452">Milroy disease</a> associated with ANGPT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0002624, Abnormal venous morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0002624, Abnormal venous morphology</dd></dl><dl><dt>Phenotypic similarity 0.407 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ANGPT2">ANGPT2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000260, abnormal angiogenesis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000260, abnormal angiogenesis</dd></dl><dl><dt>Phenotypic similarity 0.381 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=ANGPT2">ANGPT2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001192, heart edematous, abnormal</dd></dl><dl><dt>Proximity score 0.514 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ANGPT2%0DFOXC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FOXC2</a> and phenotypic similarity 0.614 to <a href="http://www.omim.org/entry/153400">Lymphedema-distichiasis syndrome</a> associated with FOXC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.514 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ANGPT2%0DFOXC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FOXC2</a> and phenotypic similarity 0.765 to mouse mutant of FOXC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002652, thin myocardium</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/619369">OMIM:619369</a> Lymphatic malformation 10 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79452">ORPHA:79452</a> Milroy disease - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.514</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.090</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.449 to ORPHA:79452 Milroy disease</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>8-6385150-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs144373407">rs144373407</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/735359">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.090</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000325203.5">ANGPT2:ENST00000325203.5:c.492C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000338312.6">ANGPT2:ENST00000338312.6:c.336C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000415216.1">ANGPT2:ENST00000415216.1:c.492C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000523120.1">ANGPT2:ENST00000523120.1:c.492C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344683.5">ANGPT2:ENST00000344683.5:c.2214+27700G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.5723%</dd>
                                    <dd>gnomAD_E_AMR: 0.0434%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.5397%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RAD9A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000172613">RAD9A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.508</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.096</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.492 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RAD9A">RAD9A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RAD9A%0DTREX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TREX1</a> and phenotypic similarity 0.770 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=51">Aicardi-Goutières syndrome</a> associated with TREX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001640, Cardiomegaly</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RAD9A%0DTREX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TREX1</a> and phenotypic similarity 0.526 to mouse mutant of TREX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003141, cardiac fibrosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003141, cardiac fibrosis</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.508</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.096</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>11-67164725-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200201923">rs200201923</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.096</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000307980.2">RAD9A:ENST00000307980.2:c.948T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0124%</dd>
                                    <dd>gnomAD_E_AMR: 0.0550%</dd>
                                    <dd>gnomAD_E_EAS: 0.2013%</dd>
                                    <dd>gnomAD_E_NFE: 0.0309%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_EAS: 0.2567%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NGFR" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000064300">NGFR</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.473 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NGFR">NGFR</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001914, hemorrhage</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NGFR%0DNTF3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NTF3</a> and phenotypic similarity 0.691 to mouse mutant of NTF3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010405, ostium secundum atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-47590176-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201266194">rs201266194</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/717336">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000172229.3">NGFR:ENST00000172229.3:c.1089G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000504201.1">NGFR:ENST00000504201.1:c.807G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0125%</dd>
                                    <dd>gnomAD_E_AMR: 0.0927%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SCRIB" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000180900">SCRIB</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.496 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SCRIB">SCRIB</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000288, abnormal pericardium morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000288, abnormal pericardium morphology</dd></dl><dl><dt>Phenotypic similarity 0.277 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=SCRIB">SCRIB</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0002734, cranial vasculature malformed, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0002734, cranial vasculature malformed, abnormal</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SCRIB%0DVANGL2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to VANGL2</a> and phenotypic similarity 0.654 to mouse mutant of VANGL2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000267, abnormal heart development</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SCRIB%0DVANGL2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to VANGL2</a> and phenotypic similarity 0.536 to fish mutant of VANGL2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0006965, Kupffer's vesicle decreased functionality, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0013833, cardiac muscle cell morphology, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>8-144893383-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201834512">rs201834512</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000320476.3">SCRIB:ENST00000320476.3:c.1039C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356994.2">SCRIB:ENST00000356994.2:c.1039C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377533.3">SCRIB:ENST00000377533.3:c.796C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_EAS: 0.0055%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GTF2H2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000145736">GTF2H2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GTF2H2%0DPHKG2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PHKG2</a> and phenotypic similarity 0.744 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=264580">Glycogen storage disease due to liver phosphorylase kinase deficiency</a> associated with PHKG2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>5-70338021-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199781191">rs199781191</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000274400.5">GTF2H2:ENST00000274400.5:c.912G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000330280.7">GTF2H2:ENST00000330280.7:c.912G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000425596.2">GTF2H2:ENST00000425596.2:c.741G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000517900.1">GTF2H2:ENST00000517900.1:c.365-6510G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SSH1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000084112">SSH1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SSH1%0DPRKD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKD1</a> and phenotypic similarity 0.749 to <a href="http://www.omim.org/entry/617364">Congenital heart defects and ectodermal dysplasia</a> associated with PRKD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0006695, Atrioventricular canal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0006695, Atrioventricular canal defect</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SSH1%0DPRKD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKD1</a> and phenotypic similarity 0.315 to fish mutant of PRKD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0008997, lymphangioblast cord malformed, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0008997, lymphangioblast cord malformed, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>12-109182346-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146128442">rs146128442</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000326495.5">SSH1:ENST00000326495.5:c.2568C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360239.3">SSH1:ENST00000360239.3:c.1632C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0694%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_EAS: 0.0220%</dd>
                                    <dd>gnomAD_E_NFE: 0.0046%</dd>
                                    <dd>gnomAD_E_SAS: 0.0208%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BRIP1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136492">BRIP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.512</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.090</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.844 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with BRIP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.512 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BRIP1%0DPALB2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PALB2</a> and phenotypic similarity 0.844 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with PALB2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.512 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BRIP1%0DPALB2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PALB2</a> and phenotypic similarity 0.386 to mouse mutant of PALB2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003229, abnormal vitelline vasculature morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003229, abnormal vitelline vasculature morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/114480">OMIM:114480</a> Breast cancer, early-onset, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/609054">OMIM:609054</a> Fanconi anemia, complementation group J - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=145">ORPHA:145</a> Hereditary breast and/or ovarian cancer syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">ORPHA:84</a> Fanconi anemia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.512</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.090</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-59761308-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202228407">rs202228407</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/184008">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.090</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000259008.2">BRIP1:ENST00000259008.2:c.3099T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.5464%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RALBP1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000017797">RALBP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RALBP1%0DRRAS2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RRAS2</a> and phenotypic similarity 0.839 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=648">Noonan syndrome</a> associated with RRAS2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RALBP1%0DRRAS2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RRAS2</a> and phenotypic similarity 0.303 to mouse mutant of RRAS2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002792, abnormal retina vasculature morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002792, abnormal retina vasculature morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>18-9535836-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201580280">rs201580280</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000019317.4">RALBP1:ENST00000019317.4:c.1869C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000383432.3">RALBP1:ENST00000383432.3:c.1869C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0702%</dd>
                                    <dd>gnomAD_E_NFE: 0.0082%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="F5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198734">F5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.411 to <a href="http://www.omim.org/entry/601367">Stroke, ischemic, susceptibility to</a> associated with F5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001297, Stroke</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001297, Stroke</dd></dl><dl><dt>Phenotypic similarity 0.483 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=F5">F5</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001914, hemorrhage</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=F5%0DLEFTY2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LEFTY2</a> and phenotypic similarity 0.678 to mouse mutant of LEFTY2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=F5%0DLEFTY2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LEFTY2</a> and phenotypic similarity 0.336 to fish mutant of LEFTY2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0010508, heart primordium fused with left side right side heart primordium, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0010508, heart primordium fused with left side right side heart primordium, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/188055">OMIM:188055</a> Thrombophilia 2 due to activated protein C resistance - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/188055">OMIM:188055</a> Thrombophilia, susceptibility to, due to factor V Leiden (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/227400">OMIM:227400</a> Factor V deficiency - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/600880">OMIM:600880</a> Budd-Chiari syndrome (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/601367">OMIM:601367</a> Stroke, ischemic, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/614389">OMIM:614389</a> Pregnancy loss, recurrent, susceptibility to, 1 (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=131">ORPHA:131</a> Budd-Chiari syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=326">ORPHA:326</a> Congenital factor V deficiency - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-169509687-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs138504020">rs138504020</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/711302">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367796.3">F5:ENST00000367796.3:c.4656T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367797.3">F5:ENST00000367797.3:c.4641T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0318%</dd>
                                    <dd>gnomAD_E_NFE: 0.0238%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PRORP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100890">PRORP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.669</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.345 to <a href="http://www.omim.org/entry/619737">Combined oxidative phosphorylation deficiency 54</a> associated with PRORP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000815, Hypergonadotropic hypogonadism</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/619737">OMIM:619737</a> Combined oxidative phosphorylation deficiency 54 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.669</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-35593137-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs45626032">rs45626032</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3043006">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.669</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000250377.7">PRORP:ENST00000250377.7:c.401C&gt;T:p.(Ala134Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000321130.10">PRORP:ENST00000321130.10:c.686C&gt;T:p.(Ala229Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000534898.4">PRORP:ENST00000534898.4:c.686C&gt;T:p.(Ala229Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000603544.1">PRORP:ENST00000603544.1:c.686C&gt;T:p.(Ala229Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000604948.1">PRORP:ENST00000604948.1:c.401C&gt;T:p.(Ala134Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000605870.1">PRORP:ENST00000605870.1:c.-131+1021C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.77952904</dd>
                                    
                                    <dd>REVEL: 0.367</dd>
                                    <dd>MVP: 0.780</dd>
                                    <dd>ALPHA_MISSENSE: 0.382</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0437%</dd>
                                    <dd>gnomAD_E_AMR: 0.1359%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.1629%</dd>
                                    <dd>gnomAD_E_SAS: 0.7154%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.1620%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SSTR3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000278195">SSTR3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SSTR3%0DMPDZ&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MPDZ</a> and phenotypic similarity 0.619 to <a href="http://www.omim.org/entry/615219">Hydrocephalus, congenital, 2, with or without brain or eye anomalies</a> associated with MPDZ.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0030718, Right atrial enlargement</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SSTR3%0DMPDZ&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MPDZ</a> and phenotypic similarity 0.424 to mouse mutant of MPDZ.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002833, increased heart weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002833, increased heart weight</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>22-37603213-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199668206">rs199668206</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000328544.3">SSTR3:ENST00000328544.3:c.630G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000402501.1">SSTR3:ENST00000402501.1:c.630G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LRP2BP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000109771">LRP2BP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRP2BP%0DLRP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRP2</a> and phenotypic similarity 0.651 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2143">Donnai-Barrow syndrome</a> associated with LRP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRP2BP%0DLRP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRP2</a> and phenotypic similarity 0.719 to mouse mutant of LRP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004056, abnormal myocardium compact layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRP2BP%0DLRP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRP2</a> and phenotypic similarity 0.487 to fish mutant of LRP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0007443, visual system decreased functionality, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>4-186288388-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs138661740">rs138661740</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000328559.7">LRP2BP:ENST00000328559.7:c.990G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000362004.3">LRP2BP:ENST00000362004.3:c.996G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000505916.1">LRP2BP:ENST00000505916.1:c.990G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000510776.1">LRP2BP:ENST00000510776.1:c.912G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="POLA2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000014138">POLA2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=POLA2%0DDNA2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DNA2</a> and phenotypic similarity 0.626 to <a href="http://www.omim.org/entry/615156">Progressive external ophthalmoplegia with mitochondrial DNA deletions, autosomal dominant 6</a> associated with DNA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001655, Patent foramen ovale</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001655, Patent foramen ovale</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>11-65064656-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201640265">rs201640265</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265465.3">POLA2:ENST00000265465.3:c.1713C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0185%</dd>
                                    <dd>gnomAD_E_AMR: 0.1446%</dd>
                                    <dd>gnomAD_E_NFE: 0.0035%</dd>
                                    <dd>gnomAD_E_SAS: 0.0098%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TNFRSF10C" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000173535">TNFRSF10C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TNFRSF10C%0DFADD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FADD</a> and phenotypic similarity 0.604 to <a href="http://www.omim.org/entry/613759">Immunodeficiency 90 with encephalopathy, functional hyposplenia, and hepatic dysfunction</a> associated with FADD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TNFRSF10C%0DFADD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FADD</a> and phenotypic similarity 0.554 to mouse mutant of FADD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005329, abnormal myocardium layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002652, thin myocardium</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>8-22974370-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs151239314">rs151239314</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356864.3">TNFRSF10C:ENST00000356864.3:c.606G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540813.1">TNFRSF10C:ENST00000540813.1:c.300G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0111%</dd>
                                    <dd>gnomAD_E_EAS: 0.0080%</dd>
                                    <dd>gnomAD_E_NFE: 0.0020%</dd>
                                    <dd>gnomAD_E_SAS: 0.0040%</dd>
                                    <dd>gnomAD_G_NFE: 0.0072%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CENPO" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000138092">CENPO</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.455 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CENPO">CENPO</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002833, increased heart weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002833, increased heart weight</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CENPO%0DCENPE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CENPE</a> and phenotypic similarity 0.823 to <a href="http://www.omim.org/entry/616051">?Microcephaly 13, primary, autosomal recessive</a> associated with CENPE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001723, Restrictive cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001723, Restrictive cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-25038499-A-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201226760">rs201226760</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000260662.1">CENPO:ENST00000260662.1:c.468A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380834.2">CENPO:ENST00000380834.2:c.468A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000473706.1">CENPO:ENST00000473706.1:c.450A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0260%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PTPN7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143851">PTPN7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PTPN7%0DMAPK7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MAPK7</a> and phenotypic similarity 0.747 to mouse mutant of MAPK7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004056, abnormal myocardium compact layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-202127273-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201226697">rs201226697</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308986.5">PTPN7:ENST00000308986.5:c.279C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000309017.3">PTPN7:ENST00000309017.3:c.594C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367279.4">PTPN7:ENST00000367279.4:c.396C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544762.1">PTPN7:ENST00000544762.1:c.-69C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543735.1">PTPN7:ENST00000543735.1:c.-207-322C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0031%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RGPD8" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169629">RGPD8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.101</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RGPD8%0DPRKCZ&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKCZ</a> and phenotypic similarity 0.740 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1606">1p36 deletion syndrome</a> associated with PRKCZ.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RGPD8%0DPRKCZ&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKCZ</a> and phenotypic similarity 0.272 to fish mutant of PRKCZ.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0008757, intersegmental vessel has extra parts of type filopodium endothelial tip cell, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0008757, intersegmental vessel has extra parts of type filopodium endothelial tip cell, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.101</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-113152295-C-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200687738">rs200687738</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PM2_Supporting, BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.101</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302558.3">RGPD8:ENST00000302558.3:c.2502G&gt;C:p.(Lys834Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000330575.5">RGPD8:ENST00000330575.5:c.2502G&gt;C:p.(Lys834Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409750.1">RGPD8:ENST00000409750.1:c.2082G&gt;C:p.(Lys694Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1007</dd>
                                    
                                    <dd>REVEL: 0.031</dd>
                                    <dd>MVP: 0.014</dd>
                                    <dd>ALPHA_MISSENSE: 0.101</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>2-113152247-C-T [0|1]</b>
                        
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.100</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302558.3">RGPD8:ENST00000302558.3:c.2550G&gt;A:p.(=)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000330575.5">RGPD8:ENST00000330575.5:c.2550G&gt;A:p.(=)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409750.1">RGPD8:ENST00000409750.1:c.2130G&gt;A:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ITPR3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000096433">ITPR3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.094</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ITPR3%0DANK1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANK1</a> and phenotypic similarity 0.823 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=822">Hereditary spherocytosis</a> associated with ANK1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001723, Restrictive cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001723, Restrictive cardiomyopathy</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ITPR3%0DANK1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANK1</a> and phenotypic similarity 0.488 to mouse mutant of ANK1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000274, enlarged heart</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000274, enlarged heart</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/222100">OMIM:222100</a> Diabetes, type 1, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/620111">OMIM:620111</a> Charcot-Marie-Tooth disease, demyelinating, type 1J - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.094</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>6-33632922-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs138821995">rs138821995</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/713893">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.094</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374316.5">ITPR3:ENST00000374316.5:c.1341C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000605930.1">ITPR3:ENST00000605930.1:c.1341C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0923%</dd>
                                    <dd>gnomAD_E_AMR: 0.0925%</dd>
                                    <dd>gnomAD_E_NFE: 0.3676%</dd>
                                    <dd>gnomAD_E_SAS: 0.0229%</dd>
                                    <dd>gnomAD_G_AFR: 0.0688%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.3502%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BMP8A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183682">BMP8A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BMP8A%0DBMPR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BMPR2</a> and phenotypic similarity 0.501 to <a href="http://www.omim.org/entry/178600">Pulmonary hypertension, familial primary, 1, with or without HHT</a> associated with BMPR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001667, Right ventricular hypertrophy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001667, Right ventricular hypertrophy</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BMP8A%0DBMPR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BMPR2</a> and phenotypic similarity 0.721 to mouse mutant of BMPR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004056, abnormal myocardium compact layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010420, muscular ventricular septal defect</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BMP8A%0DBMPR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BMPR2</a> and phenotypic similarity 0.403 to fish mutant of BMPR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001192, heart edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-39987190-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs115627781">rs115627781</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000331593.5">BMP8A:ENST00000331593.5:c.816A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0698%</dd>
                                    <dd>gnomAD_E_NFE: 0.0049%</dd>
                                    <dd>gnomAD_E_SAS: 0.0361%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="COL5A3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000080573">COL5A3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.095</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=COL5A3%0DCOL7A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COL7A1</a> and phenotypic similarity 0.823 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79408">Autosomal recessive generalized dystrophic epidermolysis bullosa, severe form</a> associated with COL7A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.095</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-10089265-C-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs188093166">rs188093166</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.095</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264828.3">COL5A3:ENST00000264828.3:c.3003G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.3046%</dd>
                                    <dd>gnomAD_G_AMR: 0.2364%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="EHD3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000013016">EHD3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EHD3%0DCAV1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAV1</a> and phenotypic similarity 0.899 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=528">Congenital generalized lipodystrophy</a> associated with CAV1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EHD3%0DCAV1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAV1</a> and phenotypic similarity 0.655 to mouse mutant of CAV1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003548, pulmonary hypertension</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010725, thin interventricular septum</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EHD3%0DCAV1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAV1</a> and phenotypic similarity 0.412 to fish mutant of CAV1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001192, heart edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-31472288-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs61749077">rs61749077</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322054.5">EHD3:ENST00000322054.5:c.456C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541626.1">EHD3:ENST00000541626.1:c.456C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.0035%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SMG6" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000070366">SMG6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SMG6%0DTERT&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TERT</a> and phenotypic similarity 0.823 to <a href="http://www.omim.org/entry/613989">Dyskeratosis congenita, autosomal dominant 2</a> associated with TERT.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SMG6%0DTERT&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TERT</a> and phenotypic similarity 0.302 to fish mutant of TERT.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0003232, pericardial cavity edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0003232, pericardial cavity edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-2202520-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202060632">rs202060632</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263073.6">SMG6:ENST00000263073.6:c.1527T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544865.1">SMG6:ENST00000544865.1:c.1434T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0087%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="IP6K1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000176095">IP6K1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=IP6K1%0DKLK2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KLK2</a> and phenotypic similarity 0.662 to mouse mutant of KLK2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001544, abnormal cardiovascular system physiology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010725, thin interventricular septum</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>3-49785297-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200393807">rs200393807</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000321599.4">IP6K1:ENST00000321599.4:c.177C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000468463.1">IP6K1:ENST00000468463.1:c.177C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395238.1">IP6K1:ENST00000395238.1:c.-272-9442C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000460540.1">IP6K1:ENST00000460540.1:c.-272-9442C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MFHAS1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000147324">MFHAS1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MFHAS1%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.603 to <a href="http://www.omim.org/entry/255960">Myxoma, intracardiac</a> associated with PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0006689, Bacterial endocarditis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0006691, Pulmonic valve myxoma</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MFHAS1%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.473 to mouse mutant of PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003929, decreased heart rate variability</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003929, decreased heart rate variability</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>8-8748850-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200026075">rs200026075</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000276282.6">MFHAS1:ENST00000276282.6:c.1719C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0116%</dd>
                                    <dd>gnomAD_E_NFE: 0.0195%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="OR4L1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000176246">OR4L1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR4L1%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.782 to <a href="http://www.omim.org/entry/608068">Neutrophilic dermatosis, acute febrile</a> associated with MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>14-20528467-C-T [0|1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000315683.1">OR4L1:ENST00000315683.1:c.264C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZSWIM4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000132003">ZSWIM4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.666</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.666</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-13915887-A-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199644245">rs199644245</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2364799">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.666</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000254323.2">ZSWIM4:ENST00000254323.2:c.637A&gt;C:p.(Thr213Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000440752.2">ZSWIM4:ENST00000440752.2:c.-213A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6774</dd>
                                    
                                    <dd>REVEL: 0.357</dd>
                                    <dd>MVP: 0.597</dd>
                                    <dd>ALPHA_MISSENSE: 0.677</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0723%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BHMT2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000132840">BHMT2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BHMT2%0DAHCY&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AHCY</a> and phenotypic similarity 0.837 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=88618">S-adenosylhomocysteine hydrolase deficiency</a> associated with AHCY.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BHMT2%0DAHCY&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AHCY</a> and phenotypic similarity 0.428 to mouse mutant of AHCY.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001613, abnormal vasodilation</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001613, abnormal vasodilation</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BHMT2%0DAHCY&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AHCY</a> and phenotypic similarity 0.427 to fish mutant of AHCY.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0005663, gall bladder decreased functionality, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>5-78373413-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201966732">rs201966732</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000255192.3">BHMT2:ENST00000255192.3:c.144A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000521567.1">BHMT2:ENST00000521567.1:c.144A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0087%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TRHDE" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000072657">TRHDE</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRHDE%0DNEUROD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NEUROD2</a> and phenotypic similarity 0.618 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1934">Early infantile epileptic encephalopathy</a> associated with NEUROD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>12-72666702-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200824580">rs200824580</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261180.4">TRHDE:ENST00000261180.4:c.144C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0033%</dd>
                                    <dd>gnomAD_E_NFE: 0.0011%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PARD6G" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178184">PARD6G</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.486 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=PARD6G">PARD6G</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001450, myocardium malformed, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001450, myocardium malformed, abnormal</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PARD6G%0DPRKCZ&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKCZ</a> and phenotypic similarity 0.740 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1606">1p36 deletion syndrome</a> associated with PRKCZ.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PARD6G%0DPRKCZ&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKCZ</a> and phenotypic similarity 0.272 to fish mutant of PRKCZ.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0008757, intersegmental vessel has extra parts of type filopodium endothelial tip cell, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0008757, intersegmental vessel has extra parts of type filopodium endothelial tip cell, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>18-77918074-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199834037">rs199834037</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000353265.3">PARD6G:ENST00000353265.3:c.711C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000585422.1">PARD6G:ENST00000585422.1:c.97-8628G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000586421.1">PARD6G:ENST00000586421.1:c.97-2325G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000587254.1">PARD6G:ENST00000587254.1:c.97-2325G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589574.1">PARD6G:ENST00000589574.1:c.96+11937G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.1666%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ATP9B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000166377">ATP9B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ATP9B%0DRIPPLY3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RIPPLY3</a> and phenotypic similarity 0.632 to mouse mutant of RIPPLY3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010402, ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>18-77013422-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199718576">rs199718576</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000307671.7">ATP9B:ENST00000307671.7:c.1149G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000426216.2">ATP9B:ENST00000426216.2:c.1149G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0146%</dd>
                                    <dd>gnomAD_E_NFE: 0.0026%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FRMD8" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000126391">FRMD8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FRMD8%0DLRP5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRP5</a> and phenotypic similarity 0.651 to <a href="http://www.omim.org/entry/259770">Osteoporosis-pseudoglioma syndrome</a> associated with LRP5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FRMD8%0DLRP5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRP5</a> and phenotypic similarity 0.476 to mouse mutant of LRP5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001914, hemorrhage</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>11-65156839-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142184687">rs142184687</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2641955">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317568.5">FRMD8:ENST00000317568.5:c.93C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000416776.2">FRMD8:ENST00000416776.2:c.93C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355991.5">FRMD8:ENST00000355991.5:c.85+2247C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0839%</dd>
                                    <dd>gnomAD_E_NFE: 0.1494%</dd>
                                    <dd>gnomAD_G_NFE: 0.1428%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CD63" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135404">CD63</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.516</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.081</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.516 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CD63%0DLAMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LAMP2</a> and phenotypic similarity 0.964 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=34587">Glycogen storage disease due to LAMP-2 deficiency</a> associated with LAMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dl><dt>Proximity score 0.516 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CD63%0DLAMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LAMP2</a> and phenotypic similarity 0.534 to mouse mutant of LAMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001544, abnormal cardiovascular system physiology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0021157, increased heart left ventricle wall thickness</dd></dl><dl><dt>Proximity score 0.516 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CD63%0DLAMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LAMP2</a> and phenotypic similarity 0.522 to fish mutant of LAMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0138138, trabecular layer has extra parts of type cardiac muscle cell, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0103127, atrium increased area, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.516</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.081</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>12-56122084-G-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs3138131">rs3138131</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.081</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000257857.4">CD63:ENST00000257857.4:c.42C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000420846.3">CD63:ENST00000420846.3:c.42C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000549117.1">CD63:ENST00000549117.1:c.42C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000552692.1">CD63:ENST00000552692.1:c.42C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000552754.1">CD63:ENST00000552754.1:c.42C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1181%</dd>
                                    <dd>gnomAD_E_AMR: 0.4687%</dd>
                                    <dd>gnomAD_E_NFE: 0.8622%</dd>
                                    <dd>gnomAD_E_SAS: 0.3517%</dd>
                                    <dd>gnomAD_G_AFR: 0.1493%</dd>
                                    <dd>gnomAD_G_AMR: 0.7075%</dd>
                                    <dd>gnomAD_G_NFE: 0.7073%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NOLC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000166197">NOLC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.323 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=NOLC1">NOLC1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NOLC1%0DPOLR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POLR1A</a> and phenotypic similarity 0.777 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1200">Burn-McKeown syndrome</a> associated with POLR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001671, Abnormal cardiac septum morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NOLC1%0DPOLR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POLR1A</a> and phenotypic similarity 0.323 to fish mutant of POLR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>10-103916985-A-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199809715">rs199809715</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405356.1">NOLC1:ENST00000405356.1:c.216A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000488254.2">NOLC1:ENST00000488254.2:c.219A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000605788.1">NOLC1:ENST00000605788.1:c.216A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000603742.1">NOLC1:ENST00000603742.1:c.-510+154A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1185%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MACC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183742">MACC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MACC1%0DSATB2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SATB2</a> and phenotypic similarity 0.617 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=251028">SATB2-associated syndrome due to a chromosomal rearrangement</a> associated with SATB2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>7-20199486-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146810957">rs146810957</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000332878.4">MACC1:ENST00000332878.4:c.498C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400331.5">MACC1:ENST00000400331.5:c.498C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589011.1">MACC1:ENST00000589011.1:c.498C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0185%</dd>
                                    <dd>gnomAD_E_AMR: 0.0087%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.0425%</dd>
                                    <dd>gnomAD_E_SAS: 0.0131%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_NFE: 0.0194%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="STAT6" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000166888">STAT6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.092</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=STAT6%0DPKD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKD1</a> and phenotypic similarity 0.501 to <a href="http://www.omim.org/entry/173900">Polycystic kidney disease 1</a> associated with PKD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001634, Mitral valve prolapse</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001634, Mitral valve prolapse</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=STAT6%0DPKD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKD1</a> and phenotypic similarity 0.708 to mouse mutant of PKD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002189, abnormal myocardial trabeculae morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=STAT6%0DPKD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKD1</a> and phenotypic similarity 0.413 to fish mutant of PKD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001192, heart edematous, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/620532">OMIM:620532</a> Hyper-IgE syndrome 6, autosomal dominant, with recurrent infections - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2126">ORPHA:2126</a> Solitary fibrous tumor (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.092</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>12-57493602-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs35182390">rs35182390</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/719355">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.092</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000300134.3">STAT6:ENST00000300134.3:c.1692C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454075.3">STAT6:ENST00000454075.3:c.1692C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537215.2">STAT6:ENST00000537215.2:c.1362C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538913.2">STAT6:ENST00000538913.2:c.1362C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543873.2">STAT6:ENST00000543873.2:c.1692C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556155.1">STAT6:ENST00000556155.1:c.1692C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0554%</dd>
                                    <dd>gnomAD_E_AMR: 0.1794%</dd>
                                    <dd>gnomAD_E_EAS: 0.1794%</dd>
                                    <dd>gnomAD_E_NFE: 0.4539%</dd>
                                    <dd>gnomAD_E_SAS: 0.1928%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_EAS: 0.0642%</dd>
                                    <dd>gnomAD_G_NFE: 0.4796%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MAP4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000047849">MAP4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAP4%0DSMAD6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMAD6</a> and phenotypic similarity 0.581 to <a href="http://www.omim.org/entry/614823">Aortic valve disease 2</a> associated with SMAD6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001642, Pulmonic stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001655, Patent foramen ovale</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAP4%0DSMAD6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMAD6</a> and phenotypic similarity 0.701 to mouse mutant of SMAD6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010418, perimembraneous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010420, muscular ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>3-47898943-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150980381">rs150980381</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264724.11">MAP4:ENST00000264724.11:c.2331G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360240.6">MAP4:ENST00000360240.6:c.3126G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000383737.4">MAP4:ENST00000383737.4:c.2103G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395734.3">MAP4:ENST00000395734.3:c.3126G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000420772.2">MAP4:ENST00000420772.2:c.2205G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000426837.2">MAP4:ENST00000426837.2:c.6561G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441748.2">MAP4:ENST00000441748.2:c.468G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0723%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GGPS1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000152904">GGPS1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.381</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.296 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GGPS1">GGPS1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010097, abnormal retina blood vessel morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010097, abnormal retina blood vessel morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GGPS1%0DCOQ2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COQ2</a> and phenotypic similarity 0.831 to <a href="http://www.omim.org/entry/607426">Coenzyme Q10 deficiency, primary, 1</a> associated with COQ2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GGPS1%0DCOQ2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COQ2</a> and phenotypic similarity 0.460 to mouse mutant of COQ2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002834, decreased heart weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002834, decreased heart weight</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/619518">OMIM:619518</a> Muscular dystrophy, congenital hearing loss, and ovarian insufficiency syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.381</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-235505849-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146421372">rs146421372</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.381</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000282841.5">GGPS1:ENST00000282841.5:c.665G&gt;C:p.(Trp222Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358966.2">GGPS1:ENST00000358966.2:c.665G&gt;C:p.(Trp222Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391855.2">GGPS1:ENST00000391855.2:c.503G&gt;C:p.(Trp168Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000476121.1">GGPS1:ENST00000476121.1:c.665G&gt;C:p.(Trp222Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000488594.1">GGPS1:ENST00000488594.1:c.665G&gt;C:p.(Trp222Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.40424877</dd>
                                    
                                    <dd>REVEL: 0.342</dd>
                                    <dd>MVP: 0.404</dd>
                                    <dd>ALPHA_MISSENSE: 0.316</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0318%</dd>
                                    <dd>gnomAD_E_NFE: 0.0290%</dd>
                                    <dd>gnomAD_E_SAS: 0.1143%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.3538%</dd>
                                    <dd>gnomAD_G_NFE: 0.0259%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SH3TC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000125089">SH3TC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.472 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SH3TC1">SH3TC1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003233, prolonged QT interval</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003233, prolonged QT interval</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SH3TC1%0DSSR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SSR2</a> and phenotypic similarity 0.651 to mouse mutant of SSR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004057, thin myocardium compact layer</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010420, muscular ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>4-8229551-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201128171">rs201128171</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245105.3">SH3TC1:ENST00000245105.3:c.2130C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539824.1">SH3TC1:ENST00000539824.1:c.1902C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0956%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LRP4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000134569">LRP4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.556 to <a href="http://www.omim.org/entry/212780">Cenani-Lenz syndactyly syndrome</a> associated with LRP4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001642, Pulmonic stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001642, Pulmonic stenosis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRP4%0DMEOX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEOX1</a> and phenotypic similarity 0.651 to <a href="http://www.omim.org/entry/214300">Klippel-Feil syndrome 2</a> associated with MEOX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRP4%0DMEOX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEOX1</a> and phenotypic similarity 0.255 to fish mutant of MEOX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0004056, posterior cardinal vein increased diameter, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0004056, posterior cardinal vein increased diameter, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/212780">OMIM:212780</a> Cenani-Lenz syndactyly syndrome - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/614305">OMIM:614305</a> Sclerosteosis 2 - autosomal dominant/recessive</dd>
<dd><a href="http://www.omim.org/entry/616304">OMIM:616304</a> ?Myasthenic syndrome, congenital, 17 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3152">ORPHA:3152</a> Sclerosteosis - autosomal dominant/recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3258">ORPHA:3258</a> Cenani-Lenz syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98913">ORPHA:98913</a> Postsynaptic congenital myasthenic syndromes - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.315 to OMIM:614305 Sclerosteosis 2</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>11-46911047-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199674492">rs199674492</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/467783">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378623.1">LRP4:ENST00000378623.1:c.2130G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                    <dd>gnomAD_E_NFE: 0.0062%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="OR51T1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000176900">OR51T1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR51T1%0DH4C9&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to H4C9</a> and phenotypic similarity 0.618 to <a href="http://www.omim.org/entry/619951">Tessadori-Bicknell-van Haaften neurodevelopmental syndrome 4</a> associated with H4C9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001684, Secundum atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001684, Secundum atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>11-4903811-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs149770343">rs149770343</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322049.1">OR51T1:ENST00000322049.1:c.682C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380378.1">OR51T1:ENST00000380378.1:c.763C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380390.1">OR51T1:ENST00000380390.1:c.-144-105487C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0123%</dd>
                                    <dd>gnomAD_E_AMR: 0.1129%</dd>
                                    <dd>gnomAD_E_NFE: 0.0617%</dd>
                                    <dd>gnomAD_G_NFE: 0.0324%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="OR5F1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000149133">OR5F1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR5F1%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.782 to <a href="http://www.omim.org/entry/608068">Neutrophilic dermatosis, acute febrile</a> associated with MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>11-55761736-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200555184">rs200555184</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000278409.1">OR5F1:ENST00000278409.1:c.366G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0318%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SHD" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105251">SHD</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.096</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SHD%0DABL1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ABL1</a> and phenotypic similarity 0.627 to <a href="http://www.omim.org/entry/617602">Congenital heart defects and skeletal malformations syndrome</a> associated with ABL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SHD%0DABL1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ABL1</a> and phenotypic similarity 0.663 to mouse mutant of ABL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004056, abnormal myocardium compact layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010724, thick interventricular septum</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.096</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-4288258-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200878149">rs200878149</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.096</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543264.2">SHD:ENST00000543264.2:c.735G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000599689.1">SHD:ENST00000599689.1:c.717-2186G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.2401%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="POLR3B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000013503">POLR3B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.088</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.352 to <a href="http://www.omim.org/entry/614381">Leukodystrophy, hypomyelinating, 8, with or without oligodontia and/or hypogonadotropic hypogonadism</a> associated with POLR3B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000044, Hypogonadotropic hypogonadism</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=POLR3B%0DNFIX&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NFIX</a> and phenotypic similarity 0.648 to <a href="http://www.omim.org/entry/602535">Marshall-Smith syndrome</a> associated with NFIX.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/614381">OMIM:614381</a> Leukodystrophy, hypomyelinating, 8, with or without oligodontia and/or hypogonadotropic hypogonadism - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/619742">OMIM:619742</a> Charcot-Marie-Tooth disease, demyelinating, type 1I - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=88637">ORPHA:88637</a> Hypomyelination-hypogonadotropic hypogonadism-hypodontia syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.088</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.316 to OMIM:619742 Charcot-Marie-Tooth disease, demyelinating, type 1I</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>12-106820986-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs141885523">rs141885523</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/306933">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.088</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000228347.4">POLR3B:ENST00000228347.4:c.1113T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539066.1">POLR3B:ENST00000539066.1:c.939T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.6180%</dd>
                                    <dd>gnomAD_G_AMR: 0.3589%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC35C2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000080189">SLC35C2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.096</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC35C2%0DMGAT2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MGAT2</a> and phenotypic similarity 0.655 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79329">MGAT2-CDG</a> associated with MGAT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001627, Abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC35C2%0DMGAT2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MGAT2</a> and phenotypic similarity 0.449 to mouse mutant of MGAT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000465, gastrointestinal hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000465, gastrointestinal hemorrhage</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.096</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>20-44983845-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201199146">rs201199146</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.096</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000243896.2">SLC35C2:ENST00000243896.2:c.444C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372227.1">SLC35C2:ENST00000372227.1:c.444C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372229.1">SLC35C2:ENST00000372229.1:c.45C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372230.5">SLC35C2:ENST00000372230.5:c.444C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543605.1">SLC35C2:ENST00000543605.1:c.531C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317734.8">SLC35C2:ENST00000317734.8:c.409-28C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0248%</dd>
                                    <dd>gnomAD_E_AMR: 0.2635%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.0345%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="OR52K2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000181963">OR52K2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.096</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR52K2%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.782 to <a href="http://www.omim.org/entry/608068">Neutrophilic dermatosis, acute febrile</a> associated with MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.096</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>11-4471139-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150771960">rs150771960</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.096</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000325719.4">OR52K2:ENST00000325719.4:c.570G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0123%</dd>
                                    <dd>gnomAD_E_AMR: 0.2428%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.0176%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DOC2A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000149927">DOC2A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.096</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DOC2A%0DRAB3GAP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAB3GAP2</a> and phenotypic similarity 0.753 to <a href="http://www.omim.org/entry/212720">Martsolf syndrome 1</a> associated with RAB3GAP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DOC2A%0DRAB3GAP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAB3GAP2</a> and phenotypic similarity 0.477 to mouse mutant of RAB3GAP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005333, decreased heart rate</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0005333, decreased heart rate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.096</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>16-30018617-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199801773">rs199801773</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.096</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000350119.4">DOC2A:ENST00000350119.4:c.531C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000564944.1">DOC2A:ENST00000564944.1:c.531C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000564979.1">DOC2A:ENST00000564979.1:c.531C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.2748%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NLRP3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162711">NLRP3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.095</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.457 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=575">Muckle-Wells syndrome</a> associated with NLRP3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0002633, Vasculitis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0002633, Vasculitis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NLRP3%0DFLII&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FLII</a> and phenotypic similarity 0.837 to <a href="http://www.omim.org/entry/620635">Cardiomyopathy, dilated, 2J</a> associated with FLII.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001684, Secundum atrial septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NLRP3%0DFLII&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FLII</a> and phenotypic similarity 0.415 to fish mutant of FLII.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0008853, fast muscle cell myotome decreased functionality, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/120100">OMIM:120100</a> Familial cold inflammatory syndrome 1 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/148200">OMIM:148200</a> Keratoendothelitis fugax hereditaria - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/191900">OMIM:191900</a> Muckle-Wells syndrome - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/607115">OMIM:607115</a> CINCA syndrome - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/617772">OMIM:617772</a> Deafness, autosomal dominant 34, with or without inflammation - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1451">ORPHA:1451</a> CINCA syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=47045">ORPHA:47045</a> Familial cold urticaria - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=575">ORPHA:575</a> Muckle-Wells syndrome - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.095</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.457 to ORPHA:575 Muckle-Wells syndrome</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-247592950-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs183128734">rs183128734</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/296950">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.095</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336119.3">NLRP3:ENST00000336119.3:c.2220C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366496.2">NLRP3:ENST00000366496.2:c.2220C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366497.2">NLRP3:ENST00000366497.2:c.2220C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391828.3">NLRP3:ENST00000391828.3:c.2220C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000348069.2">NLRP3:ENST00000348069.2:c.2156+4049C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391827.2">NLRP3:ENST00000391827.2:c.2156+4049C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.2891%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZC3H4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130749">ZC3H4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.094</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.486 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ZC3H4">ZC3H4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001914, hemorrhage</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZC3H4%0DDDX11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DDX11</a> and phenotypic similarity 0.631 to <a href="http://www.omim.org/entry/613398">Warsaw breakage syndrome</a> associated with DDX11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZC3H4%0DDDX11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DDX11</a> and phenotypic similarity 0.449 to mouse mutant of DDX11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004251, failure of heart looping</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0004251, failure of heart looping</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.094</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-47571119-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200248556">rs200248556</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.094</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000253048.5">ZC3H4:ENST00000253048.5:c.2406C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0083%</dd>
                                    <dd>gnomAD_E_AMR: 0.3423%</dd>
                                    <dd>gnomAD_E_NFE: 0.0021%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CLUH" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000132361">CLUH</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.094</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CLUH%0DIDH2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IDH2</a> and phenotypic similarity 0.837 to <a href="http://www.omim.org/entry/613657">D-2-hydroxyglutaric aciduria 2</a> associated with IDH2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CLUH%0DIDH2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IDH2</a> and phenotypic similarity 0.386 to mouse mutant of IDH2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010144, abnormal tumor vascularization</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010144, abnormal tumor vascularization</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.094</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-2601273-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147854747">rs147854747</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.094</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000435359.1">CLUH:ENST00000435359.1:c.1764C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538975.1">CLUH:ENST00000538975.1:c.1764C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000570628.2">CLUH:ENST00000570628.2:c.1764C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0492%</dd>
                                    <dd>gnomAD_E_AMR: 0.3299%</dd>
                                    <dd>gnomAD_E_NFE: 0.3685%</dd>
                                    <dd>gnomAD_E_SAS: 0.0074%</dd>
                                    <dd>gnomAD_G_AFR: 0.0690%</dd>
                                    <dd>gnomAD_G_AMR: 0.3538%</dd>
                                    <dd>gnomAD_G_NFE: 0.2593%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="N4BP2L2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000244754">N4BP2L2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.093</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=N4BP2L2%0DKDM5B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KDM5B</a> and phenotypic similarity 0.661 to <a href="http://www.omim.org/entry/618109">Intellectual developmental disorder, autosomal recessive 65</a> associated with KDM5B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.093</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>13-33017480-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs139308263">rs139308263</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.093</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357505.6">N4BP2L2:ENST00000357505.6:c.1149A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399396.3">N4BP2L2:ENST00000399396.3:c.1194A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000504114.1">N4BP2L2:ENST00000504114.1:c.1149A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000446957.2">N4BP2L2:ENST00000446957.2:c.1698-795A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0839%</dd>
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                    <dd>gnomAD_E_NFE: 0.4034%</dd>
                                    <dd>gnomAD_G_AFR: 0.0689%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.3629%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF408" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000175213">ZNF408</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.093</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.334 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=791">Retinitis pigmentosa</a> associated with ZNF408.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000135, Hypogonadism</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Phenotypic similarity 0.317 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=ZNF408">ZNF408</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0009554, vascular sprouts morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0009554, vascular sprouts morphology, abnormal</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF408%0DFANCM&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FANCM</a> and phenotypic similarity 0.844 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with FANCM.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/616468">OMIM:616468</a> ?Exudative vitreoretinopathy 6 (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/616469">OMIM:616469</a> Retinitis pigmentosa 72 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=791">ORPHA:791</a> Retinitis pigmentosa - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=891">ORPHA:891</a> Familial exudative vitreoretinopathy - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.093</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.319 to ORPHA:891 Familial exudative vitreoretinopathy</dt>
                                
                                    <dt>Phenotypic similarity 0.319 to OMIM:616468 ?Exudative vitreoretinopathy 6</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>11-46726747-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs144957649">rs144957649</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/713577">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.093</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311764.2">ZNF408:ENST00000311764.2:c.1497G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0699%</dd>
                                    <dd>gnomAD_E_AMR: 0.2027%</dd>
                                    <dd>gnomAD_E_NFE: 0.4100%</dd>
                                    <dd>gnomAD_E_SAS: 0.0163%</dd>
                                    <dd>gnomAD_G_AFR: 0.0345%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.2789%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CCT4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000115484">CCT4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.085</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CCT4%0DPEX19&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PEX19</a> and phenotypic similarity 0.783 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=772">Infantile Refsum disease</a> associated with PEX19.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.085</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-62103327-G-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs115496486">rs115496486</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.085</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394440.3">CCT4:ENST00000394440.3:c.820C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538252.1">CCT4:ENST00000538252.1:c.652C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544079.1">CCT4:ENST00000544079.1:c.730C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544185.1">CCT4:ENST00000544185.1:c.370C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1173%</dd>
                                    <dd>gnomAD_E_AMR: 0.0759%</dd>
                                    <dd>gnomAD_E_NFE: 0.7320%</dd>
                                    <dd>gnomAD_E_SAS: 0.0230%</dd>
                                    <dd>gnomAD_G_AFR: 0.1607%</dd>
                                    <dd>gnomAD_G_AMR: 0.2364%</dd>
                                    <dd>gnomAD_G_NFE: 0.7198%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SFRP5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000120057">SFRP5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.090</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SFRP5%0DWNT4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to WNT4</a> and phenotypic similarity 0.641 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=139466">SERKAL syndrome</a> associated with WNT4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0030680, Abnormal cardiovascular system morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SFRP5%0DWNT4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to WNT4</a> and phenotypic similarity 0.490 to mouse mutant of WNT4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0030992, atrioventricular cushion hypoplasia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0030992, atrioventricular cushion hypoplasia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.090</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>10-99529459-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs144832598">rs144832598</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.090</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000266066.3">SFRP5:ENST00000266066.3:c.573C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0752%</dd>
                                    <dd>gnomAD_E_AMR: 0.1132%</dd>
                                    <dd>gnomAD_E_EAS: 0.0218%</dd>
                                    <dd>gnomAD_E_NFE: 0.4986%</dd>
                                    <dd>gnomAD_E_SAS: 0.0461%</dd>
                                    <dd>gnomAD_G_AFR: 0.1377%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.5509%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="WAPL" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000062650">WAPL</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.508</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.082</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=WAPL%0DNIPBL&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NIPBL</a> and phenotypic similarity 0.653 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=199">Cornelia de Lange syndrome</a> associated with NIPBL.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0030680, Abnormal cardiovascular system morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=WAPL%0DNIPBL&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NIPBL</a> and phenotypic similarity 0.750 to mouse mutant of NIPBL.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004056, abnormal myocardium compact layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=WAPL%0DNIPBL&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NIPBL</a> and phenotypic similarity 0.323 to fish mutant of NIPBL.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.508</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.082</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-88260455-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201850920">rs201850920</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.082</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000298767.5">WAPL:ENST00000298767.5:c.545A&gt;G:p.(His182Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.08231511</dd>
                                    
                                    <dd>REVEL: 0.066</dd>
                                    <dd>MVP: 0.082</dd>
                                    <dd>ALPHA_MISSENSE: 0.079</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0069%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GSTA4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170899">GSTA4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.086</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GSTA4%0DGPX4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GPX4</a> and phenotypic similarity 0.756 to <a href="http://www.omim.org/entry/250220">Spondylometaphyseal dysplasia, Sedaghatian type</a> associated with GPX4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0012819, Myocarditis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.086</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>6-52849342-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs45507994">rs45507994</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/779335">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.086</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370959.1">GSTA4:ENST00000370959.1:c.334T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370960.1">GSTA4:ENST00000370960.1:c.55T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541324.1">GSTA4:ENST00000541324.1:c.55T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1538%</dd>
                                    <dd>gnomAD_E_AMR: 0.2111%</dd>
                                    <dd>gnomAD_E_NFE: 0.7131%</dd>
                                    <dd>gnomAD_E_SAS: 0.0588%</dd>
                                    <dd>gnomAD_G_AFR: 0.1033%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.6286%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CDH24" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000139880">CDH24</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.089</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CDH24%0DCDH2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDH2</a> and phenotypic similarity 0.713 to <a href="http://www.omim.org/entry/618929">Agenesis of corpus callosum, cardiac, ocular, and genital syndrome</a> associated with CDH2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001651, Dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0006695, Atrioventricular canal defect</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CDH24%0DCDH2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDH2</a> and phenotypic similarity 0.536 to mouse mutant of CDH2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005329, abnormal myocardium layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000274, enlarged heart</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CDH24%0DCDH2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDH2</a> and phenotypic similarity 0.263 to fish mutant of CDH2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000019, intersegmental vessel malformed, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000019, intersegmental vessel malformed, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.089</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>14-23523353-G-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs139832060">rs139832060</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.089</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000267383.5">CDH24:ENST00000267383.5:c.969C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397359.3">CDH24:ENST00000397359.3:c.969C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000487137.2">CDH24:ENST00000487137.2:c.969C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000554034.1">CDH24:ENST00000554034.1:c.969C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0448%</dd>
                                    <dd>gnomAD_E_AMR: 0.1202%</dd>
                                    <dd>gnomAD_E_NFE: 0.3521%</dd>
                                    <dd>gnomAD_E_SAS: 0.4091%</dd>
                                    <dd>gnomAD_G_AFR: 0.0575%</dd>
                                    <dd>gnomAD_G_AMR: 0.5896%</dd>
                                    <dd>gnomAD_G_NFE: 0.2921%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="WHAMM" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000156232">WHAMM</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.089</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=WHAMM%0DWASHC5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to WASHC5</a> and phenotypic similarity 0.681 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=7">3C syndrome</a> associated with WASHC5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0006695, Atrioventricular canal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0006695, Atrioventricular canal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=WHAMM%0DWASHC5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to WASHC5</a> and phenotypic similarity 0.490 to fish mutant of WASHC5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0006461, skeletal muscle decreased functionality, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.089</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>15-83478684-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201191404">rs201191404</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3190529">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.089</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000286760.4">WHAMM:ENST00000286760.4:c.206C&gt;T:p.(Pro69Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.0899</dd>
                                    
                                    <dd>REVEL: 0.036</dd>
                                    <dd>MVP: 0.061</dd>
                                    <dd>ALPHA_MISSENSE: 0.090</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0064%</dd>
                                    <dd>gnomAD_E_NFE: 0.0518%</dd>
                                    <dd>gnomAD_G_NFE: 0.0196%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TANC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000115183">TANC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.088</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TANC1%0DCAMK2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAMK2A</a> and phenotypic similarity 0.651 to <a href="http://www.omim.org/entry/617798">Intellectual developmental disorder, autosomal dominant 53</a> associated with CAMK2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TANC1%0DCAMK2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAMK2A</a> and phenotypic similarity 0.295 to fish mutant of CAMK2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0005428, heart tube inverted, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0005428, heart tube inverted, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.088</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-160027069-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148444366">rs148444366</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.088</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263635.6">TANC1:ENST00000263635.6:c.1104G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454300.1">TANC1:ENST00000454300.1:c.786G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.6405%</dd>
                                    <dd>gnomAD_G_AMR: 0.4717%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KLHL40" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000157119">KLHL40</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.255</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.365</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.511 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=KLHL40">KLHL40</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005333, decreased heart rate</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002188, small heart</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/615348">OMIM:615348</a> Nemaline myopathy 8, autosomal recessive - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=171430">ORPHA:171430</a> Severe congenital nemaline myopathy - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.255</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.365</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-42727306-C-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200216330">rs200216330</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.365</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000287777.4">KLHL40:ENST00000287777.4:c.196C&gt;A:p.(Pro66Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.36512005</dd>
                                    
                                    <dd>REVEL: 0.052</dd>
                                    <dd>MVP: 0.365</dd>
                                    <dd>ALPHA_MISSENSE: 0.089</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LMO7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136153">LMO7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.085</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.440 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=LMO7">LMO7</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000107, heart morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000107, heart morphology, abnormal</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LMO7%0DACTN2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTN2</a> and phenotypic similarity 0.902 to <a href="http://www.omim.org/entry/612158">Cardiomyopathy, hypertrophic, 23, with or without LVNC</a> associated with ACTN2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0006685, Endocardial fibrosis</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LMO7%0DACTN2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTN2</a> and phenotypic similarity 0.647 to mouse mutant of ACTN2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010418, perimembraneous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010418, perimembraneous ventricular septal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LMO7%0DACTN2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTN2</a> and phenotypic similarity 0.513 to fish mutant of ACTN2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0004526, cardiac muscle cell shape, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0008027, atrium lacks parts or has fewer parts of type sarcomere cardiac muscle cell, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.085</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>13-76397771-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147546485">rs147546485</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/722966">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.085</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000321797.8">LMO7:ENST00000321797.8:c.2012C&gt;T:p.(Thr671Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341547.4">LMO7:ENST00000341547.4:c.1865C&gt;T:p.(Thr622Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357063.3">LMO7:ENST00000357063.3:c.2867C&gt;T:p.(Thr956Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377534.3">LMO7:ENST00000377534.3:c.2867C&gt;T:p.(Thr956Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000465261.2">LMO7:ENST00000465261.2:c.2012C&gt;T:p.(Thr671Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000526202.1">LMO7:ENST00000526202.1:c.1562C&gt;T:p.(Thr521Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.10739988</dd>
                                    
                                    <dd>REVEL: 0.040</dd>
                                    <dd>MVP: 0.107</dd>
                                    <dd>ALPHA_MISSENSE: 0.104</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.9227%</dd>
                                    <dd>gnomAD_E_AMR: 0.0463%</dd>
                                    <dd>gnomAD_E_NFE: 0.0141%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.8494%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="HGS" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185359">HGS</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.081</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.437 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=HGS">HGS</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004187, cardia bifida</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0004187, cardia bifida</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=HGS%0DUBE4B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UBE4B</a> and phenotypic similarity 0.740 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1606">1p36 deletion syndrome</a> associated with UBE4B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=HGS%0DUBE4B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UBE4B</a> and phenotypic similarity 0.533 to mouse mutant of UBE4B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005329, abnormal myocardium layer morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000274, enlarged heart</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.081</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-79662950-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148054575">rs148054575</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.081</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329138.4">HGS:ENST00000329138.4:c.1314C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1856%</dd>
                                    <dd>gnomAD_E_AMR: 0.2851%</dd>
                                    <dd>gnomAD_E_NFE: 0.8660%</dd>
                                    <dd>gnomAD_E_SAS: 0.1371%</dd>
                                    <dd>gnomAD_G_AFR: 0.1378%</dd>
                                    <dd>gnomAD_G_AMR: 0.4717%</dd>
                                    <dd>gnomAD_G_NFE: 0.6553%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SH2D2A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000027869">SH2D2A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.083</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SH2D2A%0DSMAD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMAD2</a> and phenotypic similarity 0.665 to <a href="http://www.omim.org/entry/619657">Congenital heart defects, multiple types, 8, with or without heterotaxy</a> associated with SMAD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001651, Dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011579, Unbalanced atrioventricular canal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SH2D2A%0DSMAD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMAD2</a> and phenotypic similarity 0.481 to mouse mutant of SMAD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000267, abnormal heart development</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000267, abnormal heart development</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SH2D2A%0DSMAD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMAD2</a> and phenotypic similarity 0.291 to fish mutant of SMAD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0004060, cranial vasculature morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0004060, cranial vasculature morphology, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.083</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-156785810-C-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs145574758">rs145574758</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1176211">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.083</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368198.3">SH2D2A:ENST00000368198.3:c.57G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368199.3">SH2D2A:ENST00000368199.3:c.111G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392306.2">SH2D2A:ENST00000392306.2:c.111G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392302.2">SH2D2A:ENST00000392302.2:c.9+180C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0802%</dd>
                                    <dd>gnomAD_E_AMR: 0.4025%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.8017%</dd>
                                    <dd>gnomAD_E_SAS: 0.3436%</dd>
                                    <dd>gnomAD_G_AFR: 0.0803%</dd>
                                    <dd>gnomAD_G_AMR: 0.3538%</dd>
                                    <dd>gnomAD_G_EAS: 0.0643%</dd>
                                    <dd>gnomAD_G_NFE: 0.4995%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="POU4F2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000151615">POU4F2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.084</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=POU4F2%0DJAG2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to JAG2</a> and phenotypic similarity 0.837 to <a href="http://www.omim.org/entry/619566">Muscular dystrophy, limb-girdle, autosomal recessive 27</a> associated with JAG2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.084</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>4-147560460-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs189899086">rs189899086</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.084</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000281321.3">POU4F2:ENST00000281321.3:c.168C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0669%</dd>
                                    <dd>gnomAD_E_AMR: 0.0817%</dd>
                                    <dd>gnomAD_E_EAS: 0.7908%</dd>
                                    <dd>gnomAD_E_NFE: 0.1424%</dd>
                                    <dd>gnomAD_E_SAS: 0.0826%</dd>
                                    <dd>gnomAD_G_AFR: 0.0287%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KIFC3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000140859">KIFC3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.083</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIFC3%0DTUBG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TUBG1</a> and phenotypic similarity 0.627 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=261183">15q11.2 microdeletion syndrome</a> associated with TUBG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001627, Abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.083</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>16-57798091-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs139069247">rs139069247</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.083</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379655.4">KIFC3:ENST00000379655.4:c.1581C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421376.2">KIFC3:ENST00000421376.2:c.1164C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445690.2">KIFC3:ENST00000445690.2:c.1581C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000465878.2">KIFC3:ENST00000465878.2:c.1164C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539578.1">KIFC3:ENST00000539578.1:c.1407C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540079.2">KIFC3:ENST00000540079.2:c.1275C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541240.1">KIFC3:ENST00000541240.1:c.1647C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543930.1">KIFC3:ENST00000543930.1:c.1155C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000562903.1">KIFC3:ENST00000562903.1:c.1164C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.8103%</dd>
                                    <dd>gnomAD_E_AMR: 0.0376%</dd>
                                    <dd>gnomAD_E_EAS: 0.3159%</dd>
                                    <dd>gnomAD_E_NFE: 0.0062%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.8272%</dd>
                                    <dd>gnomAD_G_AMR: 0.1182%</dd>
                                    <dd>gnomAD_G_EAS: 0.1928%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TRIM22" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000132274">TRIM22</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.083</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRIM22%0DNOD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NOD2</a> and phenotypic similarity 0.601 to <a href="http://www.omim.org/entry/617321">Yao syndrome</a> associated with NOD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001714, Ventricular hypertrophy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001714, Ventricular hypertrophy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.083</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>11-5729419-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs112606816">rs112606816</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.083</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379965.3">TRIM22:ENST00000379965.3:c.790T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380027.1">TRIM22:ENST00000380027.1:c.-440-22795A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1550%</dd>
                                    <dd>gnomAD_E_AMR: 0.4258%</dd>
                                    <dd>gnomAD_E_NFE: 0.7602%</dd>
                                    <dd>gnomAD_E_SAS: 0.1046%</dd>
                                    <dd>gnomAD_G_AFR: 0.1148%</dd>
                                    <dd>gnomAD_G_AMR: 0.8255%</dd>
                                    <dd>gnomAD_G_NFE: 0.7067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="APOD" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000189058">APOD</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.079</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.477 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=APOD">APOD</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000343, abnormal response to cardiac infarction</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000343, abnormal response to cardiac infarction</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=APOD%0DJAG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to JAG1</a> and phenotypic similarity 0.610 to <a href="http://www.omim.org/entry/617992">?Deafness, congenital heart defects, and posterior embryotoxon</a> associated with JAG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=APOD%0DJAG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to JAG1</a> and phenotypic similarity 0.745 to mouse mutant of JAG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010402, ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=APOD%0DJAG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to JAG1</a> and phenotypic similarity 0.256 to fish mutant of JAG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0106477, ventral wall of dorsal aorta decreased distribution, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0106477, ventral wall of dorsal aorta decreased distribution, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.079</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>3-195306219-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs5953">rs5953</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.079</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343267.3">APOD:ENST00000343267.3:c.114C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1599%</dd>
                                    <dd>gnomAD_E_AMR: 0.4425%</dd>
                                    <dd>gnomAD_E_NFE: 0.8301%</dd>
                                    <dd>gnomAD_E_SAS: 0.3920%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_AMR: 0.9456%</dd>
                                    <dd>gnomAD_G_NFE: 0.6676%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZPLD1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170044">ZPLD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.647</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.647</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>3-102187979-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs58997897">rs58997897</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.647</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000306176.1">ZPLD1:ENST00000306176.1:c.981G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000466937.1">ZPLD1:ENST00000466937.1:c.933G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000491959.1">ZPLD1:ENST00000491959.1:c.933G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.8802%</dd>
                                    <dd>gnomAD_E_AMR: 0.3866%</dd>
                                    <dd>gnomAD_E_NFE: 0.1114%</dd>
                                    <dd>gnomAD_E_SAS: 0.0106%</dd>
                                    <dd>gnomAD_G_AFR: 0.5744%</dd>
                                    <dd>gnomAD_G_AMR: 0.8255%</dd>
                                    <dd>gnomAD_G_NFE: 0.0907%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="USP42" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000106346">USP42</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=6.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.078</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.323 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=USP42">USP42</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=USP42%0DMRPS14&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MRPS14</a> and phenotypic similarity 0.962 to <a href="http://www.omim.org/entry/618378">?Combined oxidative phosphorylation deficiency 38</a> associated with MRPS14.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=6.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.078</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>7-6194230-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199684634">rs199684634</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2657297">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.078</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000306177.5">USP42:ENST00000306177.5:c.3045C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1058%</dd>
                                    <dd>gnomAD_E_AMR: 0.5567%</dd>
                                    <dd>gnomAD_E_NFE: 0.7387%</dd>
                                    <dd>gnomAD_E_SAS: 0.1707%</dd>
                                    <dd>gnomAD_G_AFR: 0.0578%</dd>
                                    <dd>gnomAD_G_AMR: 0.7229%</dd>
                                    <dd>gnomAD_G_NFE: 0.9569%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SOX7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171056">SOX7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=6.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.489</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.091</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.489 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SOX7">SOX7</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0008825, abnormal cardiac epithelial to mesenchymal transition</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001787, pericardial edema</dd></dl><dl><dt>Phenotypic similarity 0.261 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=SOX7">SOX7</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0004469, dorsal aorta malformed, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0004469, dorsal aorta malformed, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=6.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.489</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.091</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>8-10583476-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs113624178">rs113624178</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/779341">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.091</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000304501.1">SOX7:ENST00000304501.1:c.939C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000553390.1">SOX7:ENST00000553390.1:c.1095C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000554914.1">SOX7:ENST00000554914.1:c.1095C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0554%</dd>
                                    <dd>gnomAD_E_AMR: 0.2530%</dd>
                                    <dd>gnomAD_E_NFE: 0.5186%</dd>
                                    <dd>gnomAD_E_SAS: 0.3158%</dd>
                                    <dd>gnomAD_G_AFR: 0.0689%</dd>
                                    <dd>gnomAD_G_NFE: 0.4281%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MINDY2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000128923">MINDY2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=6.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.477</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.477 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MINDY2">MINDY2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005333, decreased heart rate</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0005333, decreased heart rate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=6.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.477</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>15-59064041-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202092838">rs202092838</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000450403.2">MINDY2:ENST00000450403.2:c.447A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000559228.1">MINDY2:ENST00000559228.1:c.447A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0031%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MTUS1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000129422">MTUS1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=6.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.467</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.467 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MTUS1">MTUS1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001625, cardiac hypertrophy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001625, cardiac hypertrophy</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=6.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.467</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>8-17612900-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199854201">rs199854201</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262102.6">MTUS1:ENST00000262102.6:c.417G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381862.3">MTUS1:ENST00000381862.3:c.417G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381869.3">MTUS1:ENST00000381869.3:c.417G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000519263.1">MTUS1:ENST00000519263.1:c.417G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                    <dd>gnomAD_E_NFE: 0.0124%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AMPD3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000133805">AMPD3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=6.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.339</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AMPD3%0DADK&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ADK</a> and phenotypic similarity 0.606 to <a href="http://www.omim.org/entry/614300">Hypermethioninemia due to adenosine kinase deficiency</a> associated with ADK.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001642, Pulmonic stenosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001684, Secundum atrial septal defect</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=45">ORPHA:45</a> Adenosine monophosphate deaminase deficiency - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=6.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.339</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-10503740-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202051288">rs202051288</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2238271">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.339</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396554.3">AMPD3:ENST00000396554.3:c.584C&gt;T:p.(Ala195Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000444303.2">AMPD3:ENST00000444303.2:c.80C&gt;T:p.(Ala27Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3513982</dd>
                                    
                                    <dd>REVEL: 0.005</dd>
                                    <dd>MVP: 0.351</dd>
                                    <dd>ALPHA_MISSENSE: 0.074</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0124%</dd>
                                    <dd>gnomAD_E_AMR: 0.0318%</dd>
                                    <dd>gnomAD_E_EAS: 0.0272%</dd>
                                    <dd>gnomAD_E_NFE: 0.0106%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DDB2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000134574">DDB2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=6.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.255</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.332</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.382 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=910">Xeroderma pigmentosum</a> associated with DDB2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000135, Hypogonadism</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0000524, Conjunctival telangiectasia</dd></dl><dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DDB2%0DDCAF8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DCAF8</a> and phenotypic similarity 0.837 to <a href="http://www.omim.org/entry/610100">?Giant axonal neuropathy 2, autosomal dominant</a> associated with DCAF8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DDB2%0DDCAF8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DCAF8</a> and phenotypic similarity 0.512 to mouse mutant of DCAF8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000266, abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000266, abnormal heart morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/278740">OMIM:278740</a> Xeroderma pigmentosum, group E, DDB-negative subtype - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=910">ORPHA:910</a> Xeroderma pigmentosum - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=6.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.255</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.332</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-47259728-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs143049891">rs143049891</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/133961">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:1, LB:6)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.332</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000256996.4">DDB2:ENST00000256996.4:c.1228G&gt;A:p.(Ala410Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378600.3">DDB2:ENST00000378600.3:c.661G&gt;A:p.(Ala221Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378603.3">DDB2:ENST00000378603.3:c.1036G&gt;A:p.(Ala346Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378601.3">DDB2:ENST00000378601.3:c.*315G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.35772473</dd>
                                    
                                    <dd>REVEL: 0.137</dd>
                                    <dd>MVP: 0.358</dd>
                                    <dd>ALPHA_MISSENSE: 0.092</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1328%</dd>
                                    <dd>gnomAD_E_AMR: 0.1920%</dd>
                                    <dd>gnomAD_E_NFE: 0.4301%</dd>
                                    <dd>gnomAD_E_SAS: 0.0169%</dd>
                                    <dd>gnomAD_G_AFR: 0.0688%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.3243%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ANO9" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185101">ANO9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=6.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.619</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=6.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.619</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-433904-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs137970750">rs137970750</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.619</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000332826.6">ANO9:ENST00000332826.6:c.115G&gt;A:p.(Val39Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6426</dd>
                                    
                                    <dd>REVEL: 0.317</dd>
                                    <dd>MVP: 0.503</dd>
                                    <dd>ALPHA_MISSENSE: 0.643</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1028%</dd>
                                    <dd>gnomAD_E_NFE: 0.0781%</dd>
                                    <dd>gnomAD_E_SAS: 0.1221%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.0324%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BRAT1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000106009">BRAT1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.618</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.275 to <a href="http://www.omim.org/entry/614498">Rigidity and multifocal seizure syndrome, lethal neonatal</a> associated with BRAT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0003487, Babinski sign</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/614498">OMIM:614498</a> Rigidity and multifocal seizure syndrome, lethal neonatal - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/618056">OMIM:618056</a> Neurodevelopmental disorder with cerebellar atrophy and with or without seizures - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.618</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-2580986-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs141807337">rs141807337</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/585486">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.618</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340611.4">BRAT1:ENST00000340611.4:c.1267G&gt;A:p.(Val423Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6203616</dd>
                                    
                                    <dd>REVEL: 0.028</dd>
                                    <dd>MVP: 0.620</dd>
                                    <dd>ALPHA_MISSENSE: 0.085</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0068%</dd>
                                    <dd>gnomAD_E_AMR: 0.0265%</dd>
                                    <dd>gnomAD_E_NFE: 0.0066%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BOD1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000145919">BOD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.455</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.455 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=BOD1">BOD1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002833, increased heart weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002833, increased heart weight</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.455</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>5-173040244-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150573630">rs150573630</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000285908.5">BOD1:ENST00000285908.5:c.252C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311086.4">BOD1:ENST00000311086.4:c.252C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000480951.1">BOD1:ENST00000480951.1:c.237+2959C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TMPRSS13" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000137747">TMPRSS13</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.594</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.594</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-117789345-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs61900347">rs61900347</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.594</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430170.2">TMPRSS13:ENST00000430170.2:c.230C&gt;G:p.(Ala77Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445164.2">TMPRSS13:ENST00000445164.2:c.230C&gt;G:p.(Ala77Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524993.1">TMPRSS13:ENST00000524993.1:c.230C&gt;G:p.(Ala77Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000526090.1">TMPRSS13:ENST00000526090.1:c.230C&gt;G:p.(Ala77Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000528626.1">TMPRSS13:ENST00000528626.1:c.230C&gt;G:p.(Ala77Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6316026</dd>
                                    
                                    <dd>REVEL: 0.172</dd>
                                    <dd>MVP: 0.632</dd>
                                    <dd>ALPHA_MISSENSE: 0.093</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0226%</dd>
                                    <dd>gnomAD_E_AMR: 0.0032%</dd>
                                    <dd>gnomAD_E_EAS: 0.2504%</dd>
                                    <dd>gnomAD_E_NFE: 0.0171%</dd>
                                    <dd>gnomAD_E_SAS: 0.0035%</dd>
                                    <dd>gnomAD_G_AFR: 0.0131%</dd>
                                    <dd>gnomAD_G_AMR: 0.1244%</dd>
                                    <dd>gnomAD_G_EAS: 0.3634%</dd>
                                    <dd>gnomAD_G_NFE: 0.0140%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MTTP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000138823">MTTP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.307</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.512 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=14">Abetalipoproteinemia</a> associated with MTTP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001640, Cardiomegaly</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001640, Cardiomegaly</dd></dl><dl><dt>Phenotypic similarity 0.486 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MTTP">MTTP</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001914, hemorrhage</dd></dl><dl><dt>Phenotypic similarity 0.285 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=MTTP">MTTP</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0005758, subintestinal vein morphology, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0005758, subintestinal vein morphology, abnormal</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MTTP%0DNR2F2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NR2F2</a> and phenotypic similarity 0.692 to <a href="http://www.omim.org/entry/615779">Congenital heart defects, multiple types, 4</a> associated with NR2F2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0006695, Atrioventricular canal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0006695, Atrioventricular canal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MTTP%0DNR2F2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NR2F2</a> and phenotypic similarity 0.486 to mouse mutant of NR2F2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001914, hemorrhage</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MTTP%0DNR2F2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NR2F2</a> and phenotypic similarity 0.398 to fish mutant of NR2F2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0010244, trunk vasculature non-functional, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/200100">OMIM:200100</a> Abetalipoproteinemia - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=14">ORPHA:14</a> Abetalipoproteinemia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.307</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-100512433-T-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150319930">rs150319930</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1064236">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.307</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265517.5">MTTP:ENST00000265517.5:c.543T&gt;G:p.(His181Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457717.1">MTTP:ENST00000457717.1:c.543T&gt;G:p.(His181Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000511045.1">MTTP:ENST00000511045.1:c.624T&gt;G:p.(His208Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.30701694</dd>
                                    
                                    <dd>REVEL: 0.050</dd>
                                    <dd>MVP: 0.307</dd>
                                    <dd>ALPHA_MISSENSE: 0.088</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PAPSS2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198682">PAPSS2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.256</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.294</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.512 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PAPSS2%0DFLAD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FLAD1</a> and phenotypic similarity 0.837 to <a href="http://www.omim.org/entry/255100">Lipid storage myopathy due to flavin adenine dinucleotide synthetase deficiency</a> associated with FLAD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/612847">OMIM:612847</a> Brachyolmia 4 with mild epiphyseal and metaphyseal changes - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.256</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.294</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-89503310-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs34097979">rs34097979</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/537719">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.294</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361175.4">PAPSS2:ENST00000361175.4:c.1388C&gt;T:p.(Ala463Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427144.2">PAPSS2:ENST00000427144.2:c.1400C&gt;T:p.(Ala467Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456849.1">PAPSS2:ENST00000456849.1:c.1403C&gt;T:p.(Ala468Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.36195868</dd>
                                    
                                    <dd>REVEL: 0.151</dd>
                                    <dd>MVP: 0.362</dd>
                                    <dd>ALPHA_MISSENSE: 0.126</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.8736%</dd>
                                    <dd>gnomAD_E_AMR: 0.0289%</dd>
                                    <dd>gnomAD_E_EAS: 0.2399%</dd>
                                    <dd>gnomAD_E_NFE: 0.0345%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.8039%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TTC38" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000075234">TTC38</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.583</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.583</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>22-46681153-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199903244">rs199903244</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.583</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381031.3">TTC38:ENST00000381031.3:c.811G&gt;A:p.(Ala271Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445282.2">TTC38:ENST00000445282.2:c.637G&gt;A:p.(Ala213Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5834878</dd>
                                    
                                    <dd>REVEL: 0.547</dd>
                                    <dd>MVP: 0.583</dd>
                                    <dd>ALPHA_MISSENSE: 0.320</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MCM7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000166508">MCM7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.287</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.422 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=MCM7">MCM7</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001192, heart edematous, abnormal</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MCM7%0DCDC45&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDC45</a> and phenotypic similarity 0.737 to <a href="http://www.omim.org/entry/617063">Meier-Gorlin syndrome 7</a> associated with CDC45.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001674, Complete atrioventricular canal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001674, Complete atrioventricular canal defect</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2512">ORPHA:2512</a> Autosomal recessive primary microcephaly - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.287</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-99697002-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150084480">rs150084480</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3124391">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.287</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000303887.5">MCM7:ENST00000303887.5:c.301G&gt;A:p.(Val101Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343023.6">MCM7:ENST00000343023.6:c.301G&gt;A:p.(Val101Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354230.3">MCM7:ENST00000354230.3:c.-303G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.2924235</dd>
                                    
                                    <dd>REVEL: 0.073</dd>
                                    <dd>MVP: 0.292</dd>
                                    <dd>ALPHA_MISSENSE: 0.102</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0637%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CCDC81" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000149201">CCDC81</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.570</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.570</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-86126317-G-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs34331151">rs34331151</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.570</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000278487.3">CCDC81:ENST00000278487.3:c.858G&gt;C:p.(Gln286His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354755.1">CCDC81:ENST00000354755.1:c.1383G&gt;C:p.(Gln461His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445632.2">CCDC81:ENST00000445632.2:c.1653G&gt;C:p.(Gln551His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000528728.1">CCDC81:ENST00000528728.1:c.858G&gt;C:p.(Gln286His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.592</dd>
                                    
                                    <dd>REVEL: 0.229</dd>
                                    <dd>MVP: 0.408</dd>
                                    <dd>ALPHA_MISSENSE: 0.592</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0369%</dd>
                                    <dd>gnomAD_E_AMR: 0.1301%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.2454%</dd>
                                    <dd>gnomAD_E_SAS: 0.0621%</dd>
                                    <dd>gnomAD_G_AFR: 0.0689%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.1750%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CKAP5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000175216">CKAP5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CKAP5%0DSDCCAG8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SDCCAG8</a> and phenotypic similarity 0.744 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=110">Bardet-Biedl syndrome</a> associated with SDCCAG8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001627, Abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CKAP5%0DSDCCAG8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SDCCAG8</a> and phenotypic similarity 0.360 to mouse mutant of SDCCAG8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000820, abnormal choroid plexus morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000820, abnormal choroid plexus morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FIVE_PRIME_UTR_INTRON_VARIANT</span>
                            <b>SNV</b>
                            <b>11-46842829-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200773918">rs200773918</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000312055.5">CKAP5:ENST00000312055.5:c.-37-10A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000415402.1">CKAP5:ENST00000415402.1:c.-37-10A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529230.1">CKAP5:ENST00000529230.1:c.-37-10A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0123%</dd>
                                    <dd>gnomAD_E_SAS: 0.0035%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LCE3E" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185966">LCE3E</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LCE3E%0DPCSK5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PCSK5</a> and phenotypic similarity 0.728 to mouse mutant of PCSK5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000644, dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FIVE_PRIME_UTR_INTRON_VARIANT</span>
                            <b>SNV</b>
                            <b>1-152538717-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs77965260">rs77965260</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368789.1">LCE3E:ENST00000368789.1:c.-21-12C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0330%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_G_AMR: 0.1185%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CHI3L2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000064886">CHI3L2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.560</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.560</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-111778701-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200826910">rs200826910</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.560</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369744.2">CHI3L2:ENST00000369744.2:c.631C&gt;T:p.(Pro211Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369748.4">CHI3L2:ENST00000369748.4:c.661C&gt;T:p.(Pro221Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445067.2">CHI3L2:ENST00000445067.2:c.661C&gt;T:p.(Pro221Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000466741.1">CHI3L2:ENST00000466741.1:c.424C&gt;T:p.(Pro142Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524472.1">CHI3L2:ENST00000524472.1:c.424C&gt;T:p.(Pro142Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5717633</dd>
                                    
                                    <dd>REVEL: 0.068</dd>
                                    <dd>MVP: 0.572</dd>
                                    <dd>ALPHA_MISSENSE: 0.172</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1388%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="UBR4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000127481">UBR4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.386</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.081</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.386 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=UBR4">UBR4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003229, abnormal vitelline vasculature morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003229, abnormal vitelline vasculature morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.386</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.081</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-19500088-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs41273197">rs41273197</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/709355">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.081</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375217.2">UBR4:ENST00000375217.2:c.3010T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375226.2">UBR4:ENST00000375226.2:c.3010T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375254.3">UBR4:ENST00000375254.3:c.3010T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375267.2">UBR4:ENST00000375267.2:c.3010T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1723%</dd>
                                    <dd>gnomAD_E_AMR: 0.2025%</dd>
                                    <dd>gnomAD_E_NFE: 0.8844%</dd>
                                    <dd>gnomAD_E_SAS: 0.1339%</dd>
                                    <dd>gnomAD_G_AFR: 0.1951%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.7259%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TMEM184A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000164855">TMEM184A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.473</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.473</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-1589956-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201867640">rs201867640</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.473</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000297477.5">TMEM184A:ENST00000297477.5:c.461G&gt;A:p.(Arg154His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5010603</dd>
                                    
                                    <dd>REVEL: 0.346</dd>
                                    <dd>MVP: 0.501</dd>
                                    <dd>ALPHA_MISSENSE: 0.149</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_AMR: 0.3493%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LINGO1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169783">LINGO1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.471</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.298 to <a href="http://www.omim.org/entry/618103">Intellectual developmental disorder, autosomal recessive 64</a> associated with LINGO1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001257, Spasticity</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618103">OMIM:618103</a> Intellectual developmental disorder, autosomal recessive 64 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.471</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>15-77907398-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202233236">rs202233236</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.471</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355300.6">LINGO1:ENST00000355300.6:c.851G&gt;A:p.(Arg284His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000561030.1">LINGO1:ENST00000561030.1:c.833G&gt;A:p.(Arg278His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.47946694</dd>
                                    
                                    <dd>REVEL: 0.331</dd>
                                    <dd>MVP: 0.479</dd>
                                    <dd>ALPHA_MISSENSE: 0.149</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PIDD1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000177595">PIDD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.450</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/619827">OMIM:619827</a> Intellectual developmental disorder, autosomal recessive 75, with neuropsychiatric features and variant lissencephaly - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.450</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-800563-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs113725585">rs113725585</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.450</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000347755.5">PIDD1:ENST00000347755.5:c.2021G&gt;A:p.(Arg674His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000411829.2">PIDD1:ENST00000411829.2:c.2021G&gt;A:p.(Arg674His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5680727</dd>
                                    
                                    <dd>REVEL: 0.062</dd>
                                    <dd>MVP: 0.568</dd>
                                    <dd>ALPHA_MISSENSE: 0.130</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.2121%</dd>
                                    <dd>gnomAD_E_AMR: 0.5417%</dd>
                                    <dd>gnomAD_E_NFE: 0.9287%</dd>
                                    <dd>gnomAD_E_SAS: 0.1503%</dd>
                                    <dd>gnomAD_G_AFR: 0.1378%</dd>
                                    <dd>gnomAD_G_AMR: 0.5896%</dd>
                                    <dd>gnomAD_G_NFE: 0.7075%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="P2RY11" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000244165">P2RY11</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.448</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2073">ORPHA:2073</a> Narcolepsy type 1 (susceptibility)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.448</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-10224579-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201039858">rs201039858</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.448</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000321826.4">P2RY11:ENST00000321826.4:c.290A&gt;G:p.(Tyr97Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.45608968</dd>
                                    
                                    <dd>REVEL: 0.178</dd>
                                    <dd>MVP: 0.456</dd>
                                    <dd>ALPHA_MISSENSE: 0.389</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0870%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PPAN-P2RY11" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000243207">PPAN-P2RY11</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.448</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.448</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-10224579-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201039858">rs201039858</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.448</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393796.4">PPAN-P2RY11:ENST00000393796.4:c.1550A&gt;G:p.(Tyr517Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428358.1">PPAN-P2RY11:ENST00000428358.1:c.*49A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.45608968</dd>
                                    
                                    <dd>REVEL: 0.178</dd>
                                    <dd>MVP: 0.456</dd>
                                    <dd>ALPHA_MISSENSE: 0.389</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0870%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DHDDS" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000117682">DHDDS</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.298</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.101</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.334 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=791">Retinitis pigmentosa</a> associated with DHDDS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000135, Hypogonadism</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/613861">OMIM:613861</a> ?Congenital disorder of glycosylation, type 1bb (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/613861">OMIM:613861</a> Retinitis pigmentosa 59 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/617836">OMIM:617836</a> Developmental delay and seizures with or without movement abnormalities - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=442835">ORPHA:442835</a> Non-specific early-onset epileptic encephalopathy - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=791">ORPHA:791</a> Retinitis pigmentosa - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.298</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.101</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.298 to ORPHA:442835 Non-specific early-onset epileptic encephalopathy</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-26774179-T-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200852161">rs200852161</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.101</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374185.3">DHDDS:ENST00000374185.3:c.570T&gt;G:p.(Ile190Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000236342.7">DHDDS:ENST00000236342.7:c.542+28T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360009.2">DHDDS:ENST00000360009.2:c.542+28T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427245.2">DHDDS:ENST00000427245.2:c.542+28T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000525682.2">DHDDS:ENST00000525682.2:c.440+1256T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000526219.1">DHDDS:ENST00000526219.1:c.425+28T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1014</dd>
                                    
                                    <dd>REVEL: 0.049</dd>
                                    <dd>MVP: 0.043</dd>
                                    <dd>ALPHA_MISSENSE: 0.101</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CARD9" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187796">CARD9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.144</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CARD9%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.782 to <a href="http://www.omim.org/entry/608068">Neutrophilic dermatosis, acute febrile</a> associated with MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/212050">OMIM:212050</a> Immunodeficiency 103, susceptibility to fungal infection - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.144</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-139264888-T-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs114895119">rs114895119</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/365844">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:1, LB:3, B:1)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.144</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000315908.7">CARD9:ENST00000315908.7:c.809A&gt;T:p.(Glu270Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371732.5">CARD9:ENST00000371732.5:c.809A&gt;T:p.(Glu270Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371734.3">CARD9:ENST00000371734.3:c.809A&gt;T:p.(Glu270Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.15698698</dd>
                                    
                                    <dd>REVEL: 0.040</dd>
                                    <dd>MVP: 0.157</dd>
                                    <dd>ALPHA_MISSENSE: 0.116</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0324%</dd>
                                    <dd>gnomAD_E_AMR: 0.3162%</dd>
                                    <dd>gnomAD_E_NFE: 0.4885%</dd>
                                    <dd>gnomAD_E_SAS: 0.4093%</dd>
                                    <dd>gnomAD_G_AFR: 0.0575%</dd>
                                    <dd>gnomAD_G_NFE: 0.4482%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GAS2L2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000270765">GAS2L2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.300</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.079</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.582 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with GAS2L2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001627, Abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011535, Abnormal atrial arrangement</dd></dl><dl><dt>Phenotypic similarity 0.600 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GAS2L2">GAS2L2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010418, perimembraneous ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010418, perimembraneous ventricular septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GAS2L2%0DCPLANE2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CPLANE2</a> and phenotypic similarity 0.685 to mouse mutant of CPLANE2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000644, dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010406, common atrium</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618449">OMIM:618449</a> ?Ciliary dyskinesia, primary, 41 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">ORPHA:244</a> Primary ciliary dyskinesia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.300</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.079</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-34072386-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs78157254">rs78157254</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2647672">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.079</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000254466.6">GAS2L2:ENST00000254466.6:c.2130T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000587565.1">GAS2L2:ENST00000587565.1:c.2082T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1427%</dd>
                                    <dd>gnomAD_E_AMR: 0.5110%</dd>
                                    <dd>gnomAD_E_NFE: 0.9264%</dd>
                                    <dd>gnomAD_E_SAS: 0.5736%</dd>
                                    <dd>gnomAD_G_AFR: 0.1036%</dd>
                                    <dd>gnomAD_G_AMR: 0.4717%</dd>
                                    <dd>gnomAD_G_NFE: 0.7533%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GAB1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000109458">GAB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.282</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.563 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GAB1">GAB1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000280, thin ventricular wall</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GAB1%0DMAP2K1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MAP2K1</a> and phenotypic similarity 0.921 to <a href="http://www.omim.org/entry/615279">Cardiofaciocutaneous syndrome 3</a> associated with MAP2K1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GAB1%0DMAP2K1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MAP2K1</a> and phenotypic similarity 0.406 to mouse mutant of MAP2K1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003544, abnormal vascular endothelial cell migration</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000260, abnormal angiogenesis</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/605428">OMIM:605428</a> ?Deafness, autosomal recessive 26 (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.282</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>4-144361324-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142207353">rs142207353</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262994.4">GAB1:ENST00000262994.4:c.1374A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262995.4">GAB1:ENST00000262995.4:c.1374A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000505913.1">GAB1:ENST00000505913.1:c.1065A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0145%</dd>
                                    <dd>gnomAD_E_NFE: 0.0070%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="EXPH5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000110723">EXPH5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.130</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EXPH5%0DEXOC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EXOC2</a> and phenotypic similarity 0.612 to <a href="http://www.omim.org/entry/619306">Neurodevelopmental disorder with dysmorphic facies and cerebellar hypoplasia</a> associated with EXOC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EXPH5%0DEXOC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EXOC2</a> and phenotypic similarity 0.323 to fish mutant of EXOC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/615028">OMIM:615028</a> Epidermolysis bullosa simplex 4, localized or generalized intermediate, autosomal recessive - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.130</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-108380449-G-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs114567088">rs114567088</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Strong]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.130</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265843.4">EXPH5:ENST00000265843.4:c.5785C&gt;G:p.(Pro1929Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428840.1">EXPH5:ENST00000428840.1:c.5557C&gt;G:p.(Pro1853Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000443411.1">EXPH5:ENST00000443411.1:c.5221C&gt;G:p.(Pro1741Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000525344.1">EXPH5:ENST00000525344.1:c.5764C&gt;G:p.(Pro1922Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1303883</dd>
                                    
                                    <dd>REVEL: 0.011</dd>
                                    <dd>MVP: 0.130</dd>
                                    <dd>ALPHA_MISSENSE: 0.071</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0061%</dd>
                                    <dd>gnomAD_E_NFE: 0.0403%</dd>
                                    <dd>gnomAD_E_SAS: 0.0207%</dd>
                                    <dd>gnomAD_G_NFE: 0.0194%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FSTL4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000053108">FSTL4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.411</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.411</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-132535433-G-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147873655">rs147873655</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2348891">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.411</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265342.7">FSTL4:ENST00000265342.7:c.1883C&gt;A:p.(Thr628Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.414205</dd>
                                    
                                    <dd>REVEL: 0.205</dd>
                                    <dd>MVP: 0.414</dd>
                                    <dd>ALPHA_MISSENSE: 0.396</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0147%</dd>
                                    <dd>gnomAD_E_AMR: 0.0394%</dd>
                                    <dd>gnomAD_E_NFE: 0.0600%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0454%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SAMD5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000203727">SAMD5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.398</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.398</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-147885588-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs140643252">rs140643252</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2349471">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.398</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367474.1">SAMD5:ENST00000367474.1:c.518G&gt;A:p.(Arg173His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.4131132</dd>
                                    
                                    <dd>REVEL: 0.109</dd>
                                    <dd>MVP: 0.413</dd>
                                    <dd>ALPHA_MISSENSE: 0.107</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0616%</dd>
                                    <dd>gnomAD_E_AMR: 0.0318%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.0845%</dd>
                                    <dd>gnomAD_E_SAS: 0.0425%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.0778%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="JPH3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154118">JPH3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.387</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/606438">OMIM:606438</a> Huntington disease-like 2 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98934">ORPHA:98934</a> Huntington disease-like 2 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.387</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-87678601-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199967240">rs199967240</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.387</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000284262.2">JPH3:ENST00000284262.2:c.1120G&gt;A:p.(Ala374Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.38936606</dd>
                                    
                                    <dd>REVEL: 0.194</dd>
                                    <dd>MVP: 0.389</dd>
                                    <dd>ALPHA_MISSENSE: 0.113</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0350%</dd>
                                    <dd>gnomAD_E_NFE: 0.0046%</dd>
                                    <dd>gnomAD_E_SAS: 0.0099%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MCM5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100297">MCM5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.254</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.509 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MCM5%0DRNASEH2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RNASEH2A</a> and phenotypic similarity 0.770 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=51">Aicardi-Goutières syndrome</a> associated with RNASEH2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001640, Cardiomegaly</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617564">OMIM:617564</a> ?Meier-Gorlin syndrome 8 (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.254</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>22-35806821-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs2307343">rs2307343</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1566364">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000216122.4">MCM5:ENST00000216122.4:c.837C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382011.5">MCM5:ENST00000382011.5:c.708C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0861%</dd>
                                    <dd>gnomAD_E_AMR: 0.0173%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.1149%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TIMM50" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105197">TIMM50</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.254</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.298 to <a href="http://www.omim.org/entry/617698">3-methylglutaconic aciduria, type IX</a> associated with TIMM50.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001257, Spasticity</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Phenotypic similarity 0.466 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TIMM50">TIMM50</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0004485, increased response of heart to induced stress</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0004485, increased response of heart to induced stress</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TIMM50%0DDNAJC19&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DNAJC19</a> and phenotypic similarity 0.850 to <a href="http://www.omim.org/entry/610198">3-methylglutaconic aciduria, type V</a> associated with DNAJC19.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617698">OMIM:617698</a> 3-methylglutaconic aciduria, type IX - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=505216">ORPHA:505216</a> 3-methylglutaconic aciduria type 9 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.254</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-39978787-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201425913">rs201425913</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1579434">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000314349.4">TIMM50:ENST00000314349.4:c.1092C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544017.1">TIMM50:ENST00000544017.1:c.444C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000599794.1">TIMM50:ENST00000599794.1:c.195C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000607714.1">TIMM50:ENST00000607714.1:c.783C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0492%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.0176%</dd>
                                    <dd>gnomAD_E_SAS: 0.0947%</dd>
                                    <dd>gnomAD_G_NFE: 0.0324%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RPIA" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000153574">RPIA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.298 to <a href="http://www.omim.org/entry/608611">Ribose 5-phosphate isomerase deficiency</a> associated with RPIA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001257, Spasticity</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RPIA%0DPGM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PGM1</a> and phenotypic similarity 0.788 to <a href="http://www.omim.org/entry/614921">Congenital disorder of glycosylation, type It</a> associated with PGM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/608611">OMIM:608611</a> Ribose 5-phosphate isomerase deficiency - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-88991444-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201068417">rs201068417</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1543832">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000283646.4">RPIA:ENST00000283646.4:c.228C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0032%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC5A9" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000117834">SLC5A9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.384</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.384</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-48695114-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200484428">rs200484428</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.384</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000236495.5">SLC5A9:ENST00000236495.5:c.562A&gt;G:p.(Ile188Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000420136.2">SLC5A9:ENST00000420136.2:c.466A&gt;G:p.(Ile156Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000438567.2">SLC5A9:ENST00000438567.2:c.487A&gt;G:p.(Ile163Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000533824.1">SLC5A9:ENST00000533824.1:c.550A&gt;G:p.(Ile184Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.391</dd>
                                    
                                    <dd>REVEL: 0.391</dd>
                                    <dd>MVP: 0.362</dd>
                                    <dd>ALPHA_MISSENSE: 0.086</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1186%</dd>
                                    <dd>gnomAD_G_AMR: 0.1182%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CCDC40" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000141519">CCDC40</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.262</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.088</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.582 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with CCDC40.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001627, Abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011535, Abnormal atrial arrangement</dd></dl><dl><dt>Proximity score 0.523 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CCDC40%0DCCDC39&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCDC39</a> and phenotypic similarity 0.582 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with CCDC39.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001627, Abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011535, Abnormal atrial arrangement</dd></dl><dl><dt>Proximity score 0.523 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CCDC40%0DCCDC39&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCDC39</a> and phenotypic similarity 0.707 to mouse mutant of CCDC39.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000644, dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010403, atrial septal defect</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/613808">OMIM:613808</a> Ciliary dyskinesia, primary, 15 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">ORPHA:244</a> Primary ciliary dyskinesia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.262</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.088</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-78013697-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs185006459">rs185006459</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/226490">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.088</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000269318.5">CCDC40:ENST00000269318.5:c.180G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374876.4">CCDC40:ENST00000374876.4:c.180G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374877.3">CCDC40:ENST00000374877.3:c.180G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397545.4">CCDC40:ENST00000397545.4:c.180G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.6265%</dd>
                                    <dd>gnomAD_E_AMR: 0.0116%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_AFR: 0.5513%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SRP68" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167881">SRP68</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SRP68%0DLARP7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LARP7</a> and phenotypic similarity 0.651 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=319671">Alazami syndrome</a> associated with LARP7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001631, Atrial septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001631, Atrial septal defect</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SRP68%0DLARP7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LARP7</a> and phenotypic similarity 0.474 to fish mutant of LARP7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000724, atrium dilated, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000724, atrium dilated, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/620534">OMIM:620534</a> ?Neutropenia, severe congenital, 10, autosomal recessive (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-74068543-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200249656">rs200249656</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000307877.2">SRP68:ENST00000307877.2:c.30C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539137.1">SRP68:ENST00000539137.1:c.30C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355113.5">SRP68:ENST00000355113.5:c.-170C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_EAS: 0.0063%</dd>
                                    <dd>gnomAD_E_SAS: 0.0036%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="VPS13C" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000129003">VPS13C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.256</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.094</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.332 to <a href="http://www.omim.org/entry/616840">Parkinson disease 23, autosomal recessive, early onset</a> associated with VPS13C.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0007256, Abnormal pyramidal sign</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.511 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=VPS13C%0DPCSK5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PCSK5</a> and phenotypic similarity 0.728 to mouse mutant of PCSK5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000644, dextrocardia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010402, ventricular septal defect</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/616840">OMIM:616840</a> Parkinson disease 23, autosomal recessive, early onset - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2828">ORPHA:2828</a> Young-onset Parkinson disease - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.256</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.094</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>15-62169232-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs149307561">rs149307561</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/789076">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.094</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000249837.3">VPS13C:ENST00000249837.3:c.10035A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261517.5">VPS13C:ENST00000261517.5:c.10164A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395896.4">VPS13C:ENST00000395896.4:c.10164A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395898.3">VPS13C:ENST00000395898.3:c.10035A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0436%</dd>
                                    <dd>gnomAD_E_AMR: 0.0653%</dd>
                                    <dd>gnomAD_E_EAS: 0.0055%</dd>
                                    <dd>gnomAD_E_NFE: 0.3502%</dd>
                                    <dd>gnomAD_E_SAS: 0.1427%</dd>
                                    <dd>gnomAD_G_AFR: 0.0689%</dd>
                                    <dd>gnomAD_G_NFE: 0.2856%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PSMC3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165916">PSMC3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PSMC3%0DPRICKLE1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRICKLE1</a> and phenotypic similarity 0.275 to <a href="http://www.omim.org/entry/612437">Epilepsy, progressive myoclonic 1B</a> associated with PRICKLE1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0003487, Babinski sign</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PSMC3%0DPRICKLE1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRICKLE1</a> and phenotypic similarity 0.724 to mouse mutant of PRICKLE1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000278, abnormal myocardial fiber morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010403, atrial septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PSMC3%0DPRICKLE1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRICKLE1</a> and phenotypic similarity 0.419 to fish mutant of PRICKLE1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0005185, liver decreased functionality, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/619354">OMIM:619354</a> ?Deafness, cataract, impaired intellectual development, and polyneuropathy (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>11-47446729-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs149818746">rs149818746</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000298852.3">PSMC3:ENST00000298852.3:c.228C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602866.1">PSMC3:ENST00000602866.1:c.180C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530912.1">PSMC3:ENST00000530912.1:c.160-467C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0116%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="IL37" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000125571">IL37</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=IL37%0DIL6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IL6</a> and phenotypic similarity 0.409 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=85414">Systemic-onset juvenile idiopathic arthritis</a> associated with IL6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001701, Pericarditis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001701, Pericarditis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=IL37%0DIL6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IL6</a> and phenotypic similarity 0.607 to mouse mutant of IL6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001544, abnormal cardiovascular system physiology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000281, abnormal interventricular septum morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/619398">OMIM:619398</a> ?Inflammatory bowel disease (infantile ulcerative colitis) 31 (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-113676314-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs184728003">rs184728003</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263326.3">IL37:ENST00000263326.3:c.585C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311328.2">IL37:ENST00000311328.2:c.507C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349806.3">IL37:ENST00000349806.3:c.402C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000352179.3">IL37:ENST00000352179.3:c.522C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000353225.3">IL37:ENST00000353225.3:c.465C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0983%</dd>
                                    <dd>gnomAD_E_NFE: 0.0053%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RFXAP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000133111">RFXAP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.369 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RFXAP">RFXAP</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0005542, cornea vascularization</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0005542, cornea vascularization</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RFXAP%0DKDM6A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KDM6A</a> and phenotypic similarity 0.740 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2322">Kabuki syndrome</a> associated with KDM6A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001671, Abnormal cardiac septum morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001671, Abnormal cardiac septum morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RFXAP%0DKDM6A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KDM6A</a> and phenotypic similarity 0.672 to mouse mutant of KDM6A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0010503, myocardial trabeculae hypoplasia</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010725, thin interventricular septum</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RFXAP%0DKDM6A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KDM6A</a> and phenotypic similarity 0.323 to fish mutant of KDM6A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0000038, pericardium edematous, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=572">ORPHA:572</a> Immunodeficiency by defective expression of MHC class II - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>13-37393971-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201205150">rs201205150</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1153187">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000255476.2">RFXAP:ENST00000255476.2:c.477G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0999%</dd>
                                    <dd>gnomAD_E_NFE: 0.0143%</dd>
                                    <dd>gnomAD_G_NFE: 0.0195%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GPI" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105220">GPI</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.255</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.093</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.455 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GPI">GPI</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0002833, increased heart weight</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0002833, increased heart weight</dd></dl><dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GPI%0DTPI1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TPI1</a> and phenotypic similarity 0.962 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=868">Triose phosphate-isomerase deficiency</a> associated with TPI1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001639, Hypertrophic cardiomyopathy</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/613470">OMIM:613470</a> Hemolytic anemia, nonspherocytic, due to glucose phosphate isomerase deficiency - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.255</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.093</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-34868756-C-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1062480">rs1062480</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/780175">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.093</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356487.5">GPI:ENST00000356487.5:c.603C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000415930.3">GPI:ENST00000415930.3:c.636C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000586425.1">GPI:ENST00000586425.1:c.603C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0738%</dd>
                                    <dd>gnomAD_E_AMR: 0.1070%</dd>
                                    <dd>gnomAD_E_NFE: 0.4273%</dd>
                                    <dd>gnomAD_E_SAS: 0.0915%</dd>
                                    <dd>gnomAD_G_AFR: 0.1149%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.4280%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CLCA4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000016602">CLCA4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CLCA4%0DCALM2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CALM2</a> and phenotypic similarity 0.697 to <a href="http://www.omim.org/entry/616249">Long QT syndrome 15</a> associated with CALM2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0030682, Left ventricular noncompaction</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0030682, Left ventricular noncompaction</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CLCA4%0DCALM2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CALM2</a> and phenotypic similarity 0.483 to mouse mutant of CALM2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001636, irregular heartbeat</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001636, irregular heartbeat</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=586">ORPHA:586</a> Cystic fibrosis (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-87045902-A-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1932809">rs1932809</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370563.3">CLCA4:ENST00000370563.3:c.2634A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0326%</dd>
                                    <dd>gnomAD_E_AMR: 0.0440%</dd>
                                    <dd>gnomAD_E_EAS: 0.0169%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_SAS: 0.0066%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_EAS: 0.0656%</dd>
                                    <dd>gnomAD_G_NFE: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TOGARAM1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198718">TOGARAM1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.243</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.105</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.548 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=475">Joubert syndrome</a> associated with TOGARAM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001696, Situs inversus totalis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001696, Situs inversus totalis</dd></dl><dl><dt>Phenotypic similarity 0.486 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TOGARAM1">TOGARAM1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0001914, hemorrhage</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0001914, hemorrhage</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/619185">OMIM:619185</a> Joubert syndrome 37 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=475">ORPHA:475</a> Joubert syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.243</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.105</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-45481254-A-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs145006760">rs145006760</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1703106">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.105</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361462.2">TOGARAM1:ENST00000361462.2:c.3214A&gt;T:p.(Ile1072Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361577.3">TOGARAM1:ENST00000361577.3:c.3214A&gt;T:p.(Ile1072Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382233.2">TOGARAM1:ENST00000382233.2:c.2981A&gt;T:p.(Tyr994Phe)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1075</dd>
                                    
                                    <dd>REVEL: 0.085</dd>
                                    <dd>MVP: 0.040</dd>
                                    <dd>ALPHA_MISSENSE: 0.108</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0438%</dd>
                                    <dd>gnomAD_E_AMR: 0.0503%</dd>
                                    <dd>gnomAD_E_NFE: 0.1476%</dd>
                                    <dd>gnomAD_E_SAS: 0.0536%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.0519%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ST3GAL3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000126091">ST3GAL3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.095</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ST3GAL3%0DRELB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RELB</a> and phenotypic similarity 0.681 to mouse mutant of RELB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003325, decreased liver function</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/611090">OMIM:611090</a> Intellectual developmental disorder, autosomal recessive 12 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/615006">OMIM:615006</a> Developmental and epileptic encephalopathy 15 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3451">ORPHA:3451</a> Infantile spasms syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.095</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-44365315-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201204481">rs201204481</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/199121">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:1, LB:1, B:1)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.095</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262915.3">ST3GAL3:ENST00000262915.3:c.867C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000332628.6">ST3GAL3:ENST00000332628.6:c.567C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000347631.2">ST3GAL3:ENST00000347631.2:c.705C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000351035.3">ST3GAL3:ENST00000351035.3:c.774C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000353126.3">ST3GAL3:ENST00000353126.3:c.660C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361392.4">ST3GAL3:ENST00000361392.4:c.660C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361400.4">ST3GAL3:ENST00000361400.4:c.612C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361746.4">ST3GAL3:ENST00000361746.4:c.867C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372368.2">ST3GAL3:ENST00000372368.2:c.822C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372369.1">ST3GAL3:ENST00000372369.1:c.660C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372372.2">ST3GAL3:ENST00000372372.2:c.774C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372374.2">ST3GAL3:ENST00000372374.2:c.567C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372375.2">ST3GAL3:ENST00000372375.2:c.822C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000531993.1">ST3GAL3:ENST00000531993.1:c.612C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000533933.1">ST3GAL3:ENST00000533933.1:c.660C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000330208.2">ST3GAL3:ENST00000330208.2:c.397+5166C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335430.6">ST3GAL3:ENST00000335430.6:c.509+380C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361812.4">ST3GAL3:ENST00000361812.4:c.442+5166C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372362.2">ST3GAL3:ENST00000372362.2:c.397+5166C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372367.1">ST3GAL3:ENST00000372367.1:c.554+380C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372377.4">ST3GAL3:ENST00000372377.4:c.397+5166C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000528371.1">ST3GAL3:ENST00000528371.1:c.509+380C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000531451.1">ST3GAL3:ENST00000531451.1:c.349+5166C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000531816.1">ST3GAL3:ENST00000531816.1:c.255-30489C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545417.1">ST3GAL3:ENST00000545417.1:c.442+5166C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372365.1">ST3GAL3:ENST00000372365.1:c.*22+354C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372366.1">ST3GAL3:ENST00000372366.1:c.*22+354C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.3238%</dd>
                                    <dd>gnomAD_E_EAS: 0.0217%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LZTS1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000061337">LZTS1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.254</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.093</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LZTS1%0DNFIX&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NFIX</a> and phenotypic similarity 0.648 to <a href="http://www.omim.org/entry/602535">Marshall-Smith syndrome</a> associated with NFIX.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001629, Ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/133239">OMIM:133239</a> Esophageal squamous cell carcinoma, somatic - somatic</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.254</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.093</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>8-20111016-C-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs141146887">rs141146887</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1588990">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.093</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265801.6">LZTS1:ENST00000265801.6:c.426G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381569.1">LZTS1:ENST00000381569.1:c.426G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000522290.1">LZTS1:ENST00000522290.1:c.426G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.4132%</dd>
                                    <dd>gnomAD_E_AMR: 0.0380%</dd>
                                    <dd>gnomAD_G_AFR: 0.4131%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SKIC3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198677">SKIC3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.096</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.621 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84064">Syndromic diarrhea</a> associated with SKIC3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001627, Abnormal heart morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001629, Ventricular septal defect</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SKIC3%0DMIPEP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MIPEP</a> and phenotypic similarity 0.957 to <a href="http://www.omim.org/entry/617228">Combined oxidative phosphorylation deficiency 31</a> associated with MIPEP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001639, Hypertrophic cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0030682, Left ventricular noncompaction</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/222470">OMIM:222470</a> Trichohepatoenteric syndrome 1 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84064">ORPHA:84064</a> Syndromic diarrhea - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.096</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>5-94876493-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200646357">rs200646357</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1600627">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.096</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358746.2">SKIC3:ENST00000358746.2:c.444A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.2429%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ALMS1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000116127">ALMS1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.256</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.083</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.745 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=64">Alström syndrome</a> associated with ALMS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001685, Myocardial fibrosis</dd></dl><dl><dt>Phenotypic similarity 0.511 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=ALMS1">ALMS1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0005415, cardiac ventricle morphology, abnormal</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ALMS1%0DSDCCAG8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SDCCAG8</a> and phenotypic similarity 0.744 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=110">Bardet-Biedl syndrome</a> associated with SDCCAG8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001627, Abnormal heart morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ALMS1%0DSDCCAG8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SDCCAG8</a> and phenotypic similarity 0.360 to mouse mutant of SDCCAG8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000820, abnormal choroid plexus morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0000820, abnormal choroid plexus morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/203800">OMIM:203800</a> Alstrom syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=64">ORPHA:64</a> Alström syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.256</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.083</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-73828402-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201728850">rs201728850</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/581362">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                                <span>(VUS:5, LB:3)</span>
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.083</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264448.6">ALMS1:ENST00000264448.6:c.11950A&gt;G:p.(Ile3984Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409009.1">ALMS1:ENST00000409009.1:c.11824A&gt;G:p.(Ile3942Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.08469159</dd>
                                    
                                    <dd>MVP: 0.085</dd>
                                    <dd>ALPHA_MISSENSE: 0.065</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0064%</dd>
                                    <dd>gnomAD_E_AMR: 0.1677%</dd>
                                    <dd>gnomAD_E_NFE: 0.0194%</dd>
                                    <dd>gnomAD_E_SAS: 0.0163%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="XPC" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154767">XPC</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.253</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.082</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.382 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=910">Xeroderma pigmentosum</a> associated with XPC.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0000135, Hypogonadism</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0000524, Conjunctival telangiectasia</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=XPC%0DXRCC4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to XRCC4</a> and phenotypic similarity 0.777 to <a href="http://www.omim.org/entry/616541">Short stature, microcephaly, and endocrine dysfunction</a> associated with XRCC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001644, Dilated cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001644, Dilated cardiomyopathy</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/278720">OMIM:278720</a> Xeroderma pigmentosum, group C - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=910">ORPHA:910</a> Xeroderma pigmentosum - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.253</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.082</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-14200136-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200157068">rs200157068</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.082</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000285021.7">XPC:ENST00000285021.7:c.1247G&gt;A:p.(Arg416His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449060.2">XPC:ENST00000449060.2:c.1136G&gt;A:p.(Arg379His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.0825</dd>
                                    
                                    <dd>REVEL: 0.027</dd>
                                    <dd>MVP: 0.081</dd>
                                    <dd>ALPHA_MISSENSE: 0.083</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PSMC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100764">PSMC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.081</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.605 to <a href="http://www.omim.org/entry/620071">?Birk-Aharoni syndrome</a> associated with PSMC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0011623, Muscular ventricular septal defect</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0011623, Muscular ventricular septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PSMC1%0DPRICKLE1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRICKLE1</a> and phenotypic similarity 0.275 to <a href="http://www.omim.org/entry/612437">Epilepsy, progressive myoclonic 1B</a> associated with PRICKLE1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0003487, Babinski sign</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PSMC1%0DPRICKLE1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRICKLE1</a> and phenotypic similarity 0.724 to mouse mutant of PRICKLE1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0000278, abnormal myocardial fiber morphology</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0010403, atrial septal defect</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PSMC1%0DPRICKLE1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRICKLE1</a> and phenotypic similarity 0.419 to fish mutant of PRICKLE1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0005185, liver decreased functionality, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/620071">OMIM:620071</a> ?Birk-Aharoni syndrome (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.081</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>14-90729738-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs143172121">rs143172121</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/770241">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.081</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261303.8">PSMC1:ENST00000261303.8:c.231A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543772.2">PSMC1:ENST00000543772.2:c.12A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0847%</dd>
                                    <dd>gnomAD_E_AMR: 0.8709%</dd>
                                    <dd>gnomAD_E_NFE: 0.3287%</dd>
                                    <dd>gnomAD_E_SAS: 0.4243%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_AMR: 0.7075%</dd>
                                    <dd>gnomAD_G_NFE: 0.3110%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DTYMK" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000168393">DTYMK</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.079</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.323 to <a href="http://www.omim.org/entry/619847">Neurodegeneration, childhood-onset, with progressive microcephaly</a> associated with DTYMK.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001257, Spasticity</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DTYMK%0DRRM2B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RRM2B</a> and phenotypic similarity 0.786 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=329336">Adult-onset chronic progressive external ophthalmoplegia with mitochondrial myopathy</a> associated with RRM2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001638, Cardiomyopathy</dd><dd>HP:0001671, Abnormal cardiac septum morphology - HP:0001638, Cardiomyopathy</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DTYMK%0DRRM2B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RRM2B</a> and phenotypic similarity 0.526 to mouse mutant of RRM2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - MP:0003141, cardiac fibrosis</dd><dd>HP:0001671, Abnormal cardiac septum morphology - MP:0003141, cardiac fibrosis</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/619847">OMIM:619847</a> Neurodegeneration, childhood-onset, with progressive microcephaly - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.079</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-242617882-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150300178">rs150300178</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2652117">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.079</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000305784.2">DTYMK:ENST00000305784.2:c.513G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.9412%</dd>
                                    <dd>gnomAD_E_AMR: 0.5204%</dd>
                                    <dd>gnomAD_E_NFE: 0.1187%</dd>
                                    <dd>gnomAD_E_SAS: 0.0098%</dd>
                                    <dd>gnomAD_G_AFR: 0.8607%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.0778%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TBCD" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000141556">TBCD</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.217</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.337 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=496641">Early-onset progressive diffuse brain atrophy-microcephaly-muscle weakness-optic atrophy syndrome</a> associated with TBCD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0002445, Tetraplegia</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl><dt>Phenotypic similarity 0.435 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=TBCD">TBCD</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0001671, Abnormal cardiac septum morphology - ZP:0001192, heart edematous, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617193">OMIM:617193</a> Encephalopathy, progressive, early-onset, with brain atrophy and thin corpus callosum - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=496641">ORPHA:496641</a> Early-onset progressive diffuse brain atrophy-microcephaly-muscle weakness-optic atrophy syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.217</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-80887358-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201850842">rs201850842</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/747992">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355528.4">TBCD:ENST00000355528.4:c.2973C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539345.2">TBCD:ENST00000539345.2:c.2973C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0195%</dd>
                                    <dd>gnomAD_E_AMR: 0.0116%</dd>
                                    <dd>gnomAD_E_NFE: 0.0080%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TEX47" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000164645">TEX47</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.343</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.343</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-88424221-T-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200802688">rs200802688</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.343</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000297203.2">TEX47:ENST00000297203.2:c.36A&gt;T:p.(Lys12Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333190.4">TEX47:ENST00000333190.4:c.108+34823T&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3788475</dd>
                                    
                                    <dd>REVEL: 0.092</dd>
                                    <dd>MVP: 0.379</dd>
                                    <dd>ALPHA_MISSENSE: 0.236</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.5255%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AMR: 0.2364%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNG1A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000172785">ZNG1A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.327</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.327</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-178895-A-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200809115">rs200809115</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PM2_Supporting, BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.327</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356521.4">ZNG1A:ENST00000356521.4:c.75T&gt;G:p.(Ile25Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377400.4">ZNG1A:ENST00000377400.4:c.75T&gt;G:p.(Ile25Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377447.3">ZNG1A:ENST00000377447.3:c.75T&gt;G:p.(Ile25Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382393.1">ZNG1A:ENST00000382393.1:c.75T&gt;G:p.(Ile25Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382447.4">ZNG1A:ENST00000382447.4:c.75T&gt;G:p.(Ile25Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000314367.10">ZNG1A:ENST00000314367.10:c.-132T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382389.1">ZNG1A:ENST00000382389.1:c.-132T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000431099.2">ZNG1A:ENST00000431099.2:c.-132T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3274013</dd>
                                    
                                    <dd>REVEL: 0.094</dd>
                                    <dd>MVP: 0.327</dd>
                                    <dd>ALPHA_MISSENSE: 0.092</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TBC1D2B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167202">TBC1D2B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.283</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/619323">OMIM:619323</a> Neurodevelopmental disorder with seizures and gingival overgrowth - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=397973">ORPHA:397973</a> Intellectual disability-obesity-prognathism-eye and skin anomalies syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.283</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>15-78290635-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200408968">rs200408968</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.283</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000300584.3">TBC1D2B:ENST00000300584.3:c.2759G&gt;A:p.(Arg920Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409931.3">TBC1D2B:ENST00000409931.3:c.2707G&gt;A:p.(Asp903Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3513</dd>
                                    
                                    <dd>REVEL: 0.128</dd>
                                    <dd>ALPHA_MISSENSE: 0.351</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1136%</dd>
                                    <dd>gnomAD_E_AMR: 0.1505%</dd>
                                    <dd>gnomAD_E_EAS: 0.0956%</dd>
                                    <dd>gnomAD_E_NFE: 0.8898%</dd>
                                    <dd>gnomAD_E_SAS: 0.5285%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ALS2CL" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178038">ALS2CL</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.258</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.258</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-46713507-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201171398">rs201171398</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.258</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000318962.4">ALS2CL:ENST00000318962.4:c.2551C&gt;T:p.(Arg851Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000383742.3">ALS2CL:ENST00000383742.3:c.592C&gt;T:p.(Arg198Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000415953.1">ALS2CL:ENST00000415953.1:c.2551C&gt;T:p.(Arg851Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.26266214</dd>
                                    
                                    <dd>REVEL: 0.033</dd>
                                    <dd>MVP: 0.263</dd>
                                    <dd>ALPHA_MISSENSE: 0.089</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0260%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CCNJ" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000107443">CCNJ</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.255</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.255</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-97817688-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs193097883">rs193097883</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.255</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265992.5">CCNJ:ENST00000265992.5:c.809G&gt;A:p.(Arg270Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000403870.3">CCNJ:ENST00000403870.3:c.806G&gt;A:p.(Arg269Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000465148.2">CCNJ:ENST00000465148.2:c.842G&gt;A:p.(Arg281Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000534974.1">CCNJ:ENST00000534974.1:c.809G&gt;A:p.(Arg270Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.2551177</dd>
                                    
                                    <dd>REVEL: 0.038</dd>
                                    <dd>MVP: 0.255</dd>
                                    <dd>ALPHA_MISSENSE: 0.068</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                    <dd>gnomAD_E_SAS: 0.0098%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TBC1D28" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000189375">TBC1D28</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.246</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.246</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-18541968-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199658782">rs199658782</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.246</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000345096.4">TBC1D28:ENST00000345096.4:c.245T&gt;C:p.(Leu82Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405044.1">TBC1D28:ENST00000405044.1:c.245T&gt;C:p.(Leu82Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.25216284</dd>
                                    
                                    <dd>REVEL: 0.079</dd>
                                    <dd>MVP: 0.252</dd>
                                    <dd>ALPHA_MISSENSE: 0.211</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0129%</dd>
                                    <dd>gnomAD_E_AMR: 0.0492%</dd>
                                    <dd>gnomAD_E_NFE: 0.1731%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                    <dd>gnomAD_G_NFE: 0.1427%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CBARP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000099625">CBARP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.223</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.223</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-1234788-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200775133">rs200775133</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.223</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382477.2">CBARP:ENST00000382477.2:c.469T&gt;C:p.(Ser157Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000215376.6">CBARP:ENST00000215376.6:c.438-47T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000590083.1">CBARP:ENST00000590083.1:c.456-47T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.224</dd>
                                    
                                    <dd>REVEL: 0.224</dd>
                                    <dd>MVP: 0.126</dd>
                                    <dd>ALPHA_MISSENSE: 0.101</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0169%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SAMD7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187033">SAMD7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.221</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.221</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-169656188-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs138055938">rs138055938</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.221</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335556.3">SAMD7:ENST00000335556.3:c.1235C&gt;T:p.(Thr412Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428432.2">SAMD7:ENST00000428432.2:c.1235C&gt;T:p.(Thr412Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.24896431</dd>
                                    
                                    <dd>REVEL: 0.052</dd>
                                    <dd>MVP: 0.249</dd>
                                    <dd>ALPHA_MISSENSE: 0.090</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.4618%</dd>
                                    <dd>gnomAD_E_AMR: 0.1648%</dd>
                                    <dd>gnomAD_E_NFE: 0.0361%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.5967%</dd>
                                    <dd>gnomAD_G_AMR: 0.2364%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PCLO" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000186472">PCLO</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.205</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.298 to <a href="http://www.omim.org/entry/608027">?Pontocerebellar hypoplasia, type 3</a> associated with PCLO.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001639, Hypertrophic cardiomyopathy - HP:0001257, Spasticity</dd><dd>HP:0001671, Abnormal cardiac septum morphology -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/608027">OMIM:608027</a> ?Pontocerebellar hypoplasia, type 3 (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.205</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-82784832-C-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs77721383">rs77721383</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1578962">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.205</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333891.9">PCLO:ENST00000333891.9:c.1125G&gt;C:p.(Gln375His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423517.2">PCLO:ENST00000423517.2:c.1125G&gt;C:p.(Gln375His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.23244226</dd>
                                    
                                    <dd>REVEL: 0.044</dd>
                                    <dd>MVP: 0.232</dd>
                                    <dd>ALPHA_MISSENSE: 0.209</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0667%</dd>
                                    <dd>gnomAD_E_AMR: 0.4976%</dd>
                                    <dd>gnomAD_E_EAS: 0.6306%</dd>
                                    <dd>gnomAD_E_NFE: 0.1643%</dd>
                                    <dd>gnomAD_E_SAS: 0.2636%</dd>
                                    <dd>gnomAD_G_AMR: 0.1263%</dd>
                                    <dd>gnomAD_G_EAS: 0.1312%</dd>
                                    <dd>gnomAD_G_NFE: 0.0498%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>7-82532000-T-C [1|0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs182078347">rs182078347</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    <div class="col-sm-12">
                        <b>ClinVar: </b>
                        
                        
                            
                            
                            
                            
                            
                            <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                            <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/708091">(CRITERIA_PROVIDED_CONFLICTING_INTERPRETATIONS)</a>
                            <span>(VUS:1, LB:2)</span>
                        
                    </div>
                    <div class="col-sm-6">
                        <b>Variant score: 0.082</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333891.9">PCLO:ENST00000333891.9:c.13495A&gt;G:p.(Ile4499Val)</a>
                                
                            </dd>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423517.2">PCLO:ENST00000423517.2:c.13495A&gt;G:p.(Ile4499Val)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            <dd>Best Score: 0.10259893</dd>
                            
                            <dd>REVEL: 0.063</dd>
                            <dd>MVP: 0.103</dd>
                            <dd>ALPHA_MISSENSE: 0.101</dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>gnomAD_E_AFR: 0.0194%</dd>
                            <dd>gnomAD_E_AMR: 0.9067%</dd>
                            <dd>gnomAD_G_AMR: 0.7075%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GPRC5D" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000111291">GPRC5D</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.201</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.201</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-13095214-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148322311">rs148322311</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.201</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000228887.1">GPRC5D:ENST00000228887.1:c.944G&gt;A:p.(Gly315Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396333.3">GPRC5D:ENST00000396333.3:c.896-1431G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.20208623</dd>
                                    
                                    <dd>REVEL: 0.100</dd>
                                    <dd>MVP: 0.202</dd>
                                    <dd>ALPHA_MISSENSE: 0.185</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0231%</dd>
                                    <dd>gnomAD_E_NFE: 0.0185%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PKHD1L1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000205038">PKHD1L1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.179</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.179</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-110410693-C-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs188649530">rs188649530</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2342996">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.179</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378402.5">PKHD1L1:ENST00000378402.5:c.928C&gt;A:p.(Pro310Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.18</dd>
                                    
                                    <dd>REVEL: 0.180</dd>
                                    <dd>MVP: 0.030</dd>
                                    <dd>ALPHA_MISSENSE: 0.056</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0113%</dd>
                                    <dd>gnomAD_E_NFE: 0.0070%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="QRICH2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000129646">QRICH2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.176</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618341">OMIM:618341</a> Spermatogenic failure 35 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.176</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-74288421-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs6501878">rs6501878</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3049624">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.176</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262765.5">QRICH2:ENST00000262765.5:c.1889T&gt;C:p.(Ile630Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.2093</dd>
                                    
                                    <dd>REVEL: 0.017</dd>
                                    <dd>MVP: 0.014</dd>
                                    <dd>ALPHA_MISSENSE: 0.209</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.4606%</dd>
                                    <dd>gnomAD_E_AMR: 0.0148%</dd>
                                    <dd>gnomAD_E_EAS: 0.0112%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                    <dd>gnomAD_E_SAS: 0.0067%</dd>
                                    <dd>gnomAD_G_AFR: 0.7791%</dd>
                                    <dd>gnomAD_G_AMR: 0.1241%</dd>
                                    <dd>gnomAD_G_NFE: 0.0138%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>17-74287229-T-C [0|1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs149430693">rs149430693</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    <div class="col-sm-12">
                        <b>ClinVar: </b>
                        
                        
                            
                            <span class="label label-info">LIKELY_BENIGN</span>
                            
                            
                            
                            
                            <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3041329">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                            
                        
                    </div>
                    <div class="col-sm-6">
                        <b>Variant score: 0.098</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262765.5">QRICH2:ENST00000262765.5:c.3081A&gt;G:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>gnomAD_E_AMR: 0.0838%</dd>
                            <dd>gnomAD_E_NFE: 0.0211%</dd>
                            <dd>gnomAD_E_SAS: 0.1437%</dd>
                            <dd>gnomAD_G_AMR: 0.1179%</dd>
                            <dd>gnomAD_G_NFE: 0.0130%</dd>
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>17-74288422-T-C [1|0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs6501879">rs6501879</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    <div class="col-sm-12">
                        <b>ClinVar: </b>
                        
                        
                            
                            <span class="label label-info">LIKELY_BENIGN</span>
                            
                            
                            
                            
                            <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3049319">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                            
                        
                    </div>
                    <div class="col-sm-6">
                        <b>Variant score: 0.095</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262765.5">QRICH2:ENST00000262765.5:c.1888A&gt;G:p.(Ile630Val)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            <dd>Best Score: 0.1145</dd>
                            
                            <dd>REVEL: 0.007</dd>
                            <dd>MVP: 0.014</dd>
                            <dd>ALPHA_MISSENSE: 0.115</dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>gnomAD_E_AFR: 0.4608%</dd>
                            <dd>gnomAD_E_AMR: 0.0207%</dd>
                            <dd>gnomAD_E_EAS: 0.0281%</dd>
                            <dd>gnomAD_E_NFE: 0.0152%</dd>
                            <dd>gnomAD_E_SAS: 0.0200%</dd>
                            <dd>gnomAD_G_AFR: 0.8226%</dd>
                            <dd>gnomAD_G_AMR: 0.1316%</dd>
                            <dd>gnomAD_G_NFE: 0.0477%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DEUP1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165325">DEUP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.166</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.166</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-93141548-G-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs139654175">rs139654175</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.166</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000298050.3">DEUP1:ENST00000298050.3:c.1478G&gt;T:p.(Arg493Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000525646.1">DEUP1:ENST00000525646.1:c.704G&gt;T:p.(Arg235Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1689333</dd>
                                    
                                    <dd>REVEL: 0.021</dd>
                                    <dd>MVP: 0.169</dd>
                                    <dd>ALPHA_MISSENSE: 0.104</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1321%</dd>
                                    <dd>gnomAD_E_NFE: 0.0478%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.1233%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FAM81A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000157470">FAM81A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.153</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.153</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>15-59813464-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs143217762">rs143217762</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.153</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000288228.5">FAM81A:ENST00000288228.5:c.994G&gt;A:p.(Val332Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.18486798</dd>
                                    
                                    <dd>REVEL: 0.141</dd>
                                    <dd>MVP: 0.185</dd>
                                    <dd>ALPHA_MISSENSE: 0.061</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0259%</dd>
                                    <dd>gnomAD_E_AMR: 0.4436%</dd>
                                    <dd>gnomAD_E_EAS: 0.0951%</dd>
                                    <dd>gnomAD_E_NFE: 0.0373%</dd>
                                    <dd>gnomAD_E_SAS: 0.0459%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_AMR: 0.8255%</dd>
                                    <dd>gnomAD_G_EAS: 0.1923%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LCA5L" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000157578">LCA5L</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.153</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.153</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>21-40778319-C-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201717530">rs201717530</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2384505">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.153</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000288350.3">LCA5L:ENST00000288350.3:c.1502G&gt;C:p.(Gly501Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358268.2">LCA5L:ENST00000358268.2:c.1502G&gt;C:p.(Gly501Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380671.2">LCA5L:ENST00000380671.2:c.1502G&gt;C:p.(Gly501Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541890.1">LCA5L:ENST00000541890.1:c.452-169C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.15839623</dd>
                                    
                                    <dd>REVEL: 0.028</dd>
                                    <dd>MVP: 0.158</dd>
                                    <dd>ALPHA_MISSENSE: 0.074</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0185%</dd>
                                    <dd>gnomAD_E_AMR: 0.2169%</dd>
                                    <dd>gnomAD_G_AFR: 0.0461%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GARIN3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170613">GARIN3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.142</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.142</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-156593062-T-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs144784850">rs144784850</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.142</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302938.4">GARIN3:ENST00000302938.4:c.118A&gt;C:p.(Ile40Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1422</dd>
                                    
                                    <dd>REVEL: 0.049</dd>
                                    <dd>MVP: 0.140</dd>
                                    <dd>ALPHA_MISSENSE: 0.142</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0116%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ADGRG5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000159618">ADGRG5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.137</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.137</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-57609392-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150635750">rs150635750</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.137</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340339.4">ADGRG5:ENST00000340339.4:c.1529C&gt;T:p.(Ser510Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349457.3">ADGRG5:ENST00000349457.3:c.1529C&gt;T:p.(Ser510Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.14142282</dd>
                                    
                                    <dd>REVEL: 0.080</dd>
                                    <dd>MVP: 0.141</dd>
                                    <dd>ALPHA_MISSENSE: 0.094</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.2031%</dd>
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.1608%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SH2D4B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178217">SH2D4B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.101</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.101</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-82403836-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs113555087">rs113555087</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.101</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000470604.2">SH2D4B:ENST00000470604.2:c.1291G&gt;A:p.(Glu431Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313455.4">SH2D4B:ENST00000313455.4:c.929G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339284.2">SH2D4B:ENST00000339284.2:c.1073G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1114</dd>
                                    
                                    <dd>ALPHA_MISSENSE: 0.111</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.4624%</dd>
                                    <dd>gnomAD_E_AMR: 0.1396%</dd>
                                    <dd>gnomAD_E_NFE: 0.0309%</dd>
                                    <dd>gnomAD_G_AFR: 0.5169%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.0324%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PRUNE2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000106772">PRUNE2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>9-79318378-G-A [1|0]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376718.3">PRUNE2:ENST00000376718.3:c.8151C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428286.1">PRUNE2:ENST00000428286.1:c.7074C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC2A11" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000133460">SLC2A11</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>22-24210736-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201945538">rs201945538</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000316185.8">SLC2A11:ENST00000316185.8:c.198C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000345044.6">SLC2A11:ENST00000345044.6:c.189C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000398356.2">SLC2A11:ENST00000398356.2:c.210C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000403208.3">SLC2A11:ENST00000403208.3:c.189C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405847.1">SLC2A11:ENST00000405847.1:c.189C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000433835.3">SLC2A11:ENST00000433835.3:c.84C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0029%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KRT78" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170423">KRT78</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>12-53242613-G-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199880767">rs199880767</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000304620.4">KRT78:ENST00000304620.4:c.102C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_EAS: 0.0055%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GPATCH11" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000152133">GPATCH11</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-37319078-T-C [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201830084">rs201830084</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000281932.5">GPATCH11:ENST00000281932.5:c.27T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409774.1">GPATCH11:ENST00000409774.1:c.414T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000608836.1">GPATCH11:ENST00000608836.1:c.336T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0058%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AHNAK2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185567">AHNAK2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>14-105412386-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200059681">rs200059681</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333244.5">AHNAK2:ENST00000333244.5:c.9402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000557457.1">AHNAK2:ENST00000557457.1:c.-220-5071G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0175%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZZEF1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000074755">ZZEF1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-4015916-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201639173">rs201639173</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381638.2">ZZEF1:ENST00000381638.2:c.1053C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0062%</dd>
                                    <dd>gnomAD_E_AMR: 0.0260%</dd>
                                    <dd>gnomAD_E_NFE: 0.0203%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0130%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF777" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196453">ZNF777</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>7-149151302-A-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200964721">rs200964721</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000247930.4">ZNF777:ENST00000247930.4:c.873T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0124%</dd>
                                    <dd>gnomAD_E_AMR: 0.0549%</dd>
                                    <dd>gnomAD_E_NFE: 0.0554%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0389%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FBN3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000142449">FBN3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-8167639-T-C [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200978594">rs200978594</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/2080483">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000270509.2">FBN3:ENST00000270509.2:c.5058A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000600128.1">FBN3:ENST00000600128.1:c.5058A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000601739.1">FBN3:ENST00000601739.1:c.5058A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1012%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MAS1L" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204687">MAS1L</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>6-29454651-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs141570832">rs141570832</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377127.3">MAS1L:ENST00000377127.3:c.1029G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0752%</dd>
                                    <dd>gnomAD_E_NFE: 0.0053%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PCDHGC5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000240764">PCDHGC5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>5-140869311-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs185263705">rs185263705</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000252087.1">PCDHGC5:ENST00000252087.1:c.504C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394576.2">PCDHGC5:ENST00000394576.2:c.2425-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000517417.1">PCDHGC5:ENST00000517417.1:c.2422-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308177.3">PCDHGC5:ENST00000308177.3:c.2431-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000306593.1">PCDHGC5:ENST00000306593.1:c.2442+2129C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000252085.3">PCDHGC5:ENST00000252085.3:c.2425-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000520790.1">PCDHGC5:ENST00000520790.1:c.2419-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000518069.1">PCDHGC5:ENST00000518069.1:c.2422-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000518325.1">PCDHGC5:ENST00000518325.1:c.2425-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000517434.1">PCDHGC5:ENST00000517434.1:c.2425-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000398604.2">PCDHGC5:ENST00000398604.2:c.2425-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000398610.2">PCDHGC5:ENST00000398610.2:c.2437-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000398587.2">PCDHGC5:ENST00000398587.2:c.2434-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000518882.1">PCDHGC5:ENST00000518882.1:c.1879-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000522605.1">PCDHGC5:ENST00000522605.1:c.2422-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000519479.1">PCDHGC5:ENST00000519479.1:c.2398-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000398594.2">PCDHGC5:ENST00000398594.2:c.2416-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000523390.1">PCDHGC5:ENST00000523390.1:c.2410-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000253812.6">PCDHGC5:ENST00000253812.6:c.2425-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000573521.1">PCDHGC5:ENST00000573521.1:c.2425-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000576222.1">PCDHGC5:ENST00000576222.1:c.2416-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000571252.1">PCDHGC5:ENST00000571252.1:c.2422-5063C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0463%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PLEKHN1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187583">PLEKHN1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.096</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.096</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-909314-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs138435605">rs138435605</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.096</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379407.3">PLEKHN1:ENST00000379407.3:c.1431C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379409.2">PLEKHN1:ENST00000379409.2:c.1692C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379410.3">PLEKHN1:ENST00000379410.3:c.1536C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1009%</dd>
                                    <dd>gnomAD_E_AMR: 0.1509%</dd>
                                    <dd>gnomAD_E_NFE: 0.2816%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.0460%</dd>
                                    <dd>gnomAD_G_NFE: 0.2270%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DAGLB" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000164535">DAGLB</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.095</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.095</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>7-6456263-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs145934843">rs145934843</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.095</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000297056.6">DAGLB:ENST00000297056.6:c.1422G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421761.2">DAGLB:ENST00000421761.2:c.654G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000425398.2">DAGLB:ENST00000425398.2:c.1035G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000436575.1">DAGLB:ENST00000436575.1:c.1299G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428902.2">DAGLB:ENST00000428902.2:c.1005+36G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0249%</dd>
                                    <dd>gnomAD_E_AMR: 0.2927%</dd>
                                    <dd>gnomAD_E_NFE: 0.1062%</dd>
                                    <dd>gnomAD_E_SAS: 0.0799%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_NFE: 0.0778%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF532" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000074657">ZNF532</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.095</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.095</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>18-56586284-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148024897">rs148024897</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/3024966">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.095</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336078.4">ZNF532:ENST00000336078.4:c.765C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589288.1">ZNF532:ENST00000589288.1:c.765C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000591083.1">ZNF532:ENST00000591083.1:c.765C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000591230.1">ZNF532:ENST00000591230.1:c.765C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000591808.1">ZNF532:ENST00000591808.1:c.765C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0928%</dd>
                                    <dd>gnomAD_E_AMR: 0.0724%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.3220%</dd>
                                    <dd>gnomAD_E_SAS: 0.0915%</dd>
                                    <dd>gnomAD_G_AFR: 0.0460%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.2658%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DNAH7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000118997">DNAH7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.094</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/620356">OMIM:620356</a> Ciliary dyskinesia, primary, 50 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.094</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-196753123-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200813965">rs200813965</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.094</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000312428.6">DNAH7:ENST00000312428.6:c.5265C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.3827%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SUSD3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000157303">SUSD3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.091</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.091</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-95838156-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201944177">rs201944177</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.091</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375469.1">SUSD3:ENST00000375469.1:c.140G&gt;A:p.(Arg47His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375472.3">SUSD3:ENST00000375472.3:c.179G&gt;A:p.(Arg60His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.091</dd>
                                    
                                    <dd>REVEL: 0.091</dd>
                                    <dd>MVP: 0.076</dd>
                                    <dd>ALPHA_MISSENSE: 0.054</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0347%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DIP2C" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000151240">DIP2C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.090</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.090</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>10-518437-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148545889">rs148545889</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/1534431">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.090</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000280886.6">DIP2C:ENST00000280886.6:c.210C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381496.3">DIP2C:ENST00000381496.3:c.-112C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0123%</dd>
                                    <dd>gnomAD_E_AMR: 0.5581%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.4717%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF512B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196700">ZNF512B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.089</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.089</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>20-62594009-C-T [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200049445">rs200049445</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.089</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000217130.3">ZNF512B:ENST00000217130.3:c.2094G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369888.1">ZNF512B:ENST00000369888.1:c.2094G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000450537.1">ZNF512B:ENST00000450537.1:c.2094G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.5976%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0010%</dd>
                                    <dd>gnomAD_E_SAS: 0.0034%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MPPE1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154889">MPPE1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.086</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.086</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>18-11886739-C-G [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs45522032">rs45522032</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.086</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399978.2">MPPE1:ENST00000399978.2:c.720G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000588072.1">MPPE1:ENST00000588072.1:c.717G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000309976.9">MPPE1:ENST00000309976.9:c.678+177G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317235.7">MPPE1:ENST00000317235.7:c.678+177G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344987.7">MPPE1:ENST00000344987.7:c.679-119G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0691%</dd>
                                    <dd>gnomAD_E_AMR: 0.7001%</dd>
                                    <dd>gnomAD_E_NFE: 0.2918%</dd>
                                    <dd>gnomAD_G_AFR: 0.1263%</dd>
                                    <dd>gnomAD_G_AMR: 0.2358%</dd>
                                    <dd>gnomAD_G_NFE: 0.2204%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SPACA1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000118434">SPACA1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.084</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/620490">OMIM:620490</a> ?Spermatogenic failure 85 (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.084</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>6-88769299-G-A [0|1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs117078336">rs117078336</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.084</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000237201.1">SPACA1:ENST00000237201.1:c.603G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0557%</dd>
                                    <dd>gnomAD_E_AMR: 0.4432%</dd>
                                    <dd>gnomAD_E_NFE: 0.7786%</dd>
                                    <dd>gnomAD_E_SAS: 0.4271%</dd>
                                    <dd>gnomAD_G_AFR: 0.1033%</dd>
                                    <dd>gnomAD_G_AMR: 0.3538%</dd>
                                    <dd>gnomAD_G_NFE: 0.6157%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="COL14A1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187955">COL14A1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.078</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79501">ORPHA:79501</a> Punctate palmoplantar keratoderma type 1 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.078</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>8-121344388-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs115085969">rs115085969</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/730579">(CRITERIA_PROVIDED_SINGLE_SUBMITTER)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.078</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000247781.3">COL14A1:ENST00000247781.3:c.4383G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000297848.3">COL14A1:ENST00000297848.3:c.4668G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000309791.4">COL14A1:ENST00000309791.4:c.4668G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.8860%</dd>
                                    <dd>gnomAD_E_AMR: 0.0955%</dd>
                                    <dd>gnomAD_E_NFE: 0.0176%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.9759%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CFAP44" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000206530">CFAP44</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.072</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617593">OMIM:617593</a> Spermatogenic failure 20 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.072</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-113119440-C-T [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs140356033">rs140356033</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/710199">(CRITERIA_PROVIDED_MULTIPLE_SUBMITTERS_NO_CONFLICTS)</a>
                                
                                
                        </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.072</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295868.2">CFAP44:ENST00000295868.2:c.1426G&gt;A:p.(Val476Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393845.2">CFAP44:ENST00000393845.2:c.1426G&gt;A:p.(Val476Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.0798</dd>
                                    
                                    <dd>REVEL: 0.055</dd>
                                    <dd>MVP: 0.055</dd>
                                    <dd>ALPHA_MISSENSE: 0.080</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0555%</dd>
                                    <dd>gnomAD_E_AMR: 0.0964%</dd>
                                    <dd>gnomAD_E_NFE: 0.5221%</dd>
                                    <dd>gnomAD_E_SAS: 0.0819%</dd>
                                    <dd>gnomAD_G_AFR: 0.0691%</dd>
                                    <dd>gnomAD_G_AMR: 0.1182%</dd>
                                    <dd>gnomAD_G_NFE: 0.3892%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CSRNP3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178662">CSRNP3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.070</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.070</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-166535754-G-A [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs183252532">rs183252532</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.070</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000314499.7">CSRNP3:ENST00000314499.7:c.1249G&gt;A:p.(Ala417Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000342316.4">CSRNP3:ENST00000342316.4:c.1249G&gt;A:p.(Ala417Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409420.1">CSRNP3:ENST00000409420.1:c.1345G&gt;A:p.(Ala449Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.071</dd>
                                    
                                    <dd>REVEL: 0.071</dd>
                                    <dd>MVP: 0.043</dd>
                                    <dd>ALPHA_MISSENSE: 0.068</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.1156%</dd>
                                    <dd>gnomAD_E_EAS: 0.0054%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AMR: 0.1179%</dd>
                                    <dd>gnomAD_G_NFE: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="WDR47" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000085433">WDR47</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=1.0E0)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=1.0E0)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FIVE_PRIME_UTR_INTRON_VARIANT</span>
                            <b>SNV</b>
                            <b>1-109566159-A-G [1|0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200099921">rs200099921</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357672.3">WDR47:ENST00000357672.3:c.-9-16T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361054.3">WDR47:ENST00000361054.3:c.-9-16T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369962.3">WDR47:ENST00000369962.3:c.-9-16T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369965.4">WDR47:ENST00000369965.4:c.-9-16T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400794.3">WDR47:ENST00000400794.3:c.-9-16T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0090%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div> <!-- end gene scores -->
            
        </div>
    </div>

    

    <div id="about">
        <h3>About</h3>
        <p>
            The Exomizer is a Java program that functionally annotates variants from whole-exome
            sequencing data starting from a VCF file (version 4). The functional annotation code is
            based on <a href="https://github.com/charite/jannovar/">Jannovar</a> and uses
            <a href="https://genome.ucsc.edu/">UCSC</a> KnownGene transcript definitions and
            hg19 genomic coordinates
        </p>
        <p>
            Variants are prioritized according to user-defined criteria on variant frequency, pathogenicity,
            quality, inheritance pattern, and model organism phenotype data. Predicted pathogenicity data
            was extracted from the <a href="https://www.ncbi.nlm.nih.gov/pubmed/21520341">dbNSFP</a> resource.
        </p>
        <p>
            Developed by the Computational Biology and Bioinformatics group at the
            <a href="https://genetik.charite.de/">Institute for Medical Genetics and Human Genetics</a> of the
            <a href="https://www.charite.de">Charit&eacute; - Universit&auml;tsmedizin Berlin</a>, the Mouse
            Informatics Group at the <a href="https://www.sanger.ac.uk/">Sanger Institute</a> and the Smedley group at
            <a href="https://qmul.ac.uk/whri">Queen Mary University of London</a>.
        </p>


        <p>Problems, suggestions, or comments? Please <a href="mailto:peter.robinson@charite.de">let us know</a></p>
    </div>
</div>
</body>
</html>
